{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json, re, requests\n",
    "import os\n",
    "import copy\n",
    "import csv\n",
    "from datetime import date\n",
    "import settings\n",
    "from importlib import reload\n",
    "reload(settings)\n",
    "from settings import http_settings, file_settings\n",
    "\n",
    "\n",
    "# This function can be used for the first time on to_biotools.json output from Pub2tools\n",
    "# as well as it can be rerun on the preprints json in the future to check for new publications\n",
    "# if there are new publications in low_tools_prp_date.json file, they will be removed from this file\n",
    "# and moved to the publications file\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "WRITE_TO_DB = False\n",
    "to_curate = 100 #how many of high_tools to write to csv file to be manually curated (int or str) ('all')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Check if all files are available"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "json_file = (file_settings['path_to_pub2tools_month'] + '/' + file_settings['json_tools']).replace('//', '/')\n",
    "if not(os.path.exists(json_file) and os.path.isfile(json_file)):\n",
    "    print(\"can't find json tools file\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "pub2tools_file=(file_settings['path_to_pub2tools_month'] + '/' + file_settings['pub2tools_log']).replace('//', '/')  #check if log file exists in output folder\n",
    "if not(os.path.exists(pub2tools_file) and os.path.isfile(pub2tools_file)):\n",
    "    print(\"can't find pub2tools log file\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "preprints_file=file_settings['preprints_path'] \n",
    "if not(os.path.exists(preprints_file) and os.path.isfile(preprints_file)):\n",
    "    print(\"can't find pub2tools log file\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "lowtools_file=file_settings['lowtools_path'] \n",
    "if not(os.path.exists(lowtools_file) and os.path.isfile(lowtools_file)):\n",
    "    print(\"can't find pub2tools log file\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Process_scrap workflow"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Get token for further validation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "def login_prod(http_settings):\n",
    "    headers_token = {\n",
    "        'Content-Type': 'application/json'\n",
    "        }\n",
    "    user = json.dumps({\n",
    "        'username': http_settings['username'],\n",
    "        'password': http_settings['password']\n",
    "    })\n",
    "\n",
    "    token_r = requests.post(http_settings['host_prod'] + http_settings['login'] + http_settings['json'], headers=headers_token, data=user)\n",
    "    token = json.loads(token_r.text)['key']\n",
    "    return token"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Process Pub2tools output (only high tools)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "def process_tools(json_file):\n",
    "    high_tools = []\n",
    "    with open(json_file) as jf:\n",
    "        data = json.load(jf)\n",
    "        tools = data['list']\n",
    "        for tool in tools:\n",
    "        #    tool['confidence_flag'] = confidence_dict[tool['name']].replace('_',' ')\n",
    "            tool['editPermission'] = {'type': 'public'}\n",
    "        \n",
    "            biotoolsID = re.sub('[^a-zA-Z0-9_~ .-]*', '',tool['name'])\n",
    "            biotoolsID = re.sub('[ ]+','-', biotoolsID)\n",
    "            biotoolsID = 'pub2tools2023__' + biotoolsID\n",
    "            tool['biotoolsID'] = biotoolsID.lower()\n",
    "\n",
    "            if tool['confidence_flag'].lower() == 'high':\n",
    "                #tool['date']= check_date(pub2tools_file)\n",
    "                url_d='{d}{bt}'.format(d=http_settings['dev'],bt=tool['biotoolsID'])  #create tool_link\n",
    "                tool['tool_link']=url_d\n",
    "                high_tools.append(tool)\n",
    "    return high_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Validate processed tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "def validate_tool(tool, token, http_settings):\n",
    "    url = '{h}{t}{v}{f}'.format(h=http_settings['host_prod'], t=http_settings['tool'],v=http_settings['validate'], f=http_settings['json'])   \n",
    "    headers = {\n",
    "        'Content-Type': 'application/json',\n",
    "        'Authorization': 'Token ' + token\n",
    "    }\n",
    "\n",
    "    r = requests.post(url, headers=headers, data=json.dumps(tool))\n",
    "    if r.status_code >= 200 and r.status_code <= 299:\n",
    "        return (True, r.text)\n",
    "    return (False, r.text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "def validate_tools(tools, token, http_settings):\n",
    "    tools_count = len(tools)\n",
    "    to_add = []\n",
    "    problem_tools = []    \n",
    "    for tool in tools: \n",
    "        print(tool['confidence_flag'])\n",
    "        (valid, txt) = validate_tool(tool, token, http_settings)\n",
    "        #valid=True   #For testing\n",
    "        if valid:  \n",
    "            to_add.append(tool)            \n",
    "            print(\"Tool with name {name} is valid.\".format(name=tool['name']))\n",
    "        else:\n",
    "            print(\"Tool with name:{name} has the errors: {errors}\".format(name=tool['name'],errors=txt))\n",
    "            print('Checking if there is an error with the name {name} ... '.format(name=tool['name']))\n",
    "            if (is_html_error(txt)):\n",
    "                print(\"It's an html error message ... \")\n",
    "            else:\n",
    "                e = json.loads(txt)\n",
    "                if type(e) is dict and e.get('name') != None:\n",
    "                    print('There is an error with the name {name}'.format(name=tool['name']))\n",
    "                    tool_temp = copy.deepcopy(tool)\n",
    "                    tool_temp['name'] = tool_temp['name'] + '_autogenerated'\n",
    "                    print('Trying to fix problem by changing tool name to {name}'.format(name=tool_temp['name']))\n",
    "                    (valid, txt) = validate_tool(tool_temp, token, http_settings)\n",
    "                    if valid:\n",
    "                        print('The error was fixed by changing the name to {name}'.format(name=tool_temp['name']))\n",
    "                        tool['name'] = tool_temp['name']\n",
    "                        to_add.append(tool)            \n",
    "                    else:\n",
    "                        print('The error could not be fixed')\n",
    "                        problem_tools.append({'tool_name':tool['name'],'error':txt})\n",
    "                else:\n",
    "                    print('There was a different error')\n",
    "                    problem_tools.append({'tool_name':tool['name'],'error':txt})\n",
    "        print('-----------------')       \n",
    "    print(\"Total tools validated: {added} out of a total of: {total} \".format(added=len(to_add), total=tools_count))\n",
    "    if len(problem_tools) > 0:\n",
    "        print(\"{problem} tools with problems:\".format(problem=len(problem_tools)))\n",
    "        print(problem_tools)\n",
    "    else:\n",
    "        print(\"No tools with problems\")\n",
    "    return to_add"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "def is_html_error(message):\n",
    "    return '<html' in message.lower() or '<body' in message.lower()  "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Check for preprints among validated, and check file with all preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "def identify_preprints(tools, json_file):\n",
    "    new_pubs=[]\n",
    "    for tool in tools:\n",
    "        url_pub,is_preprint=identify_preprint(tool)\n",
    "        tool['publication_link']=url_pub\n",
    "        tool['is_preprint']=is_preprint\n",
    "        if not is_preprint:\n",
    "            new_pubs.append(tool)\n",
    "    print(new_pubs)\n",
    "    print(\"There are {preprints} preprints and {pubs} pubs in tools\".format(pubs=len(new_pubs), preprints=len(tools)-len(new_pubs)))\n",
    "    with open(json_file, 'r') as prp_json_file:\n",
    "        preprints = json.load(prp_json_file)\n",
    "    tools_prp = preprints['list']\n",
    "    number_of_pp=len(tools_prp)\n",
    "    print(\"There are {preprints} preprints in the json file. Checking if something was published.....\".format(preprints=number_of_pp))\n",
    "    updated_pubs=[]\n",
    "    for tool in tools_prp:\n",
    "        url_pub,is_preprint=identify_preprint(tool)\n",
    "        tool['publication_link']=url_pub\n",
    "        tool['is_preprint']=is_preprint\n",
    "        if not is_preprint:\n",
    "            updated_pubs.append(tool)\n",
    "    print(updated_pubs)\n",
    "    print(\"There are {published} published tools in the json file. \".format(published=len(updated_pubs)))\n",
    "    print(\"Updating json.....\")\n",
    "    to_upload = updated_pubs + new_pubs #First newly published preprints, then tools from current month\n",
    "    #preprints.extend(updated_pubs)\n",
    "        #pub_json = {\"count\":len(pubs),\"list\":pubs}\n",
    "    preprints_new = [tool for tool in tools if tool not in new_pubs]\n",
    "    preprints_old = [tool for tool in tools_prp if tool not in updated_pubs]\n",
    "    preprints_data = preprints_old + preprints_new\n",
    "    pp_json = {\"count\":len(preprints_data),\"list\":preprints_data}\n",
    "    with open(json_file, 'w') as preprints_json_file:\n",
    "            json.dump(pp_json, preprints_json_file, indent=4)\n",
    "    number_of_pp=sum(1 for tool in tools if tool.get('is_preprint') == True)\n",
    "    print(\"There are {preprints} preprints in the file after identification\".format(preprints=number_of_pp))\n",
    "    return to_upload\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [],
   "source": [
    "def search_europe_pmc(query):\n",
    "    \"\"\"Search Europe PMC and return the JSON response.\"\"\"\n",
    "    api_endpoint = \"https://www.ebi.ac.uk/europepmc/webservices/rest/search\"\n",
    "    params = {\n",
    "        'query': query,\n",
    "        'format': 'json',\n",
    "        'resultType': 'core'\n",
    "    }\n",
    "    response = requests.get(api_endpoint, params=params)\n",
    "    if response.status_code == 200:\n",
    "        return response.json()\n",
    "    else:\n",
    "        return None\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "def identify_preprint(tool):\n",
    "    \"\"\"\n",
    "    Identify if a publication associated with a tool is a preprint.\n",
    "    Args:\n",
    "    - tool (dict): A dictionary containing tool metadata (Pub2Tools output).\n",
    "    Returns:\n",
    "    - pub_link (str): A link to the publication.\n",
    "    - is_preprint (bool): True if the publication is a preprint, False otherwise.\n",
    "    \"\"\"\n",
    "    if 'doi' in tool['publication'][0]:\n",
    "        # Extract DOI from publication metadata if exists\n",
    "        doi=tool['publication'][0]['doi']\n",
    "        print(tool['name'])\n",
    "        data = search_europe_pmc(f'DOI:\"{doi}\"')        \n",
    "        if data.get('hitCount', 0) != 0:  # If DOI is found in Europe PMC database\n",
    "            result=data['resultList']['result'] \n",
    "            if (result[0].get('source')=='PPR' and result[0].get('commentCorrectionList') is None): # Check if the source is PPR and there are no comment corrections\n",
    "                is_preprint=True\n",
    "                pub_link= f\"https://doi.org/{doi}\"\n",
    "            elif (result[0].get('source')=='PPR' and result[0].get('commentCorrectionList') is not None): \n",
    "                if (result[0].get('commentCorrectionList').get('commentCorrection')[0].get('source')=='PPR'): # Check if the first comment correction source is PPR\n",
    "                    is_preprint=True\n",
    "                    pub_link= f\"https://doi.org/{doi}\"\n",
    "                else:  # Extract external ID and search for potential match\n",
    "                    ext_id=result[0].get('commentCorrectionList').get('commentCorrection')[0].get('id')\n",
    "                    potential_match=search_europe_pmc(f'ext_id:\"{ext_id}\" NOT DOI:\"{doi}\"')\n",
    "                    if potential_match or potential_match.get('hitCount', 0) != 0:\n",
    "                        tool.pop('publication')\n",
    "                        pubs={}\n",
    "                        if ('doi' in potential_match['resultList']['result'][0].keys()):\n",
    "                            new_doi=potential_match['resultList']['result'][0]['doi']\n",
    "                            pubs['doi']=new_doi\n",
    "                            pub_link= f\"https://doi.org/{new_doi}\"\n",
    "                            print(\"Tool {name} has a published version. Changing from {doi} to {new_doi} \".format(name=tool['name'], doi=doi, new_doi=new_doi))\n",
    "                        if ('pmid' in potential_match['resultList']['result'][0].keys()): \n",
    "                            new_pmid = potential_match['resultList']['result'][0]['pmid']\n",
    "                            pub_link = f\"https://pubmed.ncbi.nlm.nih.gov/{new_pmid}/\" \n",
    "                            pubs['pmid'] = new_pmid\n",
    "                            print(\"Tool {name} has a published version. Changing from {doi} to {pmid} \".format(name=tool['name'], doi=doi, pmid=new_pmid))   \n",
    "                        if ('pmcid' in potential_match['resultList']['result'][0].keys()):\n",
    "                            new_pmcid = potential_match['resultList']['result'][0]['pmcid']\n",
    "                            pubs['pmcid'] = new_pmcid \n",
    "                        tool['publication']=[pubs]\n",
    "                        is_preprint=False\n",
    "                    else:  # If there is no match in a database (even though there is correction list), we can't say much about it\n",
    "                        is_preprint=True\n",
    "                        pub_link= f\"https://doi.org/{doi}\"\n",
    "            else:   # Source is other than preprint, that's always publication (?)\n",
    "                pub_link= f\"https://doi.org/{doi}\"\n",
    "                is_preprint=False\n",
    "\n",
    "        else:  #No result in Europe PMC database\n",
    "            pub_link= f\"https://doi.org/{doi}\"\n",
    "            is_preprint=False  #Is it better to assume that this is preprint or not preprint? \n",
    "    else:\n",
    "        print(\"There is no DOI for this tool\")\n",
    "        pmid=tool['publication'][0]['pmid']\n",
    "        pub_link= f\"https://pubmed.ncbi.nlm.nih.gov/{pmid}/\" \n",
    "        # It can still be preprint, but it's safer to assume it's not and double-check it with curation.  \n",
    "        is_preprint=False\n",
    "    return pub_link, is_preprint"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "From tools that are not preprints, create curation csv_file of first TO_CURATE tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "def generate_csv_pub(tools,to_curate):\n",
    "    publications = []\n",
    "    if to_curate=='all':\n",
    "        for tool in tools:\n",
    "            if not tool['is_preprint']:\n",
    "                publications.append(tool)\n",
    "    else:\n",
    "        counter = 0\n",
    "        for tool in tools:\n",
    "            if not tool['is_preprint']:\n",
    "                publications.append(tool)\n",
    "                counter += 1\n",
    "                if counter == to_curate:\n",
    "                    break\n",
    "    file_date=check_date(pub2tools_file)  #function to return date\n",
    "    file_name='pub2tools_{year}_{month}.csv'.format(year=file_date[0],month=file_date[1]) #name .csv file\n",
    "    print(\" Writing {to_curate} files which is {pubs} publications to a {filename} \".format(to_curate=to_curate, pubs=len(publications), filename=file_name))\n",
    "    with open(file_name,'w') as fileobj: #write to .csv file\n",
    "        writerobj=csv.writer(fileobj)\n",
    "        writerobj.writerow(['tool_link','tool_name','homepage','publication_link'])\n",
    "        for tool in publications:\n",
    "            writerobj.writerow([tool['tool_link'],tool['name'],tool['homepage'],tool['publication_link']])\n",
    "    leftover_tools=[tool for tool in tools if tool not in publications]\n",
    "    tools_to_add=publications\n",
    "    return tools_to_add, leftover_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "For naming csv file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "def check_date(pub2tools_file):     #function to generate date\n",
    "    log_file=open(pub2tools_file,'r')\n",
    "    textfile=log_file.read()\n",
    "    log_file.close()\n",
    "    date=re.findall(\"--month (\\d+)-(\\d+)\",textfile)  #from command for pub2tools\n",
    "    return date[0] "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Update json for all of the low, published tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "def generate_json(tools, lowtools_file):\n",
    "    #low_tools = {\"count\":len(tools),\"list\":tools}\n",
    "    with open(json_file, 'r') as lowtools_json_file:\n",
    "        lowtools = json.load(lowtools_json_file)\n",
    "    tools_low = lowtools['list']\n",
    "    tools_update =tools_low + tools\n",
    "    lowtools_json = {\"count\":len(tools_update),\"list\":tools_update}\n",
    "    with open(lowtools_file, 'w') as low_tools_json_file:\n",
    "        json.dump(lowtools_json, low_tools_json_file, indent=4)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [],
   "source": [
    "def add_tools(tools, token, http_settings, WRITE_TO_DB):\n",
    "    print(WRITE_TO_DB)\n",
    "    if not(WRITE_TO_DB):\n",
    "        print(\"Write flag is False, exiting...\")\n",
    "        return    \n",
    "    url = '{h}{t}{f}'.format(h=http_settings['host_prod'], t=http_settings['tool'], f=http_settings['json'])\n",
    "    headers = {\n",
    "        'Content-Type': 'application/json',\n",
    "        'Authorization': 'Token ' + token\n",
    "    }\n",
    "    tools_count = len(tools)\n",
    "    problem_tools = []\n",
    "    ok_tools = 0\n",
    "    for tool in tools:\n",
    "        r = requests.post(url, headers=headers, data=json.dumps(tool))\n",
    "        \n",
    "        if r.status_code >= 200 and r.status_code <= 299:\n",
    "            print(tool['biotoolsID'], 'Added', r.status_code)\n",
    "            ok_tools += 1\n",
    "        else:\n",
    "            print(\"An Error:\",tool['biotoolsID'], r.text)\n",
    "            problem_tools.append({'tool_id':tool['biotoolsID'],'error':r.text})\n",
    "        print('--------------')\n",
    "        time.sleep(2)\n",
    "\n",
    "            \n",
    "    print(\"Total tools added: {added} out of a total of: {total} \".format(added=ok_tools, total=tools_count))\n",
    "    if len(problem_tools) > 0:\n",
    "        print(\"{problem} tools with problems:\".format(problem=len(problem_tools)))\n",
    "        print(problem_tools)\n",
    "    else:\n",
    "        print(\"No tools with problems\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [],
   "source": [
    "tools = process_tools(json_file)                        # STEP 1. READ PUB2TOOLS OUTPUT JSON FILE\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [],
   "source": [
    "token = login_prod(http_settings)                       # STEP 2. GET TOKEN\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "high\n",
      "Tool with name Motifizer is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CanSig is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CoDMI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:PPVED has the errors: {\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]}]}]}\n",
      "Checking if there is an error with the name PPVED ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name HGDTI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name KDmarkers is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name HiCHub is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TIGER is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name IsoAligner is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:RNAinsecta has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name RNAinsecta ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name:CACSV has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name CACSV ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name SiCoDEA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:Rye has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name Rye ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name OGAR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SHARCQ is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ASTool is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name IMPatienT is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TB-Net is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name EcoFun-MAP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SageNet is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:DEPP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name DEPP ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name CLIN-X is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DeepDNAbP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Methylartist is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name RLTLBO is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BANYAN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AttentionDTA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MDGNN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CRISPRedict is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PolypSeg+ is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name m5Cpred-XS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ImmuMethy is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MicrobioSee is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SEACells is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name fastman is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AutoScore-Imbalance is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ASQ is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:OptDesign has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]},{},{}]}]}\n",
      "Checking if there is an error with the name OptDesign ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name RetroSnake is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ToxPi is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:DeepSARS has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{},{}]}]}\n",
      "Checking if there is an error with the name DeepSARS ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name:StORF-Reporter has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name StORF-Reporter ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name scAEGAN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DeepBtoD is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Neuroscout is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GlycoQL is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:ARBRE has the errors: {\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]}]}]}\n",
      "Checking if there is an error with the name ARBRE ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name RustNet is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SmGDB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Onlinemeta is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SISS-Geo is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TITAN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TubULAR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name HiveRel is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ROADMAPS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SNARER is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name E2Style is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name OC_Finder is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name citationchaser is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Spiky is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name surge is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ZZS similarity tool is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BANKSY is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Pocket2Drug is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name IPOscore is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TCR-L is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name epiAneufinder is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Rizoma is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MOOD is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ClinicaDL is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DF-SSmVEP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name hemispheR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MainSEL is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GenMPI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name RRGP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name COPILOT is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name iCatcher is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name COVID-19-associated ARDS non-viral ARDS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AutoDesigner is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MILNP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PeakVI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DTSyn is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name LinQ-View is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name WalkIm is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PhenoBERT is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CPIELA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:LANDMark has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}\n",
      "Checking if there is an error with the name LANDMark ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name PCRMS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:Boquila has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name Boquila ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name MPI-GWAS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:PyMiner has the errors: {\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]}]}]}\n",
      "Checking if there is an error with the name PyMiner ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name MAPPER is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DeepTMHMM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DeSmoke-LAP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MAECI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MetaRelSubNetVis is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Read2Tree is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PregTox is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GEAR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CANon is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AI4ACP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:PipeCoV has the errors: {\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Checking if there is an error with the name PipeCoV ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name PredNitro is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name scTSSR2 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ENDS assumptions is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PanGu Drug Model is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:Phertilizer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name Phertilizer ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name recolorize is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Fast-Higashi is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name KMSubtraction is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name LIPSHOK is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AJILE12 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MASI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BiGAMi is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name nsink is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TransPhos is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Kalmag is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CSR-Net is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name RFPDR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SSPNet is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name FollicleFinder is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ceCLC is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name EEG MEG is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CIGER is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MLHC-COVID-19 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CPA-PM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Gradient-k is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name gACSON is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Aegypti-Atlas is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Psitrum is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MISS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name mHealth literacy mHealth literacy is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Gifsy-1 artAB gogB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DeepDynaTree is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name IETIF is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name multiplestressR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TSFD-Net is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name STKD is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name LuminoCell is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:DeePiCt has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name DeePiCt ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name covid-cancer is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SemanticCAP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GII 2P16 GII 4P16 noroviruses is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DFLM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TCSP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AHLS-pred is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name funRiceGenes is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CVVB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SGP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name NEUROeSTIMator is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SE-BLTCNN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SOKS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MANNERS-DB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name RLSDSPCA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CEBA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name RPfam is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name FastDMF is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name sponge gourd watermelon is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name U-RISC is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DWPPI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CPGL is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Sophora flavescens Kushen decoction is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MetaDR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GMM-NEA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TurboBC is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DiaNat-DB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name COVID-19 non-COVID-19 wards Sudan Zambia is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name sup- sup is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SPHARM-Net is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:ShrinkCRISPR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name ShrinkCRISPR ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name:SAM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name SAM ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name WFA-GPU is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name HergSPred is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name A3D database is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ingrained is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Research COVID-19 air pollution is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name rheuma2025 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name FateCompass is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CoMI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Biocode is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name FRET-lines is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name scMM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name DNE is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Movement Disorders is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name eGRM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AIM is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TMJOAI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PADF is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SIMUS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MISP SRH is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Telerehabilitation physical disabilities movement impairment is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Cure VCP Scientific Conference 2021 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Gegen Qinlian Decoction is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MANE is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name PicassoNet is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:QuasiFlow has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name QuasiFlow ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name P-FLUX is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BCOVIDOA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BiWFA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GDLPA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:UMIcountR has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{}]}]}\n",
      "Checking if there is an error with the name UMIcountR ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name iVp iLAE iBF is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name switchable CycleGAN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name la readmision por deshidratacion tanto is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MaxSpliZer is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name KDM4B is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name Antilogic is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name HisCoM-Kernel is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name subN LiBasub3 subN NaBasub3 subN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name fern phylogeny is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AB-DB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SOAR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TurboBFS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name NVivo is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name SARS-ANI is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TWIENG is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name ConfoundR is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MuLHiTA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TAGNet is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name HFP-QSMGAN is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name LightIoT is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name AsgeneDB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name LEV4REC is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name VIGIRISKS is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name pubassistant.ch is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name TcellInflamedDetector is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name BioAct is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CRIA is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MCDataset is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name exp2GO is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name MultiGATAE is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name FullMonteWeb is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name iCVI-ARTMAP is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name CRDB is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name:PriPath has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Checking if there is an error with the name PriPath ... \n",
      "There was a different error\n",
      "-----------------\n",
      "high\n",
      "Tool with name Siglec-9 is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name gRDF is valid.\n",
      "-----------------\n",
      "high\n",
      "Tool with name GROMACS-SWAXS is valid.\n",
      "-----------------\n",
      "Total tools validated: 218 out of a total of: 238 \n",
      "20 tools with problems:\n",
      "[{'tool_name': 'PPVED', 'error': '{\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]}]}]}'}, {'tool_name': 'RNAinsecta', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'CACSV', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'Rye', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'DEPP', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'OptDesign', 'error': '{\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]},{},{}]}]}'}, {'tool_name': 'DeepSARS', 'error': '{\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{},{}]}]}'}, {'tool_name': 'StORF-Reporter', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'ARBRE', 'error': '{\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]}]}]}'}, {'tool_name': 'LANDMark', 'error': '{\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}'}, {'tool_name': 'Boquila', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'PyMiner', 'error': '{\"function\":[{\"operation\":[{},{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]}]}]}'}, {'tool_name': 'PipeCoV', 'error': '{\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}'}, {'tool_name': 'Phertilizer', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'DeePiCt', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'ShrinkCRISPR', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'SAM', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'QuasiFlow', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}, {'tool_name': 'UMIcountR', 'error': '{\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{}]}]}'}, {'tool_name': 'PriPath', 'error': '{\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}'}]\n"
     ]
    }
   ],
   "source": [
    "tools_v = validate_tools(tools, token, http_settings)   # STEP 3. VALIDATE TOOLS FROM PUB2TOOLS OUTPUT\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Motifizer\n",
      "CanSig\n",
      "CoDMI\n",
      "HGDTI\n",
      "KDmarkers\n",
      "HiCHub\n",
      "TIGER\n",
      "IsoAligner\n",
      "SiCoDEA\n",
      "OGAR\n",
      "SHARCQ\n",
      "ASTool\n",
      "IMPatienT\n",
      "TB-Net\n",
      "EcoFun-MAP\n",
      "SageNet\n",
      "CLIN-X\n",
      "DeepDNAbP\n",
      "Methylartist\n",
      "RLTLBO\n",
      "BANYAN\n",
      "AttentionDTA\n",
      "MDGNN\n",
      "CRISPRedict\n",
      "PolypSeg+\n",
      "m5Cpred-XS\n",
      "ImmuMethy\n",
      "MicrobioSee\n",
      "SEACells\n",
      "fastman\n",
      "AutoScore-Imbalance\n",
      "ASQ\n",
      "RetroSnake\n",
      "ToxPi\n",
      "scAEGAN\n",
      "DeepBtoD\n",
      "Neuroscout\n",
      "GlycoQL\n",
      "RustNet\n",
      "SmGDB\n",
      "Onlinemeta\n",
      "SISS-Geo\n",
      "TITAN\n",
      "TubULAR\n",
      "HiveRel\n",
      "ROADMAPS\n",
      "SNARER\n",
      "E2Style\n",
      "OC_Finder\n",
      "citationchaser\n",
      "Spiky\n",
      "surge\n",
      "ZZS similarity tool\n",
      "BANKSY\n",
      "Pocket2Drug\n",
      "IPOscore\n",
      "TCR-L\n",
      "epiAneufinder\n",
      "Rizoma\n",
      "MOOD\n",
      "ClinicaDL\n",
      "DF-SSmVEP\n",
      "hemispheR\n",
      "MainSEL\n",
      "GenMPI\n",
      "RRGP\n",
      "COPILOT\n",
      "iCatcher\n",
      "COVID-19-associated ARDS non-viral ARDS\n",
      "AutoDesigner\n",
      "MILNP\n",
      "PeakVI\n",
      "DTSyn\n",
      "LinQ-View\n",
      "WalkIm\n",
      "PhenoBERT\n",
      "CPIELA\n",
      "PCRMS\n",
      "MPI-GWAS\n",
      "MAPPER\n",
      "DeepTMHMM\n",
      "DeSmoke-LAP\n",
      "MAECI\n",
      "MetaRelSubNetVis\n",
      "Read2Tree\n",
      "PregTox\n",
      "GEAR\n",
      "CANon\n",
      "AI4ACP\n",
      "PredNitro\n",
      "scTSSR2\n",
      "ENDS assumptions\n",
      "PanGu Drug Model\n",
      "recolorize\n",
      "Fast-Higashi\n",
      "KMSubtraction\n",
      "LIPSHOK\n",
      "AJILE12\n",
      "MASI\n",
      "BiGAMi\n",
      "nsink\n",
      "TransPhos\n",
      "Kalmag\n",
      "CSR-Net\n",
      "RFPDR\n",
      "SSPNet\n",
      "FollicleFinder\n",
      "ceCLC\n",
      "EEG MEG\n",
      "CIGER\n",
      "MLHC-COVID-19\n",
      "CPA-PM\n",
      "Gradient-k\n",
      "gACSON\n",
      "Aegypti-Atlas\n",
      "Psitrum\n",
      "MISS\n",
      "mHealth literacy mHealth literacy\n",
      "Gifsy-1 artAB gogB\n",
      "DeepDynaTree\n",
      "IETIF\n",
      "multiplestressR\n",
      "TSFD-Net\n",
      "STKD\n",
      "LuminoCell\n",
      "covid-cancer\n",
      "SemanticCAP\n",
      "GII 2P16 GII 4P16 noroviruses\n",
      "DFLM\n",
      "TCSP\n",
      "AHLS-pred\n",
      "funRiceGenes\n",
      "CVVB\n",
      "SGP\n",
      "NEUROeSTIMator\n",
      "SE-BLTCNN\n",
      "SOKS\n",
      "MANNERS-DB\n",
      "RLSDSPCA\n",
      "CEBA\n",
      "RPfam\n",
      "FastDMF\n",
      "sponge gourd watermelon\n",
      "U-RISC\n",
      "DWPPI\n",
      "CPGL\n",
      "Sophora flavescens Kushen decoction\n",
      "MetaDR\n",
      "GMM-NEA\n",
      "TurboBC\n",
      "DiaNat-DB\n",
      "COVID-19 non-COVID-19 wards Sudan Zambia\n",
      "sup- sup\n",
      "SPHARM-Net\n",
      "WFA-GPU\n",
      "HergSPred\n",
      "A3D database\n",
      "ingrained\n",
      "Research COVID-19 air pollution\n",
      "rheuma2025\n",
      "FateCompass\n",
      "CoMI\n",
      "Biocode\n",
      "FRET-lines\n",
      "scMM\n",
      "DNE\n",
      "Movement Disorders\n",
      "eGRM\n",
      "AIM\n",
      "TMJOAI\n",
      "PADF\n",
      "SIMUS\n",
      "MISP SRH\n",
      "Telerehabilitation physical disabilities movement impairment\n",
      "Cure VCP Scientific Conference 2021\n",
      "Gegen Qinlian Decoction\n",
      "MANE\n",
      "PicassoNet\n",
      "P-FLUX\n",
      "BCOVIDOA\n",
      "BiWFA\n",
      "GDLPA\n",
      "iVp iLAE iBF\n",
      "switchable CycleGAN\n",
      "la readmision por deshidratacion tanto\n",
      "MaxSpliZer\n",
      "KDM4B\n",
      "Antilogic\n",
      "HisCoM-Kernel\n",
      "subN LiBasub3 subN NaBasub3 subN\n",
      "fern phylogeny\n",
      "AB-DB\n",
      "SOAR\n",
      "TurboBFS\n",
      "NVivo\n",
      "SARS-ANI\n",
      "TWIENG\n",
      "ConfoundR\n",
      "MuLHiTA\n",
      "TAGNet\n",
      "HFP-QSMGAN\n",
      "LightIoT\n",
      "AsgeneDB\n",
      "LEV4REC\n",
      "VIGIRISKS\n",
      "pubassistant.ch\n",
      "TcellInflamedDetector\n",
      "BioAct\n",
      "CRIA\n",
      "MCDataset\n",
      "exp2GO\n",
      "MultiGATAE\n",
      "FullMonteWeb\n",
      "iCVI-ARTMAP\n",
      "CRDB\n",
      "Siglec-9\n",
      "gRDF\n",
      "GROMACS-SWAXS\n",
      "[{'name': 'CoDMI', 'description': 'Survival analysis using the Covid-Death Mean-Imputation (CoDMI) algorithm.\\n\\nPurpose: We illustrate a clinical application of Covid-Death Mean-Imputation (CoDMI) algorithm in survival analysis. Material:. and methods. We analyzed 94 patients treated for primary locally advanced rectal cancer (LARC). Overall survival was calculated in months from diagnosis to first event (last follow-up death). Because Covid-19 death events potentially bias survival estimation, to eliminate skewed data due to Covid-19 death events the observed lifetime of Covid-19 cases was replaced by its corresponding expected lifetime in absence of the Covid-19 event using CoDMI algorithm. In a traditional Kaplan-Meier approach, patient died of Covid-19 (DoC) can be: i) excluded to the cohort, or ii) counted as censored (Cen), or iii) considered as died of disease (DoD). CoDMI algorithm offers an additional, more satisfactory option: iv) DoC events are mean-imputed by the Kaplan-Meier estimator\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/alef-innovation/codmi', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3503', 'term': 'Incident curve plotting'}, {'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3421', 'term': 'Surgery'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__codmi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__codmi', 'publication': [{'doi': '10.21873/invivo.13043', 'pmid': '36309383', 'pmcid': 'PMC9677754'}], 'publication_link': 'https://doi.org/10.21873/invivo.13043', 'is_preprint': False}, {'name': 'HGDTI', 'description': 'predicting drug-target interaction by using information aggregation based on heterogeneous graph neural network.\\n\\nBackground In research on new drug discovery, the traditional wet experiment has a long period. Predicting drug-target interaction (DTI) in silico can greatly narrow the scope of search of candidate medications. Excellent algorithm model may be more effective in revealing the potential connection between drug and target in the bioinformatics network composed of drugs, proteins and other related data. Results. In this work, we have developed a heterogeneous graph neural network model, named as HGDTI, which includes a learning phase of network node embedding and a training phase of DTI classification\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://bioinfo.jcu.edu.cn/hgdti', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}, {'uri': 'http://edamontology.org/operation_0478', 'term': 'Molecular docking'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'}, {'uri': 'http://edamontology.org/topic_2275', 'term': 'Molecular modelling'}], 'publication': [{'doi': '10.1186/S12859-022-04655-5', 'pmid': '35413800', 'pmcid': 'PMC9004085'}], 'credit': [{'name': 'Xuan Xiao', 'email': 'jdzxiaoxuan@163.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__hgdti', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hgdti', 'publication_link': 'https://doi.org/10.1186/S12859-022-04655-5', 'is_preprint': False}, {'name': 'KDmarkers', 'description': 'A biomarker database for investigating epigenetic methylation and gene expression levels in Kawasaki disease.\\n\\nKawasaki disease (KD) is a form of acute systemic vasculitis that primarily affects children and has become the most common cause of acquired heart disease. While the etiopathogenesis of KD remains unknown, the diagnostic criteria of KD have been well established. Nevertheless, the diagnosis of KD is currently based on subjective clinical symptoms, and no molecular biomarker is yet available. We have previously performed and combined methylation array (Illumina HumanMethylation450 BeadChip) and transcriptome array (Affymetrix GeneChip Human Transcriptome Array 2.0) to identify genes that are differentially methylated expressed in KD patients compared with control subjects. We have found that decreased methylation levels combined with elevated gene expression can indicate genes (e.g., toll-like receptors and CD177) involved in the disease mechanisms of KD\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://cosbi.ee.ncku.edu.tw/KDmarkers/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3207', 'term': 'Gene methylation analysis'}, {'uri': 'http://edamontology.org/operation_3223', 'term': 'Differential gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3919', 'term': 'Methylation calling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3295', 'term': 'Epigenetics'}, {'uri': 'http://edamontology.org/topic_3518', 'term': 'Microarray experiment'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}], 'publication': [{'doi': '10.1016/J.CSBJ.2022.02.032', 'pmid': '35356542', 'pmcid': 'PMC8931344'}], 'credit': [{'name': 'Ho-Chang Kuo', 'email': 'dr.hckuo@gmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__kdmarkers', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kdmarkers', 'publication_link': 'https://doi.org/10.1016/J.CSBJ.2022.02.032', 'is_preprint': False}, {'name': 'TIGER', 'description': 'A Web Portal of Tumor Immunotherapy Gene Expression Resource.\\n\\nTumor Immunotherapy Gene Expression Resource.\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/tiger-dna, bio.tools/tiger', 'homepage': 'http://tiger.canceromics.org/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2436', 'term': 'Gene-set enrichment analysis'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_0313', 'term': 'Expression profile clustering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}], 'publication': [{'doi': '10.1101/2022.04.10.487568'}], 'credit': [{'name': 'Penghui Zhou', 'email': 'zhaoan@zjcc.org.cn', 'typeEntity': 'Person'}, {'name': 'Jian Ren', 'email': 'zhouph@sysucc.org.cn', 'typeEntity': 'Person'}, {'name': 'An Zhao', 'email': 'renjian@sysucc.org.cn', 'typeEntity': 'Person'}, {'name': 'Zhixiang Zuo', 'email': 'zuozhx@sysucc.org.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tiger', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tiger', 'publication_link': 'https://doi.org/10.1101/2022.04.10.487568', 'is_preprint': False}, {'name': 'SiCoDEA', 'description': 'a simple, fast and complete app for analyzing the effect of individual drugs and their combinations.\\n\\nThe administration of combinations of drugs is a method widely used in the treatment of different pathologies as it can lead to an increase in the therapeutic effect and a reduction in the dose compared to the administration of the single drugs. For these reasons, it is of interest to study combinations of drugs and in particular to determine whether a specific combination has a synergistic, antagonistic or additive effect, i.e greater, less than or equal to the effect expected by the sum of the individual drugs. For this purpose, various mathematical models have been developed, which use different methods to evaluate the synergy of a combination of drugs. Most of these methods are based on the Loewe Additivity Principle (Loewe et al., 1953), which has its key step in choosing the model used for predicting the effect of individual drugs', 'homepage': 'https://sicodea.shinyapps.io/shiny/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426', 'term': 'Modelling and simulation'}, {'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3375', 'term': 'Drug metabolism'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}], 'language': ['R'], 'credit': [{'name': 'Giulio Spinozzi', 'email': 'giulio.spinozzi@unipg.it', 'typeEntity': 'Person'}, {'name': 'Maria Paola Martelli', 'email': 'maria.martelli@unipg.it', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sicodea', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sicodea', 'publication': [{'doi': '10.3390/biom12070904', 'pmid': '35883460', 'pmcid': 'PMC9313187'}], 'publication_link': 'https://doi.org/10.3390/biom12070904', 'is_preprint': False}, {'name': 'OGAR', 'description': \"An open-source tool for studying the psychology of virtual art museum visits.\\n\\nTo expand the tools available to arts researchers in psychology, we present the Open Gallery for Arts Research (OGAR), a free, open-source tool for studying visitor behavior within an online gallery environment. OGAR is highly extensible, allowing researchers to modify the environment to test different hypotheses, and it affords assessing a wide range of outcome variables. After describing the tool and its development, we present a proof-of-concept study that evaluates OGAR's usability and performance and illustrates some ways that it can be used to study the psychology of virtual visits. With a sample of 44 adults from an online participant panel who freely explored OGAR, we observed that OGAR had good usability based on high scores on the System Usability Scale and rare instances of self-reported nausea, among other usability markers\\n\\n||| HOMEPAGE BROKEN!\", 'homepage': 'https://github.com/mboerwinkle/OGAR', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}, {'uri': 'http://edamontology.org/topic_3520', 'term': 'Proteomics experiment'}, {'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'}], 'publication': [{'doi': '10.3758/S13428-022-01857-W', 'pmid': '35469088', 'pmcid': 'PMC9037585'}], 'credit': [{'name': 'Rebekah M. Rodriguez-Boerwinkle', 'email': 'rmrodriguez@uncg.edu', 'orcidid': 'https://orcid.org/0000-0002-9836-1175', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ogar', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ogar', 'publication_link': 'https://doi.org/10.3758/S13428-022-01857-W', 'is_preprint': False}, {'name': 'SHARCQ', 'description': 'A Semi-Automated Workflow for Brain Slice Histology Alignment, Registration, and Cell Quantification (SHARCQ).\\n\\nA tool for Slice Histology Alignment, Registration, and Cell Quantification.\\n\\n1) Start with an imperfect slice image.\\n\\nHistological slices are imperfect, making brain regions difficult to accurately delineate. Hand-drawing region borders is a painstakingly slow process. With user-selected landmark points, SHARCQ warps the slice image to match the reference atlas. After this so-called atlas registration is complete, brain region borders are overlaid onto the registered slice.\\n\\n2) Click anatomical landmarks at the correct atlas location.', 'homepage': 'https://www.root-lab.org/sharcq', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}], 'language': ['MATLAB'], 'publication': [{'doi': '10.1523/ENEURO.0483-21.2022', 'pmid': '35396257', 'pmcid': 'PMC9034756'}], 'credit': [{'name': 'David H. Root', 'email': 'david.root@colorado.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sharcq', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sharcq', 'publication_link': 'https://doi.org/10.1523/ENEURO.0483-21.2022', 'is_preprint': False}, {'name': 'ASTool', 'description': \"An Easy-to-Use Tool to Accurately Identify Alternative Splicing Events from Plant RNA-Seq Data.\\n\\nAlternative splicing events detection from RNA-Seq data.\\n\\nAlternative splicing (AS) includes four main types: intron retention (IR), exon skipping (ES), alternative 5' splice sites (A5SS) and alternative 3' splice sites (A3SS). ASTool is a simple and easy-to-use bioinformatics tool for detecting AS events. And ASTool is developed to detect percent spliced in (PSI) for each AS event. Generally, IR is the most prevalent form in Plants. Accordingly, ASTool can detect novel IR events based on known splice sites.\", 'homepage': 'http://zzdlab.com/ASTool/index.php', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0264', 'term': 'Alternative splicing prediction'}, {'uri': 'http://edamontology.org/operation_0446', 'term': 'Exonic splicing enhancer prediction'}, {'uri': 'http://edamontology.org/operation_3680', 'term': 'RNA-Seq analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3320', 'term': 'RNA splicing'}, {'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_0091', 'term': 'Bioinformatics'}], 'publication': [{'doi': '10.3390/IJMS23084079', 'pmid': '35456896', 'pmcid': 'PMC9031537'}], 'credit': [{'name': 'Ziding Zhang', 'email': 'zidingzhang@cau.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__astool', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__astool', 'publication_link': 'https://doi.org/10.3390/IJMS23084079', 'is_preprint': False}, {'name': 'TB-Net', 'description': 'A Tailored, Self-Attention Deep Convolutional Neural Network Design for Detection of Tuberculosis Cases From Chest X-Ray Images.\\n\\nTuberculosis (TB) remains a global health problem, and is the leading cause of death from an infectious disease. A crucial step in the treatment of tuberculosis is screening high risk populations and the early detection of the disease, with chest x-ray (CXR) imaging being the most widely-used imaging modality. As such, there has been significant recent interest in artificial intelligence-based TB screening solutions for use in resource-limited scenarios where there is a lack of trained healthcare workers with expertise in CXR interpretation. Motivated by this pressing need and the recent recommendation by the World Health Organization (WHO) for the use of computer-aided diagnosis of TB in place of a human reader, we introduce TB-Net, a self-attention deep convolutional neural network tailored for TB case screening\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/darwinai/TuberculosisNet', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2428', 'term': 'Validation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'}, {'uri': 'http://edamontology.org/topic_3400', 'term': 'Allergy, clinical immunology and immunotherapeutics'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}], 'publication': [{'doi': '10.3389/FRAI.2022.827299', 'pmid': '35464996', 'pmcid': 'PMC9022489'}], 'credit': [{'name': 'Alexander Wong', 'email': 'a28wong@uwaterloo.ca', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tb-net', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tb-net', 'publication_link': 'https://doi.org/10.3389/FRAI.2022.827299', 'is_preprint': False}, {'name': 'CLIN-X', 'description': 'pre-trained language models and a study on cross-task transfer for concept extraction in the clinical domain.\\n\\nMotivation The field of natural language processing (NLP) has recently seen a large change toward using pre-trained language models for solving almost any task. Despite showing great improvements in benchmark datasets for various tasks, these models often perform sub-optimal in non-standard domains like the clinical domain where a large gap between pre-training documents and target documents is observed. In this article, we aim at closing this gap with domain-specific training of the language model and we investigate its effect on a diverse set of downstream tasks and settings. Results. We introduce the pre-trained CLIN-X (Clinical XLM-R) language models and show how CLIN-X outperforms other pre-trained transformer models by a large margin for 10 clinical concept extraction tasks from two languages\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/boschresearch/clin_x/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'}, {'uri': 'http://edamontology.org/operation_0306', 'term': 'Text mining'}, {'uri': 'http://edamontology.org/operation_1812', 'term': 'Parsing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0218', 'term': 'Natural language processing'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}], 'publication': [{'doi': '10.1093/BIOINFORMATICS/BTAC297', 'pmid': '35485748'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__clin-x', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__clin-x', 'publication_link': 'https://doi.org/10.1093/BIOINFORMATICS/BTAC297', 'is_preprint': False}, {'name': 'DeepDNAbP', 'description': 'A deep learning-based hybrid approach to improve the identification of deoxyribonucleic acid-binding proteins.\\n\\nAccurate identification of DNA-binding proteins (DBPs) is critical for both understanding protein function and drug design. DBPs also play essential roles in different kinds of biological activities such as DNA replication, repair, transcription, and splicing. As experimental identification of DBPs is time-consuming and sometimes biased toward prediction, constructing an effective DBP model represents an urgent need, and computational methods that can accurately predict potential DBPs based on sequence information are highly desirable. In this paper, a novel predictor called DeepDNAbP has been developed to accurately predict DBPs from sequences using a convolutional neural network (CNN) model\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://deepdbp.sblog360.blog/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3900', 'term': 'DNA-binding protein prediction'}, {'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3767', 'term': 'Protein identification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}], 'publication': [{'doi': '10.1016/J.COMPBIOMED.2022.105433', 'pmid': '35378437'}], 'credit': [{'name': 'Watshara Shoombuatong', 'email': 'watshara.sho@mahidol.ac.th', 'orcidid': 'https://orcid.org/0000-0002-3394-8709', 'typeEntity': 'Person'}, {'name': 'Md Mehedi Hasan', 'email': 'mhasan1@tulane.edu', 'orcidid': 'https://orcid.org/0000-0003-4952-0739', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__deepdnabp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__deepdnabp', 'publication_link': 'https://doi.org/10.1016/J.COMPBIOMED.2022.105433', 'is_preprint': False}, {'name': 'Methylartist', 'description': 'tools for visualizing modified bases from nanopore sequence data.\\n\\nSummary Methylartist is a consolidated suite of tools for processing, visualizing and analysing nanopore-derived modified base calls. All detectable methylation types (e.g. 5mCpG, 5hmC, 6mA) are supported, enabling integrated study of base pairs when modified naturally or as part of an experimental protocol. Availability and implementation. Methylartist is implemented in Python and is installable via PyPI and bioconda. Source code and test data are available at https: github.com adamewing methylartist. Supplementary information. Supplementary data are available at Bioinformatics online\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/adamewing/methylartist', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3185', 'term': 'Base-calling'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3919', 'term': 'Methylation calling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}, {'uri': 'http://edamontology.org/topic_3295', 'term': 'Epigenetics'}, {'uri': 'http://edamontology.org/topic_0091', 'term': 'Bioinformatics'}], 'language': ['Python'], 'publication': [{'doi': '10.1093/BIOINFORMATICS/BTAC292', 'pmid': '35482479', 'pmcid': 'PMC9154218'}], 'credit': [{'name': 'Adam D Ewing', 'email': 'adam.ewing@mater.uq.edu.au', 'orcidid': 'https://orcid.org/0000-0002-4544-994X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__methylartist', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__methylartist', 'publication_link': 'https://doi.org/10.1093/BIOINFORMATICS/BTAC292', 'is_preprint': False}, {'name': 'RLTLBO', 'description': 'An Improved Teaching-Learning-Based Optimization Algorithm with Reinforcement Learning Strategy for Solving Optimization Problems.\\n\\nThis paper presents an improved teaching-learning-based optimization (TLBO) algorithm for solving optimization problems, called RLTLBO. First, a new learning mode considering the effect of the teacher is presented. Second, the Q-Learning method in reinforcement learning (RL) is introduced to build a switching mechanism between two different learning modes in the learner phase. Finally, ROBL is adopted after both the teacher and learner phases to improve the local optima avoidance ability of RLTLBO. These two strategies effectively enhance the convergence speed and accuracy of the proposed algorithm. RLTLBO is analyzed on 23 standard benchmark functions and eight CEC2017 test functions to verify the optimization performance\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/WangShuang92/RLTLBO', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.1155/2022/1535957', 'pmid': '35371212', 'pmcid': 'PMC8970903'}], 'credit': [{'name': 'Shuang Wang', 'email': 'wang_shuang9279@163.com', 'orcidid': 'https://orcid.org/0000-0001-5933-074X', 'typeEntity': 'Person'}, {'name': 'Heming Jia', 'email': 'jiaheminglucky99@126.com', 'orcidid': 'https://orcid.org/0000-0002-4339-8464', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rltlbo', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rltlbo', 'publication_link': 'https://doi.org/10.1155/2022/1535957', 'is_preprint': False}, {'name': 'AttentionDTA', 'description': 'Drug-Target Binding Affinity Prediction by Sequence-Based Deep Learning With Attention Mechanism.\\n\\nThe identification of drug-target relations (DTRs) is substantial in drug development. A large number of methods treat DTRs as drug-target interactions (DTIs), a binary classification problem. The main drawback of these methods are the lack of reliable negative samples and the absence of many important aspects of DTR, including their dose dependence and quantitative affinities. With increasing number of publications of drug-protein binding affinity data recently, DTRs prediction can be viewed as a regression problem of drug-target affinities (DTAs) which reflects how tightly the drug binds to the target and can present more detailed and specific information than DTIs. The growth of affinity data enables the use of deep learning architectures, which have been shown to be among the state-of-the-art methods in binding affinity prediction\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/zhaoqichang/AttentionDTA_TCBB', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}, {'uri': 'http://edamontology.org/operation_3092', 'term': 'Protein feature detection'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1109/TCBB.2022.3170365', 'pmid': '35471889'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__attentiondta', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__attentiondta', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3170365', 'is_preprint': False}, {'name': 'MDGNN', 'description': 'Microbial Drug Prediction Based on Heterogeneous Multi-Attention Graph Neural Network.\\n\\nHuman beings are now facing one of the largest public health crises in history with the outbreak of COVID-19. Traditional drug discovery could not keep peace with newly discovered infectious diseases. The prediction of drug-virus associations not only provides insights into the mechanism of drug-virus interactions, but also guides the screening of potential antiviral drugs. We develop a deep learning algorithm based on the graph convolutional networks (MDGNN) to predict potential antiviral drugs. MDGNN is consisted of new node-level attention and feature-level attention mechanism and shows its effectiveness compared with other comparative algorithms. MDGNN integrates the global information of the graph in the process of information aggregation by introducing the attention at node and feature level to graph convolution\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Pijiangsheng/MDGNN', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}], 'publication': [{'doi': '10.3389/FMICB.2022.819046', 'pmid': '35464940', 'pmcid': 'PMC9021438'}], 'credit': [{'name': 'Yang Zhang', 'email': 'zhangyang07@hit.edu.cn', 'typeEntity': 'Person'}, {'name': 'Junyi Li', 'email': 'lijunyi@hit.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mdgnn', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mdgnn', 'publication_link': 'https://doi.org/10.3389/FMICB.2022.819046', 'is_preprint': False}, {'name': 'PolypSeg+', 'description': 'A Lightweight Context-Aware Network for Real-Time Polyp Segmentation.\\n\\nAutomatic polyp segmentation from colonoscopy videos is a prerequisite for the development of a computer-assisted colon cancer examination and diagnosis system. However, it remains a very challenging task owing to the large variation of polyps, the low contrast between polyps and background, and the blurring boundaries of polyps. More importantly, real-time performance is a necessity of this task, as it is anticipated that the segmented results can be immediately presented to the doctor during the colonoscopy intervention for his her prompt decision and action. It is difficult to develop a model with powerful representation capability, yielding satisfactory segmentation results and, simultaneously, maintaining real-time performance\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/szu-zzb/polypsegplus', 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}], 'publication': [{'doi': '10.1109/TCYB.2022.3162873', 'pmid': '35417366'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__polypseg', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__polypseg', 'publication_link': 'https://doi.org/10.1109/TCYB.2022.3162873', 'is_preprint': False}, {'name': 'm5Cpred-XS', 'description': 'A New Method for Predicting RNA m5C Sites Based on XGBoost and SHAP.\\n\\nWelcome to m5Cpred-xs. please click to enter the m5Cpred-xs tool to start a task...', 'homepage': 'http://m5cpred-xs.zhulab.org.cn/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3936', 'term': 'Feature selection'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0099', 'term': 'RNA'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}], 'publication': [{'doi': '10.3389/FGENE.2022.853258', 'pmid': '35432446', 'pmcid': 'PMC9005994'}], 'credit': [{'name': 'Yong Zhang', 'email': 'yongzhang@ahau.edu.cn', 'typeEntity': 'Person'}, {'name': 'Xiaolei Zhu', 'email': 'xlzhu_mdl@hotmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__m5cpred-xs', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__m5cpred-xs', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.853258', 'is_preprint': False}, {'name': 'ImmuMethy', 'description': 'database of DNA methylation plasticity at a single cytosine resolution in human blood and immune cells.\\n\\nDifferential DNA methylation is a feature of numerous physiological and pathological processes. However, the extent to which single-base cytosine methylation modifies cellular responses to various stimuli has not been well characterized. In this study, we carried out a systematic analysis of methylome data derived from human blood and immune cells and constructed the ImmuMethy database. ImmuMethy allows interrogation of DNA methylation plasticity (MPL) at the single cytosine level. MPL, which refers to the variability of DNA methylation, is quantitatively measured in multiple ways, such as quartiles and standard deviations\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://immudb.bjmu.edu.cn/immumethy/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3204', 'term': 'Methylation analysis'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}, {'uri': 'http://edamontology.org/operation_0417', 'term': 'PTM site prediction'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3295', 'term': 'Epigenetics'}, {'uri': 'http://edamontology.org/topic_3674', 'term': 'Methylated DNA immunoprecipitation'}, {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'}], 'publication': [{'doi': '10.1093/DATABASE/BAAC020', 'pmid': '35363305', 'pmcid': 'PMC9216548'}], 'credit': [{'name': 'Pingzhang Wang', 'email': 'wangpzh@bjmu.edu.cn', 'orcidid': 'https://orcid.org/0000-0003-1746-7216', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__immumethy', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__immumethy', 'publication_link': 'https://doi.org/10.1093/DATABASE/BAAC020', 'is_preprint': False}, {'name': 'MicrobioSee', 'description': 'A Web-Based Visualization Toolkit for Multi-Omics of Microbiology.\\n\\nA visualization tool for microbial multiomics.\\n\\nProvides a variety of themes, statistical methods for choice, length and width can be set freely.\\n\\nMicrobioSee: a web-based visualization toolkit for multi-omics of microbiology. Frontiers in Genetics. 2022. 13: 853612.', 'homepage': 'https://microbiosee.gxu.edu.cn', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3301', 'term': 'Microbiology'}, {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'publication': [{'doi': '10.3389/FGENE.2022.853612', 'pmid': '35464838', 'pmcid': 'PMC9024144'}], 'credit': [{'name': 'Jianping Liao', 'typeEntity': 'Person'}, {'name': 'Chengjian Jiang', 'email': 'jiangcj0520@vip.163.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__microbiosee', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__microbiosee', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.853612', 'is_preprint': False}, {'name': 'AutoScore-Imbalance', 'description': 'An interpretable machine learning tool for development of clinical scores with rare events data.\\n\\nBackground Medical decision-making impacts both individual and public health. Clinical scores are commonly used among various decision-making models to determine the degree of disease deterioration at the bedside. AutoScore was proposed as a useful clinical score generator based on machine learning and a generalized linear model. However, its current framework still leaves room for improvement when addressing unbalanced data of rare events. Methods. Using machine intelligence approaches, we developed AutoScore-Imbalance, which comprises three components: training dataset optimization, sample weight optimization, and adjusted AutoScore\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/nliulab/AutoScore-Imbalance', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3305', 'term': 'Public health and epidemiology'}, {'uri': 'http://edamontology.org/topic_3418', 'term': 'Paediatrics'}, {'uri': 'http://edamontology.org/topic_3063', 'term': 'Medical informatics'}], 'publication': [{'doi': '10.1016/J.JBI.2022.104072', 'pmid': '35421602'}], 'credit': [{'name': 'Nan Liu', 'email': 'liu.nan@duke-nus.edu.sg', 'orcidid': 'https://orcid.org/0000-0003-3610-4883', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__autoscore-imbalance', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__autoscore-imbalance', 'publication_link': 'https://doi.org/10.1016/J.JBI.2022.104072', 'is_preprint': False}, {'name': 'ToxPi', 'description': \"creating, viewing, and sharing integrative visualizations for geospatial data using ArcGIS.\\n\\nCreate ToxPi Prioritization Models.\\n\\nToxPi : Toxicological Prioritization Index.\\n\\nEnables users to build 'ToxPi' prioritization models and provides functionality within the grid framework for plotting ToxPi graphs. 'toxpiR' allows for more customization than the 'ToxPi GUI' (<https://toxpi.org>) and integration into existing workflows for greater ease-of-use, reproducibility, and transparency. toxpiR package behaves nearly identically to the GUI; the package documentation includes notes about all differences. The vignettes download example files from <https://github.com/ToxPi/ToxPi-example-files>.\\n\\nThe ToxPi interface is distributed as a single compressed .zip download.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE ToxPiGIS Toolkit (bio.tools/toxpigis_toolkit), ToxPiGIS\", 'homepage': 'http://www.toxpi.org', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'}, {'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2840', 'term': 'Toxicology'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}, {'uri': 'http://edamontology.org/topic_3400', 'term': 'Allergy, clinical immunology and immunotherapeutics'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'language': ['Python'], 'license': 'GPL-3.0', 'link': [{'url': 'https://CRAN.R-project.org/package=toxpiR', 'type': ['Repository']}], 'publication': [{'doi': '10.1038/S41370-022-00433-W', 'pmid': '35474345', 'pmcid': 'PMC9039976'}], 'credit': [{'name': 'David M. Reif', 'email': 'dmreif@ncsu.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__toxpi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__toxpi', 'publication_link': 'https://doi.org/10.1038/S41370-022-00433-W', 'is_preprint': False}, {'name': 'scAEGAN', 'description': 'Unification of Single-Cell Genomics Data by Adversarial Learning of Latent Space Correspondences.\\n\\nABSTRACT Recent progress in Single-Cell Genomics have produced different library protocols and techniques for profiling of one or more data modalities in individual cells. Machine learning methods have separately addressed specific integration challenges (libraries, samples, paired-unpaired data modalities). We formulate an unifying data-driven methodology addressing all these challenges. To this end, we design a hybrid architecture using an autoencoder (AE) network together with adversarial learning by a cycleGAN (cGAN) network, jointly referred to as scAEGAN. The AE learns a low-dimensional embedding of each condition, whereas the cGAN learns a non-linear mapping between the AE representations. The core insight is that the AE respects each sample’s uniqueness, whereas the cGAN exploits the distributional data similarity in the latent space\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/sumeer1/scAEGAN', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3935', 'term': 'Dimensionality reduction'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}], 'credit': [{'name': 'Jesper Tegner', 'email': 'jesper.tegner@kaust.edu.sa', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__scaegan', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__scaegan', 'publication': [{'doi': '10.1371/journal.pone.0281315', 'pmid': '36735690', 'pmcid': 'PMC9897517'}], 'publication_link': 'https://doi.org/10.1371/journal.pone.0281315', 'is_preprint': False}, {'name': 'DeepBtoD', 'description': 'Improved RNA-binding proteins prediction via integrated deep learning.\\n\\nRNA-binding proteins (RBPs) have crucial roles in various cellular processes such as alternative splicing and gene regulation. Therefore, the analysis and identification of RBPs is an essential issue. However, although many computational methods have been developed for predicting RBPs, a few studies simultaneously consider local and global information from the perspective of the RNA sequence. Facing this challenge, we present a novel method called DeepBtoD, which predicts RBPs directly from RNA sequences. First, a [Formula: see text]-BtoD encoding is designed, which takes into account the composition of [Formula: see text]-nucleotides and their relative positions and forms a local module. Second, we designed a multi-scale convolutional module embedded with a self-attentive mechanism, the ms-focusCNN, which is used to further learn more effective, diverse, and discriminative high-level features\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://175.27.228.227/DeepBtoD/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3901', 'term': 'RNA-binding protein prediction'}, {'uri': 'http://edamontology.org/operation_3902', 'term': 'RNA binding site prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3320', 'term': 'RNA splicing'}, {'uri': 'http://edamontology.org/topic_0128', 'term': 'Protein interactions'}], 'publication': [{'doi': '10.1142/S0219720022500068', 'pmid': '35451938'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__deepbtod', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__deepbtod', 'publication_link': 'https://doi.org/10.1142/S0219720022500068', 'is_preprint': False}, {'name': 'Neuroscout', 'description': 'unified platform for generalizable and reproducible fMRI research.\\n\\nPreprocessed (fmriprep) life fmri dataset.\\n\\nPreprocessed (fmriprep) raiders fmri dataset.', 'homepage': 'https://neuroscout.org', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'}, {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3316', 'term': 'Computer science'}], 'link': [{'url': 'https://neuroscout.org/builder', 'type': ['Other']}, {'url': 'https://neuroscout.github.io/neuroscout-paper/', 'type': ['Other']}, {'url': 'https://neuroscout.org/datasets', 'type': ['Other']}, {'url': 'https://github.com/neuroscout-datasets', 'type': ['Repository']}, {'url': 'https://neuroscout.org/predictors', 'type': ['Other']}], 'documentation': [{'url': 'https://neuroscout.org/api', 'type': ['API documentation']}], 'publication': [{'doi': '10.1101/2022.04.05.487222'}], 'credit': [{'name': 'Alejandro de la Vega', 'email': 'aleph4@gmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__neuroscout', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__neuroscout', 'publication_link': 'https://doi.org/10.1101/2022.04.05.487222', 'is_preprint': False}, {'name': 'GlycoQL', 'description': 'Daponte, V.; Hayes, C.; Mariethoz, J.; Lisacek, F.\\n\\nSubstructure located at the root.\\n\\nDealing with the Ambiguity of Glycan Substructure Search.\\n\\nMatch with exact same number of residue.\\n\\nFederated Queries & GlycoQL Query Patterns.', 'homepage': 'https://beta.glyconnect.expasy.org/glycoql/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_0570', 'term': 'Structure visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_0601', 'term': 'Protein modifications'}, {'uri': 'http://edamontology.org/topic_0166', 'term': 'Protein structural motifs and surfaces'}], 'link': [{'url': 'https://glyconnect.expasy.org/glystreem/wiki', 'type': ['Other']}, {'url': 'https://glyconnect.expasy.org/glycoql/', 'type': ['Other']}], 'credit': [{'email': 'frederique.lisacek@sib.swiss', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__glycoql', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__glycoql', 'publication': [{'doi': '10.1093/bioinformatics/btac500', 'pmid': '36124803'}], 'publication_link': 'https://doi.org/10.1093/bioinformatics/btac500', 'is_preprint': False}, {'name': 'SmGDB', 'description': 'genome database of Salvia miltiorrhiza.\\n\\nBackground Salvia miltiorrhiza is an important traditional Chinese medicinal (TCM) plant and a model plant in the genetic study of TCM. A series of omics related to Danshen have been published. Integrating, managing, storing, and sharing data has become an urgent problem to be solved in S. miltiorrhiza genetic studies. Objectives. The genome database is the link for the exchange, acquisition, and use of different omics data between data producers and users, maximizing value and utilization of data. Methods. The genome database included DSS3 genome and five RNA-Seq data. The back-end performs data search and retrieval through the LAMP (Linux, Apache, MySQL, PHP) framework. Results\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35460467', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0622', 'term': 'Genomics'}, {'uri': 'http://edamontology.org/topic_3407', 'term': 'Endocrinology and metabolism'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'language': ['SQL', 'PHP'], 'publication': [{'doi': '10.1007/S13258-022-01251-Y', 'pmid': '35460467'}], 'credit': [{'name': 'Xingfeng Li', 'email': 'lixf@sdau.edu.cn', 'typeEntity': 'Person'}, {'name': 'Zhenqiao Song', 'email': 'szq@sdau.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__smgdb', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__smgdb', 'publication_link': 'https://doi.org/10.1007/S13258-022-01251-Y', 'is_preprint': False}, {'name': 'SISS-Geo', 'description': 'Leveraging Citizen Science to Monitor Wildlife Health Risks in Brazil.\\n\\nSistema de Informação em Saúde Silvestre.\\n\\nCentro de Informação em Saúde Silvestre.\\n\\nTem Bicho e Tem Gente no SISS-Geo.\\n\\nEm 2024 queremos conhecer melhor nossos colaboradores e colaboradoras e compartilhar suas experiências com o uso da plataforma.\\n\\nInformation on Wild Health System - Fiocruz / LNCC.\\n\\nConheça o SISS‑Geo, Sistema de Informação em Saúde Silvestre.\\n\\nThe Information System in Health Silvestre - SISS Geo exists to generate from georeferenced animal observations, disease warning models in wildlife, especially those with human involvement potential, enabling surveillance and prevention to health actions.\\n\\nServiços e Informações do Brasil.\\n\\nO Sistema de Informação em Saúde Silvestre - SISS-Geo da FIOCRUZ é desenvolvido pela Plataforma Institucional Biodiversidade e Saúde Silvestre, com apoio do Laboratório Nacional de Computação Científica.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE SISS-Geo SISS-Geo', 'homepage': 'http://sissgeo.lncc.br', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3461', 'term': 'Virulence prediction'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0611', 'term': 'Electron microscopy'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}, {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}, {'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'}, {'uri': 'http://edamontology.org/topic_3344', 'term': 'Biomedical science'}], 'link': [{'url': 'http://www.biodiversidade.ciss.fiocruz.br', 'type': ['Other']}, {'url': 'https://play.google.com/store/apps/details?id=siss.ui', 'type': ['Other']}], 'publication': [{'doi': '10.1007/S41666-019-00055-2', 'pmid': '35415433', 'pmcid': 'PMC8982816'}], 'credit': [{'name': 'Luiz M. R. Gadelha', 'email': 'lgadelha@lncc.br', 'orcidid': 'https://orcid.org/0000-0002-8122-9522', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__siss-geo', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__siss-geo', 'publication_link': 'https://doi.org/10.1007/S41666-019-00055-2', 'is_preprint': False}, {'name': 'HiveRel', 'description': 'hexagons visualization for relationship-based knowledge acquisition.\\n\\nThe growing abundance in complex network data models is constantly increasing the challenges for non-expert users who perform an effective exploratory search in large data collections. In such domains, users search for entities related to a topic of interest and acquire knowledge by investigating the relationships between these entities. Designers, in turn, are challenged by the need to provide tools that enable convenient search and exploration to facilitate productive performance on the task. For this purpose, we introduce HiveRel, a search system that presents search results as tiled hexagons on a map-like surface with center-out relevance ordering and allows on-demand display of relationships between search results. HiveRel’s user interface is based on theoretical principles that reflect how users acquire knowledge through relationships\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://hive-rel-new.appspot.com/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3071', 'term': 'Biological databases'}, {'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'}], 'publication': [{'doi': '10.1007/S42486-022-00097-3', 'pmcid': 'PMC8997236'}], 'credit': [{'name': 'Guy Shani', 'email': 'shanigu@bgu.ac.il', 'orcidid': 'https://orcid.org/0000-0002-3655-7148', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__hiverel', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hiverel', 'publication_link': 'https://doi.org/10.1007/S42486-022-00097-3', 'is_preprint': False}, {'name': 'ROADMAPS', 'description': 'An Online Database of Response Data, Dosing Regimens, and Toxicities of Approved Oncology Drugs as Single Agents to Guide Preclinical In Vivo Studies.\\n\\nDevelopmental Therapeutics Program (DTP).\\n\\nROADMAPS provides publicly accessible preclinical data evaluating the effects of 70 different agents on 140 different xenograft models.\\n\\nOffice of the Associate Director.\\n\\nPreclinical Therapeutics Grants Branch.\\n\\nHollingshead MG, Greenberg N, Ahalt MG, Camalier R, Johnson BC, Collins JM, and Doroshow JH. ROADMAPS: An online database of response data, dosing regimens, and toxicities of approved oncology drugs as single agents to guide preclinical in vivo studies. Cancer Res. 2022 Apr 26. Online ahead of print. PMID: 35472132.', 'homepage': 'http://dtp.cancer.gov/databases_tools/roadmaps.htm', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}, {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_3375', 'term': 'Drug metabolism'}, {'uri': 'http://edamontology.org/topic_3374', 'term': 'Biotherapeutics'}], 'publication': [{'doi': '10.1158/0008-5472.CAN-21-4151', 'pmid': '35472132', 'pmcid': 'PMC9203935'}], 'credit': [{'name': 'Melinda G. Hollingshead', 'email': 'hollingm@mail.nih.gov', 'orcidid': 'https://orcid.org/0000-0002-1207-1397', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__roadmaps', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__roadmaps', 'publication_link': 'https://doi.org/10.1158/0008-5472.CAN-21-4151', 'is_preprint': False}, {'name': 'SNARER', 'description': 'new molecular descriptors for SNARE proteins classification.\\n\\nBackground SNARE proteins play an important role in different biological functions. This study aims to investigate the contribution of a new class of molecular descriptors (called SNARER) related to the chemical-physical properties of proteins in order to evaluate the performance of binary classifiers for SNARE proteins. Results. We constructed a SNARE proteins balanced dataset, D128, and an unbalanced one, DUNI, on which we tested and compared the performance of the new descriptors presented here in combination with the feature sets (GAAC, CTDT, CKSAAP and 188D) already present in the literature. The machine learning algorithms used were Random Forest, k-Nearest Neighbors and AdaBoost and oversampling and subsampling techniques were applied to the unbalanced dataset. The addition of the SNARER descriptors increases the precision for all considered ML algorithms\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/luigidibiasi/snarer', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_3478', 'term': 'Phylogenetic reconstruction'}, {'uri': 'http://edamontology.org/operation_0321', 'term': 'Protein structure validation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0623', 'term': 'Gene and protein families'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}], 'publication': [{'doi': '10.1186/S12859-022-04677-Z', 'pmid': '35462533', 'pmcid': 'PMC9035248'}], 'credit': [{'name': 'Alessia Auriemma Citarella', 'email': 'aauriemmacitarella@unisa.it', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__snarer', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__snarer', 'publication_link': 'https://doi.org/10.1186/S12859-022-04677-Z', 'is_preprint': False}, {'name': 'E2Style', 'description': 'Improve the Efficiency and Effectiveness of StyleGAN Inversion.\\n\\nThis paper studies the problem of StyleGAN inversion, which plays an essential role in enabling the pretrained StyleGAN to be used for real image editing tasks. The goal of StyleGAN inversion is to find the exact latent code of the given image in the latent space of StyleGAN. This problem has a high demand for quality and efficiency. Existing optimization-based methods can produce high-quality results, but the optimization often takes a long time. On the contrary, forward-based methods are usually faster but the quality of their results is inferior. In this paper, we present a new feed-forward network \"E2Style\" for StyleGAN inversion, with significant improvement in terms of efficiency and effectiveness\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/wty-ustc/e2style', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3096', 'term': 'Editing'}, {'uri': 'http://edamontology.org/operation_1812', 'term': 'Parsing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'publication': [{'doi': '10.1109/TIP.2022.3167305', 'pmid': '35439133'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__e2style', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__e2style', 'publication_link': 'https://doi.org/10.1109/TIP.2022.3167305', 'is_preprint': False}, {'name': 'OC_Finder', 'description': 'Osteoclast segmentation, counting, and classification using watershed and deep learning.\\n\\nOsteoclasts are multinucleated cells that exclusively resorb bone matrix proteins and minerals on the bone surface. They differentiate from monocyte macrophage-lineage cells in the presence of osteoclastogenic cytokines such as the receptor activator of nuclear factor-κB ligand (RANKL) and are stained positive for tartrate-resistant acid phosphatase (TRAP). In vitro, osteoclast formation assays are commonly used to assess the capacity of osteoclast precursor cells for differentiating into osteoclasts wherein the number of TRAP-positive multinucleated cells are counted as osteoclasts. Osteoclasts are manually identified on cell culture dishes by human eyes, which is a labor-intensive process. Moreover, the manual procedure is not objective and result in lack of reproducibility', 'homepage': 'https://bit.ly/OC_Finder', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3552', 'term': 'Microscope image visualisation'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}, {'uri': 'http://edamontology.org/operation_3553', 'term': 'Image annotation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_0108', 'term': 'Protein expression'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.3389/FBINF.2022.819570', 'pmid': '35474753', 'pmcid': 'PMC9038109'}], 'credit': [{'name': 'Daisuke Kihara', 'email': 'dkihara@purdue.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__oc_finder', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__oc_finder', 'publication_link': 'https://doi.org/10.3389/FBINF.2022.819570', 'is_preprint': False}, {'name': 'citationchaser', 'description': \"A tool for transparent and efficient forward and backward citation chasing in systematic searching.\\n\\nSystematic searching aims to find all possibly relevant research from multiple sources, the basis for an unbiased and comprehensive evidence base. Along with bibliographic databases, systematic reviewers use a variety of additional methods to minimise procedural bias. Citation chasing exploits connections between research articles to identify relevant records for a review by making use of explicit mentions of one article within another. Citation chasing is a popular supplementary search method because it helps to build on the work of primary research and review authors. It does so by identifying potentially relevant studies that might otherwise not be retrieved by other search methods; for example, because they did not use the review authors' search terms in the specified combinations in their titles, abstracts, or keywords\", 'homepage': 'https://estech.shinyapps.io/citationchasertest/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}, {'uri': 'http://edamontology.org/topic_3071', 'term': 'Biological databases'}], 'language': ['R'], 'link': [{'url': 'https://estech.shinyapps.io/citationchaser', 'type': ['Other']}], 'publication': [{'doi': '10.1002/JRSM.1563', 'pmid': '35472127'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__citationchaser', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__citationchaser', 'publication_link': 'https://doi.org/10.1002/JRSM.1563', 'is_preprint': False}, {'name': 'Spiky', 'description': 'An ImageJ Plugin for Data Analysis of Functional Cardiac and Cardiomyocyte Studies.\\n\\nA peak analyzer plugin for ImageJ.\\n\\nSpiky (A peak analyzer plugin for ImageJ).\\n\\n\"Spiky: an ImageJ plugin for data analysis of functional cardiac and cardiomyocyte studies.\" Côme Pasqualin, François Gannier, Angele Yu, David Benoist, Ian Findlay, Romain Bordy, Pierre Bredeloux, Véronique Maupoil.\\n\\n||| NAME (Spiky) SIMILAR TO (PUB. DIFFERENT) bio.tools/spiky (spiky)', 'homepage': 'http://pccv.univ-tours.fr/ImageJ/Spiky', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3215', 'term': 'Peak detection'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}, {'uri': 'http://edamontology.org/operation_3222', 'term': 'Peak calling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}], 'language': ['Java'], 'publication': [{'doi': '10.3390/JIMAGING8040095', 'pmid': '35448222', 'pmcid': 'PMC9026474'}], 'credit': [{'name': 'Côme Pasqualin', 'email': 'come.pasqualin@univ-tours.fr', 'orcidid': 'https://orcid.org/0000-0001-8625-1617', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__spiky', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__spiky', 'publication_link': 'https://doi.org/10.3390/JIMAGING8040095', 'is_preprint': False}, {'name': 'surge', 'description': 'a fast open-source chemical graph generator.\\n\\nBrendan D. McKay, Christoph Steinbeck and Mehmet Aziz Yirik.\\n\\nSurge is an open-source program for generating the graphs of chemical structures.\\n\\nSurge is a chemical structure generator based on the canonical generation path method. Surge uses the Nauty package to compute automorphism groups of graphs and generates all non-isomorphic constitutional isomers of a given molecular formula. Surge is available under a liberal open source license.\\n\\nThe paper describing the program is: Brendan D. McKay, Mehmet Aziz Yirik and Christoph Steinbeck, Surge: a fast open-source chemical graph generator, Journal of Cheminformatics, 14, article 24 (2002).', 'homepage': 'https://structuregenerator.github.io', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3938', 'term': 'Virtual screening'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2258', 'term': 'Cheminformatics'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3370', 'term': 'Analytical chemistry'}], 'link': [{'url': 'https://users.cecs.anu.edu.au/~bdm/surge/', 'type': ['Other']}], 'publication': [{'doi': '10.1186/S13321-022-00604-9', 'pmid': '35461261', 'pmcid': 'PMC9034616'}], 'credit': [{'name': 'Brendan D. McKay', 'email': 'brendan.mckay@anu.edu.au', 'orcidid': 'https://orcid.org/0000-0002-3553-0496', 'typeEntity': 'Person'}, {'name': 'Christoph Steinbeck', 'email': 'christoph.steinbeck@uni-jena.de', 'orcidid': 'https://orcid.org/0000-0001-6966-0814', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__surge', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__surge', 'publication_link': 'https://doi.org/10.1186/S13321-022-00604-9', 'is_preprint': False}, {'name': 'ZZS similarity tool', 'description': \"The online tool for similarity screening to identify chemicals of potential concern.\\n\\nRijksinstituut voor Volksgezondheid en Milieu.\\n\\nZZS-similarity tool Risico's van Stoffen.\", 'homepage': 'https://rvszoeksysteem.rivm.nl/ZzsSimilarityTool', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3938', 'term': 'Virtual screening'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2840', 'term': 'Toxicology'}, {'uri': 'http://edamontology.org/topic_3343', 'term': 'Compound libraries and screening'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'}], 'publication': [{'doi': '10.1002/JCC.26859', 'pmid': '35403727', 'pmcid': 'PMC9322536'}], 'credit': [{'name': 'Pim N. H. Wassenaar', 'email': 'pim.wassenaar@rivm.nl', 'orcidid': 'https://orcid.org/0000-0001-8155-861X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__zzs-similarity-tool', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__zzs-similarity-tool', 'publication_link': 'https://doi.org/10.1002/JCC.26859', 'is_preprint': False}, {'name': 'Pocket2Drug', 'description': 'An Encoder-Decoder Deep Neural Network for the Target-Based Drug Design.\\n\\nComputational modeling is an essential component of modern drug discovery. One of its most important applications is to select promising drug candidates for pharmacologically relevant target proteins. Because of continuing advances in structural biology, putative binding sites for small organic molecules are being discovered in numerous proteins linked to various diseases. These valuable data offer new opportunities to build efficient computational models predicting binding molecules for target sites through the application of data mining and machine learning. In particular, deep neural networks are powerful techniques capable of learning from complex data in order to make informed drug binding predictions. In this communication, we describe Pocket2Drug, a deep graph neural network model to predict binding molecules for a given a ligand binding site\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/shiwentao00/Pocket2Drug', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3897', 'term': 'Ligand-binding site prediction'}, {'uri': 'http://edamontology.org/operation_0482', 'term': 'Protein-ligand docking'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}], 'publication': [{'doi': '10.3389/FPHAR.2022.837715', 'pmid': '35359869', 'pmcid': 'PMC8962739'}], 'credit': [{'name': 'Michal Brylinski', 'email': 'michal@brylinski.org', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__pocket2drug', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pocket2drug', 'publication_link': 'https://doi.org/10.3389/FPHAR.2022.837715', 'is_preprint': False}, {'name': 'IPOscore', 'description': 'An interactive web-based platform for postoperative surgical complications analysis and prediction in the oncology domain.\\n\\nBackground The performance of traditional risk score systems to predict (post)-operative outcomes is limited. This weakness reduces confidence in its use to support clinical risk mitigation decisions. However, the rapid growth of health data in the last years offers principles to deal with some of these limitations. In this regard, the data allows the extraction of relevant information for both patients stratification and the rigorous identification of associated risk factors. The patients can then be targeted to specific preoperative optimization programs, thus contributing to the reduction of associated morbidity and mortality. Objectives. The main goal of this work is, therefore, to provide a clinical decision support system (CDSS) based on data-driven modeling methods for surgical risk prediction specific for cancer patients in Portugal. Results', 'homepage': 'https://iposcore.org', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0303', 'term': 'Fold recognition'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3473', 'term': 'Data mining'}], 'publication': [{'doi': '10.1016/J.CMPB.2022.106754', 'pmid': '35364482'}], 'credit': [{'name': 'Rafael S Costa', 'email': 'rs.costa@fct.unl.pt', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__iposcore', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__iposcore', 'publication_link': 'https://doi.org/10.1016/J.CMPB.2022.106754', 'is_preprint': False}, {'name': 'TCR-L', 'description': 'an analysis tool for evaluating the association between the T-cell receptor repertoire and clinical phenotypes.\\n\\nBackground T cell receptors (TCRs) play critical roles in adaptive immune responses, and recent advances in genome technology have made it possible to examine the T cell receptor (TCR) repertoire at the individual sequence level. The analysis of the TCR repertoire with respect to clinical phenotypes can yield novel insights into the etiology and progression of immune-mediated diseases. However, methods for association analysis of the TCR repertoire have not been well developed. Methods. We introduce an analysis tool, TCR-L, for evaluating the association between the TCR repertoire and disease outcomes\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/rksyouyou/TCRL-pkg', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_0416', 'term': 'Epitope mapping'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_2830', 'term': 'Immunoproteins and antigens'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}], 'publication': [{'doi': '10.1186/S12859-022-04690-2', 'pmid': '35484495', 'pmcid': 'PMC9052542'}], 'credit': [{'name': 'Qianchuan He', 'email': 'qhe@fredhutch.org', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tcr-l', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tcr-l', 'publication_link': 'https://doi.org/10.1186/S12859-022-04690-2', 'is_preprint': False}, {'name': 'Rizoma', 'description': 'a new comprehensive database on traditional uses of Chilean native plants.\\n\\nContiene registros de usos para más de 700 especies de plantas nativas de Chile.', 'homepage': 'https://ceab-rizoma.com/database/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0610', 'term': 'Ecology'}, {'uri': 'http://edamontology.org/topic_3071', 'term': 'Biological databases'}], 'publication': [{'doi': '10.3897/BDJ.10.E80002', 'pmid': '35437394', 'pmcid': 'PMC8917114'}], 'credit': [{'name': 'Sebastián Cordero', 'email': 'sebastian.cordero.r@gmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rizoma', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rizoma', 'publication_link': 'https://doi.org/10.3897/BDJ.10.E80002', 'is_preprint': False}, {'name': 'MOOD', 'description': 'A Public Benchmark for Out-of-Distribution Detection and Localization on Medical Images.\\n\\nDetecting Out-of-Distribution (OoD) data is one of the greatest challenges in safe and robust deployment of machine learning algorithms in medicine. When the algorithms encounter cases that deviate from the distribution of the training data, they often produce incorrect and over-confident predictions. OoD detection algorithms aim to catch erroneous predictions in advance by analysing the data distribution and detecting potential instances of failure. Moreover, flagging OoD cases may support human readers in identifying incidental findings. Due to the increased interest in OoD algorithms, benchmarks for different domains have recently been established. In the medical imaging domain, for which reliable predictions are often essential, an open benchmark has been missing\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/mic-dkfz/mood', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0303', 'term': 'Fold recognition'}, {'uri': 'http://edamontology.org/operation_3283', 'term': 'Anonymisation'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3303', 'term': 'Medicine'}, {'uri': 'http://edamontology.org/topic_3452', 'term': 'Tomography'}], 'publication': [{'doi': '10.1109/TMI.2022.3170077', 'pmid': '35468060'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mood', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mood', 'publication_link': 'https://doi.org/10.1109/TMI.2022.3170077', 'is_preprint': False}, {'name': 'ClinicaDL', 'description': 'An open-source deep learning software for reproducible neuroimaging processing.\\n\\nFramework for the reproducible processing of neuroimaging data with deep learning methods.\\n\\nThis repository hosts ClinicaDL, the deep learning extension of Clinica, a python library to process neuroimaging data in BIDS format.\\n\\nDocumentation : Tutorial : Forum.\\n\\nTo access the full documentation of the project, follow the link https://clinicadl.readthedocs.io/. If you find a problem when using it or if you want to provide us feedback, please open an issue or write on the forum.', 'homepage': 'http://aramislab.paris.inria.fr/files/data/models/dl/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0335', 'term': 'Formatting'}, {'uri': 'http://edamontology.org/operation_3431', 'term': 'Deposition'}, {'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}], 'language': ['Python'], 'license': 'MIT', 'link': [{'url': 'http://pypi.org/project/clinicadl', 'type': ['Repository']}], 'publication': [{'doi': '10.1016/J.CMPB.2022.106818', 'pmid': '35483271'}], 'credit': [{'name': 'Ninon Burgos', 'email': 'ninon.burgos@cnrs.fr', 'orcidid': 'https://orcid.org/0000-0002-4668-2006', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__clinicadl', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__clinicadl', 'publication_link': 'https://doi.org/10.1016/J.CMPB.2022.106818', 'is_preprint': False}, {'name': 'DF-SSmVEP', 'description': 'Dual Frequency Aggregated Steady-State Motion Visual Evoked Potential Design with Bifold Canonical Correlation Analysis.\\n\\nRecent advancements in Electroencephalographic (EEG) sensor technologies and signal processing algorithms have paved the way for further evolution of Brain Computer Interfaces (BCI) in several practical applications, ranging from rehabilitation systems to smart consumer technologies. When it comes to Signal Processing (SP) for BCI, there has been a surge of interest on Steady-State motion Visual Evoked Potentials (SSmVEP), where motion stimulation is used to address key issues associated with conventional light flashing flickering. Such benefits, however, come with the price of being less accurate and having a lower Information Transfer Rate (ITR). From this perspective, this paper focuses on the design of a novel SSmVEP paradigm without using resources such as trial time, phase, and or number of targets to enhance the ITR\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/raykakarimi/DFSSMVEP', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3299', 'term': 'Evolutionary biology'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}], 'publication': [{'doi': '10.3390/S22072568', 'pmid': '35408182', 'pmcid': 'PMC9003394'}], 'credit': [{'name': 'Arash Mohammadi', 'email': 'arash.mohammadi@concordia.ca', 'orcidid': 'https://orcid.org/0000-0003-1972-7923', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__df-ssmvep', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__df-ssmvep', 'publication_link': 'https://doi.org/10.3390/S22072568', 'is_preprint': False}, {'name': 'MainSEL', 'description': 'Record Linkage based Patient Intersection Cardinality for Rare Disease Studies using Mainzelliste and Secure Multi-Party Computation.\\n\\nBackground: The low number of patients of any rare diseases poses a difficult problem for medical research: apart from some specialized biobanks and disease registries, potential study participants’ information are disjoint and distributed over many medical institutions. Whenever some of those facilities are in close proximity, a significant overlap of patients can reasonably be expected, further complicating statistical study feasibility assessments and data gathering. Due to the sensitive nature of medical records and identifying data, data transfer and joint computations are often legally forbidden or associated with prohibitive amounts of effort\\n\\n||| HOMEPAGE BROKEN!\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/mainsel', 'homepage': 'https://github.com/medicalinformatics/MainSEL', 'topic': [{'uri': 'http://edamontology.org/topic_3325', 'term': 'Rare diseases'}, {'uri': 'http://edamontology.org/topic_3337', 'term': 'Biobank'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}, {'uri': 'http://edamontology.org/topic_3063', 'term': 'Medical informatics'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mainsel', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mainsel', 'publication': [{'doi': '10.1186/s12967-022-03671-6', 'pmid': '36209221', 'pmcid': 'PMC9547637'}], 'publication_link': 'https://doi.org/10.1186/s12967-022-03671-6', 'is_preprint': False}, {'name': 'RRGP', 'description': 'A model for collaborative multidisciplinary practice-research partnerships.\\n\\nBackground: Little attention has been given to the process of implementing or evaluating a structured academic clinician (University-Health Service) research capacity building model within healthcare settings. We have developed a model for collaborative multidisciplinary practice-research partnerships called Research Ready Grant Program (RRGP). The RRGP is informed by Cooke (1)’s Research Capacity Building (RCB) framework and principles. The aim of the study outlined in this protocol is to conduct a process and outcome evaluation of the program. We will explore how the RRGP structured mentor model contributes to research capacity building (RCB) of clinician-led multidisciplinary research teams. We will identify key factors at the organization, team, and individual levels that affect research capacity of health professionals working in one regional health service district\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://doi.org/10.21203/RS.3.RS-1558595/V1', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}, {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}, {'uri': 'http://edamontology.org/topic_3376', 'term': 'Medicines research and development'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rrgp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rrgp', 'publication': [{'doi': '10.1186/s12961-022-00870-x', 'pmid': '35698128', 'pmcid': 'PMC9195363'}], 'publication_link': 'https://doi.org/10.1186/s12961-022-00870-x', 'is_preprint': False}, {'name': 'COPILOT', 'description': 'The COPILOT Raw Illumina Genotyping QC Protocol.\\n\\nContainerised Workflow for Processing Illumina Genotyping Data.\\n\\nWebsites for you and your projects, hosted directly from your GitHub repository. Just edit, push, and your changes are live.\\n\\nWebsites for you and your projects.\\n\\nCOPILOT is a Containerised wOrkflow for Processing ILlumina genOtyping daTa. The container is deployable on multiple platforms and allows users to quickly and effortlessly prepare, pre-process and understand Illumina microarray SNP data for data analysis through an informative and interactive summary report.\\n\\nHosted directly from your GitHub repository. Just edit, push, and your changes are live.\\n\\nHere we introduce COPILOT, a Containerised wOrkflow for Processing ILlumina genOtyping daTa.\\n\\nReady to get started.\\n\\nDr Hamel Patel hamel.patel@kcl.ac.uk.', 'homepage': 'https://khp-informatics.github.io/COPILOT/index.html', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}, {'uri': 'http://edamontology.org/operation_0484', 'term': 'SNP detection'}, {'uri': 'http://edamontology.org/operation_3096', 'term': 'Editing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3518', 'term': 'Microarray experiment'}, {'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'}, {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}], 'link': [{'url': 'http://github.io/COPILOT/', 'type': ['Other']}], 'documentation': [{'url': 'https://khp-informatics.github.io/COPILOT/COPILOT_user_guide.html', 'type': ['User manual']}], 'publication': [{'doi': '10.1002/CPZ1.373', 'pmid': '35452565'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__copilot', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__copilot', 'publication_link': 'https://doi.org/10.1002/CPZ1.373', 'is_preprint': False}, {'name': 'iCatcher', 'description': \"A neural network approach for automated coding of young children's eye movements.\\n\\nInfants' looking behaviors are often used for measuring attention, real-time processing, and learning-often using low-resolution videos. Despite the ubiquity of gaze-related methods in developmental science, current analysis techniques usually involve laborious post hoc coding, imprecise real-time coding, or expensive eye trackers that may increase data loss and require a calibration phase. As an alternative, we propose using computer vision methods to perform automatic gaze estimation from low-resolution videos. At the core of our approach is a neural network that classifies gaze directions in real time. We compared our method, called iCatcher, to manually annotated videos from a prior study in which infants looked at one of two pictures on a screen. We demonstrated that the accuracy of iCatcher approximates that of human annotators and that it replicates the prior study's results\\n\\n||| HOMEPAGE BROKEN!\", 'homepage': 'https://github.com/yoterel/iCatcher', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}, {'uri': 'http://edamontology.org/topic_3418', 'term': 'Paediatrics'}], 'publication': [{'doi': '10.1111/INFA.12468', 'pmid': '35416378', 'pmcid': 'PMC9320879'}], 'credit': [{'name': 'Yotam Erel', 'email': 'erelyotam@gmail.com', 'orcidid': 'https://orcid.org/0000-0001-8319-5111', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__icatcher', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__icatcher', 'publication_link': 'https://doi.org/10.1111/INFA.12468', 'is_preprint': False}, {'name': 'COVID-19-associated ARDS non-viral ARDS', 'description': 'Repurposing Multiple-Molecule Drugs for COVID-19-Associated Acute Respiratory Distress Syndrome and Non-Viral Acute Respiratory Distress Syndrome via a Systems Biology Approach and a DNN-DTI Model Based on Five Drug Design Specifications.\\n\\nThe coronavirus disease 2019 (COVID-19) epidemic is currently raging around the world at a rapid speed. Among COVID-19 patients, SARS-CoV-2-associated acute respiratory distress syndrome (ARDS) is the main contribution to the high ratio of morbidity and mortality. However, clinical manifestations between SARS-CoV-2-associated ARDS and non-SARS-CoV-2-associated ARDS are quite common, and their therapeutic treatments are limited because the intricated pathophysiology having been not fully understood\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE ARDS non-viral ARDS, COVID-19-associated ARDS non-viral', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35409008', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3223', 'term': 'Differential gene expression profiling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}], 'publication': [{'doi': '10.3390/IJMS23073649', 'pmid': '35409008', 'pmcid': 'PMC8998971'}], 'credit': [{'name': 'Bor-Sen Chen', 'email': 'bschen@ee.nthu.edu.tw', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__covid-19-associated-ards-non-viral-ards', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__covid-19-associated-ards-non-viral-ards', 'publication_link': 'https://doi.org/10.3390/IJMS23073649', 'is_preprint': False}, {'name': 'AutoDesigner', 'description': '<i>De Novo</i> Design Algorithm for Rapidly Exploring Large Chemical Space for Lead Optimization.\\n\\nThe lead optimization stage of a drug discovery program generally involves the design, synthesis, and assaying of hundreds to thousands of compounds. The design phase is usually carried out via traditional medicinal chemistry approaches and or structure-based drug design (SBDD) when suitable structural information is available. Two of the major limitations of this approach are (1) difficulty in rapidly designing potent molecules that adhere to myriad project criteria, or the multiparameter optimization (MPO) problem, and (2) the relatively small number of molecules explored compared to the vast size of chemical space. To address these limitations, we have developed AutoDesigner, a <i>de novo< i> design algorithm. AutoDesigner employs a cloud-native, multistage search algorithm to carry out successive rounds of chemical space exploration and filtering\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35417149', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_0479', 'term': 'Backbone modelling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3303', 'term': 'Medicine'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}], 'publication': [{'doi': '10.1021/ACS.JCIM.2C00072', 'pmid': '35417149'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__autodesigner', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__autodesigner', 'publication_link': 'https://doi.org/10.1021/ACS.JCIM.2C00072', 'is_preprint': False}, {'name': 'MILNP', 'description': 'Plant lncRNA-miRNA Interaction Prediction Based on Improved Linear Neighborhood Similarity and Label Propagation.\\n\\nKnowledge of the interactions between long non-coding RNAs (lncRNAs) and microRNAs (miRNAs) is the basis of understanding various biological activities and designing new drugs. Previous computational methods for predicting lncRNA-miRNA interactions lacked for plants, and they suffer from various limitations that affect the prediction accuracy and their applicability. Research on plant lncRNA-miRNA interactions is still in its infancy. In this paper, we propose an accurate predictor, MILNP, for predicting plant lncRNA-miRNA interactions based on improved linear neighborhood similarity measurement and linear neighborhood propagation algorithm\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/HerSwain/gra/tree/MILNP', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3792', 'term': 'miRNA expression analysis'}, {'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_0463', 'term': 'miRNA target prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0659', 'term': 'Functional, regulatory and non-coding RNA'}, {'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.3389/FPLS.2022.861886', 'pmid': '35401586', 'pmcid': 'PMC8990282'}], 'credit': [{'name': 'Xiangzheng Fu', 'typeEntity': 'Person'}, {'name': 'Peng Wang', 'email': 'wangpenglw@126.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__milnp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__milnp', 'publication_link': 'https://doi.org/10.3389/FPLS.2022.861886', 'is_preprint': False}, {'name': 'PeakVI', 'description': 'A deep generative model for single-cell chromatin accessibility analysis.\\n\\nSingle-cell ATAC sequencing (scATAC-seq) is a powerful and increasingly popular technique to explore the regulatory landscape of heterogeneous cellular populations. However, the high noise levels, degree of sparsity, and scale of the generated data make its analysis challenging. Here, we present PeakVI, a probabilistic framework that leverages deep neural networks to analyze scATAC-seq data. PeakVI fits an informative latent space that preserves biological heterogeneity while correcting batch effects and accounting for technical effects, such as library size and region-specific biases. In addition, PeakVI provides a technique for identifying differential accessibility at a single-region resolution, which can be used for cell-type annotation as well as identification of key <i>cis< i>-regulatory elements\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35475224', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3935', 'term': 'Dimensionality reduction'}, {'uri': 'http://edamontology.org/operation_3222', 'term': 'Peak calling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3940', 'term': 'Chromosome conformation capture'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}], 'publication': [{'doi': '10.1016/J.CRMETH.2022.100182', 'pmid': '35475224', 'pmcid': 'PMC9017241'}], 'credit': [{'name': 'Nir Yosef', 'email': 'niryosef@berkeley.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__peakvi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__peakvi', 'publication_link': 'https://doi.org/10.1016/J.CRMETH.2022.100182', 'is_preprint': False}, {'name': 'DTSyn', 'description': 'a dual-transformer-based neural network to predict synergistic drug combinations.\\n\\nDrug combination therapies are superior to monotherapy for cancer treatment in many ways when addressing tumor heterogeneity issue. For wet-lab experiment, screening out novel synergistic drug pairs is challenging due to the enormous searching space of possible drug pairs. Thus, computational methods have been developed to predict drug pairs with potential synergistic function. Notwithstanding the success of current models, the power of generalization to other datasets as wells as understanding of mechanism for chemical-chemical interaction or chemical-sample interaction are lack of study, hindering current algorithms from real application. In this paper, we proposed a deep neural model termed DTSyn (Dual Transformer model for drug pair Synergy prediction) based on multi-head attention mechanism to identify novel drug combinations\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://doi.org/10.1101/2022.03.29.486200', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}, {'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}], 'credit': [{'name': 'Guodong Zhao', 'email': 'zhaoguodong@baidu.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__dtsyn', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dtsyn', 'publication': [{'doi': '10.1093/bib/bbac302', 'pmid': '35915050'}], 'publication_link': 'https://doi.org/10.1093/bib/bbac302', 'is_preprint': False}, {'name': 'LinQ-View', 'description': 'Improved integration of single-cell transcriptome and surface protein expression by LinQ-View.\\n\\njoint analysis for multi-modal data.\\n\\nLinQ-View: Joint analysis for multi-modal data Wilson Lab The University of Chicago.\\n\\nIntegrating multiple modalities for precise cell heterogeneity identification.\\n\\n“LinQ-View” was designed as a joint single cell analysis strategy that could integrate information from both transcriptome and surface protein markers for cell heterogeneity identification.', 'homepage': 'https://wilsonimmunologylab.github.io/LinQView', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_0313', 'term': 'Expression profile clustering'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_0108', 'term': 'Protein expression'}, {'uri': 'http://edamontology.org/topic_0166', 'term': 'Protein structural motifs and surfaces'}, {'uri': 'http://edamontology.org/topic_2830', 'term': 'Immunoproteins and antigens'}], 'publication': [{'doi': '10.1016/J.CRMETH.2021.100056', 'pmid': '35475142', 'pmcid': 'PMC9017149'}], 'credit': [{'name': 'Aly A. Khan', 'email': 'aakhan@uchicago.edu', 'typeEntity': 'Person'}, {'name': 'Patrick C. Wilson', 'email': 'wilsonp@uchicago.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__linq-view', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__linq-view', 'publication_link': 'https://doi.org/10.1016/J.CRMETH.2021.100056', 'is_preprint': False}, {'name': 'WalkIm', 'description': 'Compact image-based encoding for high-performance classification of biological sequences using simple tuning-free CNNs.\\n\\nThe classification of biological sequences is an open issue for a variety of data sets, such as viral and metagenomics sequences. Therefore, many studies utilize neural network tools, as the well-known methods in this field, and focus on designing customized network structures. However, a few works focus on more effective factors, such as input encoding method or implementation technology, to address accuracy and efficiency issues in this area. Therefore, in this work, we propose an image-based encoding method, called as WalkIm, whose adoption, even in a simple neural network, provides competitive accuracy and superior efficiency, compared to the existing classification methods (e.g. VGDC, CASTOR, and DLM-CNN) for a variety of biological sequences\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35427371', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0363', 'term': 'Reverse complement'}, {'uri': 'http://edamontology.org/operation_3472', 'term': 'k-mer counting'}, {'uri': 'http://edamontology.org/operation_0292', 'term': 'Sequence alignment'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_0781', 'term': 'Virology'}, {'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}], 'publication': [{'doi': '10.1371/JOURNAL.PONE.0267106', 'pmid': '35427371', 'pmcid': 'PMC9012348'}], 'credit': [{'name': 'Somayyeh Koohi', 'email': 'koohi@sharfi.edu', 'orcidid': 'https://orcid.org/0000-0002-3105-2511', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__walkim', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__walkim', 'publication_link': 'https://doi.org/10.1371/JOURNAL.PONE.0267106', 'is_preprint': False}, {'name': 'PhenoBERT', 'description': 'A Combined Deep Learning Method for Automated Recognition of Human Phenotype Ontology.\\n\\nAutomated recognition of Human Phenotype Ontology (HPO) terms from clinical texts is of significant interest to the field of clinical data mining. In this study, we develop a combined deep learning method named PhenoBERT for this purpose. PhenoBERT uses BERT, currently the state-of-the-art NLP model, as its core model for evaluating whether a clinically relevant text segment (CTS) could be represented by an HPO term. However, to avoid unnecessary comparison of a CTS with each of ∼14,000 HPO terms using BERT, we introduce a two-levels CNN module consisting of a series of CNN models organized at two levels in PhenoBERT. For a given CTS, the CNN module produces only a short list of candidate HPO terms for BERT to evaluate, significantly improving the computational efficiency\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35471885', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0306', 'term': 'Text mining'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0218', 'term': 'Natural language processing'}, {'uri': 'http://edamontology.org/topic_3452', 'term': 'Tomography'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_3473', 'term': 'Data mining'}], 'publication': [{'doi': '10.1109/TCBB.2022.3170301', 'pmid': '35471885'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__phenobert', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__phenobert', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3170301', 'is_preprint': False}, {'name': 'CPIELA', 'description': 'Computational Prediction of Plant Protein-Protein Interactions by Ensemble Learning Approach From Protein Sequences and Evolutionary Information.\\n\\nIdentification and characterization of plant protein-protein interactions (PPIs) are critical in elucidating the functions of proteins and molecular mechanisms in a plant cell. Although experimentally validated plant PPIs data have become increasingly available in diverse plant species, the high-throughput techniques are usually expensive and labor-intensive. With the incredibly valuable plant PPIs data accumulating in public databases, it is progressively important to propose computational approaches to facilitate the identification of possible PPIs. In this article, we propose an effective framework for predicting plant PPIs by combining the position-specific scoring matrix (PSSM), local optimal-oriented pattern (LOOP), and ensemble rotation forest (ROF) model\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35360876', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2492', 'term': 'Protein interaction prediction'}, {'uri': 'http://edamontology.org/operation_2464', 'term': 'Protein-protein binding site prediction'}, {'uri': 'http://edamontology.org/operation_0481', 'term': 'Loop modelling'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0128', 'term': 'Protein interactions'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3957', 'term': 'Protein interaction experiment'}, {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}], 'publication': [{'doi': '10.3389/FGENE.2022.857839', 'pmid': '35360876', 'pmcid': 'PMC8963800'}], 'credit': [{'name': 'Li-Ping Li', 'typeEntity': 'Person'}, {'name': 'Bo Zhang', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__cpiela', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cpiela', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.857839', 'is_preprint': False}, {'name': 'PCRMS', 'description': 'a database of predicted cis-regulatory modules and constituent transcription factor binding sites in genomes.\\n\\nMore accurate and more complete predictions of cis-regulatory modules (CRMs) and constituent transcription factor (TF) binding sites (TFBSs) in genomes can facilitate characterizing functions of regulatory sequences. Here, we developed a database predicted cis-regulatory modules (PCRMS) (https: cci-bioinfo.uncc.edu) that stores highly accurate and unprecedentedly complete maps of predicted CRMs and TFBSs in the human and mouse genomes. The web interface allows the user to browse CRMs and TFBSs in an organism, find the closest CRMs to a gene, search CRMs around a gene and find all TFBSs of a TF. PCRMS can be a useful resource for the research community to characterize regulatory genomes. Database URL: https: cci-bioinfo.uncc.edu\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE cci-bioinfo', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35452518', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0445', 'term': 'Transcription factor binding site prediction'}, {'uri': 'http://edamontology.org/operation_2575', 'term': 'Binding site prediction'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_0441', 'term': 'cis-regulatory element prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}, {'uri': 'http://edamontology.org/topic_0204', 'term': 'Gene regulation'}, {'uri': 'http://edamontology.org/topic_3169', 'term': 'ChIP-seq'}, {'uri': 'http://edamontology.org/topic_3295', 'term': 'Epigenetics'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}], 'publication': [{'doi': '10.1093/DATABASE/BAAC024', 'pmid': '35452518', 'pmcid': 'PMC9216522'}], 'credit': [{'name': 'Zhengchang Su', 'email': 'zcsu@uncc.edu', 'orcidid': 'https://orcid.org/0000-0003-4636-3440', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__pcrms', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pcrms', 'publication_link': 'https://doi.org/10.1093/DATABASE/BAAC024', 'is_preprint': False}, {'name': 'MPI-GWAS', 'description': 'a supercomputing-aided permutation approach for genomewide association studies.\\n\\nPermutation testing is a robust and popular approach for significance testing in genomic research that has the advantage of reducing inflated type 1 error rates; however, its computational cost is notorious in genome-wide association studies (GWAS). Here, we developed a supercomputing-aided approach to accelerate the permutation testing for GWAS, based on the message-passing interface (MPI) on parallel computing architecture. Our application, called MPI-GWAS, conducts MPI-based permutation testing using a parallel computing approach with our supercomputing system, Nurion (8,305 compute nodes, and 563,740 central processing units [CPUs]). For 107 permutations of one locus in MPI-GWAS, it was calculated in 600 s using 2,720 CPU cores. For 107 permutations of ~30,000-50,000 loci in over 7,000 subjects, the total elapsed time was ~4 days in the Nurion supercomputer\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/hypaik/proj_MPIGWAS', 'topic': [{'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'}, {'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}], 'publication': [{'doi': '10.5808/GI.22001', 'pmid': '35399013', 'pmcid': 'PMC9001997'}], 'credit': [{'name': 'Oh-Kyoung Kwon', 'email': 'okkwon@kisti.re.kr', 'orcidid': 'https://orcid.org/0000-0001-9734-9257', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mpi-gwas', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mpi-gwas', 'publication_link': 'https://doi.org/10.5808/GI.22001', 'is_preprint': False}, {'name': 'MAPPER', 'description': 'An Open-Source, High-Dimensional Image Analysis Pipeline Unmasks Differential Regulation of <i>Drosophila</i> Wing Features.\\n\\nAn open-source, high-dimensional image analysis pipeline unmasks differential regulation of Drosophila wing features.\\n\\nMulticellular Systems and Engineering Lab.\\n\\n||| NAME (MAPPER) SIMILAR TO (PUB. DIFFERENT) bio.tools/Mapper (Mapper)', 'homepage': 'https://multicellularsystemslab.github.io/MAPPER/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3298', 'term': 'Phenomics'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3398', 'term': 'Bioengineering'}, {'uri': 'http://edamontology.org/topic_3064', 'term': 'Developmental biology'}], 'publication': [{'doi': '10.3389/FGENE.2022.869719', 'pmid': '35480325', 'pmcid': 'PMC9035675'}], 'credit': [{'name': 'Jeremiah J. Zartman', 'email': 'jzartman@nd.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mapper', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mapper', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.869719', 'is_preprint': False}, {'name': 'DeSmoke-LAP', 'description': 'improved unpaired image-to-image translation for desmoking in laparoscopic surgery.\\n\\nEPSRC Centre for Interventional and Surgical Sciences.\\n\\nThis is the publicly available dataset from robot-assisted laparoscopic hysterectomy surgery providing a benchmark for designing and validating smoke removal algorithms.\\n\\nWatch a short video about DeSmoke-LAP here.\\n\\nThe laparoscopic surgery dataset is associated with our International Journal of Computer Assisted Radiology and Surgery (IJCARS) publication titled “DeSmoke-LAP: Improved Unpaired Image-to-Image Translation for Desmoking in Laparoscopic Surgery”. The training model of the proposed method is available as an open source on Github. We propose DeSmoke-LAP, a new method for removing smoke from real robotic laparoscopic hysterectomy videos. The proposed method is based on the unpaired image-to-image cycle-consistent generative adversarial network in which two novel loss functions, namely, inter-channel discrepancies and dark channel prior.', 'homepage': 'https://www.ucl.ac.uk/interventional-surgical-sciences/weiss-open-research/weiss-open-data-server/desmoke-lap', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}, {'uri': 'http://edamontology.org/operation_3553', 'term': 'Image annotation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3421', 'term': 'Surgery'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_0108', 'term': 'Protein expression'}, {'uri': 'http://edamontology.org/topic_3794', 'term': 'RNA immunoprecipitation'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1007/S11548-022-02595-2', 'pmid': '35355212', 'pmcid': 'PMC9110497'}], 'credit': [{'name': 'Sophia Bano', 'email': 'sophia.bano@ucl.ac.uk', 'orcidid': 'https://orcid.org/0000-0003-1329-4565', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__desmoke-lap', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__desmoke-lap', 'publication_link': 'https://doi.org/10.1007/S11548-022-02595-2', 'is_preprint': False}, {'name': 'MAECI', 'description': 'A Pipeline For Generating Consensus Sequence With Nanopore Sequencing Long-read Assembly and Error Correction.\\n\\nABSTRACT Nanopore sequencing produces long reads and offers unique advantages over next-generation sequencing, especially for the assembly of draft bacterial genomes with improved completeness. However, assembly errors can occur due to data characteristics and assembly algorithms. To address these issues, we developed MAECI, a pipeline for generating consensus sequences from multiple assemblies of the same nanopore sequencing data and error correction. Systematic evaluation showed that MAECI is an efficient and effective pipeline to improve the accuracy and completeness of bacterial genome assemblies. The available codes and implementation are at https: github.com langjidong MAECI\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/langjidong/MAECI', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0525', 'term': 'Genome assembly'}, {'uri': 'http://edamontology.org/operation_0524', 'term': 'De-novo assembly'}, {'uri': 'http://edamontology.org/operation_3180', 'term': 'Sequence assembly validation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}, {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}], 'credit': [{'name': 'Jidong Lang', 'email': 'langjidong@hotmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__maeci', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__maeci', 'publication': [{'doi': '10.1371/journal.pone.0267066', 'pmid': '35594250', 'pmcid': 'PMC9122195'}], 'publication_link': 'https://doi.org/10.1371/journal.pone.0267066', 'is_preprint': False}, {'name': 'PregTox', 'description': 'A Resource of Knowledge about Drug Fetal Toxicity.\\n\\nBackground It is of vital importance to determine the safety of drugs. Pregnant women, as a special group, need to evaluate the effects of drugs on pregnant women as well as the fetus. The use of drugs during pregnancy may be subject to fetal toxicity, thus affecting the development of the fetus or even leading to stillbirth. The U. S. Food and Drug Administration (FDA) issued a toxicity rating for drugs used during pregnancy in 1979. These toxicity ratings are denoted by the letters A, B, C, D, and X. However, the query of drug pregnancy category has yet to be well established as electronic service. Results. Here, we presented PregTox, a publicly accessible resource for pregnancy category information of 1114 drugs. The PregTox database also included chemical structures, important physico-chemical properties, protein targets, and relevant signaling pathways\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://pregtox.gzhmu.edu.cn', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0224', 'term': 'Query and retrieval'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3377', 'term': 'Safety sciences'}, {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}], 'publication': [{'doi': '10.1155/2022/4284146', 'pmid': '35469349', 'pmcid': 'PMC9034948'}], 'credit': [{'name': 'Kejian Wang', 'email': 'kejian-wang@foxmail.com', 'orcidid': 'https://orcid.org/0000-0003-4221-2485', 'typeEntity': 'Person'}, {'name': 'Qing Li', 'email': '81292522@163.com', 'orcidid': 'https://orcid.org/0000-0002-8080-1435', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__pregtox', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pregtox', 'publication_link': 'https://doi.org/10.1155/2022/4284146', 'is_preprint': False}, {'name': 'GEAR', 'description': 'a protocol to advance stakeholder consensus and research priorities in geriatrics and dementia care in the emergency department.\\n\\nGEAR Network – Geriatric Emergency care Applied Research Standardization Study.\\n\\nresearch for geriatric and dementia emergency care.\\n\\nThrough research, the mission of the Geriatric Emergency care Applied Research (GEAR) network is to generate evidence to improve the emergency care of older adults and those with dementia and other cognitive impairments.\\n\\n||| NAME (GEAR) SIMILAR TO (PUB. DIFFERENT) bio.tools/gear (geaR)', 'homepage': 'https://gearnetwork.org', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3399', 'term': 'Geriatric medicine'}, {'uri': 'http://edamontology.org/topic_3403', 'term': 'Critical care medicine'}, {'uri': 'http://edamontology.org/topic_3419', 'term': 'Psychiatry'}, {'uri': 'http://edamontology.org/topic_3577', 'term': 'Personalised medicine'}, {'uri': 'http://edamontology.org/topic_3376', 'term': 'Medicines research and development'}], 'publication': [{'doi': '10.1136/BMJOPEN-2022-060974', 'pmid': '35459682', 'pmcid': 'PMC9036447'}], 'credit': [{'name': 'Ula Hwang', 'email': 'ula.hwang@yale.edu', 'orcidid': 'https://orcid.org/0000-0002-3715-3073', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gear', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gear', 'publication_link': 'https://doi.org/10.1136/BMJOPEN-2022-060974', 'is_preprint': False}, {'name': 'CANon', 'description': \"Lightweight and Practical Cyber-Attack Detection for Automotive Controller Area Networks.\\n\\nAutomotive cyber-physical systems are in transition from the closed-systems to open-networking systems. As a result, in-vehicle networks such as the controller area network (CAN) have become essential to connect to inter-vehicle networks through the various rich interfaces. Newly exposed security concerns derived from this requirement may cause in-vehicle networks to pose threats to automotive security and driver's safety. In this paper, to ensure a high level of security of the in-vehicle network for automotive CPS, we propose a novel lightweight and practical cyber defense platform, referred to as CANon (CAN with origin authentication and non-repudiation), to be enabled to detect cyber-attacks in real-time. CANon is designed based on the hierarchical approach of centralized-session management and distributed-origin authentication\\n\\n||| HOMEPAGE MISSING!\", 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35408251', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'}], 'publication': [{'doi': '10.3390/S22072636', 'pmid': '35408251', 'pmcid': 'PMC9003419'}], 'credit': [{'name': 'Youngmi Baek', 'email': 'ymbaek@cs.ac.kr', 'orcidid': 'https://orcid.org/0000-0003-3675-1343', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__canon', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__canon', 'publication_link': 'https://doi.org/10.3390/S22072636', 'is_preprint': False}, {'name': 'AI4ACP', 'description': 'Peptide-Based Drug Predictions for Cancer Therapy Using Deep Learning.\\n\\nCopyright © 2021Institute of Information Science, Academia Sinica, TAIWAN.', 'homepage': 'https://axp.iis.sinica.edu.tw/AI4ACP/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}], 'publication': [{'doi': '10.3390/PH15040422', 'pmid': '35455418', 'pmcid': 'PMC9028292'}], 'credit': [{'name': 'Chung-Yen Lin', 'email': 'cylin@iis.sinica.edu.tw', 'orcidid': 'https://orcid.org/0000-0002-4733-9488', 'typeEntity': 'Person'}, {'name': 'Shu-Hwa Chen', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ai4acp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ai4acp', 'publication_link': 'https://doi.org/10.3390/PH15040422', 'is_preprint': False}, {'name': 'PredNitro', 'description': 'Accurately predicting nitrosylated tyrosine sites using probabilistic sequence information.\\n\\nPost-translational modification (PTM) is defined as the enzymatic changes of proteins after the translation process in protein biosynthesis. Nitrotyrosine, which is one of the most important modifications of proteins, is interceded by the active nitrogen molecule. It is known to be associated with different diseases including autoimmune diseases characterized by chronic inflammation and cell damage. Currently, nitrotyrosine sites are identified using experimental approaches which are laborious and costly. In this study, we propose a new machine learning method called PredNitro to accurately predict nitrotyrosine sites. To build PredNitro, we use sequence coupling information from the neighboring amino acids of tyrosine residues along with a support vector machine as our classification technique\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://103.99.176.239/PredNitro', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0417', 'term': 'PTM site prediction'}, {'uri': 'http://edamontology.org/operation_3755', 'term': 'PTM localisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0601', 'term': 'Protein modifications'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}], 'publication': [{'doi': '10.1016/J.GENE.2022.146445', 'pmid': '35358650'}], 'credit': [{'name': 'Iman Dehzangi', 'email': 'i.dehzangi@rutgers.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__prednitro', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__prednitro', 'publication_link': 'https://doi.org/10.1016/J.GENE.2022.146445', 'is_preprint': False}, {'name': 'scTSSR2', 'description': 'Imputing Dropout Events for Single-Cell RNA Sequencing Using Fast Two-Side Self-Representation.\\n\\nThe single-cell RNA sequencing (scRNA-seq) technique begins a new era by revealing gene expression patterns at single-cell resolution, enabling studies of heterogeneity and transcriptome dynamics of complex tissues at single-cell resolution. However, existing large proportion of dropout events may hinder downstream analyses. Thus imputation of dropout events is an important step in analyzing scRNA-seq data. We develop scTSSR2, a new imputation method that combines matrix decomposition with the previously developed two-side sparse self-representation, leading to fast two-side sparse self-representation to impute dropout events in scRNA-seq data\\n\\n||| HOMEPAGE MISSING!\\n\\n||| NAME (scTSSR2) SIMILAR TO (PUB. DIFFERENT) bio.tools/sctssr (scTSSR)', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35476574', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3557', 'term': 'Imputation'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0099', 'term': 'RNA'}], 'language': ['R'], 'publication': [{'doi': '10.1109/TCBB.2022.3170587', 'pmid': '35476574'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sctssr2', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sctssr2', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3170587', 'is_preprint': False}, {'name': 'ENDS assumptions', 'description': 'an online tool for the epistemic non-parametric drug-response scoring.\\n\\nMotivation The drug sensitivity analysis is often elucidated from drug dose-response curves. These curves capture the degree of cell viability (or inhibition) over a range of induced drugs, often with parametric assumptions that are rarely validated. Results. We present a class of non-parametric models for the curve fitting and scoring of drug dose-responses. To allow a more objective representation of the drug sensitivity, these epistemic models devoid of any parametric assumptions attached to the linear fit, allow the parallel indexing such as half-maximal inhibitory concentration and area under curve. Specifically, three non-parametric models including spline (npS), monotonic and Bayesian and the parametric logistic are implemented. Other indices including maximum effective dose and drug-response span gradient pertinent to the npS are also provided to facilitate the interpretation of the fit', 'homepage': 'https://irscope.shinyapps.io/ENDS/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426', 'term': 'Modelling and simulation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3375', 'term': 'Drug metabolism'}, {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'}], 'publication': [{'doi': '10.1093/BIOINFORMATICS/BTAC217', 'pmid': '35389453', 'pmcid': 'PMC9154283'}], 'credit': [{'name': 'Ali Amiryousefi', 'email': 'ali.amiryousefi@helisnki.fi', 'orcidid': 'https://orcid.org/0000-0002-6317-3860', 'typeEntity': 'Person'}, {'name': 'Jing Tang', 'email': 'jing.tang@helisnki.fi', 'orcidid': 'https://orcid.org/0000-0001-7480-7710', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ends-assumptions', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ends-assumptions', 'publication_link': 'https://doi.org/10.1093/BIOINFORMATICS/BTAC217', 'is_preprint': False}, {'name': 'PanGu Drug Model', 'description': 'Learn a Molecule Like a Human.\\n\\nHUAWEI Cloud Pangu Drug Molecule Model.\\n\\nAccelerating drug discovery with AI.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE PanGu Drug', 'homepage': 'http://www.pangu-drug.com/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3938', 'term': 'Virtual screening'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3375', 'term': 'Drug metabolism'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}], 'publication': [{'doi': '10.1101/2022.03.31.485886'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__pangu-drug-model', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pangu-drug-model', 'publication_link': 'https://doi.org/10.1101/2022.03.31.485886', 'is_preprint': False}, {'name': 'Fast-Higashi', 'description': 'Ultrafast and interpretable single-cell 3D genome analysis with Fast-Higashi.\\n\\nSingle-cell Hi-C (scHi-C) technologies can probe three-dimensional (3D) genome structures in single cells and their cell-to-cell variability. However, existing scHi-C analysis methods are hindered by the data quality and the complex 3D genome patterns. The lack of computational scalability and interpretability poses further challenges for large-scale scHi-C analysis. Here, we introduce Fast-Higashi, an ultrafast and interpretable method based on tensor decomposition that can jointly identify cell identities and chromatin meta-interactions. Fast-Higashi is able to simultaneously model multiple tensors with unmatched features of different sizes. A new partial random walk with restart (Partial RWR) algorithm in Fast-Higashi efficiently mitigates data sparseness\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/ma-compbio/Fast-Higashi', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}, {'uri': 'http://edamontology.org/operation_3960', 'term': 'Principal component analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0632', 'term': 'Probes and primers'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'publication': [{'doi': '10.1101/2022.04.18.488683'}], 'credit': [{'name': 'Ruochi Zhang', 'email': 'ruochiz@andrew.cmu.edu', 'typeEntity': 'Person'}, {'name': 'Jian Ma', 'email': 'jianma@cs.cmu.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__fast-higashi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fast-higashi', 'publication_link': 'https://doi.org/10.1101/2022.04.18.488683', 'is_preprint': False}, {'name': 'KMSubtraction', 'description': 'reconstruction of unreported subgroup survival data utilizing published Kaplan-Meier survival curves.\\n\\nBackground Data from certain subgroups of clinical interest may not be presented in primary manuscripts or conference abstract presentations. In an effort to enable secondary data analyses, we propose a workflow to retrieve unreported subgroup survival data from published Kaplan-Meier (KM) plots. Methods. We developed KMSubtraction, an R-package that retrieves patients from unreported subgroups by matching participants on KM plots of the overall cohort to participants on KM plots of a known subgroup with follow-up time. By excluding matched patients, the opposing unreported subgroup may be retrieved. Reproducibility and limits of error of the KMSubtraction workflow were assessed by comparing unmatched patients against the original survival data of subgroups from published datasets and simulations\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/josephjzhao/KMSubtraction/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_3503', 'term': 'Incident curve plotting'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}, {'uri': 'http://edamontology.org/topic_3577', 'term': 'Personalised medicine'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'language': ['R'], 'publication': [{'doi': '10.1186/S12874-022-01567-Z', 'pmid': '35369867', 'pmcid': 'PMC8978435'}], 'credit': [{'name': 'Yiong Huak Chan', 'email': 'medcyh@nus.edu.sg', 'typeEntity': 'Person'}, {'name': 'Raghav Sundar', 'email': 'mdcragh@nus.edu.sg', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__kmsubtraction', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kmsubtraction', 'publication_link': 'https://doi.org/10.1186/S12874-022-01567-Z', 'is_preprint': False}, {'name': 'LIPSHOK', 'description': \"LIARA Portable Smart Home Kit.\\n\\nSeveral smart home architecture implementations have been proposed in the last decade. These architectures are mostly deployed in laboratories or inside real habitations built for research purposes to enable the use of ambient intelligence using a wide variety of sensors, actuators and machine learning algorithms. However, the major issues for most related smart home architectures are their price, proprietary hardware requirements and the need for highly specialized personnel to deploy such systems. To tackle these challenges, lighter forms of smart home architectures known as smart homes in a box (SHiB) have been proposed. While SHiB remain an encouraging first step towards lightweight yet affordable solutions, they still suffer from few drawbacks. Indeed, some of these kits lack hardware support for some technologies, and others do not include enough sensors and actuators to cover most smart homes' requirements\\n\\n||| HOMEPAGE BROKEN!\", 'homepage': 'https://github.com/kevinchapron/LIPSHOK-final', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2475', 'term': 'Protein architecture recognition'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3179', 'term': 'ChIP-on-chip'}], 'publication': [{'doi': '10.3390/S22082829', 'pmid': '35458814', 'pmcid': 'PMC9024625'}], 'credit': [{'name': 'Florentin Thullier', 'email': 'florentin.thullier1@uqac.ca', 'orcidid': 'https://orcid.org/0000-0002-0334-6168', 'typeEntity': 'Person'}, {'name': 'Sébastien Gaboury', 'email': 'sebastien_gaboury@uqac.ca', 'orcidid': 'https://orcid.org/0000-0001-7749-3470', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__lipshok', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__lipshok', 'publication_link': 'https://doi.org/10.3390/S22082829', 'is_preprint': False}, {'name': 'AJILE12', 'description': 'Long-term naturalistic human intracranial neural recordings and pose.\\n\\nUnderstanding the neural basis of human movement in naturalistic scenarios is critical for expanding neuroscience research beyond constrained laboratory paradigms. Here, we describe our Annotated Joints in Long-term Electrocorticography for 12 human participants (AJILE12) dataset, the largest human neurobehavioral dataset that is publicly available; the dataset was recorded opportunistically during passive clinical epilepsy monitoring. AJILE12 includes synchronized intracranial neural recordings and upper body pose trajectories across 55 semi-continuous days of naturalistic movements, along with relevant metadata, including thousands of wrist movement events and annotated behavioral states. Neural recordings are available at 500 Hz from at least 64 electrodes per participant, for a total of 1280 hours. Pose trajectories at 9 upper-body keypoints were estimated from 118 million video frames\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/BruntonUWBio/ajile12-nwb-data', 'topic': [{'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}, {'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'}], 'publication': [{'doi': '10.1038/S41597-022-01280-Y', 'pmid': '35449141', 'pmcid': 'PMC9023453'}], 'credit': [{'name': 'Bingni W. Brunton', 'email': 'bbrunton@uw.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ajile12', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ajile12', 'publication_link': 'https://doi.org/10.1038/S41597-022-01280-Y', 'is_preprint': False}, {'name': 'BiGAMi', 'description': 'Bi-Objective Genetic Algorithm Fitness Function for Feature Selection on Microbiome Datasets.\\n\\nThe relationship between the host and the microbiome, or the assemblage of microorganisms (including bacteria, archaea, fungi, and viruses), has been proven crucial for its health and disease development. The high dimensionality of microbiome datasets has often been addressed as a major difficulty for data analysis, such as the use of Machine Learning (ML) and Deep Learning (DL) models. Here we present BiGAMi, a bi-objective genetic algorithm fitness function for feature selection in microbial datasets to train high-performing phenotype classifiers. The proposed fitness function allowed us to build classifiers that outperformed the baseline performance estimated by the original studies by using as few as 0.04% to 2.32% features of the original dataset\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/mikeleske/BiGAMi', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3936', 'term': 'Feature selection'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3460', 'term': 'Taxonomic classification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}, {'uri': 'http://edamontology.org/topic_3063', 'term': 'Medical informatics'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__bigami', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__bigami', 'publication': [{'doi': '10.3390/mps5030042', 'pmid': '35645350', 'pmcid': 'PMC9149982'}], 'publication_link': 'https://doi.org/10.3390/mps5030042', 'is_preprint': False}, {'name': 'nsink', 'description': 'An R package for flow path nitrogen removal estimation.\\n\\nThe nsink package estimates cumulative nitrogen (N) removal along a specified flow path and is based on methodologies outlined in Kellogg et al. (2010). For a user-specified watershed (i.e., hydrologic unit code (HUC)), nsink downloads all required datasets from public datasets in the United States, prepares data for use, summarizes N removal along a flow path and creates several static maps. The results of an nsink analysis may be exported to standard geospatial files for use in other applications\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/usepa/nsink', 'topic': [{'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}], 'publication': [{'doi': '10.21105/JOSS.04039', 'pmid': '35355633', 'pmcid': 'PMC8958983'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__nsink', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nsink', 'publication_link': 'https://doi.org/10.21105/JOSS.04039', 'is_preprint': False}, {'name': 'TransPhos', 'description': 'A Deep-Learning Model for General Phosphorylation Site Prediction Based on Transformer-Encoder Architecture.\\n\\nProtein phosphorylation is one of the most critical post-translational modifications of proteins in eukaryotes, which is essential for a variety of biological processes. Plenty of attempts have been made to improve the performance of computational predictors for phosphorylation site prediction. However, most of them are based on extra domain knowledge or feature selection. In this article, we present a novel deep learning-based predictor, named TransPhos, which is constructed using a transformer encoder and densely connected convolutional neural network blocks, for predicting phosphorylation sites. Data experiments are conducted on the datasets of PPA (version 3.0) and Phospho. ELM\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35457080', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0417', 'term': 'PTM site prediction'}, {'uri': 'http://edamontology.org/operation_3755', 'term': 'PTM localisation'}, {'uri': 'http://edamontology.org/operation_3904', 'term': 'Protein disorder prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0601', 'term': 'Protein modifications'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}], 'language': ['Elm'], 'publication': [{'doi': '10.3390/IJMS23084263', 'pmid': '35457080', 'pmcid': 'PMC9029334'}], 'credit': [{'name': 'Xun Wang', 'email': 'wangsyun@upc.edu.cn', 'orcidid': 'https://orcid.org/0000-0002-2741-433X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__transphos', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__transphos', 'publication_link': 'https://doi.org/10.3390/IJMS23084263', 'is_preprint': False}, {'name': 'CSR-Net', 'description': 'Learning Adaptive Context Structure Representation for Robust Feature Correspondence.\\n\\nFeature matching, which refers to identifying and then corresponding the same or similar visual pattern from two or more images, is a key technique in any image processing task that requires establishing good correspondences between images. Given potential correspondences (matches) in two scenes, a novel whole-part deep learning framework, termed as Context Structure Representation Network (CSR-Net), is designed to infer the probabilities of arbitrary correspondences being inliers. Traditional approaches commonly build the local relation between correspondences by manually engineered criteria. Different from existing attempts, the main idea of our work is to learn explicitly neighborhood structure of each correspondence, allowing us to formulate the matching problem into a dynamic local structure consensus evaluation in an end-to-end fashion\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35427222', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3473', 'term': 'Data mining'}], 'publication': [{'doi': '10.1109/TIP.2022.3166284', 'pmid': '35427222'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__csr-net', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__csr-net', 'publication_link': 'https://doi.org/10.1109/TIP.2022.3166284', 'is_preprint': False}, {'name': 'RFPDR', 'description': 'a random forest approach for plant disease resistance protein prediction.\\n\\nBackground Plant innate immunity relies on a broad repertoire of receptor proteins that can detect pathogens and trigger an effective defense response. Bioinformatic tools based on conserved domain and sequence similarity are within the most popular strategies for protein identification and characterization. However, the multi-domain nature, high sequence diversity and complex evolutionary history of disease resistance (DR) proteins make their prediction a real challenge. Here we present RFPDR, which pioneers the application of Random Forest (RF) for Plant DR protein prediction. Methods. A recently published collection of experimentally validated DR proteins was used as a positive dataset, while 10x10 nested datasets, ranging from 400-4,000 non-DR proteins, were used as negative datasets\\n\\n||| HOMEPAGE BROKEN!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE RD-RFPDR', 'homepage': 'https://github.com/cvfilippi/rfpdr', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3767', 'term': 'Protein identification'}, {'uri': 'http://edamontology.org/operation_3092', 'term': 'Protein feature detection'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}], 'publication': [{'doi': '10.7717/PEERJ.11683', 'pmid': '35480565', 'pmcid': 'PMC9037127'}], 'credit': [{'name': 'Carla Valeria Filippi', 'email': 'cfilippi@fagro.edu.uy', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rfpdr', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rfpdr', 'publication_link': 'https://doi.org/10.7717/PEERJ.11683', 'is_preprint': False}, {'name': 'SSPNet', 'description': 'An interpretable 3D-CNN for classification of schizophrenia using phase maps of resting-state complex-valued fMRI data.\\n\\nConvolutional neural networks (CNNs) have shown promising results in classifying individuals with mental disorders such as schizophrenia using resting-state fMRI data. However, complex-valued fMRI data is rarely used since additional phase data introduces high-level noise though it is potentially useful information for the context of classification. As such, we propose to use spatial source phase (SSP) maps derived from complex-valued fMRI data as the CNN input. The SSP maps are not only less noisy, but also more sensitive to spatial activation changes caused by mental disorders than magnitude maps. We build a 3D-CNN framework with two convolutional layers (named SSPNet) to fully explore the 3D structure and voxel-level relationships from the SSP maps\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35397470', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_2429', 'term': 'Mapping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}, {'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'}], 'publication': [{'doi': '10.1016/J.MEDIA.2022.102430', 'pmid': '35397470'}], 'credit': [{'name': 'Qiu-Hua Lin', 'email': 'qhlin@dlut.edu.cn', 'orcidid': 'https://orcid.org/0000-0003-0145-7136', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sspnet', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sspnet', 'publication_link': 'https://doi.org/10.1016/J.MEDIA.2022.102430', 'is_preprint': False}, {'name': 'ceCLC', 'description': 'A computationally efficient clustering linear combination approach to jointly analyze multiple phenotypes for GWAS.\\n\\nThere has been an increasing interest in joint analysis of multiple phenotypes in genome-wide association studies (GWAS) because jointly analyzing multiple phenotypes may increase statistical power to detect genetic variants associated with complex diseases or traits. Recently, many statistical methods have been developed for joint analysis of multiple phenotypes in genetic association studies, including the Clustering Linear Combination (CLC) method. The CLC method works particularly well with phenotypes that have natural groupings, but due to the unknown number of clusters for a given data, the final test statistic of CLC method is the minimum p-value among all p-values of the CLC test statistics obtained from each possible number of clusters. Therefore, a simulation procedure needs to be used to evaluate the p-value of the final test statistic\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/MeidaWang/ceCLC', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}, {'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'}, {'uri': 'http://edamontology.org/operation_3791', 'term': 'Collapsing methods'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.1371/JOURNAL.PONE.0260911', 'pmid': '35482827', 'pmcid': 'PMC9049312'}], 'credit': [{'name': 'Qiuying Sha', 'email': 'qsha@mtu.edu', 'orcidid': 'https://orcid.org/0000-0002-9342-3269', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ceclc', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ceclc', 'publication_link': 'https://doi.org/10.1371/JOURNAL.PONE.0260911', 'is_preprint': False}, {'name': 'EEG MEG', 'description': 'Towards an objective evaluation of EEG/MEG source estimation methods.\\n\\nCompute spatial resolution metrics to compare MEG with EEG+MEG — MNE 1.6.1 documentation.\\n\\nGo to the end to download the full example code.\\n\\nCompute spatial resolution metrics to compare MEG with EEG+MEG#.', 'homepage': 'http://mne.tools/stable/auto_examples/inverse/resolution_metrics_eegmeg', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}, {'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3570', 'term': 'Pure mathematics'}], 'publication': [{'doi': '10.1016/J.NEUROIMAGE.2022.119177', 'pmid': '35390459'}], 'credit': [{'name': 'Olaf Hauk', 'email': 'olaf.hauk@mrc-cbu.cam.ac.uk', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__eeg-meg', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__eeg-meg', 'publication_link': 'https://doi.org/10.1016/J.NEUROIMAGE.2022.119177', 'is_preprint': False}, {'name': 'CIGER', 'description': 'Chemical-induced gene expression ranking and its application to pancreatic cancer drug repurposing.\\n\\nChemical-induced gene expression profiles provide critical information of chemicals in a biological system, thus offering new opportunities for drug discovery. Despite their success, large-scale analysis leveraging gene expressions is limited by time and cost. Although several methods for predicting gene expressions were proposed, they only focused on imputation and classification settings, which have limited applications to real-world scenarios of drug discovery. Therefore, a chemical-induced gene expression ranking (CIGER) framework is proposed to target a more realistic but more challenging setting in which overall rankings in gene expression profiles induced by <i>de novo< i> chemicals are predicted. The experimental results show that CIGER significantly outperforms existing methods in both ranking and classification metrics\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/pth1993/CIGER.Figure', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2436', 'term': 'Gene-set enrichment analysis'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3557', 'term': 'Imputation'}, {'uri': 'http://edamontology.org/operation_0313', 'term': 'Expression profile clustering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3577', 'term': 'Personalised medicine'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}], 'publication': [{'doi': '10.1016/J.PATTER.2022.100441', 'pmid': '35465231', 'pmcid': 'PMC9023899'}], 'credit': [{'name': 'Ping Zhang', 'email': 'zhang.10631@osu.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ciger', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ciger', 'publication_link': 'https://doi.org/10.1016/J.PATTER.2022.100441', 'is_preprint': False}, {'name': 'gACSON', 'description': 'gACSON software for automated segmentation and morphology analyses of myelinated axons in 3D electron microscopy.\\n\\nBackground and objective. Advances in electron microscopy (EM) now allow three-dimensional (3D) imaging of hundreds of micrometers of tissue with nanometer-scale resolution, providing new opportunities to study the ultrastructure of the brain. In this work, we introduce a freely available Matlab-based gACSON software for visualization, segmentation, assessment, and morphology analysis of myelinated axons in 3D-EM volumes of brain tissue samples. Methods. The software is equipped with a graphical user interface (GUI)\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/AndreaBehan/g-ACSON', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3450', 'term': 'Neurite measurement'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0611', 'term': 'Electron microscopy'}, {'uri': 'http://edamontology.org/topic_3385', 'term': 'Light microscopy'}, {'uri': 'http://edamontology.org/topic_3421', 'term': 'Surgery'}, {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}, {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}], 'language': ['MATLAB'], 'license': 'MIT', 'publication': [{'doi': '10.1016/J.CMPB.2022.106802', 'pmid': '35436661'}], 'credit': [{'name': 'Alejandra Sierra', 'email': 'alejandra.sierralopez@uef.fi', 'typeEntity': 'Person'}, {'name': 'Jussi Tohka', 'email': 'jussi.tohka@uef.fi', 'orcidid': 'https://orcid.org/0000-0002-1048-5860', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gacson', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gacson', 'publication_link': 'https://doi.org/10.1016/J.CMPB.2022.106802', 'is_preprint': False}, {'name': 'Aegypti-Atlas', 'description': 'A transcriptomic atlas of <i>Aedes aegypti</i> reveals detailed functional organization of major body parts and gut regional specializations in sugar-fed and blood-fed adult females.\\n\\nAegypti-Atlas: an atlas of the yellow fever mosquito and its gut regions.\\n\\nAegypti-Atlas is an interactive database that provides a comprehensive atlas of the transcriptome of different regions/tissues (head, thorax, abdomen, Malpighian tubules, ovaries, gut) of the female yellow fever mosquito, Aedes aegypti. In addition, you will find region-specific RNAseq data for the mosquito gut (crop, proventriculus, anterior midgut, posterior midgut, and hindgut).', 'homepage': 'http://aegyptiatlas.buchonlab.com/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2942', 'term': 'Treemap visualisation'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3463', 'term': 'Expression correlation analysis'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3301', 'term': 'Microbiology'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'}, {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}], 'publication': [{'doi': '10.7554/ELIFE.76132', 'pmid': '35471187', 'pmcid': 'PMC9113746'}], 'credit': [{'name': 'Nicolas Buchon', 'email': 'nicolas.buchon@cornell.edu', 'orcidid': 'https://orcid.org/0000-0003-3636-8387', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__aegypti-atlas', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__aegypti-atlas', 'publication_link': 'https://doi.org/10.7554/ELIFE.76132', 'is_preprint': False}, {'name': 'MISS', 'description': 'A pipeline for mapping of diverse cell types across the murine brain.\\n\\nThe advent of increasingly sophisticated imaging platforms has allowed for the visualization of the murine nervous system at single-cell resolution. However, current experimental approaches have not yet produced whole-brain maps of a comprehensive set of neuronal and nonneuronal types that approaches the cellular diversity of the mammalian cortex. Here, we aim to fill in this gap in knowledge with an open-source computational pipeline, Matrix Inversion and Subset Selection (MISS), that can infer quantitatively validated distributions of diverse collections of neural cell types at 200-μm resolution using a combination of single-cell RNA sequencing (RNAseq) and in situ hybridization datasets. We rigorously demonstrate the accuracy of MISS against literature expectations\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Raj-Lab-UCSF/MISS-Pipeline', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3629', 'term': 'Deisotoping'}, {'uri': 'http://edamontology.org/operation_3800', 'term': 'RNA-Seq quantification'}, {'uri': 'http://edamontology.org/operation_2429', 'term': 'Mapping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}, {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}], 'publication': [{'doi': '10.1073/PNAS.2111786119', 'pmid': '35363567', 'pmcid': 'PMC9168512'}], 'credit': [{'name': 'Ashish Raj', 'email': 'ashish.raj@ucsf.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__miss', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__miss', 'publication_link': 'https://doi.org/10.1073/PNAS.2111786119', 'is_preprint': False}, {'name': 'mHealth literacy mHealth literacy', 'description': \"Problem-Based mHealth Literacy Scale (PB-mHLS).\\n\\nBackground Mobile devices have greatly facilitated the use of digital health resources, particularly during the COVID-19 pandemic. Mobile health (mHealth) has become a common and important way to monitor and improve health conditions for people from different social classes. The ability to utilize mHealth affects its effectiveness; therefore, the widespread application of mHealth technologies calls for an instrument that can accurately measure health literacy in the era of mobile media. Objective. We aimed to (1) identify the components of mHealth literacy for ordinary users and (2) develop a systematic scale for appropriately measuring individuals' self-perceived mHealth literacy through a problem-based framework. Methods\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE mHealth literacy, mHealth literacy mHealth, literacy mHealth literacy, 001 mHealth literacy mHealth literacy\", 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35394446', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2428', 'term': 'Validation'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}, {'uri': 'http://edamontology.org/operation_3960', 'term': 'Principal component analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3063', 'term': 'Medical informatics'}, {'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}], 'publication': [{'doi': '10.2196/31459', 'pmid': '35394446', 'pmcid': 'PMC9034416'}], 'credit': [{'name': 'Pengxiang Li', 'email': 'lipx@szu.edu.cn', 'orcidid': 'https://orcid.org/0000-0003-2310-3865', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mhealth-literacy-mhealth-literacy', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mhealth-literacy-mhealth-literacy', 'publication_link': 'https://doi.org/10.2196/31459', 'is_preprint': False}, {'name': 'TSFD-Net', 'description': 'Tissue specific feature distillation network for nuclei segmentation and classification.\\n\\nNuclei segmentation and classification of hematoxylin and eosin-stained histology images is a challenging task due to a variety of issues, such as color inconsistency that results from the non-uniform manual staining operations, clustering of nuclei, and blurry and overlapping nuclei boundaries. Existing approaches involve segmenting nuclei by drawing their polygon representations or by measuring the distances between nuclei centroids. In contrast, we leverage the fact that morphological features (appearance, shape, and texture) of nuclei in a tissue vary greatly depending upon the tissue type. We exploit this information by extracting tissue specific (TS) features from raw histopathology images using the proposed tissue specific feature distillation (TSFD) backbone\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Mr-TalhaIlyas/TSFD', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1016/J.NEUNET.2022.02.020', 'pmid': '35367734'}], 'credit': [{'name': 'Talha Ilyas', 'email': 'talha@jbnu.ac.kr', 'orcidid': 'https://orcid.org/0000-0002-4168-2998', 'typeEntity': 'Person'}, {'name': 'Zubaer Ibna Mannan', 'email': 'zimannan@gmail.com', 'orcidid': 'https://orcid.org/0000-0002-5336-5252', 'typeEntity': 'Person'}, {'name': 'Abbas Khan', 'email': 'kabbas570@gmail.com', 'orcidid': 'https://orcid.org/0000-0002-2120-4729', 'typeEntity': 'Person'}, {'name': 'Sami Azam', 'email': 'Sami.Azam@cdu.edu.au', 'orcidid': 'https://orcid.org/0000-0001-7572-9750', 'typeEntity': 'Person'}, {'name': 'Hyongsuk Kim', 'email': 'hskim@jbnu.ac.kr', 'typeEntity': 'Person'}, {'name': 'Friso De Boer', 'email': 'Friso.DeBoer@cdu.edu.au', 'orcidid': 'https://orcid.org/0000-0003-4946-4175', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tsfd-net', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tsfd-net', 'publication_link': 'https://doi.org/10.1016/J.NEUNET.2022.02.020', 'is_preprint': False}, {'name': 'STKD', 'description': \"Distilling Knowledge From Synchronous Teaching for Efficient Model Compression.\\n\\nKnowledge distillation (KD) transfers discriminative knowledge from a large and complex model (known as teacher) to a smaller and faster one (known as student). Existing advanced KD methods, limited to fixed feature extraction paradigms that capture teacher's structure knowledge to guide the training of the student, often fail to obtain comprehensive knowledge to the student. Toward this end, in this article, we propose a new approach, synchronous teaching knowledge distillation (STKD), to integrate online teaching and offline teaching for transferring rich and comprehensive knowledge to the student. In the online learning stage, a blockwise unit is designed to distill the intermediate-level knowledge and high-level knowledge, which can achieve bidirectional guidance of the teacher and student networks\\n\\n||| HOMEPAGE BROKEN!\", 'homepage': 'https://github.com/nanxiaotong/STKD', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}]}], 'publication': [{'doi': '10.1109/TNNLS.2022.3164264', 'pmid': '35420989'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__stkd', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__stkd', 'publication_link': 'https://doi.org/10.1109/TNNLS.2022.3164264', 'is_preprint': False}, {'name': 'LuminoCell', 'description': 'a versatile and affordable platform for real-time monitoring of luciferase-based reporters.\\n\\nLuciferase reporter assays represent a simple and sensitive experimental system in cell and molecular biology to study multiple biological processes. However, the application of these assays is often limited by the costs of conventional luminometer instruments and the versatility of their use in different experimental conditions. Therefore, we aimed to develop a small, affordable luminometer allowing continuous measurement of luciferase activity, designed for inclusion into various kinds of tissue culture incubators. Here, we introduce LuminoCell-an open-source platform for the construction of an affordable, sensitive, and portable luminometer capable of real-time monitoring in-cell luciferase activity\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/barta-lab/LuminoCell', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310', 'term': 'Sequence assembly'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}], 'publication': [{'doi': '10.26508/LSA.202201421', 'pmid': '35440493', 'pmcid': 'PMC9018015'}], 'credit': [{'name': 'Tomáš Bárta', 'email': 'tbarta@med.muni.cz', 'orcidid': 'https://orcid.org/0000-0003-0703-066X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__luminocell', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__luminocell', 'publication_link': 'https://doi.org/10.26508/LSA.202201421', 'is_preprint': False}, {'name': 'SemanticCAP', 'description': 'Chromatin Accessibility Prediction Enhanced by Features Learning from a Language Model.\\n\\nA large number of inorganic and organic compounds are able to bind DNA and form complexes, among which drug-related molecules are important. Chromatin accessibility changes not only directly affect drug-DNA interactions, but they can promote or inhibit the expression of the critical genes associated with drug resistance by affecting the DNA binding capacity of TFs and transcriptional regulators. However, the biological experimental techniques for measuring it are expensive and time-consuming. In recent years, several kinds of computational methods have been proposed to identify accessible regions of the genome. Existing computational models mostly ignore the contextual information provided by the bases in gene sequences. To address these issues, we proposed a new solution called SemanticCAP\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://github.com/ykzhang0126/semanticCAP', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3940', 'term': 'Chromosome conformation capture'}, {'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}, {'uri': 'http://edamontology.org/topic_3656', 'term': 'Immunoprecipitation experiment'}], 'publication': [{'doi': '10.3390/GENES13040568', 'pmid': '35456374', 'pmcid': 'PMC9028922'}], 'credit': [{'name': 'Lijun Quan', 'email': 'ljquan@suda.edu.cn', 'orcidid': 'https://orcid.org/0000-0003-4551-4198', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__semanticcap', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__semanticcap', 'publication_link': 'https://doi.org/10.3390/GENES13040568', 'is_preprint': False}, {'name': 'GII 2P16 GII 4P16 noroviruses', 'description': 'Evolutionary analyses of emerging GII.2[P16] and GII.4 Sydney [P16] noroviruses.\\n\\nGII 2[P16] and GII. 4 Sydney [P16] are currently the two predominant norovirus genotypes. This study sought to clarify their evolutionary patterns by analyzing the major capsid VP1 and RNA-dependent RNA polymerase (RdRp) genes. Sequence diversities were analyzed at both nucleotide and amino acid levels. Selective pressures were evaluated with the Hyphy package in different models. Phylogenetic trees were constructed by the maximum likelihood method from full VP1 sequences, and evolutionary rates were estimated by the Bayesian Markov Chain Monte Carlo approach\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE VP1 RdRp GII 2P16 GII, GII 2P16 GII 4P16, GII 2P16 GII, RdRp GII 2P16 GII', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35450165', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0547', 'term': 'Phylogenetic inference (maximum likelihood and Bayesian methods)'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}, {'uri': 'http://edamontology.org/operation_0540', 'term': 'Phylogenetic inference (from molecular sequences)'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3293', 'term': 'Phylogenetics'}, {'uri': 'http://edamontology.org/topic_2830', 'term': 'Immunoproteins and antigens'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'publication': [{'doi': '10.1093/VE/VEAC030', 'pmid': '35450165', 'pmcid': 'PMC9019527'}], 'credit': [{'name': 'Jie-mei Yu', 'email': 'jmyu1@bjtu.edu.cn', 'url': 'https://orcid.org/0000-0003-2988-384X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gii-2p16-gii-4p16-noroviruses', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gii-2p16-gii-4p16-noroviruses', 'publication_link': 'https://doi.org/10.1093/VE/VEAC030', 'is_preprint': False}, {'name': 'DFLM', 'description': 'Predicting the Sequence Specificities of DNA-Binding Proteins by DNA Fine-Tuned Language Model With Decaying Learning Rates.\\n\\nDNA-binding proteins (DBPs) play vital roles in the regulation of biological systems. Although there are already many deep learning methods for predicting the sequence specificities of DBPs, they face two challenges as follows. Classic deep learning methods for DBPs prediction usually fail to capture the dependencies between genomic sequences since their commonly used one-hot codes are mutually orthogonal. Besides, these methods usually perform poorly when samples are inadequate. To address these two challenges, we developed a novel language model for mining DBPs using human genomic data and ChIP-seq datasets with decaying learning rates, named DNA Fine-tuned Language Model (DFLM)\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Deep-Bioinfo/DFLM', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3900', 'term': 'DNA-binding protein prediction'}, {'uri': 'http://edamontology.org/operation_3903', 'term': 'DNA binding site prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3169', 'term': 'ChIP-seq'}, {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}], 'publication': [{'doi': '10.1109/TCBB.2022.3165592', 'pmid': '35389869'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__dflm', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dflm', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3165592', 'is_preprint': False}, {'name': 'TCSP', 'description': 'a Template-Based Crystal Structure Prediction Algorithm for Materials Discovery.\\n\\nMToolbox, The boiler for materials discovery.', 'homepage': 'http://www.materialsatlas.org/crystalstructure', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0475', 'term': 'Nucleic acid structure prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0082', 'term': 'Structure prediction'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3376', 'term': 'Medicines research and development'}], 'link': [{'url': 'http://MaterialsAtlas.org', 'type': ['Other']}], 'publication': [{'doi': '10.1021/ACS.INORGCHEM.1C03879', 'pmid': '35420427'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tcsp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tcsp', 'publication_link': 'https://doi.org/10.1021/ACS.INORGCHEM.1C03879', 'is_preprint': False}, {'name': 'AHLS-pred', 'description': 'a novel sequence-based predictor of acyl-homoserine-lactone synthases using machine learning algorithms.\\n\\nAcyl-homoserine-lactones (AHLs), as the major quorum sensing (QS) signalling molecules in Gram-negative bacteria, have shown great application potential in regulating biological nutrient removal process. The identification of AHLs synthases plays an essential role in in-depth research on QS mechanisms and applications of biological wastewater treatment processes. This work proposed the first prediction model for AHLs synthases based on machine learning algorithms, namely, AHLS-pred. The training dataset AHLS1400 and the independent testing dataset AHLS132 for AHLSs prediction were first established. Three sequence-based feature extraction methods are utilized to generate feature descriptors, namely, amino acid composition, dipeptide composition and G-gap dipeptide composition respectively\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35403334', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_3802', 'term': 'Sorting'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}], 'publication': [{'doi': '10.1111/1758-2229.13068', 'pmid': '35403334'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ahls-pred', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ahls-pred', 'publication_link': 'https://doi.org/10.1111/1758-2229.13068', 'is_preprint': False}, {'name': 'funRiceGenes', 'description': 'New Data and New Features of the FunRiceGenes (Functionally Characterized Rice Genes) Database.\\n\\nA comprehensive database of functionally characterized rice genes.\\n\\n4500+ cloned rice genes [Download↓].\\n\\n6100+ gene family members [Download↓].\\n\\nInteractive application of funRiceGenes: https://venyao.xyz/funRiceGenes/.', 'homepage': 'https://funricegenes.github.io/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3431', 'term': 'Deposition'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0623', 'term': 'Gene and protein families'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}, {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'link': [{'url': 'http://funricegenes.ncpgr.cn/', 'type': ['Other']}, {'url': 'https://venyao.xyz/funRiceGenes/', 'type': ['Other']}], 'publication': [{'doi': '10.1186/S12284-022-00569-1', 'pmid': '35438356', 'pmcid': 'PMC9018921'}], 'credit': [{'name': 'Wen Yao', 'email': 'yaowen@henau.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__funricegenes', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__funricegenes', 'publication_link': 'https://doi.org/10.1186/S12284-022-00569-1', 'is_preprint': False}, {'name': 'CVVB', 'description': 'Efficient selection between hierarchical cognitive models.\\n\\nModel comparison is the cornerstone of theoretical progress in psychological research. Common practice overwhelmingly relies on tools that evaluate competing models by balancing in-sample descriptive adequacy against model flexibility, with modern approaches advocating the use of marginal likelihood for hierarchical cognitive models. Cross-validation is another popular approach but its implementation remains out of reach for cognitive models evaluated in a Bayesian hierarchical framework, with the major hurdle being its prohibitive computational cost. To address this issue, we develop novel algorithms that make variational Bayes (VB) inference for hierarchical models feasible and computationally efficient for complex cognitive models of substantive theoretical interest. It is well known that VB produces good estimates of the first moments of the parameters, which gives good predictive densities estimates\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Henry-Dao/CVVB', 'topic': [{'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}, {'uri': 'http://edamontology.org/topic_3452', 'term': 'Tomography'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'language': ['MATLAB'], 'publication': [{'doi': '10.1037/MET0000458', 'pmid': '35446049'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__cvvb', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cvvb', 'publication_link': 'https://doi.org/10.1037/MET0000458', 'is_preprint': False}, {'name': 'SGP', 'description': 'Algorithm for the Pruning of Synthesis Graphs.\\n\\nSynthesis route planning is in the core of chemical intelligence that will power the autonomous chemistry platforms. In this task, we rely on algorithms to generate possible synthesis routes with the help of retro- and forward-synthetic approaches. Generated synthesis routes can be merged into a synthesis graph which represents theoretical pathways to the target molecule. However, it is often required to modify a synthesis graph due to typical constraints. These constraints might include \"undesirable substances\", e.g., an intermediate that the chemist does not favor or substances that might be toxic. Consequently, we need to prune the synthesis graph by the elimination of such undesirable substances\\n\\n||| HOMEPAGE BROKEN!\\n\\n||| NAME (SGP) SIMILAR TO (PUB. DIFFERENT) bio.tools/sgp2 (SGP2)', 'homepage': 'https://github.com/ncats/SGP', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310', 'term': 'Sequence assembly'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3314', 'term': 'Chemistry'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1021/ACS.JCIM.1C01202', 'pmid': '35438992', 'pmcid': 'PMC9093600'}], 'credit': [{'name': 'Gergely Zahoránszky-Kőhalmi', 'email': 'gergely.zahoranszky-kohalmi@nih.gov', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sgp', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sgp', 'publication_link': 'https://doi.org/10.1021/ACS.JCIM.1C01202', 'is_preprint': False}, {'name': 'SE-BLTCNN', 'description': 'A channel attention adapted deep learning model based on PSSM for membrane protein classification.\\n\\nMembrane protein classification is a key to inferring the function of uncharacterized membrane protein. To get around the time-consuming and expensive biochemical experiments in the wet lab, there has been a lot of research focusing on developing fast and reliable bioinformatics or computer modeling methods for membrane protein prediction. However, most research is inclined to incorporate as many types of protein data as possible, yet in many cases, the number of accessible protein data types is quite limited. To solve this challenge, a channel attention adapted deep learning model that takes the position-specific scoring matrix (PSSM) as only input and its simplified version without channel attention have been developed. They are named SE-BLTCNN and BLTCNN, respectively (the abbreviations for \"SE embedded BiLSTM-TextCNN\" and \"plain BiLSTM-TextCNN\")\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Raymond-2017/membrane-protein-classifiction', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3767', 'term': 'Protein identification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0820', 'term': 'Membrane and lipoproteins'}, {'uri': 'http://edamontology.org/topic_0623', 'term': 'Gene and protein families'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1016/J.COMPBIOLCHEM.2022.107680', 'pmid': '35421797'}], 'credit': [{'name': 'Shunfang Wang', 'email': 'sfwang_66@ynu.edu.cn', 'orcidid': 'https://orcid.org/0000-0002-1927-8753', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__se-bltcnn', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__se-bltcnn', 'publication_link': 'https://doi.org/10.1016/J.COMPBIOLCHEM.2022.107680', 'is_preprint': False}, {'name': 'SOKS', 'description': 'Automatic Searching of the Optimal Kernel Shapes for Stripe-Wise Network Pruning.\\n\\nIn spite of the remarkable performance, deep convolutional neural networks (CNNs) are typically over-parameterized and computationally expensive. Network pruning has become a popular approach to reducing the storage and calculations of CNN models, which commonly prunes filters in a structured way or discards single weights without structural constraints. However, the redundancy in convolution kernels and the influence of kernel shapes on the performance of CNN models have attracted little attention. In this article, we develop a framework, termed searching of the optimal kernel shape (SOKS), to automatically search for the optimal kernel shapes and perform stripe-wise pruning (SWP). To be specific, we introduce coefficient matrices regularized by a variety of regularization terms to locate important kernel positions\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35412989', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1109/TNNLS.2022.3162067', 'pmid': '35412989'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__soks', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__soks', 'publication_link': 'https://doi.org/10.1109/TNNLS.2022.3162067', 'is_preprint': False}, {'name': 'MANNERS-DB', 'description': 'Mind Your Manners! A Dataset and a Continual Learning Approach for Assessing Social Appropriateness of Robot Actions.\\n\\nTo date, endowing robots with an ability to assess social appropriateness of their actions has not been possible. This has been mainly due to (i) the lack of relevant and labelled data and (ii) the lack of formulations of this as a lifelong learning problem. In this paper, we address these two issues. We first introduce the Socially Appropriate Domestic Robot Actions dataset (MANNERS-DB), which contains appropriateness labels of robot actions annotated by humans. Secondly, we train and evaluate a baseline Multi Layer Perceptron and a Bayesian Neural Network (BNN) that estimate social appropriateness of actions in MANNERS-DB. Finally, we formulate learning social appropriateness of actions as a continual learning problem using the uncertainty of Bayesian Neural Network parameters\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/jonastjoms/MANNERS-DB', 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.3389/FROBT.2022.669420', 'pmid': '35356061', 'pmcid': 'PMC8959540'}], 'credit': [{'name': 'Sinan Kalkan', 'email': 'skalkan@metu.edu.tr', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__manners-db', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__manners-db', 'publication_link': 'https://doi.org/10.3389/FROBT.2022.669420', 'is_preprint': False}, {'name': 'RLSDSPCA', 'description': 'Enhancing Characteristic Gene Selection and Tumor Classification by the Robust Laplacian Supervised Discriminative Sparse PCA.\\n\\nCharacteristic gene selection and tumor classification of gene expression data play major roles in genomic research. Due to the characteristics of a small sample size and high dimensionality of gene expression data, it is a common practice to perform dimensionality reduction prior to the use of machine learning-based methods to analyze the expression data. In this context, classical principal component analysis (PCA) and its improved versions have been widely used. Recently, methods based on supervised discriminative sparse PCA have been developed to improve the performance of data dimensionality reduction. However, such methods still have limitations: most of them have not taken into consideration the improvement of robustness to outliers and noise, label information, sparsity, as well as capturing intrinsic geometrical structures in one objective function', 'homepage': 'http://csbio.njust.edu.cn/bioinf/rlsdspca/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3935', 'term': 'Dimensionality reduction'}, {'uri': 'http://edamontology.org/operation_3960', 'term': 'Principal component analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}], 'publication': [{'doi': '10.1021/ACS.JCIM.1C01403', 'pmid': '35353532'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rlsdspca', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rlsdspca', 'publication_link': 'https://doi.org/10.1021/ACS.JCIM.1C01403', 'is_preprint': False}, {'name': 'CEBA', 'description': \"A Data Lake for Data Sharing and Environmental Monitoring.\\n\\nActuellement 59 jeux de données dans le catalogue CEBA.\\n\\nCloud Environnemental au Bénéfice de l'Auvergne.\\n\\nObjectif : permettre de mieux comprendre les systèmes environnementaux, leurs évolutions dans le contexte du changement climatique global et leurs interactions avec les agroécosystèmes : impact des facteurs environnementaux sur les productions agricoles et inversement.\\n\\n||| NAME (CEBA) SIMILAR TO (PUB. DIFFERENT) bio.tools/CEBA (CEBA geoportail)\", 'homepage': 'https://ceba.uca.fr', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}, {'uri': 'http://edamontology.org/topic_3071', 'term': 'Biological databases'}, {'uri': 'http://edamontology.org/topic_0610', 'term': 'Ecology'}, {'uri': 'http://edamontology.org/topic_3316', 'term': 'Computer science'}], 'language': ['JavaScript'], 'publication': [{'doi': '10.3390/S22072733', 'pmid': '35408347', 'pmcid': 'PMC9003009'}], 'credit': [{'name': 'David Sarramia', 'email': 'david.sarramia@uca.fr', 'orcidid': 'https://orcid.org/0000-0002-7062-9707', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ceba', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ceba', 'publication_link': 'https://doi.org/10.3390/S22072733', 'is_preprint': False}, {'name': 'RPfam', 'description': 'A refiner towards curated-like multiple sequence alignments of the Pfam protein families.\\n\\nHigh-quality multiple sequence alignments can provide insights into the architecture and function of protein families. The existing MSA tools often generate results inconsistent with biological distribution of conserved regions because of positioning amino acid residues and gaps only by symbols. We propose RPfam, a refiner towards curated-like MSAs for modeling the protein families in the Pfam database. RPfam refines the automatic alignments via scoring alignments based on the PFASUM matrix, restricting realignments within badly aligned blocks, optimizing the block scores by dynamic programming, and running refinements iteratively using the Simulated Annealing algorithm. Experiments show RPfam effectively refined the alignments produced by the MSA tools ClustalO and Muscle with reference to the curated seed alignments of the Pfam protein families\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'http://www.worldscinet.com/jbcb/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2089', 'term': 'Sequence alignment refinement'}, {'uri': 'http://edamontology.org/operation_0492', 'term': 'Multiple sequence alignment'}, {'uri': 'http://edamontology.org/operation_0322', 'term': 'Molecular model refinement'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0623', 'term': 'Gene and protein families'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_0080', 'term': 'Sequence analysis'}], 'publication': [{'doi': '10.1142/S0219720022400029', 'pmid': '35430947'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rpfam', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rpfam', 'publication_link': 'https://doi.org/10.1142/S0219720022400029', 'is_preprint': False}, {'name': 'sponge gourd watermelon', 'description': 'Tomato chlorosis virus infecting <i>Luffa cylindrica</i> and <i>Citrullus lanatus</i> in China.\\n\\nSponge gourd (<i>Luffa cylindrica< i>) and watermelon (<i>Citrullus lanatus< i>) are important cash crops in China. In September 2015, interveinal yellow spots and chlorosis, suspected to be caused by the tomato chlorosis virus (ToCV; genus <i>Crinivirus< i>), were observed on sponge gourd and watermelon plants in six greenhouses in the cities of Shouguang, Dezhou, and Taian (2 greenhouses in each city) of Shandong Province. The incidences of the disease in sponge gourd and watermelon greenhouses were 10% to 20%. To identify causative pathogens, 20 sponge gourd and 15 watermelon samples were collected from cucurbit plant facilities in Shandong Province, China\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE ToCV sponge gourd watermelon, ToCV infection sponge gourd watermelon, sponge gourd, gourd watermelon', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35412329', 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0632', 'term': 'Probes and primers'}, {'uri': 'http://edamontology.org/topic_3293', 'term': 'Phylogenetics'}], 'publication': [{'doi': '10.1094/PDIS-01-22-0086-PDN', 'pmid': '35412329'}], 'credit': [{'name': 'Xiaoying Liu', 'email': '17863801005@163.com', 'typeEntity': 'Person'}, {'name': 'Xiaohui Sun', 'email': 'ddxiaohuifly@163.com', 'orcidid': 'https://orcid.org/0000-0002-9359-3065', 'typeEntity': 'Person'}, {'name': 'Shusen Wang', 'email': 'wss13706465846@qq.com', 'typeEntity': 'Person'}, {'name': 'Yongguang Liu', 'email': 'ygliu425@163.com', 'typeEntity': 'Person'}, {'name': 'Huijie Dai', 'email': 'climsion@126.com', 'typeEntity': 'Person'}, {'name': 'Xiaoping Zhu', 'email': 'zhuxp@sdau.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sponge-gourd-watermelon', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sponge-gourd-watermelon', 'publication_link': 'https://doi.org/10.1094/PDIS-01-22-0086-PDN', 'is_preprint': False}, {'name': 'U-RISC', 'description': 'An Annotated Ultra-High-Resolution Electron Microscopy Dataset Challenging the Existing Deep Learning Algorithms.\\n\\nConnectomics is a developing field aiming at reconstructing the connection of the neural system at the nanometer scale. Computer vision technology, especially deep learning methods used in image processing, has promoted connectomic data analysis to a new era. However, the performance of the state-of-the-art (SOTA) methods still falls behind the demand of scientific research. Inspired by the success of ImageNet, we present an annotated ultra-high resolution image segmentation dataset for cell membrane (U-RISC), which is the largest cell membrane-annotated electron microscopy (EM) dataset with a resolution of 2.18 nm pixel. Multiple iterative annotations ensured the quality of the dataset\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/EmmaSRH/U-RISC-Data-Code', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3553', 'term': 'Image annotation'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0611', 'term': 'Electron microscopy'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}], 'publication': [{'doi': '10.3389/FNCOM.2022.842760', 'pmid': '35480847', 'pmcid': 'PMC9038176'}], 'credit': [{'name': 'Kai Du', 'email': 'kai.du@pku.edu.cn', 'typeEntity': 'Person'}, {'name': 'Tingting Jiang', 'email': 'ttjiang@pku.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__u-risc', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__u-risc', 'publication_link': 'https://doi.org/10.3389/FNCOM.2022.842760', 'is_preprint': False}, {'name': 'DWPPI', 'description': 'A Deep Learning Approach for Predicting Protein-Protein Interactions in Plants Based on Multi-Source Information With a Large-Scale Biological Network.\\n\\nThe prediction of protein-protein interactions (PPIs) in plants is vital for probing the cell function. Although multiple high-throughput approaches in the biological domain have been developed to identify PPIs, with the increasing complexity of PPI network, these methods fall into laborious and time-consuming situations. Thus, it is essential to develop an effective and feasible computational method for the prediction of PPIs in plants. In this study, we present a network embedding-based method, called DWPPI, for predicting the interactions between different plant proteins based on multi-source information and combined with deep neural networks (DNN)\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35387292', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2492', 'term': 'Protein interaction prediction'}, {'uri': 'http://edamontology.org/operation_3094', 'term': 'Protein interaction network prediction'}, {'uri': 'http://edamontology.org/operation_3092', 'term': 'Protein feature detection'}, {'uri': 'http://edamontology.org/operation_4009', 'term': 'Small molecule design'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0128', 'term': 'Protein interactions'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0632', 'term': 'Probes and primers'}, {'uri': 'http://edamontology.org/topic_2275', 'term': 'Molecular modelling'}], 'publication': [{'doi': '10.3389/FBIOE.2022.807522', 'pmid': '35387292', 'pmcid': 'PMC8978800'}], 'credit': [{'name': 'Li-Ping Li', 'typeEntity': 'Person'}, {'name': 'Chang-Qing Yu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__dwppi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dwppi', 'publication_link': 'https://doi.org/10.3389/FBIOE.2022.807522', 'is_preprint': False}, {'name': 'CPGL', 'description': 'Prediction of compound-protein interaction by integrating graph attention network with long short-term memory neural network.\\n\\nRecent advancements of artificial intelligence based on deep learning algorithms have made it possible to computationally predict compound-protein interaction (CPI) without conducting laboratory experiments. In this manuscript, we integrated a graph attention network (GAT) for compounds and a long short-term memory neural network (LSTM) for proteins, used end-to-end representation learning for both compounds and proteins, and proposed a deep learning algorithm, CPGL (CPI with GAT and LSTM) to optimize the feature extraction from compounds and proteins and to improve the model robustness and generalizability. CPGL demonstrated an excellent predictive performance and outperforms recently reported deep learning models\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/RobinDoyle/CPGL', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_2929', 'term': 'Protein fragment weight comparison'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3361', 'term': 'Laboratory techniques'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}], 'publication': [{'doi': '10.1101/2022.04.19.488691'}], 'credit': [{'name': 'Yaning Yang', 'email': 'ynyang@ustc.edu.cn', 'typeEntity': 'Person'}, {'name': 'Steven X Xu', 'email': 'sxu@genmab.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__cpgl', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cpgl', 'publication_link': 'https://doi.org/10.1101/2022.04.19.488691', 'is_preprint': False}, {'name': 'Sophora flavescens Kushen decoction', 'description': 'Mechanisms of Compound <i>Sophora flavescens</i> (Kushen) Decoction for the Treatment of Ulcerative Colitis based on Network Pharmacology and Molecular Docking Technology.\\n\\nBackground The compound Sophora flavescenes (Kushen) decoction was found to reduce the inflammatory symptom of Ulcerative Colitis (UC). However, there exists a very limited understanding of the molecular pharmacological mechanisms. Objective. This study aimed to explore the mechanism of compound Sophora flavescens (Kushen) decoction in treating ulcerative colitis from the perspective of network pharmacology. Methods\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE compound Sophora flavescens Kushen decoction, flavescens Kushen decoction, Sophora flavescens Kushen, compound Sophora flavescens Kushen', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35388762', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0478', 'term': 'Molecular docking'}, {'uri': 'http://edamontology.org/operation_0276', 'term': 'Protein interaction network analysis'}, {'uri': 'http://edamontology.org/operation_3501', 'term': 'Enrichment analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_2275', 'term': 'Molecular modelling'}, {'uri': 'http://edamontology.org/topic_0128', 'term': 'Protein interactions'}], 'publication': [{'doi': '10.2174/1573409918666220406103708', 'pmid': '35388762'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sophora-flavescens-kushen-decoction', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sophora-flavescens-kushen-decoction', 'publication_link': 'https://doi.org/10.2174/1573409918666220406103708', 'is_preprint': False}, {'name': 'MetaDR', 'description': 'Human disease prediction from microbiome data by multiple feature fusion and deep learning.\\n\\nHuman disease prediction from microbiome data has broad implications in metagenomics. It is rare for the existing methods to consider abundance profiles from both known and unknown microbial organisms, or capture the taxonomic relationships among microbial taxa, leading to significant information loss. On the other hand, deep learning has shown unprecedented advantages in classification tasks for its feature-learning ability. However, it encounters the opposite situation in metagenome-based disease prediction since high-dimensional low-sample-size metagenomic datasets can lead to severe overfitting; and black-box model fails in providing biological explanations. To circumvent the related problems, we developed MetaDR, a comprehensive machine learning-based framework that integrates various information and deep learning to predict human diseases\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Microbiods/MetaDR', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3460', 'term': 'Taxonomic classification'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}, {'uri': 'http://edamontology.org/operation_0324', 'term': 'Phylogenetic analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'}, {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'}, {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_2259', 'term': 'Systems biology'}], 'publication': [{'doi': '10.1016/J.ISCI.2022.104081', 'pmid': '35372808', 'pmcid': 'PMC8971930'}], 'credit': [{'name': 'Ka-Chun Wong', 'email': 'kc.w@cityu.edu.hk', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__metadr', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__metadr', 'publication_link': 'https://doi.org/10.1016/J.ISCI.2022.104081', 'is_preprint': False}, {'name': 'GMM-NEA', 'description': 'Reconstruction of Nuclear Ensemble Approach Electronic Spectra Using Probabilistic Machine Learning.\\n\\nThe theoretical prediction of molecular electronic spectra by means of quantum mechanical (QM) computations is fundamental to gain a deep insight into many photophysical and photochemical processes. A computational strategy that is attracting significant attention is the so-called Nuclear Ensemble Approach (NEA), that relies on generating a representative ensemble of nuclear geometries around the equilibrium structure and computing the vertical excitation energies (Δ<i>E< i>) and oscillator strengths (<i>f< i>) and <i>phenomenologically broadening< i> each transition with a line-shaped function with empirical full-width δ. Frequently, the choice of δ is carried out by visually finding the trade-off between artificial vibronic features (small δ) and over-smoothing of electronic signatures (large δ)\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/lucerlab/GMM-NEA', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}, {'uri': 'http://edamontology.org/operation_3860', 'term': 'Spectrum calculation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3314', 'term': 'Chemistry'}, {'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'}], 'publication': [{'doi': '10.1021/ACS.JCTC.2C00004', 'pmid': '35481363', 'pmcid': 'PMC9097286'}], 'credit': [{'name': 'Luis Cerdán', 'email': 'luis.cerdan@uv.es', 'orcidid': 'https://orcid.org/0000-0002-7174-2453', 'typeEntity': 'Person'}, {'name': 'Daniel Roca-Sanjuán', 'email': 'daniel.roca@uv.es', 'orcidid': 'https://orcid.org/0000-0001-6495-2770', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gmm-nea', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gmm-nea', 'publication_link': 'https://doi.org/10.1021/ACS.JCTC.2C00004', 'is_preprint': False}, {'name': 'TurboBC', 'description': 'A Memory Efficient and Scalable GPU Based Betweenness Centrality Algorithm in the Language of Linear Algebra.\\n\\nBetweenness centrality (BC) is a shortest path centrality metric used to measure the influence of individual vertices or edges on huge graphs that are used for modeling and analysis of human brain, omics data, or social networks. The application of the BC algorithm to modern graphs must deal with the size of the graphs, as well with highly irregular data-access patterns. These challenges are particularly important when the BC algorithm is implemented on Graphics Processing Units (GPU), due to the limited global memory of these processors, as well as the decrease in performance due to the load unbalance resulting from processing irregular data structures\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35440894', 'topic': [{'uri': 'http://edamontology.org/topic_3570', 'term': 'Pure mathematics'}], 'publication': [{'doi': '10.1145/3458744.3474047', 'pmid': '35440894', 'pmcid': 'PMC9015014'}], 'credit': [{'email': 'oarti001@fiu.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__turbobc', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__turbobc', 'publication_link': 'https://doi.org/10.1145/3458744.3474047', 'is_preprint': False}, {'name': 'DiaNat-DB', 'description': 'a molecular database of antidiabetic compounds from medicinal plants.\\n\\nPlease use this identifier to cite or link to this item: http://rdu.iquimica.unam.mx/handle/20.500.12214/1186.', 'homepage': 'http://rdu.iquimica.unam.mx/handle/20.500.12214/1186', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3938', 'term': 'Virtual screening'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3192', 'term': 'Sequence trimming'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_3303', 'term': 'Medicine'}, {'uri': 'http://edamontology.org/topic_2258', 'term': 'Cheminformatics'}, {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}], 'publication': [{'doi': '10.1039/D0RA10453A', 'pmid': '35424427', 'pmcid': 'PMC8694643'}], 'credit': [{'name': 'Abraham Madariaga-Mazón', 'email': 'amadariaga@iquimica.unam.mx', 'orcidid': 'https://orcid.org/0000-0002-8938-1318', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__dianat-db', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dianat-db', 'publication_link': 'https://doi.org/10.1039/D0RA10453A', 'is_preprint': False}, {'name': 'COVID-19 non-COVID-19 wards Sudan Zambia', 'description': 'O03 The impact of COVID-19 infection prevention and control measures on transmission of hospital-acquired infections and antimicrobial resistance in Africa.\\n\\nAbstract Background. Patients who develop serious illness due to COVID-19 are more likely to have bacterial coinfections, for which WHO recommends treatment with antibiotics. As a result, many countries are observing a change in antimicrobial stewardship (AMS), in addition to changes in infection prevention and control (IPC) practices such as the use of personal protective equipment, on COVID-19 wards. Few data on COVID-19 and its impact on nosocomial infections and antimicrobial resistance (AMR) are available from low and middle-income countries (LMICs)\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE COVID-19 non-COVID-19 wards Sudan, COVID-19 non-COVID-19 wards, patients COVID-19 non-COVID-19 wards Sudan, non-COVID-19 wards Sudan Zambia', 'homepage': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9040044/', 'topic': [{'uri': 'http://edamontology.org/topic_3301', 'term': 'Microbiology'}, {'uri': 'http://edamontology.org/topic_3293', 'term': 'Phylogenetics'}], 'publication': [{'doi': '10.1093/JACAMR/DLAC003.002', 'pmcid': 'PMC9040044'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__covid-19-non-covid-19-wards-sudan-zambia', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__covid-19-non-covid-19-wards-sudan-zambia', 'publication_link': 'https://doi.org/10.1093/JACAMR/DLAC003.002', 'is_preprint': False}, {'name': 'sup- sup', 'description': 'Single-Cell Transcriptome Integration Analysis Reveals the Correlation Between Mesenchymal Stromal Cells and Fibroblasts.\\n\\n<b>Background:< b> Mesenchymal stromal cells (MSCs) and fibroblasts show similar morphology, surface marker expression, and proliferation, differentiation, and immunomodulatory capacities. These similarities not only blur their cell identities but also limit their application. <b>Methods:< b> We performed single-cell transcriptome sequencing of the human umbilical cord and foreskin MSCs (HuMSCs and FSMSCs) and extracted the single-cell transcriptome data of the bone marrow and adipose MSCs (BMSCs and ADMSCs) from the Gene Expression Omnibus (GEO) database\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE sup- sup iCD45, sup- sup iCD34, sup iCD45 i sup- sup, sup- sup iCD34 i sup-', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35360851', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3223', 'term': 'Differential gene expression profiling'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3463', 'term': 'Expression correlation analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_3934', 'term': 'Cytometry'}], 'publication': [{'doi': '10.3389/FGENE.2022.798331', 'pmid': '35360851', 'pmcid': 'PMC8961367'}], 'credit': [{'name': 'Hanhua Yang', 'typeEntity': 'Person'}, {'name': 'Yufeng Liu', 'typeEntity': 'Person'}, {'name': 'Tianyou Wang', 'email': 'Wangty999@sohu.com', 'typeEntity': 'Person'}, {'name': 'Lian Ma', 'email': 'malian8965@sina.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sup--sup', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sup--sup', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.798331', 'is_preprint': False}, {'name': 'SPHARM-Net', 'description': 'Spherical Harmonics-Based Convolution for Cortical Parcellation.\\n\\nWe present a spherical harmonics-based convolutional neural network (CNN) for cortical parcellation, which we call SPHARM-Net. Recent advances in CNNs offer cortical parcellation on a fine-grained triangle mesh of the cortex. Yet, most CNNs designed for cortical parcellation employ spatial convolution that depends on extensive data augmentation and allows only predefined neighborhoods of specific spherical tessellation. On the other hand, a rotation-equivariant convolutional filter avoids data augmentation, and rotational equivariance can be achieved in spectral convolution independent of a neighborhood definition. Nevertheless, the limited resources of a modern machine enable only a finite set of spectral components that might lose geometric details\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Shape-Lab/SPHARM-Net', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1109/TMI.2022.3168670', 'pmid': '35436188'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__spharm-net', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__spharm-net', 'publication_link': 'https://doi.org/10.1109/TMI.2022.3168670', 'is_preprint': False}, {'name': 'HergSPred', 'description': 'Accurate Classification of hERG Blockers/Nonblockers with Machine-Learning Models.\\n\\nHuman oral bioavailability (HOB).\\n\\nBlood-brain barrier penetration (BBB).', 'homepage': 'http://www.icdrug.com/ICDrug/T', 'topic': [{'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}], 'publication': [{'doi': '10.1021/ACS.JCIM.2C00256', 'pmid': '35404051'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__hergspred', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hergspred', 'publication_link': 'https://doi.org/10.1021/ACS.JCIM.2C00256', 'is_preprint': False}, {'name': 'A3D database', 'description': 'structure-based predictions of protein aggregation for the human proteome.\\n\\nSummary Protein aggregation is associated with many human disorders and constitutes a major bottleneck for producing therapeutic proteins. Our knowledge of the human protein structures repertoire has dramatically increased with the recent development of the AlphaFold (AF) deep-learning method. This structural information can be used to understand better protein aggregation properties and the rational design of protein solubility. This article uses the Aggrescan3D (A3D) tool to compute the structure-based aggregation predictions for the human proteome and make the predictions available in a database form\\n\\n||| HOMEPAGE MISSING!\\n\\n||| NAME (A3D database) SIMILAR TO (PUB. DIFFERENT) bio.tools/a3d-database (A3D Database)\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE A3D', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35445695', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}, {'uri': 'http://edamontology.org/operation_4008', 'term': 'Protein design'}, {'uri': 'http://edamontology.org/operation_0409', 'term': 'Protein solubility prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}, {'uri': 'http://edamontology.org/topic_0130', 'term': 'Protein folding, stability and design'}, {'uri': 'http://edamontology.org/topic_0080', 'term': 'Sequence analysis'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}, {'uri': 'http://edamontology.org/topic_0736', 'term': 'Protein folds and structural domains'}], 'publication': [{'doi': '10.1093/BIOINFORMATICS/BTAC215', 'pmid': '35445695', 'pmcid': 'PMC9746890'}], 'credit': [{'name': 'Salvador Ventura', 'email': 'salvador.ventura@uab.cat', 'typeEntity': 'Person'}, {'name': 'Sebastian Kmiecik', 'email': 'sekmi@chem.uw.edu.pl', 'orcidid': 'https://orcid.org/0000-0001-7623-0935', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__a3d-database', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__a3d-database', 'publication_link': 'https://doi.org/10.1093/BIOINFORMATICS/BTAC215', 'is_preprint': False}, {'name': 'ingrained', 'description': 'An Automated Framework for Fusing Atomic-Scale Image Simulations into Experiments.\\n\\nTo fully leverage the power of image simulation to corroborate and explain patterns and structures in atomic resolution microscopy, an initial correspondence between the simulation and experimental image must be established at the outset of further high accuracy simulations or calculations. Furthermore, if simulation is to be used in context of highly automated processes or high-throughput optimization, the process of finding this correspondence itself must be automated. In this work, \"ingrained,\" an open-source automation framework which solves for this correspondence and fuses atomic resolution image simulations into the experimental images to which they correspond, is introduced\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/MaterialEyes/ingrained-lite', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426', 'term': 'Modelling and simulation'}, {'uri': 'http://edamontology.org/operation_3457', 'term': 'Single particle analysis'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3524', 'term': 'Simulation experiment'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}], 'publication': [{'doi': '10.1002/SMLL.202102960', 'pmid': '35384282'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ingrained', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ingrained', 'publication_link': 'https://doi.org/10.1002/SMLL.202102960', 'is_preprint': False}, {'name': 'Research COVID-19 air pollution', 'description': 'A path towards advancing exposure science.\\n\\nThe latest data for coronavirus (COVID-19) cases from the WHO COVID-19 dashboard.\\n\\nWHO Health Emergencies Programme.\\n\\nBolivia (Plurinational State of).\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE COVID-19', 'homepage': 'https://covid19.who.int/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3096', 'term': 'Editing'}, {'uri': 'http://edamontology.org/operation_3283', 'term': 'Anonymisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3305', 'term': 'Public health and epidemiology'}, {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}, {'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_3322', 'term': 'Respiratory medicine'}], 'publication': [{'doi': '10.1016/J.ENVRES.2022.113240', 'pmid': '35390303', 'pmcid': 'PMC8979614'}], 'credit': [{'name': 'Carol J Burns', 'email': 'cjburns.bec@gmail.com', 'typeEntity': 'Person'}, {'name': 'Judy S LaKind', 'email': 'lakindassoc@gmail.com', 'orcidid': 'https://orcid.org/0000-0002-0323-9178', 'typeEntity': 'Person'}, {'name': 'Josh Naiman', 'email': 'joshnaiman@gmail.com', 'typeEntity': 'Person'}, {'name': 'Denali Boon', 'email': 'denali.boon@corteva.com', 'typeEntity': 'Person'}, {'name': 'Jane E Clougherty', 'email': 'jec373@drexel.edu', 'typeEntity': 'Person'}, {'name': 'Ana M Rule', 'email': 'arule1@jhu.edu', 'typeEntity': 'Person'}, {'name': 'Angelika Zidek', 'email': 'angelika.zidek@hc-sc.gc.ca', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__research-covid-19-air-pollution', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__research-covid-19-air-pollution', 'publication_link': 'https://doi.org/10.1016/J.ENVRES.2022.113240', 'is_preprint': False}, {'name': 'rheuma2025', 'description': '[Publicity campaign rheuma2025 of the Union for Rheumatology].\\n\\nJährlich werden 70–80 neue Rheumatolog:innen für Klinik und Praxis gebraucht, um die Zukunft der Patient:innenversorgung zu sichern. #rhmtlgy.', 'homepage': 'https://rheuma2025.de', 'topic': [{'uri': 'http://edamontology.org/topic_3416', 'term': 'Musculoskeletal medicine'}, {'uri': 'http://edamontology.org/topic_3403', 'term': 'Critical care medicine'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'publication': [{'doi': '10.1007/S00393-022-01191-8', 'pmid': '35389077', 'pmcid': 'PMC8988111'}], 'credit': [{'name': 'Hanns-Martin Lorenz', 'email': 'hanns-martin.lorenz@med.uni-heidelberg.de', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__rheuma2025', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rheuma2025', 'publication_link': 'https://doi.org/10.1007/S00393-022-01191-8', 'is_preprint': False}, {'name': 'FateCompass', 'description': 'Characterization of cell-fate decision landscapes by estimating transcription factor dynamics.\\n\\nModulation of gene expression during differentiation by transcription factors promotes cell diversity. Despite their role in cell fate decisions, no experimental assays estimate their regulatory activity in a high-throughput manner and at the single-cell resolution. We present FateCompass for identifying lineage-specific transcription factors across differentiation. It uses single-cell transcriptomics data to infer differentiation trajectories and transcription factor activities. We combined a probabilistic framework with RNA velocities or a differentiation potential to estimate transition probabilities and perform stochastic simulations. Also, we learned transcription factor activities using a linear model of gene regulation. Considering dynamic changes and correlations, we identified lineage-specific regulators\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/sarajimenez/fatecompass', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_0445', 'term': 'Transcription factor binding site prediction'}, {'uri': 'http://edamontology.org/operation_2437', 'term': 'Gene regulatory network prediction'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0749', 'term': 'Transcription factors and regulatory sites'}, {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_0204', 'term': 'Gene regulation'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}], 'credit': [{'name': 'Gérard Gradwohl', 'email': 'molinan@igbmc.fr', 'typeEntity': 'Person'}, {'name': 'Nacho Molina', 'email': 'gradwohl@igbmc.fr', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__fatecompass', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fatecompass', 'publication': [{'doi': '10.1016/j.crmeth.2023.100512', 'pmid': '37533652', 'pmcid': 'PMC10391345'}], 'publication_link': 'https://doi.org/10.1016/j.crmeth.2023.100512', 'is_preprint': False}, {'name': 'CoMI', 'description': 'consensus mutual information for tissue-specific gene signatures.\\n\\nBackground The gene signatures have been considered as a promising early diagnosis and prognostic analysis to identify disease subtypes and to determine subsequent treatments. Tissue-specific gene signatures of a specific disease are an emergency requirement for precision medicine to improve the accuracy and reduce the side effects. Currently, many approaches have been proposed for identifying gene signatures for diagnosis and prognostic. However, they often lack of tissue-specific gene signatures. Results. Here, we propose a new method, consensus mutual information (CoMI) for analyzing omics data and discovering gene signatures\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35439942', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_0313', 'term': 'Expression profile clustering'}, {'uri': 'http://edamontology.org/operation_0533', 'term': 'Expression profile pathway mapping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3577', 'term': 'Personalised medicine'}, {'uri': 'http://edamontology.org/topic_3407', 'term': 'Endocrinology and metabolism'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'publication': [{'doi': '10.1186/S12859-022-04682-2', 'pmid': '35439942', 'pmcid': 'PMC9019939'}], 'credit': [{'name': 'Jinn-Moon Yang', 'email': 'moon@faculty.nctu.edu.tw', 'orcidid': 'https://orcid.org/0000-0002-3205-4391', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__comi', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__comi', 'publication_link': 'https://doi.org/10.1186/S12859-022-04682-2', 'is_preprint': False}, {'name': 'Biocode', 'description': 'A Data-Driven Procedure to Learn the Growth of Biological Networks.\\n\\nProbabilistic biological network growth models have been utilized for many tasks including but not limited to capturing mechanism and dynamics of biological growth activities, null model representation, capturing anomalies, etc. Well-known examples of these probabilistic models are Kronecker model, preferential attachment model, and duplication-based model. However, we should frequently keep developing new models to better fit and explain the observed network features while new networks are being observed. Additionally, it is difficult to develop a growth model each time we study a new network. In this paper, we propose Biocode, a framework to automatically discover novel biological growth models matching user-specified graph attributes in directed and undirected biological graphs\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/seferlab/biocode', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_1781', 'term': 'Gene regulatory network analysis'}, {'uri': 'http://edamontology.org/operation_3660', 'term': 'Metabolic network modelling'}, {'uri': 'http://edamontology.org/operation_0477', 'term': 'Protein modelling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}, {'uri': 'http://edamontology.org/topic_2259', 'term': 'Systems biology'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.1109/TCBB.2022.3165092', 'pmid': '35380966'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__biocode', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__biocode', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3165092', 'is_preprint': False}, {'name': 'FRET-lines', 'description': 'Unraveling multi-state molecular dynamics in single-molecule FRET experiments. I. Theory of FRET-lines.\\n\\nConformational dynamics of biomolecules are of fundamental importance for their function. Single-molecule studies of Förster Resonance Energy Transfer (smFRET) between a tethered donor and acceptor dye pair are a powerful tool to investigate the structure and dynamics of labeled molecules. However, capturing and quantifying conformational dynamics in intensity-based smFRET experiments remains challenging when the dynamics occur on the sub-millisecond timescale. The method of multiparameter fluorescence detection addresses this challenge by simultaneously registering fluorescence intensities and lifetimes of the donor and acceptor\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Fluorescence-Tools/FRETlines', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2476', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/operation_0244', 'term': 'Simulation analysis'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/topic_0593', 'term': 'NMR'}, {'uri': 'http://edamontology.org/topic_0736', 'term': 'Protein folds and structural domains'}, {'uri': 'http://edamontology.org/topic_0130', 'term': 'Protein folding, stability and design'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'publication': [{'doi': '10.1063/5.0089134', 'pmid': '35428384', 'pmcid': 'PMC9014241'}], 'credit': [{'name': 'Hugo Sanabria', 'email': 'hsanabr@clemson.edu', 'orcidid': 'https://orcid.org/0000-0001-7068-6827', 'typeEntity': 'Person'}, {'name': 'Claus A. M. Seidel', 'email': 'cseidel@hhu.de', 'orcidid': 'https://orcid.org/0000-0002-5171-149X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__fret-lines', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fret-lines', 'publication_link': 'https://doi.org/10.1063/5.0089134', 'is_preprint': False}, {'name': 'scMM', 'description': 'A mixture-of-experts deep generative model for integrated analysis of single-cell multiomics data.\\n\\nThe recent development of single-cell multiomics analysis has enabled simultaneous detection of multiple traits at the single-cell level, providing deeper insights into cellular phenotypes and functions in diverse tissues. However, currently, it is challenging to infer the joint representations and learn relationships among multiple modalities from complex multimodal single-cell data. Here, we present scMM, a novel deep generative model-based framework for the extraction of interpretable joint representations and crossmodal generation. scMM addresses the complexity of data by leveraging a mixture-of-experts multimodal variational autoencoder\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/kodaim1115/scMM', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3935', 'term': 'Dimensionality reduction'}, {'uri': 'http://edamontology.org/operation_3625', 'term': 'Relation extraction'}, {'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'publication': [{'doi': '10.1016/J.CRMETH.2021.100071', 'pmid': '35474667', 'pmcid': 'PMC9017195'}], 'credit': [{'name': 'Teppei Shimamura', 'email': 'shimamura@med.nagoya-u.ac.jp', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__scmm', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__scmm', 'publication_link': 'https://doi.org/10.1016/J.CRMETH.2021.100071', 'is_preprint': False}, {'name': 'DNE', 'description': 'Towards a Comprehensive Solution for a Vision-Based Digitized Neurological Examination.\\n\\nDigitized Neurological Examination.\\n\\nAn accessible vision-based exam and documentation solution for neurological biomarkers recording options and clinical applications using a smartphone/tablet device.', 'homepage': 'https://dneproject.web.illinois.edu/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_3937', 'term': 'Feature extraction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3334', 'term': 'Neurology'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1109/JBHI.2022.3167927', 'pmid': '35439148', 'pmcid': 'PMC9707344'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__dne', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dne', 'publication_link': 'https://doi.org/10.1109/JBHI.2022.3167927', 'is_preprint': False}, {'name': 'Movement Disorders', 'description': 'Recommendations of the International Parkinson and Movement Disorder Society Task Force.\\n\\nGenetic Nomenclature in Movement Disorders Study Group.\\n\\nInternational Parkinson and Movement Disorder Society.\\n\\nInternational Parkinson and Movement Disorder Society Genetic Nomenclature in Movement Disorders Study Group.\\n\\nMDS is a professional society of over 11,000 clinicians, scientists, and healthcare professionals dedicated to the care of patients with movement disorders.\\n\\nThe purpose of this study group is to provide recommendations for revising the current nomenclature for genetically determined movement disorders in order to resolve or minimize problems that are sources of confusion and perpetuate misinformation.\\n\\nWe use cookies to give you the best possible experience with our website.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Nomenclature Genetic Movement Disorders, Task Force Nomenclature', 'homepage': 'https://www.movementdisorders.org/MDS/About/Committees--Other-Groups/MDS-Task-Forces/Task-Force-on-Nomenclature-in-Movement-Disorders.htm', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3407', 'term': 'Endocrinology and metabolism'}, {'uri': 'http://edamontology.org/topic_3334', 'term': 'Neurology'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}], 'link': [{'url': 'https://www.movementdisorders.org/', 'type': ['Other']}, {'url': 'https://www.movementdisorders.org/MDS/', 'type': ['Other']}], 'publication': [{'doi': '10.1002/MDS.28982', 'pmid': '35481685'}], 'credit': [{'email': 'm.a.j.de.koning-tijssen@umcg.nl', 'typeEntity': 'Person'}, {'email': 's.van.der.veen04@umcg.nl', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__movement-disorders', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__movement-disorders', 'publication_link': 'https://doi.org/10.1002/MDS.28982', 'is_preprint': False}, {'name': 'eGRM', 'description': 'A genealogical estimate of genetic relationships.\\n\\nThe application of genetic relationships among individuals, characterized by a genetic relationship matrix (GRM), has far-reaching effects in human genetics. However, the current standard to calculate the GRM treats linked markers as independent and does not explicitly model the underlying genealogical history of the study sample. Here, we propose a coalescent-informed framework, namely the expected GRM (eGRM), to infer the expected relatedness between pairs of individuals given an ancestral recombination graph (ARG) of the sample. Through extensive simulations, we show that the eGRM is an unbiased estimate of latent pairwise genome-wide relatedness and is robust when computed with ARG inferred from incomplete genetic data. As a result, the eGRM better captures the structure of a population than the canonical GRM, even when using the same genetic information\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/Ephraim-usc/egrm', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}, {'uri': 'http://edamontology.org/operation_0487', 'term': 'Haplotype mapping'}, {'uri': 'http://edamontology.org/operation_3891', 'term': 'Essential dynamics'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3056', 'term': 'Population genetics'}, {'uri': 'http://edamontology.org/topic_3574', 'term': 'Human genetics'}, {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'}, {'uri': 'http://edamontology.org/topic_3518', 'term': 'Microarray experiment'}], 'publication': [{'doi': '10.1016/J.AJHG.2022.03.016', 'pmid': '35417677', 'pmcid': 'PMC9118131'}], 'credit': [{'name': 'Caoqi Fan', 'email': 'caoqifan@usc.edu', 'typeEntity': 'Person'}, {'name': 'Charleston W.K. Chiang', 'email': 'charleston.chiang@med.usc.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__egrm', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__egrm', 'publication_link': 'https://doi.org/10.1016/J.AJHG.2022.03.016', 'is_preprint': False}, {'name': 'AIM', 'description': 'A Mapping Program for Infrared Spectroscopy of Proteins.\\n\\nHere, we present a new analysis program, AIM, that allows extracting the vibrational amide-I Hamiltonian using molecular dynamics trajectories for protein infrared spectroscopy modeling. The constructed Hamiltonians can be used as input for spectral calculations allowing the calculation of infrared absorption spectra, vibrational circular dichroism, and two-dimensional infrared spectra. These spectroscopies allow the study of the structure and dynamics of proteins. We will explain the essence of how AIM works and give examples of the information and spectra that can be obtained with the program using the Trypsin Inhibitor as an example. AIM is freely available from GitHub, and the package contains a demonstration allowing easy introduction to the use of the program\\n\\n||| HOMEPAGE MISSING!\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/aim', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35387451', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0244', 'term': 'Simulation analysis'}, {'uri': 'http://edamontology.org/operation_2476', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/operation_0477', 'term': 'Protein modelling'}, {'uri': 'http://edamontology.org/operation_3860', 'term': 'Spectrum calculation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0593', 'term': 'NMR'}, {'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/topic_0078', 'term': 'Proteins'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}], 'publication': [{'doi': '10.1021/ACS.JCTC.2C00113', 'pmid': '35387451', 'pmcid': 'PMC9097285'}], 'credit': [{'name': 'Kim E. van Adrichem', 'email': 'k.e.van.adrichem@rug.nl', 'orcidid': 'https://orcid.org/0000-0002-2260-9269', 'typeEntity': 'Person'}, {'name': 'Thomas L. C. Jansen', 'email': 't.l.c.jansen@rug.nl', 'orcidid': 'https://orcid.org/0000-0001-6066-6080', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__aim', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__aim', 'publication_link': 'https://doi.org/10.1021/ACS.JCTC.2C00113', 'is_preprint': False}, {'name': 'TMJOAI', 'description': 'An Artificial Web-Based Intelligence Tool for Early Diagnosis of the Temporomandibular Joint Osteoarthritis.\\n\\nOsteoarthritis is a chronic disease that affects the temporomandibular joint (TMJ), causing chronic pain and disability. To diagnose patients suffering from this disease before advanced degradation of the bone, we developed a diagnostic tool called TMJOAI. This machine learning based algorithm is capable of classifying the health status TMJ in of patients using 52 clinical, biological and jaw condyle radiomic markers. The TMJOAI includes three parts. the feature preparation, selection and model evaluation\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35434730', 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'publication': [{'doi': '10.1007/978-3-030-90874-4_8', 'pmid': '35434730', 'pmcid': 'PMC9012403'}], 'credit': [{'name': 'Celia Le', 'email': 'celiale@umich.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tmjoai', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tmjoai', 'publication_link': 'https://doi.org/10.1007/978-3-030-90874-4_8', 'is_preprint': False}, {'name': 'PADF', 'description': 'A property-oriented adaptive design framework for rapid discovery of energetic molecules based on small-scale labeled datasets.\\n\\nIt remains an important challenge to apply machine learning in material discovery with limited-scale datasets available, in particular for the energetic materials. Motivated by the challenge, we developed a Property-oriented Adaptive Design Framework (PADF) to quickly design new energetic compounds with desired properties. The PADF consists of a search space, machine learning model, optimization algorithm and an evaluator based on quantum mechanical calculations. The effectiveness and generality of the PADF were assessed by two case studies on the heat of formation and heat of explosion as the target properties. 88 compounds were selected as the initial training dataset from the search space containing 84 083 compounds generated\\n\\n||| HOMEPAGE BROKEN!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE SVR', 'homepage': 'https://github.com/Alan-Xie/PADF/blob/main/Code/descriptors.py', 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'publication': [{'doi': '10.1039/D1RA03715C', 'pmid': '35478886', 'pmcid': 'PMC9037014'}], 'credit': [{'name': 'Yunhao Xie', 'email': 'xmpuscu@scu.edu.cn', 'orcidid': 'https://orcid.org/0000-0003-4924-5570', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__padf', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__padf', 'publication_link': 'https://doi.org/10.1039/D1RA03715C', 'is_preprint': False}, {'name': 'SIMUS', 'description': 'An open-source simulator for medical ultrasound imaging. Part II.\\n\\nBackground and objective. Computational ultrasound imaging has become a well-established methodology in the ultrasound community. In the accompanying paper (part I), we described a new ultrasound simulator (SIMUS) for MATLAB, which belongs to the Matlab UltraSound Toolbox (MUST). SIMUS can generate pressure fields and radiofrequency RF signals for simulations in medical ultrasound imaging. It works in a harmonic domain and uses far-field and paraxial linear equations. Methods. In this article (part II), we illustrate how SIMUS compares with other ultrasound simulators (Field II, k-Wave, FOCUS, and Verasonics) for a homogeneous medium. We designed different transmit sequences (focused, planar, and diverging wavefronts) and calculated the corresponding 2-D and 3-D (with elevation focusing) RMS pressure fields. Results. SIMUS produced pressure fields similar to those of Field II, FOCUS, and k-Wave\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35398580', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3954', 'term': 'Echography'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}], 'language': ['MATLAB'], 'publication': [{'doi': '10.1016/J.CMPB.2022.106774', 'pmid': '35398580'}], 'credit': [{'name': 'François Varray', 'email': 'francois.varray@creatis.insa-lyon.fr', 'typeEntity': 'Person'}, {'name': 'Damien Garcia', 'email': 'damien.garcia@creatis.insa-lyon.fr', 'orcidid': 'https://orcid.org/0000-0002-8552-1475', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__simus', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__simus', 'publication_link': 'https://doi.org/10.1016/J.CMPB.2022.106774', 'is_preprint': False}, {'name': 'Telerehabilitation physical disabilities movement impairment', 'description': 'Telerehabilitation for physical disabilities and movement impairment.\\n\\nThe Telerehab toolkit is a patient and practitioner guide to remote assessment and management for people with movement impairment and disability created by researchers at the University of Plymouth in consultation with practitioners and patients.\\n\\nA patient and practitioner guide to remote appointments for people with movement impairment and disability.\\n\\nTelerehabilitation (or telerehab) is the delivery of rehabilitation services via remote methods such as video-based (online) and telephone.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Telerehab, Telerehab Toolkit', 'homepage': 'https://www.plymouth.ac.uk/research/telerehab', 'topic': [{'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'}, {'uri': 'http://edamontology.org/topic_3416', 'term': 'Musculoskeletal medicine'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}], 'publication': [{'doi': '10.1111/JEP.13689', 'pmid': '35437833', 'pmcid': 'PMC9790516'}], 'credit': [{'name': 'Jenny Freeman', 'email': 'jenny.freeman@plymouth.ac.uk', 'orcidid': 'https://orcid.org/0000-0002-4072-9758', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__telerehabilitation-physical-disabilities-movement-impairment', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__telerehabilitation-physical-disabilities-movement-impairment', 'publication_link': 'https://doi.org/10.1111/JEP.13689', 'is_preprint': False}, {'name': 'Cure VCP Scientific Conference 2021', 'description': 'Molecular and clinical insights into neurodegeneration and myopathy linked to multisystem proteinopathy-1 (MSP-1).\\n\\nCure VCP Disease, Inc. aims to drive efforts to cure diseases related to mutations of the Valosin Containing Protein gene. This includes the disease Inclusion Body Myopathy associated with Paget’s disease of bone and Frontotemporal Dementia (IBMPFD).\\n\\nCure VCP Disease, Inc., is dedicated to driving the development of a cure for diseases caused by the valosin containing protein (VCP) gene, including muscle disease, bone disease, and the neurodegenerative diseases ALS, FTD, Parkinson’s, and CMT.\\n\\nWe collaborate on research and advocacy initiatives that help patients and their families. We encourage patients and doctors to connect with us.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Cure VCP Disease, Cure VCP, p97 VCP', 'homepage': 'https://www.curevcp.org', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3920', 'term': 'DNA testing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_3305', 'term': 'Public health and epidemiology'}], 'publication': [{'doi': '10.1016/J.NBD.2022.105722', 'pmid': '35405261', 'pmcid': 'PMC9169230'}], 'credit': [{'name': 'Malavika Raman', 'email': 'malavika.raman@tufts.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__cure-vcp-scientific-conference-2021', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cure-vcp-scientific-conference-2021', 'publication_link': 'https://doi.org/10.1016/J.NBD.2022.105722', 'is_preprint': False}, {'name': 'Gegen Qinlian Decoction', 'description': 'Exploration of the Mechanism of the Control of Coccidiosis in Chickens Based on Network Pharmacology and Molecular Docking With the Addition of Modified Gegen Qinlian Decoction.\\n\\nGegen Qinlian Decoction is a long-established Chinese herbal compound for the treatment of diarrhea and dysentery, while Magnolia officinalis has been demonstrated to have some anthelmintic activity. The preliminary screening of this study showed that the addition of Modified Gegen Qinlian Decoction has some effective on the prevention and treatment of coccidiosis in chickens. However, the mechanism of its treatment of chicken coccidiosis is not clear. The network pharmacology study was based on the screening of chemical components and related targets from TCMSP and PharmMapper server databases\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Gegen Qinlian Decoction fights, Qinlian Decoction, Modified Gegen Qinlian Decoction, Gegen Qinlian', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35372563', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0478', 'term': 'Molecular docking'}, {'uri': 'http://edamontology.org/operation_3501', 'term': 'Enrichment analysis'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2275', 'term': 'Molecular modelling'}, {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_3519', 'term': 'PCR experiment'}, {'uri': 'http://edamontology.org/topic_3343', 'term': 'Compound libraries and screening'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}], 'publication': [{'doi': '10.3389/FVETS.2022.849518', 'pmid': '35372563', 'pmcid': 'PMC8968990'}], 'credit': [{'name': 'Hongbin Si', 'email': 'shb2009@gxu.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gegen-qinlian-decoction', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gegen-qinlian-decoction', 'publication_link': 'https://doi.org/10.3389/FVETS.2022.849518', 'is_preprint': False}, {'name': 'MANE', 'description': 'A joint NCBI and EMBL-EBI transcript set for clinical genomics and research.\\n\\nMatched Annotation from NCBI and EMBL-EBI (MANE).\\n\\nIndex of /refseq/MANE/MANE_human.\\n\\nThe Matched Annotation from NCBI and EMBL-EBI (MANE) is a collaboration that aims to converge on human gene annotation and to produce a genome wide transcript set that includes pairs of RefSeq (NM) and Ensembl/GENCODE (ENST) transcripts that are 100% identical.\\n\\nMANE Transcripts Note: \"MANE Select\" – representative transcripts that are matched between RefSeq and Ensembl.\\n\\nTable of matches from GRCh37 showing equivalence in 5\\' UTR, 3\\' UTR and CDS in relation to MANE Select and MANE Plus Clinical transcripts on GRCh38.\\n\\nThe MANE project - The Matched Annotation from the NCBI and EMBL-EBI (MANE) is a collaborative project that aims to converge on human gene and transcript annotation and to define a genome wide set of representative transcripts and corresponding proteins for human protein-coding genes.', 'homepage': 'https://www.ncbi.nlm.nih.gov/refseq/MANE/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0362', 'term': 'Genome annotation'}, {'uri': 'http://edamontology.org/operation_3225', 'term': 'Variant classification'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_0622', 'term': 'Genomics'}, {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}, {'uri': 'http://edamontology.org/topic_0605', 'term': 'Informatics'}], 'link': [{'url': 'http://tark.ensembl.org/web/mane_project/', 'type': ['Other']}, {'url': 'http://tark.ensembl.org/web/manelist', 'type': ['Other']}, {'url': 'http://tark.ensembl.org/web/mane_GRCh37_list/', 'type': ['Other']}, {'url': 'https://ftp.ncbi.nlm.nih.gov/refseq/MANE/MANE_human/', 'type': ['Other']}], 'publication': [{'doi': '10.1038/S41586-022-04558-8', 'pmid': '35388217', 'pmcid': 'PMC9007741'}], 'credit': [{'name': 'Terence D. Murphy', 'email': 'murphyte@ncbi.nlm.nih.gov', 'orcidid': 'https://orcid.org/0000-0001-9311-9745', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mane', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mane', 'publication_link': 'https://doi.org/10.1038/S41586-022-04558-8', 'is_preprint': False}, {'name': 'PicassoNet', 'description': 'Searching Adaptive Architecture for Efficient Facial Landmark Localization.\\n\\nSince recent facial landmark localization methods achieve satisfying accuracy, few of them enable fast inference speed, which, however, is critical in many real-world facial applications. Existing methods typically employ complicated network structure and predict all the key points through uniform computation, which is inefficient since individual facial part might take different computation to obtain the best performance. Taking both accuracy and efficiency into consideration, we propose the PicassoNet, a lightweight cascaded facial landmark detector with adaptive computation for individual facial part. Different from the conventional cascaded methods, PicassoNet integrates refinement submodules into a single network with group convolution, where each convolution group predicts landmarks from an individual facial part\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35482689', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3927', 'term': 'Network analysis'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'publication': [{'doi': '10.1109/TNNLS.2022.3167743', 'pmid': '35482689'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__picassonet', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__picassonet', 'publication_link': 'https://doi.org/10.1109/TNNLS.2022.3167743', 'is_preprint': False}, {'name': 'P-FLUX', 'description': 'A phosphorus budget dataset spanning diverse agricultural production systems in the United States and Canada.\\n\\nSurface agronomic P budgets for 61 cropping systems using field-scale P flux data across 24 research sites in the United States and Canada.', 'homepage': 'https://doi.org/10.15482/USDA.ADC/1523365', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3431', 'term': 'Deposition'}, {'uri': 'http://edamontology.org/operation_0387', 'term': 'Molecular surface calculation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}, {'uri': 'http://edamontology.org/topic_3855', 'term': 'Environmental science'}], 'publication': [{'doi': '10.1002/JEQ2.20351', 'pmid': '35373848'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__p-flux', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__p-flux', 'publication_link': 'https://doi.org/10.1002/JEQ2.20351', 'is_preprint': False}, {'name': 'BCOVIDOA', 'description': 'A Novel Binary Coronavirus Disease Optimization Algorithm for Feature Selection.\\n\\nThe increased use of digital tools such as smart phones, Internet of Things devices, cameras, and microphones, has led to the produuction of big data. Large data dimensionality, redundancy, and irrelevance are inherent challenging problems when it comes to big data. Feature selection is a necessary process to select the optimal subset of features when addressing such problems. In this paper, the authors propose a novel Binary Coronavirus Disease Optimization Algorithm (BCOVIDOA) for feature selection, where the Coronavirus Disease Optimization Algorithm (COVIDOA) is a new optimization technique that mimics the replication mechanism used by Coronavirus when hijacking human cells. The performance of the proposed algorithm is evaluated using twenty-six standard benchmark datasets from UCI Repository. The results are compared with nine recent wrapper feature selection algorithms\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35464666', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3936', 'term': 'Feature selection'}, {'uri': 'http://edamontology.org/operation_0265', 'term': 'Frameshift detection'}, {'uri': 'http://edamontology.org/operation_3435', 'term': 'Standardisation and normalisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}, {'uri': 'http://edamontology.org/topic_3299', 'term': 'Evolutionary biology'}], 'publication': [{'doi': '10.1016/J.KNOSYS.2022.108789', 'pmid': '35464666', 'pmcid': 'PMC9014647'}], 'credit': [{'name': 'Khalid M. Hosny', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__bcovidoa', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__bcovidoa', 'publication_link': 'https://doi.org/10.1016/J.KNOSYS.2022.108789', 'is_preprint': False}, {'name': 'GDLPA', 'description': 'Gene and drug landing page aggregator.\\n\\nMotivation Many biological and biomedical researchers commonly search for information about genes and drugs to gather knowledge from these resources. For the most part, such information is served as landing pages in disparate data repositories and web portals. Results. The Gene and Drug Landing Page Aggregator (GDLPA) provides users with access to 50 gene-centric and 19 drug-centric repositories, enabling them to retrieve landing pages corresponding to their gene and drug queries. Bringing these resources together into one dashboard that directs users to the landing pages across many resources can help centralize gene- and drug-centric knowledge, as well as raise awareness of available resources that may be missed when using standard search engines. To demonstrate the utility of GDLPA, case studies for the gene klotho and the drug remdesivir were developed\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE cfde-gene-pages', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35368424', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'}, {'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3377', 'term': 'Safety sciences'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'}, {'uri': 'http://edamontology.org/topic_3379', 'term': 'Preclinical and clinical studies'}, {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}], 'publication': [{'doi': '10.1093/BIOADV/VBAC013', 'pmid': '35368424', 'pmcid': 'PMC8969666'}], 'credit': [{'name': 'Avi Ma’ayan', 'email': 'avi.maayan@mssm.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gdlpa', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gdlpa', 'publication_link': 'https://doi.org/10.1093/BIOADV/VBAC013', 'is_preprint': False}, {'name': 'iVp iLAE iBF', 'description': 'Phytochemical Composition and Protective Effect of <i>Vernonanthura polyanthes</i> Leaf against In Vivo Doxorubicin-Mediated Toxicity.\\n\\n<i>Vernonanthura polyanthes< i> (Spreng.) A. J. Vega & Dematt. (syn.: <i>Vernonia polyanthes< i> Less) is popularly known as \"assa-peixe\" and its leaves are used in folk medicine mainly to treat respiratory diseases. In this study, we evaluated the cytogenotoxic and anticytogenotoxic potential of the <i>V. polyanthes< i> leaf aqueous extract (<i>Vp< i>LAE) and its <i>n< i>-butanol fraction (<i>n-< i>BF) in the presence or absence of doxorubicin (DXR) (pre-, co-, and post-treatments) on a murine model for 24 h or 120 h. The micronucleus test (MN) and the comet assay were used to assess the cytogenotoxic and anticytogenotoxic potential of <i>Vp< i>LAE and <i>n-< i>BF (250, 500, and 1000 mg kg) administered via gavage to Swiss Webster mice\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE iVp iLAE, iLAE iBF, iVp iLAE i-BF iV polyanthes', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35458751', 'topic': [{'uri': 'http://edamontology.org/topic_3322', 'term': 'Respiratory medicine'}, {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'}, {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}, {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}, {'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}], 'publication': [{'doi': '10.3390/MOLECULES27082553', 'pmid': '35458751', 'pmcid': 'PMC9032716'}], 'credit': [{'name': 'Elisa Flávia Luiz Cardoso Bailão', 'email': 'elisa.flavia@ueg.br', 'orcidid': 'https://orcid.org/0000-0001-6737-7548', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ivp-ilae-ibf', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ivp-ilae-ibf', 'publication_link': 'https://doi.org/10.3390/MOLECULES27082553', 'is_preprint': False}, {'name': 'switchable CycleGAN', 'description': 'Multi-Contrast MRI Image Synthesis Using Switchable Cycle-Consistent Generative Adversarial Networks.\\n\\nMulti-contrast MRI images use different echo and repetition times to highlight different tissues. However, not all desired image contrasts may be available due to scan-time limitations, suboptimal signal-to-noise ratio, and or image artifacts. Deep learning approaches have brought revolutionary advances in medical image synthesis, enabling the generation of unacquired image contrasts (e.g., T1-weighted MRI images) from available image contrasts (e.g., T2-weighted images). Particularly, CycleGAN is an advanced technique for image synthesis using unpaired images. However, it requires two separate image generators, demanding more training resources and computations. Recently, a switchable CycleGAN has been proposed to address this limitation and successfully implemented using CT images\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE CycleGAN, switchable', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35453864', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}, {'uri': 'http://edamontology.org/operation_3443', 'term': 'Image analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'}, {'uri': 'http://edamontology.org/topic_3452', 'term': 'Tomography'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.3390/DIAGNOSTICS12040816', 'pmid': '35453864', 'pmcid': 'PMC9026507'}], 'credit': [{'name': 'Lili He', 'email': 'lili.he@cchmc.org', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__switchable-cyclegan', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__switchable-cyclegan', 'publication_link': 'https://doi.org/10.3390/DIAGNOSTICS12040816', 'is_preprint': False}, {'name': 'la readmision por deshidratacion tanto', 'description': 'Decreasing Readmissions After Ileostomy Creation Through a Perioperative Quality Improvement Program.\\n\\nBackground Readmission after ileostomy creation in patients undergoing colorectal surgery creates a significant burden on health care cost and patient quality of care, with a 30-day readmission rate of 40%. Objective. This study aimed to evaluate the implementation of our perioperative quality improvement program, Decreasing Readmissions After Ileostomy Creation. Design. Perioperative interventions were administered to patients who underwent ileostomy creation. Settings. A single tertiary care academic center. Patients. Eighty patients participated in this program from February 2020 to January 2021. Main outcome measures\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE como la readmision por deshidratacion, causas como la readmision por, readmision por todas las causas, la creacion de una ileostomia', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35421028', 'topic': [{'uri': 'http://edamontology.org/topic_3421', 'term': 'Surgery'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}], 'publication': [{'doi': '10.1097/DCR.0000000000002256', 'pmid': '35421028'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__la-readmision-por-deshidratacion-tanto', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__la-readmision-por-deshidratacion-tanto', 'publication_link': 'https://doi.org/10.1097/DCR.0000000000002256', 'is_preprint': False}, {'name': 'KDM4B', 'description': 'potential prognostic biomarker revealed by large-scale public databases and clinical samples in uterine corpus endometrial carcinoma.\\n\\nBackground Uterine corpus endometrial carcinoma (UCEC) is the fourth most common cancer among women worldwide. The 5 year survival rate for patients with advanced UCEC is only 17%. Lysine-specific demethylase 4B (KDM4B), a histone demethylase, is overexpressed or dysregulated in various cancers and is associated with tumor progression and poor prognosis. We performed bioinformatics analysis and <i>in vitro< i> assays to assess the role of KDM4B in UCEC. Additionally, the role of KDM4B in the tumor immune microenvironment was explored. Methods. The mRNA and protein levels of KDM4B in UCEC were evaluated using The Cancer Genome Atlas (TCGA), The Human Protein Atlas (HPA), and Gene Expression Omnibus (GEO) databases\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35485290', 'topic': [{'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}], 'publication': [{'doi': '10.1039/D1MO00287B', 'pmid': '35485290'}], 'credit': [{'name': 'Mengjun Zhang', 'email': '1427@hrbmu.edu.cn', 'orcidid': 'https://orcid.org/0000-0001-5501-4598', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__kdm4b', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kdm4b', 'publication_link': 'https://doi.org/10.1039/D1MO00287B', 'is_preprint': False}, {'name': 'Antilogic', 'description': 'new supervised machine learning software for the automatic interpretation of antibiotic susceptibility testing in clinical microbiology.\\n\\nObjective Antibiotic susceptibility testing (AST) is necessary in order to adjust empirical antibiotic treatment, but the interpretation of results requires experience and knowledge. We have developed a machine learning software that is capable of reading AST images without any human intervention and that automatically interprets the AST, based on a database of antibiograms that have been clinically validated with European Committee on Antimicrobial Susceptibility Testing rules. Methods. We built a database of antibiograms that were labelled by senior microbiologists for three species: Escherichia coli, Klebsiella pneumoniae, and Staphylococcus aureus. We then developed Antilogic, a Python software based on an original image segmentation module and supervised learning models that we trained against the database\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35398511', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3301', 'term': 'Microbiology'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}], 'language': ['Python'], 'publication': [{'doi': '10.1016/J.CMI.2022.03.035', 'pmid': '35398511'}], 'credit': [{'name': 'Sophie Alexandra Baron', 'email': 'Sophie.BARON@ap-hm.fr', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__antilogic', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__antilogic', 'publication_link': 'https://doi.org/10.1016/J.CMI.2022.03.035', 'is_preprint': False}, {'name': 'HisCoM-Kernel', 'description': 'Kernel-based hierarchical structural component models for pathway analysis.\\n\\nBIBS lab, Seoul National University.\\n\\nHierarchical structural CoMponent analysis using Kernel (HisCoM-Kernel) is a novel statistical approach for modeling nonlinear associations between biomarkers and phenotype by extending the Kernel machine regression.\\n\\nMain Page Usage (C/C++ version) Download.\\n\\nAn online exploration tool for public and private, multi-omic and clinical/phenotype data.\\n\\nDon\\'t know where to start? Jump in with one of our tutorials or \"How do I …\" walkthroughs.', 'homepage': 'http://statgen.snu.ac.kr/software/HisCom-Kernel/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659', 'term': 'Regression analysis'}, {'uri': 'http://edamontology.org/operation_3928', 'term': 'Pathway analysis'}, {'uri': 'http://edamontology.org/operation_3463', 'term': 'Expression correlation analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'}, {'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'link': [{'url': 'https://xena.ucsc.edu/', 'type': ['Other']}], 'publication': [{'doi': '10.1093/BIOINFORMATICS/BTAC276', 'pmid': '35460238'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__hiscom-kernel', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hiscom-kernel', 'publication_link': 'https://doi.org/10.1093/BIOINFORMATICS/BTAC276', 'is_preprint': False}, {'name': 'subN LiBasub3 subN NaBasub3 subN', 'description': 'First-Principles Study of Three-Dimensional Electrides Containing One-Dimensional [Ba<sub>3</sub>N]<sup>3+</sup> Chains.\\n\\nElectrides, a unique type of compound where electrons act as anions, have a high electron mobility and a low work function, which makes them promising for applications in electronic devices and high-performance catalysts. The discovery of novel electrides and the expansion of the electride family have great significance for their promising applications. Herein, we reported four three-dimensional (3D) electrides by coupling crystal structure database searches and first-principles electronic structure analysis\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Basub3 subN LiBasub3 subN NaBasub3, Basub3 subN LiBasub3 subN, subN NaBasub3 subN Nasub5 subBasub3, NaBasub3 subN Nasub5 subBasub3 subN', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35474803', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_0321', 'term': 'Protein structure validation'}, {'uri': 'http://edamontology.org/operation_0250', 'term': 'Protein property calculation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0081', 'term': 'Structure analysis'}], 'publication': [{'doi': '10.1021/ACSOMEGA.2C00956', 'pmid': '35474803', 'pmcid': 'PMC9026116'}], 'credit': [{'name': 'Tian-Nan Ye', 'typeEntity': 'Person'}, {'name': 'Xiao-Dong Wen', 'orcidid': 'https://orcid.org/0000-0001-5626-8581', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__subn-libasub3-subn-nabasub3-subn', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__subn-libasub3-subn-nabasub3-subn', 'publication_link': 'https://doi.org/10.1021/ACSOMEGA.2C00956', 'is_preprint': False}, {'name': 'fern phylogeny', 'description': 'An Open and Continuously Updated Fern Tree of Life.\\n\\n1 Ferns, with about 12,000 species, are the second most diverse lineage of vascular plants after angiosperms. They have been the subject of numerous molecular phylogenetic studies, resulting in the publication of trees for every major clade and DNA sequences from nearly half of all species. Global fern phylogenies have been published periodically, but as molecular systematics research continues at a rapid pace, these become quickly outdated. Here, we develop a mostly automated, reproducible, open pipeline to generate a continuously updated fern tree of life (FTOL) from DNA sequence data available in GenBank. Our tailored sampling strategy combines whole plastomes (few taxa, many loci) with commonly sequenced plastid regions (many taxa, few loci) to obtain a global, species-level fern phylogeny with high resolution along the backbone and maximal sampling across the tips\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE FTOL, fern phylogenies', 'homepage': 'https://fernphy.github.io', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0323', 'term': 'Phylogenetic inference'}, {'uri': 'http://edamontology.org/operation_3942', 'term': 'Tree dating'}, {'uri': 'http://edamontology.org/operation_3192', 'term': 'Sequence trimming'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'}, {'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}, {'uri': 'http://edamontology.org/topic_0194', 'term': 'Phylogenomics'}, {'uri': 'http://edamontology.org/topic_0637', 'term': 'Taxonomy'}, {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}], 'language': ['R'], 'credit': [{'name': 'Joel H. Nitta', 'email': 'joelnitta@gmail.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__fern-phylogeny', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fern-phylogeny', 'publication': [{'doi': '10.3389/fpls.2022.909768', 'pmid': '36092417', 'pmcid': 'PMC9449725'}], 'publication_link': 'https://doi.org/10.3389/fpls.2022.909768', 'is_preprint': False}, {'name': 'AB-DB', 'description': 'Force-Field parameters, MD trajectories, QM-based data, and Descriptors of Antimicrobials.\\n\\nAntibiotic resistance is a major threat to public health. The development of chemo-informatic tools to guide medicinal chemistry campaigns in the efficint design of antibacterial libraries is urgently needed. We present AB-DB, an open database of all-atom force-field parameters, molecular dynamics trajectories, quantum-mechanical properties, and curated physico-chemical descriptors of antimicrobial compounds. We considered more than 300 molecules belonging to 25 families that include the most relevant antibiotic classes in clinical use, such as β-lactams and (fluoro)quinolones, as well as inhibitors of key bacterial proteins. We provide traditional descriptors together with properties obtained with Density Functional Theory calculations. Noteworthy, AB-DB contains less conventional descriptors extracted from μs-long molecular dynamics simulations in explicit solvent\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35365662', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3938', 'term': 'Virtual screening'}, {'uri': 'http://edamontology.org/operation_2476', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/operation_3482', 'term': 'Antimicrobial resistance prediction'}, {'uri': 'http://edamontology.org/operation_3893', 'term': 'Forcefield parameterisation'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2258', 'term': 'Cheminformatics'}, {'uri': 'http://edamontology.org/topic_0209', 'term': 'Medicinal chemistry'}, {'uri': 'http://edamontology.org/topic_3305', 'term': 'Public health and epidemiology'}, {'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'}], 'publication': [{'doi': '10.1038/S41597-022-01261-1', 'pmid': '35365662', 'pmcid': 'PMC8976083'}], 'credit': [{'name': 'Giuliano Malloci', 'email': 'giuliano.malloci@dsf.unica.it', 'orcidid': 'https://orcid.org/0000-0002-5985-257X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__ab-db', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ab-db', 'publication_link': 'https://doi.org/10.1038/S41597-022-01261-1', 'is_preprint': False}, {'name': 'TurboBFS', 'description': 'GPU Based Breadth-First Search (BFS) Algorithms in the Language of Linear Algebra.\\n\\nParallel Computing and Data Science Lab (Saeed Lab).\\n\\nA scalable datascience lab led by Dr. Fahad Saeed.\\n\\nGPU-ArraySort-2.0 GPU-ArraySort-2.0 Public.\\n\\nThis version of GPU-ArraySort is capable of sorting large number of variable sized arrays, also includes some big fixes.', 'homepage': 'https://github.com/pcdslab', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3802', 'term': 'Sorting'}, {'uri': 'http://edamontology.org/operation_3646', 'term': 'Peptide database search'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3570', 'term': 'Pure mathematics'}, {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}, {'uri': 'http://edamontology.org/topic_3520', 'term': 'Proteomics experiment'}], 'publication': [{'doi': '10.1109/IPDPSW52791.2021.00084', 'pmid': '35425667', 'pmcid': 'PMC9007172'}], 'credit': [{'email': 'oarti001@fiu.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__turbobfs', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__turbobfs', 'publication_link': 'https://doi.org/10.1109/IPDPSW52791.2021.00084', 'is_preprint': False}, {'name': 'NVivo', 'description': 'NVivo qualitative data analysis software helps to discover richer insights from your qualitative & mixed methods research. Organize, store, and analyze data today!.\\n\\nThe NVivo 14 Bundle is Back – Save $280!.', 'homepage': 'https://www.qsrinternational.com/nvivo-qualitative-data-analysis-software/home', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3802', 'term': 'Sorting'}, {'uri': 'http://edamontology.org/operation_0224', 'term': 'Query and retrieval'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.5195/JMLA.2022.1271', 'pmid': '35440911', 'pmcid': 'PMC9014916'}], 'credit': [{'name': 'Kerry Dhakal', 'email': 'kerry.dhakal@osumc.edu', 'orcidid': 'https://orcid.org/0000-0001-7782-5922', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__nvivo', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nvivo', 'publication_link': 'https://doi.org/10.5195/JMLA.2022.1271', 'is_preprint': False}, {'name': 'SARS-ANI', 'description': 'A Global Open Access Dataset of Reported SARS-CoV-2 Events in Animals.\\n\\nJoint statement on the prioritization of monitoring SARS-CoV-2 infection in wildlife and preventing the formation of animal reservoirs.\\n\\nAs we enter the third year of the pandemic, SARS-CoV-2, the virus that causes COVID-19, is spreading between people at an intense level globally.\\n\\nBy Food and Agriculture Organization (FAO), World Organisation for Animal Health (OIE) and World Health Organization (WHO).\\n\\nThe SARS-ANI VIS provides intuitive insights into specific aspects of SARS-CoV-2 events in animals at-a-glance.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE SARS-CoV-2 (bio.tools/sars-cov-2)', 'homepage': 'https://www.who.int/news/item/07-03-2022-joint-statement-on-the-prioritization-of-monitoring-sars-cov-2-infection-in-wildlife-and-preventing-the-formation-of-animal-reservoirs', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3280', 'term': 'Named-entity and concept recognition'}, {'uri': 'http://edamontology.org/operation_0335', 'term': 'Formatting'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3810', 'term': 'Agricultural science'}, {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}, {'uri': 'http://edamontology.org/topic_3305', 'term': 'Public health and epidemiology'}, {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'}, {'uri': 'http://edamontology.org/topic_3452', 'term': 'Tomography'}], 'link': [{'url': 'https://vis.csh.ac.at/sars-ani/', 'type': ['Other']}], 'credit': [{'name': 'Amélie Desvars-Larrive', 'email': 'amelie.desvars@vetmeduni.ac.at', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__sars-ani', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sars-ani', 'publication': [{'doi': '10.1038/s41597-022-01543-8', 'pmid': '35871228', 'pmcid': 'PMC9308035'}], 'publication_link': 'https://doi.org/10.1038/s41597-022-01543-8', 'is_preprint': False}, {'name': 'ConfoundR', 'description': 'Biological misinterpretation of transcriptional signatures in tumour samples can unknowingly undermine mechanistic understanding and faithful alignment with preclinical data.\\n\\nConfoundR is an interactive web application developed in R with Shiny. The goals of this app are to enable users to:.\\n\\nIf you are ready to carry out this analysis, you can skip the below introductions, and examine the expression of individual genes using the Expression Boxplots page, or examine the expression of multiple genes using either a heatmap on the Expression Heatmap page or the Gene Set Enrichment Analysis (GSEA) method on the GSEA page. For more information on the datasets used in the ConfoundR app or for more details on the analysis modules available in the app see below.', 'homepage': 'https://confoundr.qub.ac.uk/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2436', 'term': 'Gene-set enrichment analysis'}, {'uri': 'http://edamontology.org/operation_0314', 'term': 'Gene expression profiling'}, {'uri': 'http://edamontology.org/operation_0315', 'term': 'Expression profile comparison'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3934', 'term': 'Cytometry'}, {'uri': 'http://edamontology.org/topic_3518', 'term': 'Microarray experiment'}], 'credit': [{'name': 'Philip D. Dunne', 'email': 'p.dunne@qub.ac.uk', 'orcidid': 'https://orcid.org/0000-0001-9160-283X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__confoundr', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__confoundr', 'publication': [{'doi': '10.1158/1078-0432.ccr-22-1102', 'pmid': '35792866', 'pmcid': 'PMC9475248'}], 'publication_link': 'https://doi.org/10.1158/1078-0432.ccr-22-1102', 'is_preprint': False}, {'name': 'MuLHiTA', 'description': 'A Novel Multiclass Classification Framework With Multibranch LSTM and Hierarchical Temporal Attention for Early Detection of Mental Stress.\\n\\nMental stress is an increasingly common psychological issue leading to diseases such as depression, addiction, and heart attack. In this study, an early detection framework based on electroencephalogram (EEG) data is developed for reducing the risk of these diseases. In existing frameworks, signals are often segmented into smaller sections prior to being input to a deep neural network. However, this approach ignores the fundamental nature of EEG signals as a carrier of valuable information (e.g., the integrity of frequency and phase, and temporal fluctuations of EEG components). As such, this type of segmenting may lead to information loss and a failure to effectively identify mental stress levels\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35385389', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_3335', 'term': 'Cardiology'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}], 'publication': [{'doi': '10.1109/TNNLS.2022.3159573', 'pmid': '35385389'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mulhita', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mulhita', 'publication_link': 'https://doi.org/10.1109/TNNLS.2022.3159573', 'is_preprint': False}, {'name': 'TAGNet', 'description': 'Learning Configurable Context Pathways for Semantic Segmentation.\\n\\nState-of-the-art semantic segmentation methods capture the relationship between pixels to facilitate contextual information exchange. Advanced methods utilize fixed pathways for context exchange, lacking the flexibility to harness the most relevant context for each pixel. In this paper, we present Configurable Context Pathways (CCPs), a novel model for establishing pathways for augmenting contextual information. In contrast to previous pathway models, CCPs are learned, leveraging configurable regions to form information flows between pairs of pixels. We propose TAGNet to adaptively configure the regions, which span over the entire image space, driven by the relationships between the remote pixels. Subsequently, the information flows along the pathways are updated gradually by the information provided by sequences of configurable regions, forming more powerful contextual information\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/dilincv/TAGNet', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3928', 'term': 'Pathway analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0602', 'term': 'Molecular interactions, pathways and networks'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'publication': [{'doi': '10.1109/TPAMI.2022.3165034', 'pmid': '35471871'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tagnet', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tagnet', 'publication_link': 'https://doi.org/10.1109/TPAMI.2022.3165034', 'is_preprint': False}, {'name': 'HFP-QSMGAN', 'description': 'QSM from homodyne-filtered phase images.\\n\\nPurpose Homodyne filtering is a standard preprocessing step in the estimation of SWI. Unfortunately, SWI is not quantitative, and QSM cannot be accurately estimated from filtered phase images. Compared with gradient-echo sequences suitable for computing QSM, SWI is more readily available and is often the only susceptibility-sensitive sequence acquired in the clinical setting. In this project, we aimed to quantify susceptibility from the homodyne-filtered phase (HFP), acquired for computing susceptibility-weighted images, using convolutional neural networks to solve the compounded problem of (1) computing the solution to the inverse dipole problem, and (2) compensating for the effects of the homodyne filtering. Methods. Two convolutional neural networks, the U-Net and a modified QSMGAN architecture (HFP-QSMGAN), were trained to predict QSM maps at different TEs from HFP images\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/mui-neuro/HFP-QSMGAN', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}, {'uri': 'http://edamontology.org/operation_3799', 'term': 'Quantification'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'}, {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1002/MRM.29260', 'pmid': '35381109', 'pmcid': 'PMC9323427'}], 'credit': [{'name': 'Vincent Beliveau', 'email': 'vincent.beliveau@i-med.ac.at', 'orcidid': 'https://orcid.org/0000-0001-7805-279X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__hfp-qsmgan', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hfp-qsmgan', 'publication_link': 'https://doi.org/10.1002/MRM.29260', 'is_preprint': False}, {'name': 'LightIoT', 'description': 'Lightweight and secure communication for energy-efficient IoT in health informatics.\\n\\nInternet of Things (IoT) is considered as a key enabler of health informatics. IoT-enabled devices are used for in-hospital and in-home patient monitoring to collect and transfer biomedical data pertaining to blood pressure, electrocardiography (ECG), blood sugar levels, body temperature, etc. Among these devices, wearables have found their presence in a wide range of healthcare applications. These devices generate data in real-time and transmit them to nearby gateways and remote servers for processing and visualization. The data transmitted by these devices are vulnerable to a range of adversarial threats, and as such, privacy and integrity need to be preserved. In this paper, we present LightIoT, a lightweight and secure communication approach for data exchanged among the devices of a healthcare infrastructure\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35449692', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}, {'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3063', 'term': 'Medical informatics'}, {'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'}], 'publication': [{'doi': '10.1109/TGCN.2021.3077318', 'pmid': '35449692', 'pmcid': 'PMC9017590'}], 'credit': [{'name': 'Fazlullah Khan', 'email': 'fazlullah@awkum.edu.pk', 'typeEntity': 'Person'}, {'name': 'Spyridon Mastorakis', 'email': 'smastorakis@unomaha.edu', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__lightiot', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__lightiot', 'publication_link': 'https://doi.org/10.1109/TGCN.2021.3077318', 'is_preprint': False}, {'name': 'AsgeneDB', 'description': 'A curated orthology arsenic metabolism gene database and computational tool for metagenome annotation.\\n\\nArsenic (As) is the most ubiquitous toxic metalloid in nature. Microbe mediated As metabolism plays an important role in the global As biogeochemical processes, greatly changing its toxicity and bioavailability. While metagenomic sequencing may advance our understanding of the As metabolism capacity of microbial communities in different environments, accurate metagenomic profiling of As metabolism remains challenging due to low coverage and inaccurate definitions of As metabolism gene families in public orthology databases. Here we developed a manually curated As metabolism gene database (AsgeneDB) comprising 414,773 representative sequences from 59 As metabolism gene families, which are affiliated with 1,653 microbial genera from 46 phyla\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Asgene', 'homepage': 'https://doi.org/10.22541/AU.164975586.65142559/V1', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0362', 'term': 'Genome annotation'}, {'uri': 'http://edamontology.org/operation_3460', 'term': 'Taxonomic classification'}, {'uri': 'http://edamontology.org/operation_3929', 'term': 'Metabolic pathway prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_3407', 'term': 'Endocrinology and metabolism'}, {'uri': 'http://edamontology.org/topic_3837', 'term': 'Metagenomic sequencing'}, {'uri': 'http://edamontology.org/topic_0623', 'term': 'Gene and protein families'}, {'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'}], 'language': ['R'], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__asgenedb', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__asgenedb', 'publication': [{'doi': '10.1093/nargab/lqac080', 'pmid': '36330044', 'pmcid': 'PMC9623898'}], 'publication_link': 'https://doi.org/10.1093/nargab/lqac080', 'is_preprint': False}, {'name': 'pubassistant.ch', 'description': 'consolidating publication profiles of researchers.\\n\\nEnter one or more of the following IDs:.\\n\\nUse Zotero to retrieve DOI of publications.\\n\\nUse Crossref for increased accuracy in matching publications.', 'homepage': 'http://pubassistant.ch', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}, {'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2815', 'term': 'Human biology'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'language': ['R'], 'publication': [{'doi': '10.12688/F1000RESEARCH.73493.3', 'pmid': '35475034', 'pmcid': 'PMC9010799'}], 'credit': [{'name': 'Mark D. Robinson', 'email': 'mark.robinson@mls.uzh.ch', 'orcidid': 'https://orcid.org/0000-0002-3048-5518', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__pubassistant.ch', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pubassistant.ch', 'publication_link': 'https://doi.org/10.12688/F1000RESEARCH.73493.3', 'is_preprint': False}, {'name': 'TcellInflamedDetector', 'description': 'an R package to distinguish T cell inflamed tumor types from non-T cell inflamed tumor types.\\n\\nA major issue in the use of immune checkpoint inhibitors is their lack of efficacy in many patients. Previous studies have reported that the T cell inflamed signature can help predict the response to immunotherapy. Thus, many studies have investigated mechanisms of immunotherapy resistance by defining the tumor microenvironment based on T cell inflamed and non-T cell inflamed subsets. Although methods of calculating T cell inflamed subsets have been developed, valid screening tools for distinguishing T cell inflamed from non-T cell inflamed subsets using gene expression data are still needed, since general researchers who are unfamiliar with the details of the equations can experience difficulties using extant scoring formulas to conduct analyses\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35399012', 'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}], 'language': ['R'], 'publication': [{'doi': '10.5808/GI.22005', 'pmid': '35399012', 'pmcid': 'PMC9001994'}], 'credit': [{'name': 'Hyun-Seok Park', 'email': 'neo@ewha.ac.kr', 'orcidid': 'https://orcid.org/0000-0002-1237-8831', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__tcellinflameddetector', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tcellinflameddetector', 'publication_link': 'https://doi.org/10.5808/GI.22005', 'is_preprint': False}, {'name': 'CRIA', 'description': 'An Interactive Gene Selection Algorithm for Cancers Prediction Based on Copy Number Variations.\\n\\nGenomic copy number variations (CNVs) are among the most important structural variations of genes found to be related to the risk of individual cancer and therefore they can be utilized to provide a clue to the research on the formation and progression of cancer. In this paper, an improved computational gene selection algorithm called CRIA (correlation-redundancy and interaction analysis based on gene selection algorithm) is introduced to screen genes that are closely related to cancer from the whole genome based on the value of gene CNVs. The CRIA algorithm mainly consists of two parts. Firstly, the main effect feature is selected out from the original feature set that has the largest correlation with the class label\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35386679', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2454', 'term': 'Gene prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3958', 'term': 'Copy number variation'}, {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}], 'publication': [{'doi': '10.3389/FPLS.2022.839044', 'pmid': '35386679', 'pmcid': 'PMC8978562'}], 'credit': [{'name': 'Dongxi Li', 'email': 'dxli0426@126.com', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__cria', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cria', 'publication_link': 'https://doi.org/10.3389/FPLS.2022.839044', 'is_preprint': False}, {'name': 'MCDataset', 'description': 'a public reference dataset of Monte Carlo simulated quantities for multilayered and voxelated tissues computed by massively parallel PyXOpto Python package.\\n\\nSignificance Current open-source Monte Carlo (MC) method implementations for light propagation modeling are many times tedious to build and require third-party licensed software that can often discourage prospective researchers in the biomedical optics community from fully utilizing the light propagation tools. Furthermore, the same drawback also limits rigorous cross-validation of physical quantities estimated by various MC codes. Aim. Proposal of an open-source tool for light propagation modeling and an easily accessible dataset to encourage fruitful communications amongst researchers and pave the way to a more consistent comparison between the available implementations of the MC method. Approach\\n\\n||| HOMEPAGE BROKEN!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE PyXOpto', 'homepage': 'https://github.com/xopto/mcdataset', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3431', 'term': 'Deposition'}, {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}, {'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'}, {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}], 'language': ['Python'], 'publication': [{'doi': '10.1117/1.JBO.27.8.083012', 'pmid': '35437973', 'pmcid': 'PMC9016074'}], 'credit': [{'name': 'Peter Naglič', 'email': 'peter.naglic@fe.uni-lj.si', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__mcdataset', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mcdataset', 'publication_link': 'https://doi.org/10.1117/1.JBO.27.8.083012', 'is_preprint': False}, {'name': 'exp2GO', 'description': 'Improving Prediction of Functions in the Gene Ontology With Expression Data.\\n\\nThe computational methods for the prediction of gene function annotations aim to automatically find associations between a gene and a set of Gene Ontology (GO) terms describing its functions. Since the hand-made curation process of novel annotations and the corresponding wet experiments validations are very time-consuming and costly procedures, there is a need for computational tools that can reliably predict likely annotations and boost the discovery of new gene functions. This work proposes a novel method for predicting annotations based on the inference of GO similarities from expression similarities. The novel method was benchmarked against other methods on several public biological datasets, obtaining the best comparative results. exp2GO effectively improved the prediction of GO annotations in comparison to state-of-the-art methods\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/sinc-lab/exp2GO', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3672', 'term': 'Gene functional annotation'}, {'uri': 'http://edamontology.org/operation_2454', 'term': 'Gene prediction'}, {'uri': 'http://edamontology.org/operation_2428', 'term': 'Validation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0089', 'term': 'Ontology and terminology'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}, {'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}], 'publication': [{'doi': '10.1109/TCBB.2022.3167245', 'pmid': '35417352'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__exp2go', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__exp2go', 'publication_link': 'https://doi.org/10.1109/TCBB.2022.3167245', 'is_preprint': False}, {'name': 'MultiGATAE', 'description': 'A Novel Cancer Subtype Identification Method Based on Multi-Omics and Attention Mechanism.\\n\\nCancer is one of the leading causes of death worldwide, which brings an urgent need for its effective treatment. However, cancer is highly heterogeneous, meaning that one cancer can be divided into several subtypes with distinct pathogenesis and outcomes. This is considered as the main problem which limits the precision treatment of cancer. Thus, cancer subtypes identification is of great importance for cancer diagnosis and treatment. In this work, we propose a deep learning method which is based on multi-omics and attention mechanism to effectively identify cancer subtypes. We first used similarity network fusion to integrate multi-omics data to construct a similarity graph\\n\\n||| HOMEPAGE BROKEN!', 'homepage': 'https://github.com/kataomoi7/multiGATAE', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_2269', 'term': 'Statistics and probability'}], 'publication': [{'doi': '10.3389/FGENE.2022.855629', 'pmid': '35391797', 'pmcid': 'PMC8979770'}], 'credit': [{'name': 'Huimin Luo', 'email': 'luohuimin@henu.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__multigatae', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__multigatae', 'publication_link': 'https://doi.org/10.3389/FGENE.2022.855629', 'is_preprint': False}, {'name': 'FullMonteWeb', 'description': \"Scalable and accessible personalized photodynamic therapy optimization with FullMonte and PDT-SPACE.\\n\\nThis is the parent group for the Lilge-Betz group's PDT development. How to get started You'll need a free Gitlab account to...\\n\\nThis is the parent group for the Lilge-Betz group's PDT development. How to get started.\\n\\nYou'll need a free Gitlab account to view projects.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE FullMonte\", 'homepage': 'https://gitlab.com/FullMonte', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337', 'term': 'Visualisation'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}, {'uri': 'http://edamontology.org/topic_3306', 'term': 'Biophysics'}, {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'publication': [{'doi': '10.1117/1.JBO.27.8.083006', 'pmid': '35380030', 'pmcid': 'PMC8978262'}], 'credit': [{'name': 'Abdul-Amir Yassine', 'email': 'abed.yassine@mail.utoronto.ca', 'orcidid': 'https://orcid.org/0000-0003-2159-447X', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__fullmonteweb', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fullmonteweb', 'publication_link': 'https://doi.org/10.1117/1.JBO.27.8.083006', 'is_preprint': False}, {'name': 'iCVI-ARTMAP', 'description': 'Using Incremental Cluster Validity Indices and Adaptive Resonance Theory Reset Mechanism to Accelerate Validation and Achieve Multiprototype Unsupervised Representations.\\n\\nThis article presents an adaptive resonance theory predictive mapping (ARTMAP) model, which uses incremental cluster validity indices (iCVIs) to perform unsupervised learning, namely, iCVI-ARTMAP. Incorporating iCVIs to the decision-making and many-to-one mapping capabilities of this adaptive resonance theory (ART)-based model can improve the choices of clusters to which samples are incrementally assigned. These improvements are accomplished by intelligently performing the operations of swapping sample assignments between clusters, splitting and merging clusters, and caching the values of variables when iCVI values need to be recomputed\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35353707', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'}, {'uri': 'http://edamontology.org/operation_2428', 'term': 'Validation'}, {'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}], 'publication': [{'doi': '10.1109/TNNLS.2022.3160381', 'pmid': '35353707'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__icvi-artmap', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__icvi-artmap', 'publication_link': 'https://doi.org/10.1109/TNNLS.2022.3160381', 'is_preprint': False}, {'name': 'CRDB', 'description': 'Integrated Platform to Gain Statistical Insight Into the Correlation Between Cancer and COVID-19.\\n\\nBackground The advancement of cancer research has been facilitated through freely available cancer literature, databases, and tools. The age of genomics and big data has given rise to the need for cooperation and data sharing in order to make efficient use of this new information in the COVID-19 pandemic. Although there are many databases for cancer research, their access is not easy owing to different ways of processing and managing the data. There is an absence of a unified platform to manage all of them in a transparent and more comprehensible way. Objective\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35430561', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2421', 'term': 'Database search'}, {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}, {'uri': 'http://edamontology.org/operation_3802', 'term': 'Sorting'}, {'uri': 'http://edamontology.org/operation_3792', 'term': 'miRNA expression analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_3068', 'term': 'Literature and language'}, {'uri': 'http://edamontology.org/topic_3489', 'term': 'Database management'}, {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}, {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'}], 'language': ['PHP', 'SQL'], 'publication': [{'doi': '10.2196/35020', 'pmid': '35430561', 'pmcid': 'PMC9191331'}], 'credit': [{'name': 'Tianshun Gao', 'email': 'gts.hust@gmail.com', 'orcidid': 'https://orcid.org/0000-0002-0466-9081', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__crdb', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__crdb', 'publication_link': 'https://doi.org/10.2196/35020', 'is_preprint': False}, {'name': 'Siglec-9', 'description': 'Putative Immune Checkpoint Marker for Cancer Progression Across Multiple Cancer Types.\\n\\nSiglec-9, a cell surface transmembrane receptor mainly expressed on B cells, CD56<sup>+< sup> NK cells, and CD4<sup>+< sup> and CD8<sup>+< sup> T cells, is strongly related to the tumor immune microenvironment. However, the expression pattern of Siglec-9 and its prognostic potential have not been investigated in a pan-cancer perspective. This study aimed to explore the association of Siglec-9 with prognosis, tumor stage, molecular subtype, and the immune microenvironment in pan-cancer. The mRNA expression of Siglec-9 was obtained from The Cancer Genome Atlas (TCGA), the Broad Institute Cancer Cell Line Encyclopedia (CCLE), and Genotype-Tissue Expression (GTEx). The relationship between Siglec-9 mRNA expression and prognosis was evaluated by the Kaplan-Meier analysis\\n\\n||| HOMEPAGE MISSING!', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35372497', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2436', 'term': 'Gene-set enrichment analysis'}, {'uri': 'http://edamontology.org/operation_3463', 'term': 'Expression correlation analysis'}, {'uri': 'http://edamontology.org/operation_3223', 'term': 'Differential gene expression profiling'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3400', 'term': 'Allergy, clinical immunology and immunotherapeutics'}, {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}, {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'}, {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'}, {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}], 'publication': [{'doi': '10.3389/FMOLB.2022.743515', 'pmid': '35372497', 'pmcid': 'PMC8968865'}], 'credit': [{'name': 'Zhongping Cheng', 'email': 'mdcheng18@263.net', 'typeEntity': 'Person'}, {'name': 'Zhengliang Gao', 'email': 'zhengliang_gao@tongji.edu.cn', 'typeEntity': 'Person'}, {'name': 'Shupeng Liu', 'email': 'lshup@tongji.edu.cn', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__siglec-9', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__siglec-9', 'publication_link': 'https://doi.org/10.3389/FMOLB.2022.743515', 'is_preprint': False}, {'name': 'gRDF', 'description': 'An Efficient Compressor with Reduced Structural Regularities That Utilizes gRePair.\\n\\nThe explosive volume of semantic data published in the Resource Description Framework (RDF) data model demands efficient management and compression with better compression ratio and runtime. Although extensive work has been carried out for compressing the RDF datasets, they do not perform well in all dimensions. However, these compressors rarely exploit the graph patterns and structural regularities of real-world datasets. Moreover, there are a variety of existing approaches that reduce the size of a graph by using a grammar-based graph compression algorithm. In this study, we introduce a novel approach named gRDF (graph repair for RDF) that uses gRePair, one of the most efficient grammar-based graph compression schemes, to compress the RDF dataset\\n\\n||| HOMEPAGE MISSING!\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE HDT HDT++ k2-trees RDF-TR gRePair', 'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=35408160', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3211', 'term': 'Genome indexing'}, {'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}], 'publication': [{'doi': '10.3390/S22072545', 'pmid': '35408160', 'pmcid': 'PMC9003471'}], 'credit': [{'name': 'Young-Koo Lee', 'email': 'yklee@khu.ac.kr', 'orcidid': 'https://orcid.org/0000-0003-2314-5395', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__grdf', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__grdf', 'publication_link': 'https://doi.org/10.3390/S22072545', 'is_preprint': False}, {'name': 'GROMACS-SWAXS', 'description': 'combining MD simulations with Bayesian inference or with the maximum entropy principle.\\n\\nWelcome to GROMACS-SWAXS, a modified GROMACS version for:.\\n\\nToggle Light / Dark / Auto color theme.\\n\\nToggle table of contents sidebar.', 'homepage': 'https://cbjh.gitlab.io/gromacs-swaxs-docs', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2476', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/operation_0244', 'term': 'Simulation analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'}, {'uri': 'http://edamontology.org/topic_2828', 'term': 'X-ray diffraction'}, {'uri': 'http://edamontology.org/topic_0611', 'term': 'Electron microscopy'}], 'publication': [{'doi': '10.1101/2022.04.05.487171'}], 'credit': [{'name': 'Jochen S. Hub', 'email': 'jochen.hub@uni-saarland.de', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__gromacs-swaxs', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gromacs-swaxs', 'publication_link': 'https://doi.org/10.1101/2022.04.05.487171', 'is_preprint': False}]\n",
      "There are 48 preprints and 170 pubs in tools\n",
      "There are 246 preprints in the json file. Checking if something was published.....\n",
      "HISTA\n",
      "TreeProfiler\n",
      "GametesOmics\n",
      "GammaGateR\n",
      "GAUSS\n",
      "UNNT\n",
      "BaRDIC\n",
      "CROTONdb\n",
      "Cell4D\n",
      "MAGPIE\n",
      "MolPhase\n",
      "RapTCR\n",
      "SMARTdb\n",
      "3Dmapper\n",
      "MacaqueNet\n",
      "plasmidEC\n",
      "ASBAR\n",
      "bistro\n",
      "GIA\n",
      "iQC\n",
      "PMTPred\n",
      "Omix\n",
      "AITeQ\n",
      "microbiomedataset\n",
      "Localgini\n",
      "EpiSegMix\n",
      "MoTR\n",
      "pyHiM\n",
      "HILAMA\n",
      "OpenCell\n",
      "omicsMIC\n",
      "ASTA-P\n",
      "ACE\n",
      "Allo\n",
      "QInfoMating\n",
      "HLAfreq\n",
      "DPKS\n",
      "ssREAD\n",
      "PTMNavigator\n",
      "IMPALA\n",
      "MPDS\n",
      "CAMPER\n",
      "starTracer\n",
      "CellsFromSpace\n",
      "CBGTPy\n",
      "VDI\n",
      "NetAn\n",
      "eMIND\n",
      "SpatialPrompt\n",
      "Retro-MTGR\n",
      "NCBench\n",
      "GeneSetR\n",
      "CONORM\n",
      "MUSTANG\n",
      "evopython\n",
      "hART\n",
      "phylociraptor\n",
      "Monkeybread\n",
      "HAVAC\n",
      "Cfold\n",
      "simmr\n",
      "AICOM-MP\n",
      "Mycotools\n",
      "SurvBal\n",
      "Het-PR\n",
      "ANMS\n",
      "SeqVerify\n",
      "SCP Viz\n",
      "magpie\n",
      "VData\n",
      "ONeSAMP 3.0\n",
      "ALL-Sum\n",
      "PubNavigator\n",
      "rotate\n",
      "InstaNovo\n",
      "SCancerRNA\n",
      "PyHFO\n",
      "SuperFeat\n",
      "MatchMaps\n",
      "tidytcells\n",
      "MIGRENE\n",
      "AutoBA\n",
      "RCoV19\n",
      "Virseqimprover\n",
      "Causal Cookbook\n",
      "GB99dms\n",
      "DNS\n",
      "UNI\n",
      "AFITBin\n",
      "ProteinFlow\n",
      "inDrops-2\n",
      "FL-VLD\n",
      "EXtra-Xwiz\n",
      "cellatlas\n",
      "MLPER\n",
      "MLDAAPP\n",
      "MRMCsamplesize\n",
      "richPathR\n",
      "OpenToolFlux\n",
      "MetFinder\n",
      "CellPolaris\n",
      "WCRC-25\n",
      "LCLUC\n",
      "Autometa\n",
      "DevCCF\n",
      "PepCNN\n",
      "knee OA\n",
      "Cell2Sentence\n",
      "ZEPPI\n",
      "candy darter\n",
      "BAllCools\n",
      "MocFormer\n",
      "bamSliceR\n",
      "iLEC-DNA\n",
      "intNMF\n",
      "ScreenDOP\n",
      "FairMobi-Net\n",
      "ZeptoCTC\n",
      "ReproduceMe\n",
      "InARes\n",
      "QPPLab\n",
      "SIDERITE\n",
      "BAllC\n",
      "MammalMethylClock\n",
      "U-Flow\n",
      "GROMACS-CIF\n",
      "BayesKAT\n",
      "VIBES\n",
      "admix-kit\n",
      "Tool admix-kit has a published version. Changing from 10.1101/2023.09.30.560263 to 10.1093/bioinformatics/btae148 \n",
      "Tool admix-kit has a published version. Changing from 10.1101/2023.09.30.560263 to 38490256 \n",
      "diplo-locus\n",
      "TARO\n",
      "cellPLATO\n",
      "PhysiCell Studio\n",
      "APOLLO\n",
      "gaiaAssociation\n",
      "skDER\n",
      "FluxNorm\n",
      "TRILL\n",
      "PFP-FM\n",
      "PhyGCN\n",
      "SOCfinder\n",
      "protoSpaceJAM\n",
      "Multioviz\n",
      "OMEinfo\n",
      "Tool OMEinfo has a published version. Changing from 10.1101/2023.10.23.563576 to 10.1093/bioadv/vbae025 \n",
      "Tool OMEinfo has a published version. Changing from 10.1101/2023.10.23.563576 to 38456126 \n",
      "MDMD\n",
      "AP-LASR\n",
      "Pindel-TD\n",
      "Segzoo\n",
      "PyDecNef\n",
      "PMGmap\n",
      "AnophelesModel\n",
      "BTR\n",
      "NAMERS\n",
      "NeoHeadHunter\n",
      "MSIsensor-RNA\n",
      "Nphos\n",
      "OncoReport\n",
      "sincFold\n",
      "TEC-miTarget\n",
      "MTGCL\n",
      "ABIapp\n",
      "BATTER\n",
      "LAVASET\n",
      "PREVIC\n",
      "PhysioFit\n",
      "GenomeSpy\n",
      "MetaQuad\n",
      "Tool MetaQuad has a published version. Changing from 10.1101/2023.10.09.561628 to 10.1093/bioadv/vbae030 \n",
      "Tool MetaQuad has a published version. Changing from 10.1101/2023.10.09.561628 to 38476299 \n",
      "TEPEAK\n",
      "Ocelli\n",
      "3DVascNet\n",
      "MSA_Limit\n",
      "cellContrast\n",
      "HIV-DRIVES\n",
      "MCHelper\n",
      "Rummagene\n",
      "ReporterScore\n",
      "TreePPL\n",
      "AleRax\n",
      "NanoLAS\n",
      "STEGO\n",
      "Listi Wiki\n",
      "CUDASW++4.0\n",
      "TEclass2\n",
      "phyloKRR\n",
      "EPDRNA\n",
      "Tool EPDRNA has a published version. Changing from 10.21203/RS.3.RS-3476519/V1 to 10.1007/s10930-024-10183-3 \n",
      "Tool EPDRNA has a published version. Changing from 10.21203/RS.3.RS-3476519/V1 to 38491248 \n",
      "NeuroMotion\n",
      "STew\n",
      "READi-Dem\n",
      "PitArray\n",
      "CellPie\n",
      "MADAME\n",
      "scRANK\n",
      "SiamEEGNet\n",
      "Graphite\n",
      "DynaMight\n",
      "SODA\n",
      "AgroNT\n",
      "PDB NextGen Archive\n",
      "TreeArchTraits\n",
      "cubeVB\n",
      "NIEND\n",
      "plastiC\n",
      "ColoRecPred\n",
      "NanoBERTa-ASP\n",
      "SeqPHASE\n",
      "Capricorn\n",
      "ConfluentFUCCI\n",
      "dismo\n",
      "immuno-cross\n",
      "CIViCutils\n",
      "climetrics\n",
      "GraphR\n",
      "RESIRE-V\n",
      "COINSTAC\n",
      "TopoRoot+\n",
      "bsAbs\n",
      "KmerSV\n",
      "AdaptiX\n",
      "FedGMMAT\n",
      "scDiff\n",
      "PhysiMeSS\n",
      "ORACLE\n",
      "ARPALData\n",
      "MIC-SQTL\n",
      "gtfsort\n",
      "AID-HS\n",
      "DiabetIA\n",
      "BFF\n",
      "TOLGAN\n",
      "KIET Annual Report\n",
      "SaProt\n",
      "VRP assembler\n",
      "Kairos\n",
      "VAE\n",
      "Int and in\n",
      "pyTME\n",
      "EBP\n",
      "AngoraPy\n",
      "CytoPipeline\n",
      "Nanotope\n",
      "FedRDR\n",
      "HSQC2STRUC\n",
      "StrastiveVI\n",
      "PhyloAcc-C\n",
      "DepMap\n",
      "scBoolSeq\n",
      "[{'name': 'admix-kit', 'description': 'An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations.\\n\\nSummary Admixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations. Availability and implementation. Admix-kit package is open-source and available at https: github.com KangchengHou admix-kit. Additionally, users can use the pipeline designed for admixed genotype simulation available at https: github.com UW-GAC admix-kit_workflow', 'homepage': 'https://github.com/KangchengHou/admix-kit', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'}, {'uri': 'http://edamontology.org/operation_3920', 'term': 'DNA testing'}, {'uri': 'http://edamontology.org/operation_0244', 'term': 'Simulation analysis'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_0625', 'term': 'Genotype and phenotype'}, {'uri': 'http://edamontology.org/topic_3056', 'term': 'Population genetics'}, {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}], 'language': ['Python'], 'link': [{'url': 'https://github.com/UW-GAC/admix-kit_workflow', 'type': ['Repository']}], 'credit': [{'name': 'Kangcheng Hou', 'email': 'houkc@ucla.edu', 'typeEntity': 'Person'}, {'name': 'Bogdan Pasaniuc', 'email': 'pasaniuc@ucla.edu', 'typeEntity': 'Person'}, {'name': 'Haoyu Zhang', 'email': 'haoyu.zhang2@nih.gov', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__admix-kit', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__admix-kit', 'publication_link': 'https://pubmed.ncbi.nlm.nih.gov/38490256/', 'is_preprint': False, 'publication': [{'doi': '10.1093/bioinformatics/btae148', 'pmid': '38490256'}]}, {'name': 'OMEinfo', 'description': 'Global Geographic Metadata for -omics Experiments.\\n\\nMicrobiome studies increasingly associate geographical features like rurality and climate types with microbiomes. However, microbiologists bioinformaticians often struggle to access and integrate rich geographical metadata from sources such as GeoTIFFs; and inconsistent definitions of rurality, for example, can hinder cross-study comparisons. To address this, we present OMEinfo, a Python-based tool for automated retrieval of consistent geographical metadata from user-provided location data. OMEinfo leverages open data sources such as the Global Human Settlement Layer, Köppen-Geiger climate classification models, and Open-Data Inventory for Anthropogenic Carbon dioxide, to ensure metadata accuracy and provenance', 'homepage': 'https://github.com/m-crown/OMEinfo/', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422', 'term': 'Data retrieval'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'}, {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_3277', 'term': 'Sample collections'}], 'language': ['Python'], 'license': 'MIT', 'credit': [{'email': 'matthew.bashton@northumbria.ac.uk', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__omeinfo', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__omeinfo', 'publication_link': 'https://pubmed.ncbi.nlm.nih.gov/38456126/', 'is_preprint': False, 'publication': [{'doi': '10.1093/bioadv/vbae025', 'pmid': '38456126', 'pmcid': 'PMC10918632'}]}, {'name': 'MetaQuad', 'description': 'Shared Informative Variants Discovery in Metagenomic Samples.\\n\\nMotivation Strain-level analysis of metagenomic data has garnered significant interest in recent years. Microbial single nucleotide polymorphisms (SNPs) are genomic variants that can reflect strain-level differences within a microbial species. The diversity and emergence of SNPs in microbial genomes may reveal evolutionary history and environmental adaptation in microbial populations. However, efficient discovery of shared polymorphic variants in a large collection metagenomic samples remains a computational challenge. Results. MetaQuad employs a density-based clustering technique to efficiently differentiate shared variants from non-polymorphic sites using shotgun metagenomic data. Empirical comparisons with other state-of-the-art methods show that MetaQuad significantly reduces the number of false-positive SNPs without greatly affecting the true-positive rate', 'homepage': 'https://github.com/holab-hku/MetaQuad', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0484', 'term': 'SNP detection'}, {'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'}, {'uri': 'http://edamontology.org/operation_3675', 'term': 'Variant filtering'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}, {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'}, {'uri': 'http://edamontology.org/topic_3837', 'term': 'Metagenomic sequencing'}, {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}], 'language': ['Python', 'R'], 'license': 'MIT', 'credit': [{'name': 'Joshua W. K. Ho', 'email': 'jwkho@hku.hk', 'typeEntity': 'Person'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__metaquad', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__metaquad', 'publication_link': 'https://pubmed.ncbi.nlm.nih.gov/38476299/', 'is_preprint': False, 'publication': [{'doi': '10.1093/bioadv/vbae030', 'pmid': '38476299'}]}, {'name': 'EPDRNA', 'description': 'a model for identifying DNA-RNA binding sites in disease-related proteins.\\n\\nEPDRNA--Prediction of DNA and RNA binding sites in disease-associated proteins.\\n\\nEnter the query protein sequence in FASTA format(maximum 10 proteins for each submission).', 'homepage': 'http://www.s-bioinformatics.cn/epdrna', 'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3902', 'term': 'RNA binding site prediction'}, {'uri': 'http://edamontology.org/operation_3903', 'term': 'DNA binding site prediction'}, {'uri': 'http://edamontology.org/operation_2464', 'term': 'Protein-protein binding site prediction'}, {'uri': 'http://edamontology.org/operation_3901', 'term': 'RNA-binding protein prediction'}]}], 'topic': [{'uri': 'http://edamontology.org/topic_3125', 'term': 'DNA binding sites'}, {'uri': 'http://edamontology.org/topic_0128', 'term': 'Protein interactions'}, {'uri': 'http://edamontology.org/topic_3534', 'term': 'Protein binding sites'}, {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}, {'uri': 'http://edamontology.org/topic_0099', 'term': 'RNA'}], 'confidence_flag': 'high', 'editPermission': {'type': 'public'}, 'biotoolsID': 'pub2tools2023__epdrna', 'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__epdrna', 'publication_link': 'https://pubmed.ncbi.nlm.nih.gov/38491248/', 'is_preprint': False, 'publication': [{'doi': '10.1007/s10930-024-10183-3', 'pmid': '38491248'}]}]\n",
      "There are 4 published tools in the json file. \n",
      "Updating json.....\n",
      "There are 48 preprints in the file after identification\n"
     ]
    }
   ],
   "source": [
    "tools_prp = identify_preprints(tools_v,preprints_file)    #STEP 4. FROM THOSE VALIDATED, IDENTIFY PREPRINTS. RERUN = FALSE - it's first time we add \"is_preprint\" flag. More about it in preprints.py file.\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " Writing 100 files which is 100 publications to a pub2tools_2022_02.csv \n"
     ]
    }
   ],
   "source": [
    "tools_to_add, tools_left = generate_csv_pub(tools_prp,100) #STEP 5. GENERATE CSV FROM TO_CURATE FIRST PUBLICATIONS. Alternative function - generate_csv_prp will use TO_CURATE first publications and identified preprints (that might be useful in case of coming back to old curation schema (curating all validated tools))\n",
    "generate_json(tools_left)                                        # STEP 6. GEENRATE TWO JSON FILES FROM TOOLS THAT WON'T BE CURATED. (leftover tools = all tools - to_curate tools). Generates file with preprints and file with publications. File with preprints can be used as input to identify_preprints later. \n",
    "#add_tools(tools_to_add, token, http_settings, WRITE_TO_DB)       # STEP 7. ADD TO_CURATE TOOLS TO DEV\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.7"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
