{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run with pub2tools output"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. Define run settings:\n",
    "\n",
    "    * _include_preprints_ (bool): set to False if preprints should not be included in the tools to add to the database.\n",
    "    * _to_curate_ (int || 'all'): number of published tools to be added to the database."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "include_preprints = False\n",
    "to_curate = 100"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "2. Define file paths:\n",
    "\n",
    "    * _json_file_: path for json file with output from Pub2Tools.\n",
    "    * _pub2tools_log_:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "json_file = \"C:/Users/aname/Downloads/to_biotools.json\"\n",
    "pub2tools_log = \"pub2tools.log\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "3. Define username and password."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "username = 'curation'\n",
    "password = 'D9v7#5KV'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "4. Authentication."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from biotools_dev import login_prod\n",
    "\n",
    "token = login_prod(username, password)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "5. Read pub2tools output and get tools with high confidence score from json file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json\n",
    "from tool_processing import process_tools\n",
    "\n",
    "with open(json_file) as jf:\n",
    "    data = json.load(jf)\n",
    "    tools = data['list']\n",
    "\n",
    "processed_tools = process_tools(tools)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Tool validation**\n",
    "\n",
    "6. Validate tools and separate them into valid and problem tools."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Tool with name:HISTA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Trackplot has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SpatialDDLS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:aenmd has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:VoDEx has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OBIA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TreeProfiler has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:KidneyGPS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ML-PLIC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:VCFshiny has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DataDTA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PTBGRP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EasySSR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:scAce has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:A3DyDB has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:KAS-Analyzer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GraffiTE has the errors: {\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Tool with name:GametesOmics has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NeuroBridge has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RgnTX has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GammaGateR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SCAR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TIPred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GAUSS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CRUSTY has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PanEffect has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UNNT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:COWID has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BaRDIC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:YoloCurvSeg has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CROTONdb has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LexMAL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RibDif2 has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Tool with name:cfOmics has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Cell4D has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Phables has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EquiPNAS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PC3T has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Onkopipe has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MAGPIE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ABC-HuMi has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MuTATE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CIRCUST has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:dynamicLM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FLIBase has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MolPhase has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RapTCR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ActivePPI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MAVEN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SMARTdb has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MuDCoD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:3Dmapper has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ePlatypus has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DeepMHCI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:T9GPred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MacaqueNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:P-DOR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:pLM-BLAST has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BiocMAP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BondGraphs.jl has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MoleculeExperiment has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FunTaxIS-lite has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:plasmidEC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ASBAR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:bistro has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GIA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:iQC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:STAR_outliers has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SincMSNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PMTPred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Tcbf has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Omix has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AITeQ has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DeepCCI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:chem16S has the errors: {\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Tool with name:microbiomedataset has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:IDLaS-NL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CoVpipe2 has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{}]}]}\n",
      "Tool with name:HACA3 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SAM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Localgini has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EpiSegMix has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MoTR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SEM3De has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:phippery has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:WebSTR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DANGER analysis has the errors: {\"function\":[{\"operation\":[{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}\n",
      "Tool with name:eDAVE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EnsemPPIS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:pyHiM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:gRNA-SeqRET has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HILAMA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OpenCell has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PHFinder has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:omicsMIC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:phyloGAN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GillesPy2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ASTA-P has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BioREx has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ACE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Allo has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:QInfoMating has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PyDESeq2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:IREDFisher has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BioThings Explorer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HLAfreq has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DPKS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PCNportal has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CCPlotR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ssREAD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Oktoberfest has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:pyPept has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:scPipe has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PTMNavigator has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SatAnalyzer has the errors: {\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Tool with name:IMPALA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Recombulator-X has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GiftBTE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ensemblQueryR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MPDS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OSCNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ECL 3.0 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BioNetGMMFit has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CAMPER has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CRISPR-COPIES has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]},{},{}]}]}\n",
      "Tool with name:BayesLDM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:mbQTL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:starTracer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ediblecity has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:INTEGRATE-Circ has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CellsFromSpace has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CBGTPy has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ScribbleDom has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Seq2Sat has the errors: {\"function\":[{\"operation\":[{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{}]}]}\n",
      "Tool with name:FrameD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:acorn has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PyCreas has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DEVOUR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PyMC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:VDI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Cyclone has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NetAn has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PDB-BRE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:COLLAGENE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:StARGate-X has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PiRATeMC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ROSes-FINDER has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:eMIND has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DefCor-Net has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SpatialPrompt has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RELIEF has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Infomovel has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HowDirty has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AllCCS2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RINmaker has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LSA-ac4C has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Retro-MTGR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Infrared has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TRAP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GRIN-Toolbox has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HPC-Atlas has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MKCL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PLAE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EasyKin has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NCBench has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GALBA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:rMSIfragment has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GeNetOntology has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ChromNetMotif has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LOGICS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GeneSetR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:STGNNks has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CONORM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PGDB has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}\n",
      "Tool with name:HQAlign has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ReconFormer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NEmecum has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BQsupports has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BG2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MUSTANG has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:evopython has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CSNN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HMCDA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UNesT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SWATH-F has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MMIP has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{}]}]}\n",
      "Tool with name:GOWDL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:hART has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RSim has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RNA-GAN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:skani has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DISCERN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:J-score has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:compleasm has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AliSim-HPC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:phylociraptor has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Monkeybread has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Graph-CSPNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Firalink has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:iHerd has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:POKE-ADDICT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ClinPrior has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RetroBioCat Database has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CONVIQT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HAVAC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Cfold has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DRING has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MACtest has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CCsup+ sup has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:simmr has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DrugMechDB has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:D-LMBmap has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DPSP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DASH has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AICOM-MP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Mycotools has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AlignStatPlot has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FexSplice has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CBPDdb has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SurvBal has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ZINBMM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ChromatoShiny has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Embassy Good Science has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SoyDBean has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ATOdb has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MEGADOCK-on-Colab has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Open Genes has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ANTI-Vea-UGR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CrMP-Sol database has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:mtPGS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Het-PR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MI-DAGSC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OntoAvida has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:WSIC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PorcineAI-enhancer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:scTOP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:causalCmprsk has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Simplicity has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DrForna has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:singleCellBase has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:1p 19qNET has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ANMS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Scorpio has the errors: {\"function\":[{\"operation\":[{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]}]}]}\n",
      "Tool with name:SeqVerify has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DSMRI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ASySD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SCP Viz has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:magpie has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GraphIdn has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BendNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HiTaxon has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PyMVPA BIDS-App has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:VData has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ONeSAMP 3.0 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BaroWavelet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:nursing workforce competencies job satisfaction has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GeleNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:iRNxQuest has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UCSF ChimeraX has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PySynthMRI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Scellpam has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ALL-Sum has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PubNavigator has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GelMap has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NeuroMabSeq has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EzMechanism has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MassSpecPreppy has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:rotate has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BEACH has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PhaseDancer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:InstaNovo has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:cloneRate has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RTlogD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EnsembleFit has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MPOA has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}\n",
      "Tool with name:SCancerRNA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PyHFO has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:metaBAT-LR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RefSeeker has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FGNN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Contact-FP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SuperFeat has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TidyTron has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MatchMaps has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MBMethPred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:tidytcells has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Aerogel has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BioDGW-CMI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:bubbleHeatmap has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TIVC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MIGRENE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EFA-Net has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AutoBA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HAVOC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:happi has the errors: {\"function\":[{\"operation\":[{},{},{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]}]}]}\n",
      "Tool with name:HMD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RCoV19 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Sensommatic has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NODD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Virseqimprover has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FFGC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Toblerone has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EASi-seq has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{},{}]}]}\n",
      "Tool with name:RIBFIND2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Causal Cookbook has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Raidionics has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Fast-SNN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BertNDA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GB99dms has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LogBB_Pred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:RibSeg v2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DNS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ICAL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UNI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HAHNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SCINPAS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CALIBRA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SciKit-SurgeryGlenoid has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SELINA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Bsup3 supX has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:cudaMMC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AFITBin has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ProteinFlow has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:inDrops-2 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FL-VLD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EXtra-Xwiz has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:cellatlas has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BridgIT+ has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3939.\"]},{},{},{}]}]}\n",
      "Tool with name:GOAT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ShIVA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MTBC 0 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FISA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MicroTom has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{}]}]}\n",
      "Tool with name:ddgRADer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:s-CRIq has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MLPER has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PairVAE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MLDAAPP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ChecMatE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ODeGP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PSF toolkit has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OHO has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:scSNPdemux has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GNBR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TIHM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:JNMR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:mStrain has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Strokecopilot has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MRMCsamplesize has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GSFA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:richPathR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:OpenToolFlux has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MetFinder has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:naplib-python has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UIBVFEDPlus-Light has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GENIE Biopharma has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CLEAN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CellPolaris has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:sup177 sup Lu-DOTATATE sup90 sup has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SARS-CoV-2 has the errors: {\"function\":[{\"operation\":[{\"general_errors\":[\"Invalid URI: http://edamontology.org/operation_3923.\"]},{},{},{},{}]}]}\n",
      "Tool with name:WCRC-25 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BSNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LCLUC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PAVE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PICO has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Autometa has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:sptotal has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Deep-cloud has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MEMINV has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DevCCF has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Mycovista has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:LibFewShot has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AGIN-ICP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DeepFruit has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:audit_cc has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PepCNN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:responsefun has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:knee OA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ADSP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Xtt CO236 Xtu CO237 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:gcCov has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Cell2Sentence has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SysMod has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:NABind has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:4D-diaXLMS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ZEPPI has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PlasPi TDM has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:candy darter has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BPL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CASPER has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AFsample has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CoMemMoRFPred has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BAllCools has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:YOLO-G has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BAWGNet has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MocFormer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CSL-Tox has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:bamSliceR has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:Scipion-EM-ProDy has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PepAnalyzer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GTSNET has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:iLEC-DNA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:AliNA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CRISPRe has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HiREX has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:intNMF has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DISCOVER-EEG has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:jCAN has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:C3VD has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ScreenDOP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CIR4MICS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:VenomPred 2.0 has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FairMobi-Net has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ZeptoCTC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ARIAS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TraumaICDBERT has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SubMDTA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ReproduceMe has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:InARes has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:QPPLab has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:IRv2-Net has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:TransPortal-TICBase has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SIDERITE has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:BAllC has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CBEdeepoff has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:B3LYP has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CIcADA has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:IoTSim has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MammalMethylClock has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:ADscreen has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:CellProfiler-plugins has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:U-Flow has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:PICOTEES has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:MitraCox has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:UFS has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:EpiGe has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:DeepLOF has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GROMACS-CIF has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GMTransformer has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:HemoCell has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:SLDL has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:GlioCova has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n",
      "Tool with name:FragAssembler has the errors: {\"biotoolsID\":[\"A resource with this ID already exists. bio.tools IDs need to be unique\"]}\n"
     ]
    }
   ],
   "source": [
    "from tool_validation import validate_tools\n",
    "\n",
    "valid_tools, problem_tools = validate_tools(processed_tools, token)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from utils.json_utils import load_tools_from_json\n",
    "\n",
    "valid_tools = load_tools_from_json('process_scrap_2023_10/low_tools_prp_2023_10.json')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'name': 'BayesKAT',\n",
       "  'description': 'Bayesian Optimal Kernel-based Test for genetic association studies reveals joint genetic effects in complex diseases.\\n\\nABSTRACT GWAS methods have identified individual SNPs significantly associated with specific phenotypes. Nonetheless, many complex diseases are polygenic and are controlled by multiple genetic variants that are usually non-linearly dependent. These genetic variants are marginally less effective and remain undetected in GWAS analysis. Kernel-based tests (KBT), which evaluate the joint effect of a group of genetic variants, are therefore critical for complex disease analysis. However, choosing different kernel functions in KBT can significantly influence the type I error control and power, and selecting the optimal kernel remains a statistically challenging task. A few existing methods suffer from inflated type 1 errors, limited scalability, inferior power, or issues of ambiguous conclusions',\n",
       "  'homepage': 'https://github.com/wangjr03/BayesKAT',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3920', 'term': 'DNA testing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3791',\n",
       "      'term': 'Collapsing methods'},\n",
       "     {'uri': 'http://edamontology.org/operation_3928',\n",
       "      'term': 'Pathway analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3517',\n",
       "    'term': 'GWAS study'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'language': ['R'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.18.562824',\n",
       "    'pmid': '37905124',\n",
       "    'pmcid': 'PMC10614916'}],\n",
       "  'credit': [{'name': 'Yuehua Cui',\n",
       "    'email': 'cuiy@msu.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Jianrong Wang',\n",
       "    'email': 'wangj164@msu.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__bayeskat',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__bayeskat',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.18.562824',\n",
       "  'is_preprint': True},\n",
       " {'name': 'VIBES',\n",
       "  'description': 'A Workflow for Annotating and Visualizing Viral Sequences Integrated into Bacterial Genomes.\\n\\nABSTRACT Bacteriophages are viruses that infect bacteria. Many bacteriophages integrate their genomes into the bacterial chromosome and become prophages. Prophages may substantially burden or benefit host bacteria fitness, acting in some cases as parasites and in others as mutualists, and have been demonstrated to increase host virulence. The increasing ease of bacterial genome sequencing provides an opportunity to deeply explore prophage prevalence and insertion sites. Here we present VIBES, a workflow intended to automate prophage annotation in complete bacterial genome sequences. VIBES provides additional context to prophage annotations by annotating bacterial genes and viral proteins in user-provided bacterial and viral genomes',\n",
       "  'homepage': 'https://github.com/TravisWheelerLab/VIBES',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0362',\n",
       "      'term': 'Genome annotation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0622', 'term': 'Genomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_3673',\n",
       "    'term': 'Whole genome sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_3316', 'term': 'Computer science'},\n",
       "   {'uri': 'http://edamontology.org/topic_0160',\n",
       "    'term': 'Sequence sites, features and motifs'}],\n",
       "  'language': ['Python', 'Perl'],\n",
       "  'license': 'BSD-3-Clause',\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562434',\n",
       "    'pmid': '37905003',\n",
       "    'pmcid': 'PMC10614876'}],\n",
       "  'credit': [{'name': 'Travis J. Wheeler',\n",
       "    'email': 'twheeler@arizona.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__vibes',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__vibes',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562434',\n",
       "  'is_preprint': True},\n",
       " {'name': 'admix-kit',\n",
       "  'description': 'An Integrated Toolkit and Pipeline for Genetic Analyses of Admixed Populations.\\n\\nSummary Admixed populations, with their unique and diverse genetic backgrounds, are often underrepresented in genetic studies. This oversight not only limits our understanding but also exacerbates existing health disparities. One major barrier has been the lack of efficient tools tailored for the special challenges of genetic study of admixed populations. Here, we present admix-kit, an integrated toolkit and pipeline for genetic analyses of admixed populations. Admix-kit implements a suite of methods to facilitate genotype and phenotype simulation, association testing, genetic architecture inference, and polygenic scoring in admixed populations. Availability and implementation. Admix-kit package is open-source and available at https: github.com KangchengHou admix-kit. Additionally, users can use the pipeline designed for admixed genotype simulation available at https: github.com UW-GAC admix-kit_workflow',\n",
       "  'homepage': 'https://github.com/KangchengHou/admix-kit',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3920', 'term': 'DNA testing'},\n",
       "     {'uri': 'http://edamontology.org/operation_0244',\n",
       "      'term': 'Simulation analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_3056',\n",
       "    'term': 'Population genetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'https://github.com/UW-GAC/admix-kit_workflow',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.09.30.560263',\n",
       "    'pmid': '37873338',\n",
       "    'pmcid': 'PMC10592849'}],\n",
       "  'credit': [{'name': 'Kangcheng Hou',\n",
       "    'email': 'houkc@ucla.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Bogdan Pasaniuc',\n",
       "    'email': 'pasaniuc@ucla.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Haoyu Zhang',\n",
       "    'email': 'haoyu.zhang2@nih.gov',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__admix-kit',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__admix-kit',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.30.560263',\n",
       "  'is_preprint': True},\n",
       " {'name': 'diplo-locus',\n",
       "  'description': 'A lightweight toolkit for inference and simulation of time-series genetic data under general diploid selection.\\n\\nLight-weight toolkit for the inference and simulation of Wright-Fisher diploid selection on independent loci from time-series data.\\n\\nThis repository hosts Python CLI tool (DiploLocus) and python package (diplo_locus) for simulating and computing log-likelihoods for time series genetic data based on the diploid Wright-Fisher diffusion.',\n",
       "  'homepage': 'https://github.com/steinrue/diplo_locus',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3227',\n",
       "      'term': 'Variant calling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0199',\n",
       "    'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_3056',\n",
       "    'term': 'Population genetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3068',\n",
       "    'term': 'Literature and language'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'link': [{'url': 'https://pypi.org/project/diplo-locus/',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.12.562101',\n",
       "    'pmid': '37905072',\n",
       "    'pmcid': 'PMC10614779'}],\n",
       "  'credit': [{'name': 'Matthias Steinrücken',\n",
       "    'email': 'steinrue@uchicago.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__diplo-locus',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__diplo-locus',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.12.562101',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TARO',\n",
       "  'description': 'tree-aggregated factor regression for microbiome data integration.\\n\\nMotivation Although the human microbiome plays a key role in health and disease, the biological mechanisms underlying the interaction between the microbiome and its host are incompletely understood. Integration with other molecular profiling data offers an opportunity to characterize the role of the microbiome and elucidate therapeutic targets. However, this remains challenging to the high dimensionality, compositionality, and rare features found in microbiome profiling data. These challenges necessitate the use of methods that can achieve structured sparsity in learning cross-platform association patterns. Results. We propose Tree-Aggregated factor RegressiOn (TARO) for the integration of microbiome and metabolomic data. We leverage information on the phylogenetic tree structure to flexibly aggregate rare features\\n\\n||| NAME (TARO) SIMILAR TO (PUB. DIFFERENT) bio.tools/taro (TarO)',\n",
       "  'homepage': 'https://github.com/amishra-stats/taro-package',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3501',\n",
       "      'term': 'Enrichment analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3697',\n",
       "    'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3293', 'term': 'Phylogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}],\n",
       "  'language': ['R', 'C++'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562792',\n",
       "    'pmid': '37904958',\n",
       "    'pmcid': 'PMC10614880'}],\n",
       "  'credit': [{'name': 'Aditya K. Mishra',\n",
       "    'email': 'amishra@flatironinstitute.org',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__taro',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__taro',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562792',\n",
       "  'is_preprint': True},\n",
       " {'name': 'cellPLATO',\n",
       "  'description': 'an unsupervised method for identifying cell behaviour in heterogeneous cell trajectory data.\\n\\nAdvances in imaging, cell segmentation, and cell tracking now routinely produce microscopy datasets of a size and complexity comparable to transcriptomics or proteomics. New tools are required to process this ‘phenomics’ type data. Cell PLasticity Analysis TOol (cellPLATO) is a Python-based analysis software designed for measurement and classification of diverse cell behaviours based on clustering of parameters of cell morphology and motility. cellPLATO is used after segmentation and tracking of cells from live cell microscopy data. The tool extracts morphological and motility metrics from each cell per timepoint, before being using them to segregate cells into behavioural subtypes with dimensionality reduction. Resultant cell tracks have a ‘behavioural ID’ for each cell per timepoint corresponding to their changing behaviour over time in a sequence',\n",
       "  'homepage': 'http://github.com/Michael-shannon/cellPLATO',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3935',\n",
       "      'term': 'Dimensionality reduction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3446',\n",
       "      'term': 'Cell migration analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_0244',\n",
       "      'term': 'Simulation analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3298', 'term': 'Phenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.28.564355',\n",
       "    'pmid': '37961659',\n",
       "    'pmcid': 'PMC10634992'}],\n",
       "  'credit': [{'name': 'Emily M. Mace',\n",
       "    'email': 'em3375@cumc.columbia.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cellplato',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cellplato',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.28.564355',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PhysiCell Studio',\n",
       "  'description': 'a graphical tool to make agent-based modeling more accessible.\\n\\nPhysiCell Studio tool to build multicellular models.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE PhysiCell (bio.tools/PhysiCell)',\n",
       "  'homepage': 'http://nanohub.org/tools/pcstudio',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3096',\n",
       "      'term': 'Editing'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2426',\n",
       "      'term': 'Modelling and simulation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'https://github.com/PhysiCell-Tools/PhysiCell-Studio',\n",
       "    'type': ['Repository']},\n",
       "   {'url': 'http://github.com/PhysiCell-Tools/PhysiCell-Studio/issues',\n",
       "    'type': ['Issue tracker']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.24.563727',\n",
       "    'pmid': '37961612',\n",
       "    'pmcid': 'PMC10634793'}],\n",
       "  'credit': [{'name': 'Paul Macklin',\n",
       "    'email': 'macklinp@iu.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__physicell-studio',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__physicell-studio',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.24.563727',\n",
       "  'is_preprint': True},\n",
       " {'name': 'APOLLO',\n",
       "  'description': 'A genome-scale metabolic reconstruction resource of 247,092 diverse human microbes spanning multiple continents, age groups, and body sites.\\n\\nSummary Computational modelling of microbiome metabolism has proved instrumental to catalyse our understanding of diet-host-microbiome-disease interactions through the interrogation of mechanistic, strain- and molecule-resolved metabolic models. We present APOLLO, a resource of 247,092 human microbial genome-scale metabolic reconstructions spanning 19 phyla and accounting for microbial genomes from 34 countries, all age groups, and five body sites. We explored the metabolic potential of the reconstructed strains and developed a machine learning classifier able to predict with high accuracy the taxonomic strain assignments\\n\\n||| HOMEPAGE MISSING!\\n\\n||| NAME (APOLLO) SIMILAR TO (PUB. DIFFERENT) bio.tools/apollo-seq (Apollo), bio.tools/apollo (Apollo Genome Annotation and Curation Tool)\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE vmh',\n",
       "  'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=37873072',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3660',\n",
       "      'term': 'Metabolic network modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3929',\n",
       "      'term': 'Metabolic pathway prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3460',\n",
       "      'term': 'Taxonomic classification'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3697',\n",
       "    'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3407',\n",
       "    'term': 'Endocrinology and metabolism'},\n",
       "   {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.02.560573',\n",
       "    'pmid': '37873072',\n",
       "    'pmcid': 'PMC10592896'}],\n",
       "  'credit': [{'name': 'Ines Thiele',\n",
       "    'email': 'ines.thiele@universityofgalway.ie',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__apollo',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__apollo',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.02.560573',\n",
       "  'is_preprint': True},\n",
       " {'name': 'gaiaAssociation',\n",
       "  'description': 'Regulatory Landscape Enrichment Analysis (RLEA) using gaiaAssociation.\\n\\nCompare ATAC-seq datasets against loci datasets to see if there is a specific enrichment between a particular loci set in a cell type.',\n",
       "  'homepage': 'https://github.com/GreallyLab/gaiaAssociation',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3501',\n",
       "      'term': 'Enrichment analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3517',\n",
       "    'term': 'GWAS study'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'documentation': [{'url': 'https://pypi.org/project/gaiaAssociation/0.6.0/#description',\n",
       "    'type': ['General']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.11.561933',\n",
       "    'pmid': '37905111',\n",
       "    'pmcid': 'PMC10614734'}],\n",
       "  'credit': [{'name': 'John M. Greally',\n",
       "    'email': 'john.greally@einsteinmed.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__gaiaassociation',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gaiaassociation',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.11.561933',\n",
       "  'is_preprint': True},\n",
       " {'name': 'skDER',\n",
       "  'description': 'microbial genome dereplication approaches for comparative and metagenomic applications.\\n\\nskDER ( https: github.com raufs skDER ) combines recent advances to efficiently estimate average nucleotide identity (ANI) between thousands of microbial genomes by skani 1 with two low-memory methods for genomic dereplication. The first method implements a dynamic algorithm to determine a concise set of representative genomes. This approach is well-suited for selecting reference genomes to align metagenomic reads onto for tracking strain presence across related microbiome samples. This is because fewer representative genomes should alleviate the concern that reads belonging to the same strain get falsely partitioned across closely related genomes',\n",
       "  'homepage': 'https://github.com/raufs/skDER',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3182',\n",
       "      'term': 'Genome alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_3198', 'term': 'Read mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3174',\n",
       "    'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0797',\n",
       "    'term': 'Comparative genomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'language': ['Python', 'C++'],\n",
       "  'license': 'BSD-3-Clause',\n",
       "  'publication': [{'doi': '10.1101/2023.09.27.559801',\n",
       "    'pmid': '38045253',\n",
       "    'pmcid': 'PMC10690176'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__skder',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__skder',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.27.559801',\n",
       "  'is_preprint': True},\n",
       " {'name': 'FluxNorm',\n",
       "  'description': 'Toolbox for metabolic flux assay normalization by<i>in situ</i>cell counting.\\n\\nABSTRACT Plate-based quantitative metabolic flux analysis has emerged as the central technology to examine cellular metabolism and mitochondrial bioenergetics. However, accurate interpretation of metabolic activity between different experimental conditions in multi-well microplates requires data normalization based on in situ cell counts. Here, we describe FluxNorm, a platform-independent semi-automated computational workflow, validated for three different cell types, to normalize cell density for accurate assessment of cellular bioenergetics',\n",
       "  'homepage': 'https://github.com/pekkurnazlab/FluxNormalyzer',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2428',\n",
       "      'term': 'Validation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3407',\n",
       "    'term': 'Endocrinology and metabolism'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3955', 'term': 'Fluxomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3340',\n",
       "    'term': 'Cell culture collection'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562314',\n",
       "    'pmid': '37873183',\n",
       "    'pmcid': 'PMC10592829'}],\n",
       "  'credit': [{'name': 'Gulcin Pekkurnaz',\n",
       "    'email': 'gpekkurnaz@ucsd.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__fluxnorm',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fluxnorm',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562314',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TRILL',\n",
       "  'description': 'Orchestrating Modular Deep-Learning Workflows for Democratized, Scalable Protein Analysis and Engineering.\\n\\nA bstract. Deep-learning models have been rapidly adopted by many fields, partly due to the deluge of data humanity has amassed. In particular, the petabases of biological sequencing data enable the unsupervised training of protein language models that learn the “language of life.” However, due to their prohibitive size and complexity, contemporary deep-learning models are often unwieldy, especially for scientists with limited machine learning backgrounds. TRILL ( TR aining and I nference using the L anguage of L ife) is a platform for creative protein design and discovery. Leveraging several state-of-the-art models such as ESM-2, DiffDock, and RFDiffusion, TRILL allows researchers to generate novel proteins, predict 3-D structures, extract high-dimensional representations of proteins, functionally classify proteins and more',\n",
       "  'homepage': 'https://trill.readthedocs.io/en/latest/home.html',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0477',\n",
       "      'term': 'Protein modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3891',\n",
       "      'term': 'Essential dynamics'},\n",
       "     {'uri': 'http://edamontology.org/operation_4008',\n",
       "      'term': 'Protein design'},\n",
       "     {'uri': 'http://edamontology.org/operation_0303',\n",
       "      'term': 'Fold recognition'},\n",
       "     {'uri': 'http://edamontology.org/operation_2476',\n",
       "      'term': 'Molecular dynamics'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_0218',\n",
       "    'term': 'Natural language processing'},\n",
       "   {'uri': 'http://edamontology.org/topic_3398', 'term': 'Bioengineering'},\n",
       "   {'uri': 'http://edamontology.org/topic_0623',\n",
       "    'term': 'Gene and protein families'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.24.563881',\n",
       "    'pmid': '37986952',\n",
       "    'pmcid': 'PMC10659302'}],\n",
       "  'credit': [{'name': 'Zachary A Martinez',\n",
       "    'email': 'martinez.zacharya@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__trill',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__trill',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.24.563881',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PFP-FM',\n",
       "  'description': 'FM-indexes are a crucial data structure in DNA alignment, but searching with them usually takes at least one random access per character in the query pattern. Ferragina and Fischer observed in 2007 that word-based indexes often use fewer random accesses than character-based indexes, and thus support faster searches. Since DNA lacks natural word boundaries, however, it is necessary to parse it somehow before applying word-based FM-indexing. Last year, Deng et al. proposed parsing genomic data by induced suffix sorting and showed the resulting word-based FM-indexes support faster counting queries than standard FM-indexes when patterns are a few thousand characters or longer. In this paper we show that using prefix-free parsing which takes parameters that let us tune the average length of the phrases instead of induced suffix sorting, gives a significant speedup for patterns of only a few hundred characters\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Accelerated FM-index',\n",
       "  'homepage': 'https://github.com/marco-oliva/pfp',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_1812',\n",
       "      'term': 'Parsing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3211',\n",
       "      'term': 'Genome indexing'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}],\n",
       "  'language': ['C++', 'Python', 'Shell'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3487536/V1',\n",
       "    'pmid': '37961504',\n",
       "    'pmcid': 'PMC10635359'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pfp-fm',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pfp-fm',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3487536/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PhyGCN',\n",
       "  'description': 'Pre-trained Hypergraph Convolutional Neural Networks with Self-supervised Learning.\\n\\nHypergraphs are powerful tools for modeling complex interactions across various domains, including biomedicine. However, learning meaningful node representations from hypergraphs remains a challenge. Existing supervised methods often lack generalizability, thereby limiting their real-world applications. We propose a new method, Pre-trained Hypergraph Convolutional Neural Networks with Self-supervised Learning (PhyGCN), which leverages hypergraph structure for self-supervision to enhance node representations. PhyGCN introduces a unique training strategy that integrates variable hyperedge sizes with self-supervised learning, enabling improved generalization to unseen data. Applications on multi-way chromatin interactions and polypharmacy side-effects demonstrate the effectiveness of PhyGCN\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://www.ncbi.nlm.nih.gov/pubmed/?term=37873233',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.01.560404',\n",
       "    'pmid': '37873233',\n",
       "    'pmcid': 'PMC10592843'}],\n",
       "  'credit': [{'name': 'Jian Ma',\n",
       "    'email': 'jianma@cs.cmu.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Quanquan Gu',\n",
       "    'email': 'qgu@cs.ucla.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__phygcn',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__phygcn',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.01.560404',\n",
       "  'is_preprint': True},\n",
       " {'name': 'SOCfinder',\n",
       "  'description': 'a genomic tool for identifying cooperative genes in bacteria.\\n\\nBacteria cooperate by working collaboratively to defend their colonies, share nutrients, and resist antibiotics. Nevertheless, our understanding of these remarkable behaviours primarily comes from studying a few well-characterized species. Consequently, there is a significant gap in our understanding of microbial cooperation, particularly in natural environments. To address this gap, we can use bioinformatic tools to identify cooperative traits and their underlying genes across diverse species. Existing tools address this challenge through two approaches. One approach is to identify genes that encode extracellular proteins, which can provide benefits to neighbouring cells. An alternative approach is to predict gene function using annotation tools. However, these tools have several limitations. Not all extracellular proteins are cooperative, and not all cooperative behaviours are controlled by extracellular proteins',\n",
       "  'homepage': 'https://github.com/lauriebelch/SOCfinder',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3672',\n",
       "      'term': 'Gene functional annotation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2454',\n",
       "      'term': 'Gene prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2489',\n",
       "      'term': 'Subcellular localisation prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3172',\n",
       "    'term': 'Metabolomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3053', 'term': 'Genetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3390',\n",
       "    'term': 'Nutritional science'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}],\n",
       "  'language': ['Python', 'Shell'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.16.562460'}],\n",
       "  'credit': [{'name': 'Laurence J. Belcher',\n",
       "    'email': 'laurence.belcher@biology.ox.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__socfinder',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__socfinder',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.16.562460',\n",
       "  'is_preprint': True},\n",
       " {'name': 'protoSpaceJAM',\n",
       "  'description': 'AN OPEN-SOURCE, CUSTOMIZABLE AND WEB-ACCESSIBLE DESIGN PLATFORM FOR CRISPR/CAS9 INSERTIONAL KNOCK-IN.\\n\\nWelcome to protoSpaceJAM’s documentation! — protoSpaceJAM 0.2 documentation.\\n\\nThe behavior and algorithm of protoSpaceJAM can be customized by:.\\n\\nBuild your job and click \"Add to the submission list\", or skip and go to step 2. Reset.\\n\\nENST IDs, terminus/chr:position *.',\n",
       "  'homepage': 'http://protospacejam.czbiohub.org',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0308',\n",
       "      'term': 'PCR primer design'},\n",
       "     {'uri': 'http://edamontology.org/operation_3192',\n",
       "      'term': 'Sequence trimming'},\n",
       "     {'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0632',\n",
       "    'term': 'Probes and primers'},\n",
       "   {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0659',\n",
       "    'term': 'Functional, regulatory and non-coding RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'}],\n",
       "  'language': ['Python', 'C', 'C++', 'Java'],\n",
       "  'license': 'BSD-3-Clause',\n",
       "  'link': [{'url': 'http://github.com/czbiohub-sf/protoSpaceJAM',\n",
       "    'type': ['Repository']},\n",
       "   {'url': 'http://github.com/czbiohub-sf/protospacejam',\n",
       "    'type': ['Repository']},\n",
       "   {'url': 'http://czbiohub-sf.github.io/protoSpaceJAM/index.html',\n",
       "    'type': ['Other']}],\n",
       "  'documentation': [{'url': 'http://github.com/czbiohub-sf/protoSpaceJAM/wiki',\n",
       "    'type': ['User manual']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.04.560793'}],\n",
       "  'credit': [{'name': 'Duo Peng',\n",
       "    'email': 'manuel.leonetti@czbiohub.org',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Manuel D. Leonetti',\n",
       "    'email': 'duo.peng@czbiohub.org',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__protospacejam',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__protospacejam',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.04.560793',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Multioviz',\n",
       "  'description': \"an interactive platform for<i>in silico</i>perturbation and interrogation of gene regulatory networks.\\n\\nCustomize map type for within a molecular level (ML) and click RUN.\\n\\nIf 'sparse' connections chosen above, upload mapping file:.\",\n",
       "  'homepage': 'http://multioviz.ccv.brown.edu',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2429',\n",
       "      'term': 'Mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_2238',\n",
       "      'term': 'Statistical calculation'},\n",
       "     {'uri': 'http://edamontology.org/operation_1781',\n",
       "      'term': 'Gene regulatory network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'},\n",
       "   {'uri': 'http://edamontology.org/topic_3053', 'term': 'Genetics'}],\n",
       "  'language': ['R'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://github.com/lcrawlab/multio-viz',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.10.561790'}],\n",
       "  'credit': [{'name': 'Lorin Crawford',\n",
       "    'email': 'lcrawford@microsoft.com',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Ashley Mae Conard',\n",
       "    'email': 'ashleyconard@microsoft.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__multioviz',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__multioviz',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.10.561790',\n",
       "  'is_preprint': True},\n",
       " {'name': 'OMEinfo',\n",
       "  'description': 'Global Geographic Metadata for -omics Experiments.\\n\\nMicrobiome studies increasingly associate geographical features like rurality and climate types with microbiomes. However, microbiologists bioinformaticians often struggle to access and integrate rich geographical metadata from sources such as GeoTIFFs; and inconsistent definitions of rurality, for example, can hinder cross-study comparisons. To address this, we present OMEinfo, a Python-based tool for automated retrieval of consistent geographical metadata from user-provided location data. OMEinfo leverages open data sources such as the Global Human Settlement Layer, Köppen-Geiger climate classification models, and Open-Data Inventory for Anthropogenic Carbon dioxide, to ensure metadata accuracy and provenance',\n",
       "  'homepage': 'https://github.com/m-crown/OMEinfo/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3697',\n",
       "    'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3277',\n",
       "    'term': 'Sample collections'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.23.563576'}],\n",
       "  'credit': [{'email': 'matthew.bashton@northumbria.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__omeinfo',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__omeinfo',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.23.563576',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MDMD',\n",
       "  'description': 'a computational model for predicting drug-related microbes based on the aggregated metapaths from a heterogeneous network.\\n\\nThe prediction of drug-related microbes.\\n\\nPlease input the chemical structure of the drug, for example:.\\n\\nEg1: (1R,2S,7R,8R)-2,18-diacetoxyasperdiol.',\n",
       "  'homepage': 'https://mdmd2023.bit1024.top/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436',\n",
       "      'term': 'Aggregation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0154',\n",
       "    'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_3379',\n",
       "    'term': 'Preclinical and clinical studies'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3400',\n",
       "    'term': 'Allergy, clinical immunology and immunotherapeutics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562158'}],\n",
       "  'credit': [{'name': 'Juan Wang',\n",
       "    'email': 'wangjuan@nclab.net',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__mdmd',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mdmd',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562158',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AP-LASR',\n",
       "  'description': 'Automated Protein Libraries from Ancestral Sequence Reconstruction.\\n\\nBackground Ancestral sequence reconstruction (ASR) provides an informative roadmap of evolutionary protein sequence space that benefits protein design and engineering in pursuit of high stability and diverse functionality. Using statistical and biological knowledge, ASR can determine the most probable ancestor among potential alternative amino acid states. However, the inherent uncertainty of ASR can be further leveraged to determine viable “nearby” ancestors with wide-ranging functionalities by sampling alternative amino acid states. Results. Here we introduce AP-LASR which i) automates ASR and ii) leverages uncertainty in ASR to generate diverse protein sequence libraries that consist of ancestral sequences and near-ancestor sequences',\n",
       "  'homepage': 'https://github.com/WoldringLabMSU/AP-LASR',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3745',\n",
       "      'term': 'Ancestral reconstruction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3478',\n",
       "      'term': 'Phylogenetic reconstruction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0492',\n",
       "      'term': 'Multiple sequence alignment'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0154',\n",
       "    'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_0623',\n",
       "    'term': 'Gene and protein families'},\n",
       "   {'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'},\n",
       "   {'uri': 'http://edamontology.org/topic_3518',\n",
       "    'term': 'Microarray experiment'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.09.561537'}],\n",
       "  'credit': [{'name': 'Daniel Woldring',\n",
       "    'email': 'woldring@egr.msu.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__ap-lasr',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ap-lasr',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.09.561537',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Pindel-TD',\n",
       "  'description': 'a tandem duplication detector based on a pattern growth approach.\\n\\nTandem duplication (TD) is a major type of structural variation (SV), and plays an important role in novel gene formation and human diseases. However, TDs are often missed or incorrectly classified as insertions by most of modern SV detection methods due to the lacking of specialized operation on TD related mutational signals. Herein, we developed a TD detection module of Pindel referred as Pindel-TD based on a TD specific pattern growth approach. Pindel-TD detects TDs with a wide size range at single nucleotide resolution. Using simulation and real read data of HG002, we demonstrate that Pindel-TD outperformed other leading methods in terms of precision, recall, F1-score and robustness. Further applying Pindel-TD on data generated from K562 cancer cell line, we identified a TD located at the seventh exon of SAGE1 , explaining its high expression',\n",
       "  'homepage': 'https://github.com/xjtu-omics/pindel',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3227',\n",
       "      'term': 'Variant calling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3196', 'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3228',\n",
       "      'term': 'Structural variation detection'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3512',\n",
       "    'term': 'Gene transcripts'},\n",
       "   {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.08.561441'}],\n",
       "  'credit': [{'name': 'Kai Ye',\n",
       "    'email': 'kaiye@xjtu.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pindel-td',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pindel-td',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.08.561441',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Segzoo',\n",
       "  'description': 'a turnkey system that summarizes genome annotations.\\n\\nSystem for turnkey analysis of semi-automated genome annotations.\\n\\nHome: https://github.com/hoffmangroup/segzoo.\\n\\nLicense: GNU General Public v2 (GPLv2).',\n",
       "  'homepage': 'https://anaconda.org/bioconda/segzoo',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0362',\n",
       "      'term': 'Genome annotation'},\n",
       "     {'uri': 'http://edamontology.org/operation_1812', 'term': 'Parsing'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0622', 'term': 'Genomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-2.0',\n",
       "  'link': [{'url': 'https://github.com/hoffmangroup/segzoo',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.559369'}],\n",
       "  'credit': [{'name': 'Michael M. Hoffman',\n",
       "    'email': 'michael.hoffman@utoronto.ca',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__segzoo',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__segzoo',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.559369',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PyDecNef',\n",
       "  'description': 'An open-source framework for fMRI-based decoded neurofeedback.\\n\\nThe pyDecNef library provides a complete framework to perform real-time fMRI decoded neurofeedback studies in Python.',\n",
       "  'homepage': 'https://pedromargolles.github.io/pyDecNef/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3384',\n",
       "    'term': 'Medical imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://github.com/pedromargolles/pyDecNef',\n",
       "    'type': ['Repository']}],\n",
       "  'download': [{'url': 'https://github.com/pedromargolles/pyDecNef/tree/main/2.decnef_training/client',\n",
       "    'type': 'Source code'},\n",
       "   {'url': 'https://github.com/pedromargolles/pyDecNef/tree/main/1.model_construction',\n",
       "    'type': 'Source code'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.02.560503'}],\n",
       "  'credit': [{'name': 'Pedro Margolles',\n",
       "    'email': 'pedromargolles@gmail.com',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'David Soto', 'email': 'd.soto@bcbl.eu', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pydecnef',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pydecnef',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.02.560503',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PMGmap',\n",
       "  'description': 'A Software Tool for Comprehensive Visualization of Coding, Non-coding and Genome Features of Plant Mitochondrial Genomes.\\n\\nGenome visualization tools are important for exploring genomic features and their interactions. Currently, visualization of the plant mitochondrial genomes (mitogenome) depends on those tools designed originally for animal mitogenomes and plant plasto.\\n\\nPlant Mitochondrial Genomes Map(PMGmap).\\n\\nthis is for holding javascript data.\\n\\nTo address these challenges, a software package plant mitochondrial genome map (PMGmap), was developed using Python programming language.\\n\\nA tool for plant mitochondrial genome visualization.',\n",
       "  'homepage': 'http://www.1kmpg.cn/pmgmap',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0573',\n",
       "      'term': 'Map drawing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3208',\n",
       "      'term': 'Genome visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0780',\n",
       "    'term': 'Plant biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'},\n",
       "   {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_3303', 'term': 'Medicine'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'https://authorea.com/users/675182/articles/673251plant-mitochondrial-genome-map-pmgmap-a-software-tool-for-comprehensive-visualizationof-coding-non-coding-and-genome-features-of-plant-mitochondrial-genomes',\n",
       "    'type': ['Other']}],\n",
       "  'publication': [{'doi': '10.22541/AU.169772240.03411454/V1'}],\n",
       "  'credit': [{'name': 'Chang Liu', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pmgmap',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pmgmap',\n",
       "  'publication_link': 'https://doi.org/10.22541/AU.169772240.03411454/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AnophelesModel',\n",
       "  'description': 'An R package to interface mosquito bionomics, human exposure and intervention effects with models of malaria intervention impact.\\n\\nAn R package to quantify the effect of vector bionomics on the impact of vector control interventions against malaria transmitted by Anopheles mosquitoes.\\n\\nAnophelesModel is an R package to quantify the impact of vector control interventions according to mosquito species characteristics.\\n\\nMonica Golumbeanu, Olivier Briët, Clara Champagne, Jeanne Lemant, Barnabas Zogo, Maximilian Gerhards, Marianne Sinka, Nakul Chitnis, Melissa Penny, Emilie Pothin, Tom Smith.\\n\\nThe AnophelesModel package can be used to parameterize a model of the mosquito feeding cycle using data about mosquito bionomics (entomological characteristics) and biting patterns, as well as human activity and intervention effects.',\n",
       "  'homepage': 'https://github.com/SwissTPH/AnophelesModel',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426',\n",
       "      'term': 'Modelling and simulation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0610', 'term': 'Ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3305',\n",
       "    'term': 'Public health and epidemiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'}],\n",
       "  'language': ['R'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://swisstph.github.io/AnophelesModel/index.html',\n",
       "    'type': ['Other']},\n",
       "   {'url': 'https://swisstph.github.io/AnophelesModel/articles/AnophelesModel.html',\n",
       "    'type': ['Other']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562838'}],\n",
       "  'credit': [{'name': 'Monica Golumbeanu',\n",
       "    'email': 'monica.golumbeanu@swisstph.ch',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__anophelesmodel',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__anophelesmodel',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562838',\n",
       "  'is_preprint': True},\n",
       " {'name': 'BTR',\n",
       "  'description': 'A Bioinformatics Tool Recommendation System.\\n\\nbioinformatics_tool_recommendation.',\n",
       "  'homepage': 'https://github.com/ryangreenj/bioinformatics_tool_recommendation',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_0218',\n",
       "    'term': 'Natural language processing'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562252'}],\n",
       "  'credit': [{'name': 'Jinze Liu',\n",
       "    'email': 'jinze.liu@vcuhealth.org',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Tingting Yu',\n",
       "    'email': 'tingting.yu@uconn.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__btr',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__btr',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562252',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NAMERS',\n",
       "  'description': 'a purpose-built reference DNA sequence database to support applied eDNA metabarcoding.\\n\\nNovel Applied eDNA Metabarcoding Reference Sequences.',\n",
       "  'homepage': 'https://namers.ca',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0308',\n",
       "      'term': 'PCR primer design'},\n",
       "     {'uri': 'http://edamontology.org/operation_3192',\n",
       "      'term': 'Sequence trimming'},\n",
       "     {'uri': 'http://edamontology.org/operation_3200',\n",
       "      'term': 'DNA barcoding'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0632',\n",
       "    'term': 'Probes and primers'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.06.561210'}],\n",
       "  'credit': [{'name': 'Kristen M. Westfall',\n",
       "    'email': 'westfall.kristenm@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__namers',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__namers',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.06.561210',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NeoHeadHunter',\n",
       "  'description': 'an algorithm for the detection, ranking and probabilistic classification of neoepitope candidates.\\n\\nBackground The manufacturing of personalized cancer vaccine requires the accurate identification of neoepitopes, abnormal peptides presented by cancer cells and recognized by the host immune system of the cancer patient. Results. We designed and developed NeoHeadHunter, a computational algorithm and pipeline to detect and rank neoepitope candidates. Unlike other algorithms, NeoHeadHunter can estimate the probability that each predicted neoepitope candidate is true positive',\n",
       "  'homepage': 'https://github.com/XuegongLab/neoheadhunter',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0252',\n",
       "      'term': 'Peptide immunogenicity prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_3930', 'term': 'Immunogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'Apache-2.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.25.563895'}],\n",
       "  'credit': [{'name': 'Xiaofei Zhao',\n",
       "    'email': 'zhaoxf22@mails.tsinghua.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__neoheadhunter',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__neoheadhunter',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.25.563895',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MSIsensor-RNA',\n",
       "  'description': 'Microsatellite Instability Detection for Bulk and Single-cell Gene Expression Data.\\n\\nMicrosatellite instability (MSI) is an indispensable biomarker in cancer immunotherapy. Currently, MSI scoring methods by high-throughput omics methods have gained popularity and demonstrated better performance than the gold standard method for MSI detection. However, the MSI detection method on expression data, especially single-cell expression data, is still lacking, limiting the scope of clinical application and prohibiting the investigation of MSI at a single-cell level. Herein, we developed MSIsensor-RNA, an accurate, robust, adaptable, and standalone software to detect MSI status based on expression values of MSI-associated genes. We demonstrated the favorable performance and promise of MSIsensor-RNA in both bulk and single-cell gene expression data in multiplatform technologies including RNA-seq, microarray, and single-cell RNA-seq',\n",
       "  'homepage': 'https://github.com/xjtu-omics/msisensor-rna',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2885',\n",
       "    'term': 'DNA polymorphism'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_3518',\n",
       "    'term': 'Microarray experiment'},\n",
       "   {'uri': 'http://edamontology.org/topic_3360', 'term': 'Biomarkers'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.09.561613'}],\n",
       "  'credit': [{'name': 'Kai Ye',\n",
       "    'email': 'kaiye@xjtu.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__msisensor-rna',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__msisensor-rna',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.09.561613',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Nphos',\n",
       "  'description': 'Database and Predictor of Protein<i>N</i>-phosphorylation.\\n\\nSimple template with a more complex navbar that includes a search form and dropdown.\\n\\nQuickSearch UniProt AC Gene Name Protein Name Search.\\n\\nNphos provides comprehensive information and predictive tool for the study of protein N-phosphorylation modification (including pHistidine, pArginine and pLysine).\\n\\ne.g. P53396 ACLY ATP-citrate synthase.\\n\\nBioinformatics-Aided Drug Discovery Group, Xiamen University.\\n\\n2021-06-04 热烈庆祝2017级博士研究生丁若凡答辩顺利结束.',\n",
       "  'homepage': 'http://ppodd.org.cn/Nphos/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0417',\n",
       "      'term': 'PTM site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'},\n",
       "     {'uri': 'http://edamontology.org/operation_2499',\n",
       "      'term': 'Splicing analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0601',\n",
       "    'term': 'Protein modifications'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3336', 'term': 'Drug discovery'},\n",
       "   {'uri': 'http://edamontology.org/topic_0080', 'term': 'Sequence analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}],\n",
       "  'link': [{'url': 'http://www.bio-add.org', 'type': ['Other']},\n",
       "   {'url': 'http://www.bio-add.org/Nphos', 'type': ['Other']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.559246'}],\n",
       "  'credit': [{'name': 'Zhi-Liang Ji',\n",
       "    'email': 'yfzhao@xmu.edu.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Yufen Zhao', 'email': 'appo@xmu.edu.cn', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__nphos',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nphos',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.559246',\n",
       "  'is_preprint': True},\n",
       " {'name': 'OncoReport',\n",
       "  'description': 'a system for integrative NGS analysis in precision medicine.\\n\\nOncoReport - an innovative open-source tool designed to generate comprehensive reports derived from Next-Generation Sequencing.\\n\\nOncoReport - interactive reports from NGS analysis.',\n",
       "  'homepage': 'https://oncoreport.atlas.dmi.unict.it',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3168',\n",
       "    'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_3577',\n",
       "    'term': 'Personalised medicine'},\n",
       "   {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3423',\n",
       "    'term': 'Complementary medicine'},\n",
       "   {'uri': 'http://edamontology.org/topic_3396', 'term': 'Systems medicine'}],\n",
       "  'language': ['PHP', 'R'],\n",
       "  'link': [{'url': 'https://github.com/knowmics-lab/oncoreport',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3456496/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__oncoreport',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__oncoreport',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3456496/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'sincFold',\n",
       "  'description': 'end-to-end learning of short- and long-range interactions for RNA folding.\\n\\nResearch Institute for Signals, Systems and Computational Intelligence.\\n\\nsincFold is an end-to-end deep learning approach for RNA folding.\\n\\nsincFold is an end-to-end deep learning approach that predicts the nucleotides contact matrix using only the RNA sequence as input. This demo provides a sincFold model pre-trained with known RNAs secondary structures.\\n\\nResearch at sinc(i) aims to develop new algorithms for machine learning, data mining, signal processing and complex systems, providing innovative technologies for advancing healthcare, bioinformatics, precision agriculture, autonomous systems and human-computer interfaces. Research facilities are located in the Facultad de Ingeniería y Ciencias Hídricas building, Universidad Nacional del Litoral , on the University Campus .',\n",
       "  'homepage': 'https://sinc.unl.edu.ar/web-demo/sincFold',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0278',\n",
       "      'term': 'RNA secondary structure prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0502',\n",
       "      'term': 'RNA secondary structure alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_2441',\n",
       "      'term': 'RNA structure prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0267',\n",
       "      'term': 'Protein secondary structure prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0659',\n",
       "    'term': 'Functional, regulatory and non-coding RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3810',\n",
       "    'term': 'Agricultural science'},\n",
       "   {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'},\n",
       "   {'uri': 'http://edamontology.org/topic_0082',\n",
       "    'term': 'Structure prediction'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://github.com/sinc-lab/sincFold/',\n",
       "    'type': ['Repository']},\n",
       "   {'url': 'http://sinc.unl.edu.ar', 'type': ['Other']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.10.561771'}],\n",
       "  'credit': [{'name': 'L.A. Bugnon',\n",
       "    'email': 'lbugnon@sinc.unl.edu.ar',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__sincfold',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__sincfold',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.10.561771',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TEC-miTarget',\n",
       "  'description': 'enhancing microRNA target prediction based on deep learning of ribonucleic acid sequences.\\n\\nBackground: MicroRNAs play a critical role in regulating gene expression by binding to specific target sites within gene transcripts, making the identification of microRNA targets a prominent focus of research. Conventional experimental methods for identifying microRNA targets are both time-consuming and expensive, prompting the development of computational tools for target prediction. However, the existing computational tools exhibit limited performance in meeting the demands of practical applications, highlighting the need to improve the performance of microRNA target prediction models. Results In this paper, we utilize the most popular natural language processing and computer vision technologies to propose a novel approach, called TEC-miTarget, for microRNA target prediction based on Transformer Encoder and convolutional neural networks',\n",
       "  'homepage': 'https://github.com/tingpeng17/TEC-miTarget',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0463',\n",
       "      'term': 'miRNA target prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0272',\n",
       "      'term': 'Residue contact prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0659',\n",
       "    'term': 'Functional, regulatory and non-coding RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_0218',\n",
       "    'term': 'Natural language processing'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3453552/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__tec-mitarget',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tec-mitarget',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3453552/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MTGCL',\n",
       "  'description': 'Multi-Task Graph Contrastive Learning for Identifying Cancer Driver Genes from Multi-omics Data.\\n\\nCancer is a complex disease that typically arises from the accumulation of mutations in driver genes. Identification of cancer driver genes is crucial for understanding the molecular mechanisms of cancer, and developing the targeted therapeutic approaches. With the development of high-throughput biological technology, a large amount of genomic data and protein interaction network data have been generated, which provides abundant data resources for identifying cancer driver genes through computational methods. Given the ability of graph neural networks to effectively integrate graph structure topology information and node features information, some graph neural network-based methods have been developed for identifying cancer driver genes',\n",
       "  'homepage': 'https://github.com/NWPU-903PR/MTGCL',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2454',\n",
       "      'term': 'Gene prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3501',\n",
       "      'term': 'Enrichment analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562159'}],\n",
       "  'credit': [{'name': 'Shao-Wu Zhang',\n",
       "    'email': 'zhangsw@nwpu.edu.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Xiaodong Cui',\n",
       "    'email': 'xiaodong.cui@nwpu.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__mtgcl',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mtgcl',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562159',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ABIapp',\n",
       "  'description': 'A user-friendly tool for precise parasitic helminth species delimitation.\\n\\nParasitic helminths are highly diverse and ubiquitously distributed in various environments and hosts. Their vast species diversity makes both morphological and molecular species delimitation challenging. We developed and validated a simple, easy-to-use application, Applying Taxonomic Boundaries for Species Identification (ABIapp), to aid in the delimitation of helminth species. ABIapp utilizes a database of cut-off genetic distances determined by the K-means algorithm to establish helminth taxonomic boundaries for the 18S and 28S rRNA genes, ITS1 and 2 regions, and the 12S and 16S rRNA, COI, COII, cytB, and ND1 genes. ABIapp is written in R, using the Shiny framework to deliver an interactive and user-friendly interface. To assess the accuracy of ABIapp, validation was undertaken both in silico and with actual specimens',\n",
       "  'homepage': 'https://github.com/slphyx/ABIApp',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3460',\n",
       "      'term': 'Taxonomic classification'},\n",
       "     {'uri': 'http://edamontology.org/operation_2428', 'term': 'Validation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0415',\n",
       "      'term': 'Nucleic acid feature detection'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3302',\n",
       "    'term': 'Parasitology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0659',\n",
       "    'term': 'Functional, regulatory and non-coding RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'},\n",
       "   {'uri': 'http://edamontology.org/topic_0637', 'term': 'Taxonomy'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'}],\n",
       "  'language': ['R'],\n",
       "  'link': [{'url': 'https://github.com/slphyx/ABI', 'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.22541/AU.169772261.15720476/V1'}],\n",
       "  'credit': [{'name': 'Sompob Saralamba', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__abiapp',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__abiapp',\n",
       "  'publication_link': 'https://doi.org/10.22541/AU.169772261.15720476/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'BATTER',\n",
       "  'description': 'Accurate Prediction of Rho-dependent and Rho-independent Transcription Terminators in Metagenomes.\\n\\nTranscription terminators mark the 3’ ends of both coding and noncoding transcripts in bacteria and play crucial roles in gene regulations (such as controlling the stoichiometry of gene expression and conditionally switching off gene expression by inducing premature termination). Recently developed experimental 3’ end mapping techniques greatly improved the current understanding of bacteria transcription termination, but these methods cannot detect transcripts that are unexpressed in the limited experimental conditions and cannot utilize the vast amount of information embedded in the rapidly growing metagenome data. Computational approaches can relieve these problems, but the development of such in-silico methods lags behind the experimental techniques',\n",
       "  'homepage': 'https://github.com/lulab/BATTER',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3482',\n",
       "      'term': 'Antimicrobial resistance prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0267',\n",
       "      'term': 'Protein secondary structure prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3280',\n",
       "      'term': 'Named-entity and concept recognition'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_3941',\n",
       "    'term': 'Metatranscriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0659',\n",
       "    'term': 'Functional, regulatory and non-coding RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.02.560326'}],\n",
       "  'credit': [{'name': 'Zhenjiang Zech Xu',\n",
       "    'email': 'zhilu@tsinghua.edu.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Zhi John Lu',\n",
       "    'email': 'zhenjiang.xu@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__batter',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__batter',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.02.560326',\n",
       "  'is_preprint': True},\n",
       " {'name': 'LAVASET',\n",
       "  'description': 'Latent Variable Stochastic Ensemble of Trees. A novel ensemble method for correlated datasets.\\n\\nMotivation Random Forests (RFs) can deal with a large number of variables, achieve reasonable prediction scores, and yield highly interpretable feature importance values. As such, RFs are appropriate models for feature selection and further dimension reduction (DR). However, RFs are often not appropriate for correlated datasets due to their mode of selecting individual features for splitting. Addressing correlation relationships in high dimensional datasets is imperative for reducing the number of variables that are assigned high importance, hence making the DR most efficient. Here, we propose the LAtent VAriable Stochastic Ensemble of Trees (LAVASET) method that derives latent variables based on the distance characteristics of each feature and aims to incorporate the correlation factor in the splitting step. Results',\n",
       "  'homepage': 'https://github.com/melkasapi/LAVASET',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3935',\n",
       "      'term': 'Dimensionality reduction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'}],\n",
       "  'language': ['Python', 'C++'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.20.563223'}],\n",
       "  'credit': [{'name': 'Melpomeni Kasapi',\n",
       "    'email': 'jmp111@ic.ac.uk',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Joram M. Posma',\n",
       "    'email': 'mk218@imperial.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__lavaset',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__lavaset',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.20.563223',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PREVIC',\n",
       "  'description': 'An adaptive parent report measure of expressive vocabulary in children between 3 and 8 years of age.\\n\\nPREVIC: Parent Report of Expressive Vocabulary in Children.\\n\\nVielen Dank für Ihr Interesse an unserem Sprach-Fragebogen für Eltern von Vorschulkindern.',\n",
       "  'homepage': 'https://ccp-odc.eva.mpg.de/previc-demo/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3436',\n",
       "      'term': 'Aggregation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3359', 'term': 'Splitting'},\n",
       "     {'uri': 'http://edamontology.org/operation_2428',\n",
       "      'term': 'Validation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0203',\n",
       "    'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_3068',\n",
       "    'term': 'Literature and language'}],\n",
       "  'language': ['JavaScript'],\n",
       "  'link': [{'url': 'https://github.com/ccp-eva/previc-demo',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.31234/OSF.IO/HVNCP'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__previc',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__previc',\n",
       "  'publication_link': 'https://doi.org/10.31234/OSF.IO/HVNCP',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PhysioFit',\n",
       "  'description': 'a software to quantify cell growth parameters and extracellular fluxes.\\n\\nWelcome to PhysioFit documentation! — PhysioFit 3.3.1 documentation.\\n\\nWelcome to the collaborative portal dedicated to metabolomics data processing,.\\n\\nanalysis and annotation for Metabolomics community.\\n\\nWorkflow4experimenters (W4E) course 2024 : Pre-Registrations Opened - Until 31 Jan.\\n\\nKeep updated with our newsletter : SUBSCRIBE.',\n",
       "  'homepage': 'https://workflow4metabolomics.org',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_2426',\n",
       "      'term': 'Modelling and simulation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3955', 'term': 'Fluxomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3172', 'term': 'Metabolomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3895', 'term': 'Synthetic biology'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://github.com/MetaSys-LISBP/PhysioFit/',\n",
       "    'type': ['Repository']}],\n",
       "  'documentation': [{'url': 'https://physiofit.readthedocs.io/',\n",
       "    'type': ['User manual']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.12.561695'}],\n",
       "  'credit': [{'name': 'Pierre Millard',\n",
       "    'email': 'pierre.millard@insa-toulouse.fr',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__physiofit',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__physiofit',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.12.561695',\n",
       "  'is_preprint': True},\n",
       " {'name': 'GenomeSpy',\n",
       "  'description': 'Deciphering Cancer Genomes with GenomeSpy.\\n\\nA visualization grammar and a GPU-accelerated rendering engine for genomic (and other) data.\\n\\nGenomeSpy is a grammar-based visualization tool for genomic and other data.\\n\\nUse GenomeSpy to make your own visualizations!.\\n\\nGenomeSpy builds upon the concepts originally introduced in The Grammar Of Graphics and later implemented in ggplot2 and Vega-Lite. The building blocks that GenomeSpy provides allow users to build tailored, interactive genomic visualizations, which can be embedded on web pages or within JavaScript applications. The carefully crafted GPU-accelerated rendering engine guarantees smoothly animated interactions and a pleasant user experience for end users. Scroll down for live examples.',\n",
       "  'homepage': 'https://genomespy.app/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_1812', 'term': 'Parsing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3695', 'term': 'Filtering'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3379',\n",
       "    'term': 'Preclinical and clinical studies'},\n",
       "   {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'}],\n",
       "  'language': ['JavaScript'],\n",
       "  'license': 'MIT',\n",
       "  'link': [{'url': 'https://csbi.ltdk.helsinki.fi/p/genomespy-preprint/',\n",
       "    'type': ['Other']},\n",
       "   {'url': 'https://observablehq.com/collection/@tuner/genomespy',\n",
       "    'type': ['Other']},\n",
       "   {'url': 'https://github.com/genome-spy/genome-spy',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.06.561159'}],\n",
       "  'credit': [{'name': 'Kari Lavikka',\n",
       "    'email': 'kari.lavikka@helsinki.fi',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Sampsa Hautaniemi',\n",
       "    'email': 'sampsa.hautaniemi@helsinki.fi',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__genomespy',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__genomespy',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.06.561159',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MetaQuad',\n",
       "  'description': 'Shared Informative Variants Discovery in Metagenomic Samples.\\n\\nMotivation Strain-level analysis of metagenomic data has garnered significant interest in recent years. Microbial single nucleotide polymorphisms (SNPs) are genomic variants that can reflect strain-level differences within a microbial species. The diversity and emergence of SNPs in microbial genomes may reveal evolutionary history and environmental adaptation in microbial populations. However, efficient discovery of shared polymorphic variants in a large collection metagenomic samples remains a computational challenge. Results. MetaQuad employs a density-based clustering technique to efficiently differentiate shared variants from non-polymorphic sites using shotgun metagenomic data. Empirical comparisons with other state-of-the-art methods show that MetaQuad significantly reduces the number of false-positive SNPs without greatly affecting the true-positive rate',\n",
       "  'homepage': 'https://github.com/holab-hku/MetaQuad',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0484',\n",
       "      'term': 'SNP detection'},\n",
       "     {'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'},\n",
       "     {'uri': 'http://edamontology.org/operation_3675',\n",
       "      'term': 'Variant filtering'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3174',\n",
       "    'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2885', 'term': 'DNA polymorphism'},\n",
       "   {'uri': 'http://edamontology.org/topic_3837',\n",
       "    'term': 'Metagenomic sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'language': ['Python', 'R'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.09.561628'}],\n",
       "  'credit': [{'name': 'Joshua W. K. Ho',\n",
       "    'email': 'jwkho@hku.hk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__metaquad',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__metaquad',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.09.561628',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TEPEAK',\n",
       "  'description': 'A novel method for identifying and characterizing polymorphic transposable elements in non-model species populations.\\n\\nTransposable elements (TEs) replicate within genomes and are an active source of genetic variability in many species. Their role in immunity and domestication underscores their biological significance. However, analyzing TEs, especially within lesser-studied and wild populations, poses considerable challenges. To address this, we introduce TEPEAK, a simple and efficient approach to identify and characterize TEs in populations without any prior sequence or loci information. In addition to processing user-submitted genomes, TEPEAK integrates with the National Center for Biotechnology Information (NCBI) Sequence Read Archive (SRA) to increase cohort sizes or incorporate proximate species. Our application of TEPEAK to 256 horse genomes spanning 11 groups reaffirmed established genetic histories and highlighted disruptions in crucial genes',\n",
       "  'homepage': 'https://github.com/mrburke00/TEPEAK',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3197',\n",
       "      'term': 'Genetic variation analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_0554',\n",
       "      'term': 'Allele frequency distribution analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_2436',\n",
       "      'term': 'Gene-set enrichment analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0199',\n",
       "    'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3297', 'term': 'Biotechnology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3056',\n",
       "    'term': 'Population genetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0798',\n",
       "    'term': 'Mobile genetic elements'},\n",
       "   {'uri': 'http://edamontology.org/topic_3293', 'term': 'Phylogenetics'}],\n",
       "  'language': ['Python', 'Shell'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562297'}],\n",
       "  'credit': [{'name': 'Ryan Layer',\n",
       "    'email': 'ryan.layer@colorado.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__tepeak',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tepeak',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562297',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Ocelli',\n",
       "  'description': 'an open-source tool for the visualization of developmental multimodal single-cell data.\\n\\nOcelli — Ocelli 0.1.1 documentation.',\n",
       "  'homepage': 'https://github.com/TabakaLab/ocelli',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3557',\n",
       "      'term': 'Imputation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0314',\n",
       "      'term': 'Gene expression profiling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3295',\n",
       "    'term': 'Epigenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3940',\n",
       "    'term': 'Chromosome conformation capture'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0108',\n",
       "    'term': 'Protein expression'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'BSD-2-Clause',\n",
       "  'documentation': [{'url': 'https://ocelli.readthedocs.io',\n",
       "    'type': ['User manual']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.05.561074'}],\n",
       "  'credit': [{'name': 'Marcin Tabaka',\n",
       "    'email': 'mtabaka@ichf.edu.pl',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__ocelli',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ocelli',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.05.561074',\n",
       "  'is_preprint': True},\n",
       " {'name': '3DVascNet',\n",
       "  'description': 'an automated software for segmentation and quantification of vascular networks in 3D.\\n\\nBackground Analysis of vascular networks is an essential step to unravel the mechanisms regulating the physiological and pathological organization of blood vessels. So far, most of the analyses are performed using 2D projections of 3D networks, a strategy that has several obvious shortcomings. For instance, it does not capture the true geometry of the vasculature, and generates artifacts on vessel connectivity. These limitations are accepted in the field because manual analysis of 3D vascular networks is a laborious and complex process that is often prohibitive for large volumes. Methods\\n\\n||| HOMEPAGE BROKEN!',\n",
       "  'homepage': 'https://github.com/HemaxiN/3DVascNet',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3443',\n",
       "      'term': 'Image analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.19.563201'}],\n",
       "  'credit': [{'email': 'cfranco@medicina.ulisboa.pt', 'typeEntity': 'Person'},\n",
       "   {'email': 'hemaxi.narotamo@tecnico.ulisboa.pt', 'typeEntity': 'Person'},\n",
       "   {'email': 'msilveira@isr.tecnico.ulisboa.pt', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__3dvascnet',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__3dvascnet',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.19.563201',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MSA_Limit',\n",
       "  'description': 'Automated evaluation of multiple sequence alignment methods to handle third generation sequencing errors.\\n\\nGitlab du laboratoire CRIStAL.',\n",
       "  'homepage': 'http://gitlab.cristal.univ-lille.fr/crohmer/msa-limit',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0492',\n",
       "      'term': 'Multiple sequence alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_0496',\n",
       "      'term': 'Global alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_0491',\n",
       "      'term': 'Pairwise sequence alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_3472',\n",
       "      'term': 'k-mer counting'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0080',\n",
       "    'term': 'Sequence analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_3361',\n",
       "    'term': 'Laboratory techniques'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.560796'}],\n",
       "  'credit': [{'name': 'Antoine Limasset',\n",
       "    'email': 'antoine.limasset@univ-lille.fr',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__msa_limit',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__msa_limit',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.560796',\n",
       "  'is_preprint': True},\n",
       " {'name': 'cellContrast',\n",
       "  'description': 'Reconstructing Spatial Relationships in Single-Cell RNA Sequencing Data via Deep Contrastive Learning.\\n\\nA vast amount of single-cell RNA-seq (SC) data has been accumulated via various studies and consortiums, but the lack of spatial information limits its analysis of complex biological activities. To bridge this gap, we introduce cellContrast, a computational method for reconstructing spatial relationships among SC cells from spatial transcriptomics (ST) reference. By adopting a contrastive learning framework and training with ST data, cellContrast projects gene expressions into a hidden space where proximate cells share similar representation values. We performed extensive benchmarking on diverse platforms, including SeqFISH, Stereo-Seq, 10X Visium, and MERSCOPE, on mouse embryo and human breast cells. The results reveal that cellContrast substantially outperforms other related methods, facilitating accurate spatial reconstruction of SC',\n",
       "  'homepage': 'https://github.com/HKU-BAL/CellContrast',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0314',\n",
       "      'term': 'Gene expression profiling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3625',\n",
       "      'term': 'Relation extraction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0099', 'term': 'RNA'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.12.562026'}],\n",
       "  'credit': [{'name': 'Ruibang Luo',\n",
       "    'email': 'rbluo@cs.hku.hk',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Yuanhua Huang',\n",
       "    'email': 'yuanhua@hku.hk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cellcontrast',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cellcontrast',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.12.562026',\n",
       "  'is_preprint': True},\n",
       " {'name': 'HIV-DRIVES',\n",
       "  'description': 'HIV Drug Resistance Identification, Variant Evaluation, & Surveillance Pipeline.\\n\\nHIV-DRIVES-validation-reports.',\n",
       "  'homepage': 'https://github.com/MicroBioGenoHub/HIV-DRIVES',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3227',\n",
       "      'term': 'Variant calling'},\n",
       "     {'uri': 'http://edamontology.org/operation_4009',\n",
       "      'term': 'Small molecule design'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3168',\n",
       "    'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'}],\n",
       "  'language': ['Shell'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'link': [{'url': 'https://github.com/MicroBioGenoHub/HIV-DRIVES-validation-reports',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.09.30.23296350'}],\n",
       "  'credit': [{'name': 'Gerald Mboowa',\n",
       "    'email': 'gmboowa@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__hiv-drives',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hiv-drives',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.30.23296350',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MCHelper',\n",
       "  'description': 'MCHelper automatically curates transposable element libraries across species.\\n\\nThe number of species with high quality genome sequences continues to increase, in part due to scaling up of multiple large scale biodiversity sequencing projects. While the need to annotate genic sequences in these genomes is widely acknowledged, the parallel need to annotate transposable element sequences that have been shown to alter genome architecture, rewire gene regulatory networks, and contribute to the evolution of host traits is becoming ever more evident. However, accurate genome-wide annotation of transposable element sequences is still technically challenging. Several de novo transposable element identification tools are now available, but manual curation of the libraries produced by these tools is needed to generate high quality genome annotations. Manual curation is time-consuming, and thus impractical for large-scale genomic studies, and lacks reproducibility',\n",
       "  'homepage': 'https://github.com/GonzalezLab/MCHelper',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0362',\n",
       "      'term': 'Genome annotation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3644',\n",
       "      'term': 'de Novo sequencing'},\n",
       "     {'uri': 'http://edamontology.org/operation_0525',\n",
       "      'term': 'Genome assembly'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3050',\n",
       "    'term': 'Biodiversity'},\n",
       "   {'uri': 'http://edamontology.org/topic_3299',\n",
       "    'term': 'Evolutionary biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3673',\n",
       "    'term': 'Whole genome sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_0780', 'term': 'Plant biology'}],\n",
       "  'language': ['Python', 'Shell', 'R'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562682'}],\n",
       "  'credit': [{'name': 'Josefa González',\n",
       "    'email': 'josefa.gonzalez@csic.es',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__mchelper',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mchelper',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562682',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Rummagene',\n",
       "  'description': 'Mining Gene Sets from Supporting Materials of PMC Publications.\\n\\nextracted from supporting tables of loading.\\n\\nto find the most similar gene sets that match your query.',\n",
       "  'homepage': 'https://rummagene.com',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2436',\n",
       "      'term': 'Gene-set enrichment analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_2454',\n",
       "      'term': 'Gene prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0224',\n",
       "      'term': 'Query and retrieval'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0203',\n",
       "    'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'},\n",
       "   {'uri': 'http://edamontology.org/topic_3169', 'term': 'ChIP-seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.560783'}],\n",
       "  'credit': [{'name': 'Avi Ma’ayan',\n",
       "    'email': 'avi.maayan@mssm.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__rummagene',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__rummagene',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.560783',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ReporterScore',\n",
       "  'description': 'Generalized Reporter Score-based Enrichment Analysis for Omics Data.\\n\\nEnrichment analysis contextualizes biological features in pathways to facilitate a systematic understanding of high-dimensional data and is widely used in biomedical research. The emerging reporter score-based analysis (RSA) method shows more promising sensitivity, as it relies on p-values instead of raw values of features. However, RSA cannot be directly applied to multi-group experimental designs and is often misused due to the lack of a proper tool. Here, we propose the Generalized Reporter Score-based Analysis (GRSA) method for multi-group and longitudinal omics data. A comparison with other popular enrichment analysis methods demonstrated that GRSA had increased sensitivity across multiple benchmark datasets. We applied GRSA to microbiome, transcriptome, and metabolome data and discovered new biological insights in omics studies\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE GRSA',\n",
       "  'homepage': 'https://github.com/Asa12138/ReporterScore',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3501',\n",
       "      'term': 'Enrichment analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3926',\n",
       "      'term': 'Pathway visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0533',\n",
       "      'term': 'Expression profile pathway mapping'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3172',\n",
       "    'term': 'Metabolomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3697', 'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'}],\n",
       "  'language': ['R', 'Shell', 'Perl'],\n",
       "  'link': [{'url': 'https://github.com/wangpeng407/ReporterScore',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562235'}],\n",
       "  'credit': [{'name': 'Chao Jiang',\n",
       "    'email': 'jiang_chao@zju.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__reporterscore',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__reporterscore',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562235',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TreePPL',\n",
       "  'description': 'A Universal Probabilistic Programming Language for Phylogenetics.\\n\\nDescription will go into a meta tag in <head />.\\n\\nA Universal Probabilistic Programming Language Inspired from Phylogenetics.\\n\\nDesigned to meet the needs of computational biologists.',\n",
       "  'homepage': 'https://treeppl.org',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0323',\n",
       "      'term': 'Phylogenetic inference'},\n",
       "     {'uri': 'http://edamontology.org/operation_3658',\n",
       "      'term': 'Statistical inference'},\n",
       "     {'uri': 'http://edamontology.org/operation_0326',\n",
       "      'term': 'Phylogenetic tree editing'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3293',\n",
       "    'term': 'Phylogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3372',\n",
       "    'term': 'Software engineering'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_3068',\n",
       "    'term': 'Literature and language'}],\n",
       "  'language': ['R', 'Python', 'OCaml'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.10.561673'}],\n",
       "  'credit': [{'name': 'Viktor Senderov',\n",
       "    'email': 'vsenderov@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__treeppl',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__treeppl',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.10.561673',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AleRax',\n",
       "  'description': 'A tool for gene and species tree co-estimation and reconciliation under a probabilistic model of gene duplication, transfer, and loss.\\n\\nABSTRACT Motivation. Genomes are a rich source of information on the pattern and process of evolution across biological scales. How best to make use of that information is an active area of research in phylogenetics. Ideally, phylogenetic methods should not only model substitutions along gene trees, which explain differences between homologous gene sequences, but also the processes that generate the gene trees themselves along a shared species tree. To conduct accurate inferences, one needs to account for uncertainty at both levels, that is, in gene trees estimated from inherently short sequences and in their diverse evolutionary histories along a shared species tree. Results',\n",
       "  'homepage': 'https://github.com/BenoitMorel/AleRax',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3947',\n",
       "      'term': 'Phylogenetic tree reconciliation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0544',\n",
       "      'term': 'Species tree construction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3478',\n",
       "      'term': 'Phylogenetic reconstruction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0553',\n",
       "      'term': 'Gene tree construction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3942',\n",
       "      'term': 'Tree dating'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3293',\n",
       "    'term': 'Phylogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0623',\n",
       "    'term': 'Gene and protein families'},\n",
       "   {'uri': 'http://edamontology.org/topic_0194', 'term': 'Phylogenomics'}],\n",
       "  'language': ['C++', 'Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'download': [{'url': 'https://cme.h-its.org/exelixis/material/alerax_data.tar.gz',\n",
       "    'type': 'Software package'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.06.561091'}],\n",
       "  'credit': [{'name': 'Benoit Morel',\n",
       "    'email': 'benoit.morel@h-its.org',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__alerax',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__alerax',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.06.561091',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NanoLAS',\n",
       "  'description': 'a Comprehensive Nanobody Database with Data Integration, Consolidation, and Application.\\n\\nNanobodies, a unique subclass of antibodies first discovered in camelid animals, are composed solely of a single heavy chain’s variable region (VHH). Their significantly reduced molecular weight, in comparison to conventional antibodies, confers numerous advantages in the treatment of various diseases. As research and applications involving nanobodies expand, the quantity of identified nanobodies is also rapidly growing. However, the existing antibody databases are deficient in type and coverage, failing to satisfy the comprehensive needs of researchers and thus, impeding progress in nanobody research. In response to this, we have amalgamated data from multiple sources to successfully assemble a new and comprehensive nanobody database',\n",
       "  'homepage': 'https://www.nanolas.cloud',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'},\n",
       "     {'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'},\n",
       "   {'uri': 'http://edamontology.org/topic_3930', 'term': 'Immunogenetics'}],\n",
       "  'publication': [{'doi': '10.1101/2023.09.27.559691'}],\n",
       "  'credit': [{'name': 'Bingding Huang',\n",
       "    'email': 'huangbingding@sztu.edu.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Xin Wang',\n",
       "    'email': 'wangxin@sztu.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__nanolas',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nanolas',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.27.559691',\n",
       "  'is_preprint': True},\n",
       " {'name': 'STEGO',\n",
       "  'description': 'an application to aid in scRNA-seq and scTCR-seq processing and analysis.\\n\\nIntroduction The hypervariable T cell receptor (TCR), created through somatic recombination, allows for recognition of a diverse array of antigens. Single sequencing technologies allow capture of both the single cell expression data (scRNA-seq) with the paired single cell TCR sequencing (scTCR-seq). However, the current analytical pipelines have limited capacity to integrate both data levels. To overcome these limitations, we developed STEGO (Single cell TCR and Expression Grouped Ontologies) Shiny R application to facilitate the complex analysis required for understanding T cells role in various conditions. Program parameters. STEGO. R application includes the Seurat quality control (QC) process, merging with Harmony, followed by semi-supervised cellular annotations with scGate\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/stego\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE STEGO R',\n",
       "  'homepage': 'https://github.com/KerryAM-R/STEGO.R',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0416',\n",
       "      'term': 'Epitope mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_0313',\n",
       "      'term': 'Expression profile clustering'},\n",
       "     {'uri': 'http://edamontology.org/operation_0314',\n",
       "      'term': 'Gene expression profiling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2830',\n",
       "    'term': 'Immunoproteins and antigens'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_0108',\n",
       "    'term': 'Protein expression'}],\n",
       "  'language': ['R'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.09.27.559702'}],\n",
       "  'credit': [{'name': 'Pieter Meysman',\n",
       "    'email': 'pieter.meysman@uantwerpen.be',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__stego',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__stego',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.27.559702',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Listi Wiki',\n",
       "  'description': 'a database for the foodborne pathogen<i>Listeria monocytogenes</i>.\\n\\nHome Categories Essential Genes Random gene Log in Register.\\n\\nGenes or proteins Interactions Expression.\\n\\nGenome browser Interaction browser Expression browser.\\n\\nFAQ User list History list Statistics People Data Site notice.',\n",
       "  'homepage': 'http://listiwiki.uni-goettingen.de/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3208',\n",
       "      'term': 'Genome visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2492',\n",
       "      'term': 'Protein interaction prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0474',\n",
       "      'term': 'Protein structure prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0203',\n",
       "    'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0128',\n",
       "    'term': 'Protein interactions'},\n",
       "   {'uri': 'http://edamontology.org/topic_3324', 'term': 'Infectious disease'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_3305',\n",
       "    'term': 'Public health and epidemiology'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.16.562455'}],\n",
       "  'credit': [{'name': 'Jörg Stülke',\n",
       "    'email': 'jstuelk@gwdg.de',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Sven Halbedel',\n",
       "    'email': 'halbedels@rki.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__listi-wiki',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__listi-wiki',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.16.562455',\n",
       "  'is_preprint': True},\n",
       " {'name': 'CUDASW++4.0',\n",
       "  'description': 'Ultra-fast GPU-based Smith-Waterman Protein Sequence Database Search.\\n\\nBackground The maximal sensitivity for local pairwise alignment makes the Smith-Waterman algorithm a popular choice for protein sequence database search. However, its quadratic time complexity makes it compute-intensive. Unfortunately, current state-of-the-art software tools are not able to leverage the massively parallel processing capabilities of modern GPUs with close-to-peak performance. This motivates the need for more efficient implementations. Results. CUDASW++4.0 is a fast software tool for scanning protein sequence databases with the Smith-Waterman algorithm on CUDA-enabled GPUs. Our approach achieves high efficiency for dynamic programming-based alignment computation by minimizing memory accesses and instructions\\n\\n||| NAME (CUDASW++4.0) SIMILAR TO (PUB. DIFFERENT) bio.tools/cudasw (CUDASW++)',\n",
       "  'homepage': 'https://github.com/asbschmidt/CUDASW4',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0495',\n",
       "      'term': 'Local alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_0338',\n",
       "      'term': 'Sequence database search'},\n",
       "     {'uri': 'http://edamontology.org/operation_0491',\n",
       "      'term': 'Pairwise sequence alignment'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0078', 'term': 'Proteins'},\n",
       "   {'uri': 'http://edamontology.org/topic_0080', 'term': 'Sequence analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_0081',\n",
       "    'term': 'Structure analysis'}],\n",
       "  'language': ['C++', 'Ada'],\n",
       "  'license': 'Apache-2.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.09.561526'}],\n",
       "  'credit': [{'name': 'Bertil Schmidt',\n",
       "    'email': 'bertil.schmidt@uni-mainz.de',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Felix Kallenborn',\n",
       "    'email': 'kallenborn@uni-mainz.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cudasw4.0',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cudasw4.0',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.09.561526',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TEclass2',\n",
       "  'description': 'Classification of transposable elements using Transformers.\\n\\nInstitute of Bioinformatics WWU Münster.\\n\\nInstitute of Bioinformatics Münster.\\n\\nTo run a TEclass2 request: please enter either a file or type/copy the sequence data in FASTA format into the \"Sequence Data\" field.',\n",
       "  'homepage': 'https://bioinformatics.uni-muenster.de/tools/teclass2/index.pl',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3695',\n",
       "      'term': 'Filtering'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0091',\n",
       "    'term': 'Bioinformatics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'},\n",
       "   {'uri': 'http://edamontology.org/topic_3299',\n",
       "    'term': 'Evolutionary biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0798',\n",
       "    'term': 'Mobile genetic elements'},\n",
       "   {'uri': 'http://edamontology.org/topic_3500', 'term': 'Zoology'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'https://github.com/IOB-Muenster/TEclass2',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562246'}],\n",
       "  'credit': [{'name': 'Wojciech Makalowski',\n",
       "    'email': 'wojmak@uni-muenster.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__teclass2',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__teclass2',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562246',\n",
       "  'is_preprint': True},\n",
       " {'name': 'phyloKRR',\n",
       "  'description': 'Non-linear phylogenetic regression using regularized kernels.\\n\\nPhylogenetic regression is a type of Generalized Least Squares (GLS) method that incorporates a covariance matrix based on the evolutionary relationships between species (i.e., phylogenetic relationships). While this method has found widespread use in hypothesis testing via comparative phylogenetic methods, such as phylogenetic ANOVA, its ability to account for non-linear relationships has received little attention. To address this issue, we utilized GLS in a high-dimensional feature space, employing linear combinations of transformed data to account for non-linearity, a common approach in kernel regression. We analyzed two biological datasets using both Radial Basis Function (RBF) and linear kernel transformations. The first dataset contained morphometric data, while the second dataset comprised discrete trait data and diversification rates as labels',\n",
       "  'homepage': 'https://github.com/Ulises-Rosas/phylokrr',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3478',\n",
       "      'term': 'Phylogenetic reconstruction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2939',\n",
       "      'term': 'Principal component visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3293',\n",
       "    'term': 'Phylogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.04.560983'}],\n",
       "  'credit': [{'name': 'Ulises Rosas-Puchuri',\n",
       "    'email': 'urosas@ou.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__phylokrr',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__phylokrr',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.04.560983',\n",
       "  'is_preprint': True},\n",
       " {'name': 'EPDRNA',\n",
       "  'description': 'a model for identifying DNA-RNA binding sites in disease-related proteins.\\n\\nEPDRNA--Prediction of DNA and RNA binding sites in disease-associated proteins.\\n\\nEnter the query protein sequence in FASTA format(maximum 10 proteins for each submission).',\n",
       "  'homepage': 'http://www.s-bioinformatics.cn/epdrna',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3902',\n",
       "      'term': 'RNA binding site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3903',\n",
       "      'term': 'DNA binding site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2464',\n",
       "      'term': 'Protein-protein binding site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3901',\n",
       "      'term': 'RNA-binding protein prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3125',\n",
       "    'term': 'DNA binding sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_0128',\n",
       "    'term': 'Protein interactions'},\n",
       "   {'uri': 'http://edamontology.org/topic_3534',\n",
       "    'term': 'Protein binding sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0099', 'term': 'RNA'}],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3476519/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__epdrna',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__epdrna',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3476519/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NeuroMotion',\n",
       "  'description': 'Open-source Simulator with Neuromechanical and Deep Network Models to Generate Surface EMG signals during Voluntary Movement.\\n\\nNeuromechanical studies investigate how the nervous system interacts with the musculoskeletal (MSK) system to generate volitional movements. Such studies have been supported by simulation models that provide insights into variables that cannot be measured experimentally and allow a large number of conditions to be tested before the experimental analysis. However, current simulation models of electromyography (EMG), a core physiological signal in neuromechanical analyses, are mainly limited to static contractions and cannot fully represent the dynamic modulation of EMG signals during volitional movements. Here, we overcome these limitations by presenting NeuroMotion, an open-source simulator that provides a full-spectrum synthesis of EMG signals during voluntary movements. NeuroMotion is comprised of three modules',\n",
       "  'homepage': 'https://github.com/shihan-ma/NeuroMotion',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426',\n",
       "      'term': 'Modelling and simulation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3891',\n",
       "      'term': 'Essential dynamics'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3300',\n",
       "    'term': 'Physiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3306', 'term': 'Biophysics'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.05.560588'}],\n",
       "  'credit': [{'name': 'Dario Farina',\n",
       "    'email': 'd.farina@imperial.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__neuromotion',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__neuromotion',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.05.560588',\n",
       "  'is_preprint': True},\n",
       " {'name': 'STew',\n",
       "  'description': 'Uncover spatially informed shared variations for single-cell spatial transcriptomics with STew.\\n\\nMotivation The recent spatial transcriptomics (ST) technologies have enabled characterization of gene expression patterns and spatial information, advancing our understanding of cell lineages within diseased tissues. Several analytical approaches have been proposed for ST data, but effectively utilizing spatial information to unveil the shared variation with gene expression remains a challenge. Results. We introduce STew, a Spatial Transcriptomic multi-viEW representative learning method, to jointly analyze spatial information and gene expression in a scalable manner, followed by a data-driven statistical framework to measure the goodness of model fit. Through benchmarking using Human DLPFC data with true manual annotations, STew achieved superior performance in both clustering accuracy and continuity of identified spatial domains compared with other methods',\n",
       "  'homepage': 'http://github.com/fanzhanglab/STew',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3432',\n",
       "      'term': 'Clustering'},\n",
       "     {'uri': 'http://edamontology.org/operation_3463',\n",
       "      'term': 'Expression correlation analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3935',\n",
       "      'term': 'Dimensionality reduction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3308',\n",
       "    'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3518',\n",
       "    'term': 'Microarray experiment'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'}],\n",
       "  'language': ['R'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.10.561789'}],\n",
       "  'credit': [{'name': 'Fan Zhang',\n",
       "    'email': 'fan.3.zhang@cuanschutz.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__stew',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__stew',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.10.561789',\n",
       "  'is_preprint': True},\n",
       " {'name': 'READi-Dem',\n",
       "  'description': 'ML-powered, web-interface tool for Robust, Efficient, Affordable Diagnosis of Dementia.\\n\\nAn ML-powered, web-interface tool for Robust, Efficient, Affordable Diagnosis of Dementia.\\n\\nREADi-Dem is a web-based application to aid in dementia screening, using cutting-edge machine learning techniques on a unique population-level dataset. We first identified the most strongly-predictive features for dementia diagnosis from our dataset, and further refined these to a set of 27 minimal features. These can be easily obtained by a patient and their caregiver, without the need for any training or time consuming cognitive tests. We then tested a range of ML models, and found that the support vector machine provided the highest classification accuracy. The trained model was incorporated into our app, which uses questionnaire answers to predict the likelihood of a dementia diagnosis for the individual.',\n",
       "  'homepage': 'https://researchmind.co.uk/readi-dem',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3891',\n",
       "      'term': 'Essential dynamics'},\n",
       "     {'uri': 'http://edamontology.org/operation_3960',\n",
       "      'term': 'Principal component analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3935',\n",
       "      'term': 'Dimensionality reduction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3399',\n",
       "    'term': 'Geriatric medicine'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}],\n",
       "  'language': ['R'],\n",
       "  'link': [{'url': 'https://github.com/sanjaysinghrathi/READi-Dem',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.23.23297405'}],\n",
       "  'credit': [{'name': 'Sanjay Rathee',\n",
       "    'email': 'sr952@cantab.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__readi-dem',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__readi-dem',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.23.23297405',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PitArray',\n",
       "  'description': 'A Non-invasive Detection of Parkinson’s Disease using PitArray.\\n\\nParkinson’s disease (PD) is a progressive neurodegenerative disorder affecting the central nervous system, often diagnosed in its advanced stages due to the absence of sensitive biomarkers. With this objective in mind, our study conducted a comprehensive analysis of differentially expressed genes (DEGs) sourced from blood-based microarray datasets to uncover potential biomarkers and developed a machine learning based classifier to conduct two step validations. By analyzing gene expression of three projects, we identified 678 DEGs, consisting of 337 genes showing upregulation and 341 genes presenting downregulation. Additionally, insights from functional enrichment and the protein-protein network analysis indicate that HLA-F , IRF-1 , and RPS28 have the potential to serve as biomarkers for diagnosing PD',\n",
       "  'homepage': 'https://github.com/Arittra95/PitArray',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3223',\n",
       "      'term': 'Differential gene expression profiling'},\n",
       "     {'uri': 'http://edamontology.org/operation_2436',\n",
       "      'term': 'Gene-set enrichment analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3360',\n",
       "    'term': 'Biomarkers'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3518',\n",
       "    'term': 'Microarray experiment'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'}],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.15.562440'}],\n",
       "  'credit': [{'name': 'Md Salimullah',\n",
       "    'email': 'dgnib.gov.bd@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pitarray',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pitarray',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.15.562440',\n",
       "  'is_preprint': True},\n",
       " {'name': 'CellPie',\n",
       "  'description': 'a fast spatial transcriptomics topic discovery method via joint factorization of gene expression and imaging data.\\n\\nMotivation Spatially resolved transcriptomics has enabled the study of expression of genes within tissues while retaining their spatial identity. The lack of single-cell resolution for most of the current high-throughput spatial transcriptomics technologies led to the development of in-silico methods, to disentangle the spatial profiles of individual cell-types. However, most current approaches ignore useful information from associated imaging data that can help to better resolve cell-types or spatial domains. Results. We present CellPie , a fast, reference-free topic modelling method, based on joint non-negative matrix factorisation between spatial RNA transcripts and histological or molecular imaging features. This synergy of the two modalities can lead to improved single-cell deconvolution and spatial clustering',\n",
       "  'homepage': 'https://github.com/ManchesterBioinference/CellPie',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3629',\n",
       "      'term': 'Deisotoping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3432', 'term': 'Clustering'},\n",
       "     {'uri': 'http://edamontology.org/operation_0314',\n",
       "      'term': 'Gene expression profiling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3308',\n",
       "    'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.09.29.560213'}],\n",
       "  'credit': [{'name': 'Sokratia Georgaka',\n",
       "    'email': 'sokratia.georgaka@manchester.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cellpie',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cellpie',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.29.560213',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MADAME',\n",
       "  'description': 'a user-friendly bioinformatic tool for data and metadata retrieval in microbiome research.\\n\\nMicrobiome research advancements have provided countless insights. Despite the massive amount of data currently stored in public repositories, these resources remain vastly underutilized due to the intricacy of data and metadata retrieval from these databases. However, leveraging data-driven approaches is crucial for microbiome research progress by overcoming variations between studies and identifying generalizable trends. We designed the open-access and user-friendly bioinformatic tool MADAME (MetADAta MicrobiomE) to streamline the data and metadata retrieval process. MADAME addresses the challenges posed by the public repositories’ current limitations, allowing users to retrieve publications associated with the accession codes of interest. Additionally, MADAME allows users to visually explore retrieved results through the generation of a comprehensive report with plots and statistics',\n",
       "  'homepage': 'https://github.com/BiomeResearchTeam/MADAME',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3697',\n",
       "    'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3810',\n",
       "    'term': 'Agricultural science'},\n",
       "   {'uri': 'http://edamontology.org/topic_0091', 'term': 'Bioinformatics'}],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.14.562335'}],\n",
       "  'credit': [{'name': 'Sara Fumagalli',\n",
       "    'email': 'sara.fumagalli@unimib.it',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Giulia Agostinetto',\n",
       "    'email': 'giulia.agostinetto@unimib.it',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__madame',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__madame',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.14.562335',\n",
       "  'is_preprint': True},\n",
       " {'name': 'scRANK',\n",
       "  'description': 'Ranking of cell clusters in a single-cell RNA-sequencing analysis framework using prior knowledge.\\n\\nPrioritization or ranking of different cell types in a scRNA-Seq framework can be performed in a variety of ways, some of these include: i) obtaining an indication of the proportion of cell types between the different conditions under study, ii) counting the number of differentially expressed genes (DEGs) between cell types and conditions in the experiment or, iii) prioritizing cell types based on prior knowledge about the conditions under study (i.e., a specific disease). These methods have drawbacks and limitations thus novel methods for improving cell ranking are required. Here we present a novel methodology that exploits prior knowledge in combination with expert-user information to accentuate cell types from a scRNA-seq analysis that yield the most biologically meaningful results',\n",
       "  'homepage': 'https://github.com/aoulas/scRANK',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3096',\n",
       "      'term': 'Editing'},\n",
       "     {'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3379',\n",
       "    'term': 'Preclinical and clinical studies'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.02.560416'}],\n",
       "  'credit': [{'name': 'Anastasis Oulas',\n",
       "    'email': 'anastasioso@cing.ac.cy',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__scrank',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__scrank',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.02.560416',\n",
       "  'is_preprint': True},\n",
       " {'name': 'SiamEEGNet',\n",
       "  'description': 'Siamese Neural Network-Based EEG Decoding for Drowsiness Detection.\\n\\nRecent advancements in deep-learning have significantly enhanced EEG-based drowsiness detection. However, most existing methods overlook the importance of relative changes in EEG signals compared to a baseline, a fundamental aspect in conventional EEG analysis including event-related potential and time-frequency spectrograms. We herein introduce SiamEEGNet, a Siamese neural network architecture designed to capture relative changes between EEG data from the baseline and a time window of interest. Our results demonstrate that SiamEEGNet is capable of robustly learning from high-variability data across multiple sessions subjects and outperforms existing model architectures in cross-subject scenarios. Furthermore, the model’s interpretability associates with previous findings of drowsiness-related EEG correlates',\n",
       "  'homepage': 'http://github.com/CECNL/SiamEEGNet',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0479',\n",
       "      'term': 'Backbone modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3937',\n",
       "      'term': 'Feature extraction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.23.563513'}],\n",
       "  'credit': [{'name': 'Chun-Shu Wei',\n",
       "    'email': 'wei@nycu.edu.tw',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__siameegnet',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__siameegnet',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.23.563513',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Graphite',\n",
       "  'description': 'painting genomes using a colored De Bruijn graph.\\n\\nThe recent growth of microbial sequence data allows comparisons at unprecedented scales, enabling tracking of strains, mobile genetic elements, or genes. Querying a genome against a large reference database can easily yield thousands of matches that are tedious to interpret and pose computational challenges. We developed Graphite that uses a colored De Bruijn graph (cDBG) to paint query genomes, selecting the local best matches along the full query length. By focusing on the closest genomic match of each query region, Graphite reduces the number of matches while providing promising leads for genomic forensics. When applied to hundreds of Campylobacter genomes we found extensive gene sharing, including a previously undetected C. coli plasmid that matched a C. jejuni chromosome\\n\\n||| NAME (Graphite) SIMILAR TO (PUB. DIFFERENT) bio.tools/graphite (graphite)',\n",
       "  'homepage': 'https://github.com/MGXlab/Graphite',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0524',\n",
       "      'term': 'De-novo assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_0224',\n",
       "      'term': 'Query and retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_3209',\n",
       "      'term': 'Genome comparison'},\n",
       "     {'uri': 'http://edamontology.org/operation_3211',\n",
       "      'term': 'Genome indexing'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0798',\n",
       "    'term': 'Mobile genetic elements'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0654', 'term': 'DNA'}],\n",
       "  'language': ['C', 'Julia'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.08.561343'}],\n",
       "  'credit': [{'name': 'Bas E. Dutilh',\n",
       "    'email': 'bedutilh@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__graphite',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__graphite',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.08.561343',\n",
       "  'is_preprint': True},\n",
       " {'name': 'DynaMight',\n",
       "  'description': 'estimating molecular motions with improved reconstruction from cryo-EM images.\\n\\nHow to deal with continuously flexing molecules is one of the biggest outstanding challenges in single-particle analysis of proteins from cryo-EM images. Here, we present DynaMight, a new software tool that estimates a continuous space of conformations in a cryo-EM data set by learning 3D deformations of a Gaussian pseudo-atomic model of a consensus structure for every particle image. Inversion of the learnt deformations is then used to obtain an improved reconstruction of the consensus structure. We illustrate the performance of DynaMight for several experimental cryo-EM data sets. We also show how error estimates on the deformations may be obtained by independently training two variational autoencoders (VAEs) on half sets of the cryo-EM data, and how regularisation of the 3D deformations through the use of atomic models may lead to important artefacts due to model bias',\n",
       "  'homepage': 'https://github.com/3dem/DynaMight',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3457',\n",
       "      'term': 'Single particle analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3891',\n",
       "      'term': 'Essential dynamics'},\n",
       "     {'uri': 'http://edamontology.org/operation_3456',\n",
       "      'term': 'Rigid body refinement'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0176',\n",
       "    'term': 'Molecular dynamics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3047', 'term': 'Molecular biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.18.562877'}],\n",
       "  'credit': [{'name': 'Johannes Schwab',\n",
       "    'email': 'schwab@mrc-lmb.cam.ac.uk',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Sjors H.W. Scheres',\n",
       "    'email': 'scheres@mrc-lmb.cam.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__dynamight',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dynamight',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.18.562877',\n",
       "  'is_preprint': True},\n",
       " {'name': 'SODA',\n",
       "  'description': 'Software to Support the Curation and Sharing of FAIR Autonomic Nervous System Data.\\n\\nSince 2014, the National Institutes of Health (NIH)’s Stimulating Peripheral Activity to Relieve Conditions (SPARC) Program has been supporting research and development of therapeutic devices that modulate electrical activity in the autonomic nervous system (ANS) to improve organ function, also known as bioelectronic medicine. To optimize the reusability of data resulting from ANS-related research, the SPARC Program also supported the development of guidelines for curating and sharing data in line with the FAIR (Findable, Accessible, Interoperable, Reusable) Principles. These guidelines are exhaustive to maximize FAIRness of data but as a result, they are difficult and time-consuming for researchers to implement\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/soda-astral, bio.tools/soda-solubility\\n\\n||| NAME (SODA) SIMILAR TO (PUB. DIFFERENT) bio.tools/soda (SODa)',\n",
       "  'homepage': 'https://github.com/fairdataihub/SODA-for-SPARC',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3303', 'term': 'Medicine'},\n",
       "   {'uri': 'http://edamontology.org/topic_0219',\n",
       "    'term': 'Data submission, annotation and curation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3376',\n",
       "    'term': 'Medicines research and development'}],\n",
       "  'language': ['JavaScript', 'Python'],\n",
       "  'license': 'MIT',\n",
       "  'documentation': [{'url': 'https://github.com/fairdataihub/SODA-for-SPARC-Docs',\n",
       "    'type': ['User manual']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562300'}],\n",
       "  'credit': [{'name': 'Bhavesh Patel',\n",
       "    'email': 'bpatel@calmi2.org',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__soda',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__soda',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562300',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AgroNT',\n",
       "  'description': 'A Foundational Large Language Model for Edible Plant Genomes.\\n\\nInstaDeepAI/agro-nucleotide-transformer-1b.\\n\\nWe’re on a journey to advance and democratize artificial intelligence through open source and open science.\\n\\nInstaDeepAI/agro-nucleotide-transformer-1b · Hugging Face.\\n\\nAgroNT is a DNA language model trained on primarily edible plant genomes. More specifically, AgroNT uses the transformer architecture with self-attention and a masked language modeling objective to leverage highly available genotype data from 48 different plant speices to learn general representations of nucleotide sequences. AgroNT contains 1 billion parameters and has a context window of 1024 tokens. AgroNt uses a non-overlapping 6-mer tokenizer to convert genomic nucletoide sequences to tokens. As a result the 1024 tokens correspond to approximately 6144 base pairs.\\n\\nInstaDeepAI/plant-genomic-benchmark.\\n\\nesm biology genomics language model plants.',\n",
       "  'homepage': 'https://huggingface.co/InstaDeepAI/agro-nt',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_0433',\n",
       "      'term': 'Splice site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3226',\n",
       "      'term': 'Variant prioritisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0438',\n",
       "      'term': 'Transcriptional regulatory element prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0780',\n",
       "    'term': 'Plant biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.24.563624'}],\n",
       "  'credit': [{'name': 'Thomas Pierrot',\n",
       "    'email': 't.pierrot@instadeep.com',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Marie Lopez',\n",
       "    'email': 'm.lopez@instadeep.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__agront',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__agront',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.24.563624',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PDB NextGen Archive',\n",
       "  'description': 'Centralising Access to Integrated Annotations and Enriched Structural Information by the Worldwide Protein Data Bank.\\n\\nIndex of /pub/databases/pdb_nextgen.\\n\\nwwPDB: Worldwide Protein Data Bank.\\n\\nSince February 01 2023, the wwPDB enriches PDB entries with additional annotation and distributes the latest versions of each entry via next generation archive (NextGen) accessible at https://files-nextgen.wwpdb.org and its mirrors in the USA, UK and Japan.\\n\\nThe PDB NextGen Repositories are updated every month on the 1st Wednesday at 00:00 UTC.\\n\\nDirectory structure of the PDB NextGen archive.',\n",
       "  'homepage': 'http://www.wwpdb.org/ftp/pdb-nextgen-archive-site',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3431',\n",
       "      'term': 'Deposition'},\n",
       "     {'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_2950',\n",
       "      'term': 'Residue distance calculation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0078', 'term': 'Proteins'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_0097',\n",
       "    'term': 'Nucleic acid structure analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_3071',\n",
       "    'term': 'Biological databases'}],\n",
       "  'link': [{'url': 'http://ftp.ebi.ac.uk/pub/databases/pdb_nextgen',\n",
       "    'type': ['Other']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.24.563739'}],\n",
       "  'credit': [{'email': 'sburley@proteomics.rutgers.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pdb-nextgen-archive',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pdb-nextgen-archive',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.24.563739',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TreeArchTraits',\n",
       "  'description': 'an R package to analyse the architectural traits of trees using TLS data.\\n\\n1 The architecture of trees is significantly influenced by their interactions, directly affecting their functioning and structural development. Different tree architectural indicators (TAT) have evolved, including these interactions’ measurements. Recent advances in Terrestrial Laser Scanning data collection make measuring the three-dimensional characteristics of trees more more efficient. The R package TreeArchTraits facilitates the processing of three-dimensional tree characteristics obtained through Terrestrial Laser Scanning, enabling the computation of various indices. A set of trees belonging to different forest conditions was used to demonstrate the TreeArchTraits potential for characterizing tree architectures. Abstract Figure',\n",
       "  'homepage': 'https://gitlab.com/Puletti/tree_arch_traits',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3365',\n",
       "    'term': 'Data architecture, analysis and design'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.1101/2023.09.29.560266'}],\n",
       "  'credit': [{'name': 'Nicola Puletti',\n",
       "    'email': 'nicola.puletti@crea.gov.it',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__treearchtraits',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__treearchtraits',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.29.560266',\n",
       "  'is_preprint': True},\n",
       " {'name': 'cubeVB',\n",
       "  'description': 'Variational Bayesian Phylogenies through Matrix Representation of Tree Space.\\n\\nABSTRACT Variational Bayesian methods work well when the distribution of interest can be represented by a Euclidian space using a bijective transformation between phylogenies and their representation. Until now, full phylogenetic inference has been hampered by the lack of such transformation. Here, we study the distance matrix as a representation of a phylogeny through single link clustering. Though such representation does not map to unique trees, restriction to a subspace we call a “cube”, makes the representation bijective at the cost of not being able to represent all possible trees. We introduce a variational Bayesian algorithm “cubeVB” based on MCMC and show through well calibrated simulation study that it is possible to recover parameters of interest like tree height and length',\n",
       "  'homepage': 'https://github.com/rbouckaert/cubevb',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0323',\n",
       "      'term': 'Phylogenetic inference'},\n",
       "     {'uri': 'http://edamontology.org/operation_0551',\n",
       "      'term': 'Phylogenetic tree topology analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3942',\n",
       "      'term': 'Tree dating'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'language': ['Java'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.19.563180'}],\n",
       "  'credit': [{'name': 'Remco R. Bouckaert',\n",
       "    'email': 'r.bouckaert@auckland.ac.nz',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cubevb',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cubevb',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.19.563180',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NIEND',\n",
       "  'description': 'Neuronal Image Enhancement through Noise Disentanglement.\\n\\nBinary releases of Vaa3D and related software components.\\n\\nv3d_external v3d_external Public.',\n",
       "  'homepage': 'https://github.com/Vaa3D',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3450',\n",
       "      'term': 'Neurite measurement'},\n",
       "     {'uri': 'http://edamontology.org/operation_3553',\n",
       "      'term': 'Image annotation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'link': [{'url': 'https://github.com/zzhmark/neuron-image-denoise-exp',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.21.563265'}],\n",
       "  'credit': [{'name': 'Yufeng Liu',\n",
       "    'email': 'yufeng_liu@seu.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__niend',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__niend',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.21.563265',\n",
       "  'is_preprint': True},\n",
       " {'name': 'plastiC',\n",
       "  'description': 'A pipeline for recovery and characterization of plastid genomes from metagenomic datasets.\\n\\nThe use of culture independent molecular methods, often referred to as metagenomics, have revolutionized the ability to explore and characterize microbial communities from diverse environmental sources. Most metagenomic workflows have been developed for identification of prokaryotic and eukaryotic community constituents, but tools for identification of plastid genomes are lacking. The endosymbiotic origin of plastids also poses challenges where plastid metagenomic assembled genomes (MAGs) may be misidentified as low-quality bacterial MAGs',\n",
       "  'homepage': 'https://github.com/Finn-Lab/plastiC',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_3460',\n",
       "      'term': 'Taxonomic classification'},\n",
       "     {'uri': 'http://edamontology.org/operation_3798',\n",
       "      'term': 'Read binning'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3174',\n",
       "    'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'language': ['Python', 'Shell'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.12688/WELLCOMEOPENRES.19589.1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__plastic',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__plastic',\n",
       "  'publication_link': 'https://doi.org/10.12688/WELLCOMEOPENRES.19589.1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ColoRecPred',\n",
       "  'description': 'Support Vector Machine based prediction models for drug repurposing and designing novel drugs for colorectal cancer.\\n\\nColoRecPred-Colorectal Cancer Drug Activity Prediction Server.\\n\\nColorectal Cancer Drug Activity Prediction Server.\\n\\nColoRecPred has been developed to predict the drug activities of the unknown compounds/molecules to alleviate Drug Development and Discovery.\\n\\nColoRecPred is a webserver developed to predict the drug activity of unknown molecules or compounds. The prediction server has been developed using AI/ML QSAR models. The webserver can be utilised by the researchers all around the world to evaluate the drug activity on the specific cell lines of colorectal cancer.',\n",
       "  'homepage': 'http://project.iith.ac.in/cgntlab/colorecpred',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_4009',\n",
       "      'term': 'Small molecule design'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3336',\n",
       "    'term': 'Drug discovery'},\n",
       "   {'uri': 'http://edamontology.org/topic_0202', 'term': 'Pharmacology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3373', 'term': 'Drug development'},\n",
       "   {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.20.563215'}],\n",
       "  'credit': [{'name': 'Rahul Kumar',\n",
       "    'email': 'rahulk@bt.iith.ac.in',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__colorecpred',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__colorecpred',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.20.563215',\n",
       "  'is_preprint': True},\n",
       " {'name': 'NanoBERTa-ASP',\n",
       "  'description': 'Predicting Nanobody Binding Epitopes Based on a Pretrained RoBERTa Model.\\n\\nNanobodies, also known as VHH or single-domain antibodies, are a unique class of antibodies that consist only of heavy chains and lack light chains. Nanobodies possess the advantages of both small molecule drugs and conventional antibodies, making them a promising class of therapeutic biopharmaceuticals. Studying the characteristics of nanobody sequences can aid the development and design of nanobodies. An important challenge in this field is accurately predicting the binding sites between nanobodies and antigens. The binding site is the region where the nanobody interacts with the antigen. The precise prediction of these binding sites is crucial for comprehending the interaction mechanism between the nanobody and the antigen, facilitating the design of effective nanobodies, as well as gaining valuable insights into the properties of nanobodies',\n",
       "  'homepage': 'https://github.com/WangLabforComputationalBiology/NanoBERTa-ASP',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0416',\n",
       "      'term': 'Epitope mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_0252',\n",
       "      'term': 'Peptide immunogenicity prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2575',\n",
       "      'term': 'Binding site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0480',\n",
       "      'term': 'Side chain modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_4009',\n",
       "      'term': 'Small molecule design'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2830',\n",
       "    'term': 'Immunoproteins and antigens'},\n",
       "   {'uri': 'http://edamontology.org/topic_0218',\n",
       "    'term': 'Natural language processing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_2275',\n",
       "    'term': 'Molecular modelling'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.09.29.560264'}],\n",
       "  'credit': [{'email': 'xin.leo.wang@outlook.com', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__nanoberta-asp',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nanoberta-asp',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.29.560264',\n",
       "  'is_preprint': True},\n",
       " {'name': 'SeqPHASE',\n",
       "  'description': 'Haxe as a Swiss knife for bioinformatic applications.\\n\\nFlot (2010) SeqPHASE: a web tool for interconverting PHASE input/output files and FASTA sequence alignments Molecular Ecology Ressources 10 (1): 162-166 [link].',\n",
       "  'homepage': 'https://eeg-ebe.github.io/SeqPHASE/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454',\n",
       "      'term': 'Phasing'},\n",
       "     {'uri': 'http://edamontology.org/operation_0292',\n",
       "      'term': 'Sequence alignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_3802', 'term': 'Sorting'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0610', 'term': 'Ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0091', 'term': 'Bioinformatics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3372',\n",
       "    'term': 'Software engineering'}],\n",
       "  'language': ['Python', 'JavaScript', 'Perl', 'Java', 'C++'],\n",
       "  'license': 'Apache-2.0',\n",
       "  'link': [{'url': 'https://github.com/eeg-ebe/SeqPHASE',\n",
       "    'type': ['Repository']}],\n",
       "  'download': [{'url': 'https://eeg-ebe.github.io/SeqPHASE/download.html',\n",
       "    'type': 'Downloads page'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.15.562406'}],\n",
       "  'credit': [{'name': 'Jean-François Flot',\n",
       "    'email': 'jean-francois.flot@ulb.be',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__seqphase',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__seqphase',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.15.562406',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Capricorn',\n",
       "  'description': 'Enhancing Hi-C contact matrices for loop detection with Capricorn.\\n\\nHigh-resolution Hi-C contact matrices reveal the detailed three-dimensional architecture of the genome, but high-coverage experimental Hi-C data are expensive to generate. On the other hand, chromatin structure analyses struggle with extremely sparse contact matrices. To address this problem, computational methods to enhance low-coverage contact matrices have been developed, but existing methods are largely based on resolution enhancement methods for natural images and hence often employ models that do not distinguish between biologically meaningful contacts, such as loops, and other stochastic contacts. We present Capricorn, a machine learning model for Hi-C resolution enhancement that incorporates small-scale chromatin features as additional views of the input Hi-C contact matrix and leverages a diffusion probability model backbone to generate a high-coverage matrix',\n",
       "  'homepage': 'https://github.com/CHNFTQ/Capricorn',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0481',\n",
       "      'term': 'Loop modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_0479',\n",
       "      'term': 'Backbone modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_3940',\n",
       "    'term': 'Chromosome conformation capture'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.25.564065'}],\n",
       "  'credit': [{'name': 'William S. Noble',\n",
       "    'email': 'william-noble@uw.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Sheng Wang',\n",
       "    'email': 'swang@cs.washington.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__capricorn',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__capricorn',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.25.564065',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ConfluentFUCCI',\n",
       "  'description': 'ConfluentFUCCI for fully-automated analysis of cell-cycle progression in a highly dense collective of migrating cells.\\n\\nA suite of tools for analyzing large scale confluent FUCCI experiments.',\n",
       "  'homepage': 'https://leogolds.github.io/ConfluentFUCCI/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3446',\n",
       "      'term': 'Cell migration analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2229',\n",
       "    'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3306', 'term': 'Biophysics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3300', 'term': 'Physiology'}],\n",
       "  'download': [{'url': 'https://github.com/leogolds/ConfluentFUCCI/tree/main',\n",
       "    'type': 'Source code'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.20.563216'}],\n",
       "  'credit': [{'name': 'Lior Atia',\n",
       "    'email': 'atialior@bgu.ac.il',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__confluentfucci',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__confluentfucci',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.20.563216',\n",
       "  'is_preprint': True},\n",
       " {'name': 'dismo',\n",
       "  'description': 'building and analysing discrete spatial ODE models.\\n\\nSummary dismo is a Python package for building and analysing discrete spatial models based on ordinary differential equations. Its primary purpose is to allow arbitrarily complex internal and transport processes to easily be mapped over multiple different regular grids. For this it features one, two and threedimensional layouts, with standard and non-standard (e.g. hexagonal or triangular) grids. Availability and implementation. https: gitlab.com qtb-hhu dismo Contact. oliver.ebenhoeh@hhu.de',\n",
       "  'homepage': 'https://gitlab.com/qtb-hhu/dismo',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2426',\n",
       "      'term': 'Modelling and simulation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'},\n",
       "   {'uri': 'http://edamontology.org/topic_2259', 'term': 'Systems biology'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562679'}],\n",
       "  'credit': [{'name': 'Oliver Ebenhöh',\n",
       "    'email': 'oliver.ebenhoeh@hhu.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__dismo',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__dismo',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562679',\n",
       "  'is_preprint': True},\n",
       " {'name': 'immuno-cross',\n",
       "  'description': 'A novel approach to identify cross-identity peptides between Epstein-Barr virus and central nervous system proteins in Guillain-Barré syndrome and multiple sclerosis.\\n\\nBackground Guillain-Barré Syndrome (GBS) and multiple sclerosis are autoimmune diseases associated with an immune system attack response against peripheral and central nervous system autoantigens, respectively. Given the potential of Epstein-Barr virus (EBV) as a risk factor for both multiple sclerosis and GBS, the present study aimed to identify crucial residues among potential EBV CD4+ T lymphocyte epitopes and nervous system proteins. Methods. Public databases (Allele Frequency Net Database, Immune Epitope Database, Genevestigator and Protein Atlas) were used to select proteins abundant in the nervous system, EBV immunogenic proteins, and HLA haplotypes. Computational tools were employed for predicting HLA-binding peptides and immunogenicity',\n",
       "  'homepage': 'https://github.com/evoMOL-Lab/immuno-cross',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0252',\n",
       "      'term': 'Peptide immunogenicity prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0416',\n",
       "      'term': 'Epitope mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3767',\n",
       "      'term': 'Protein identification'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3930',\n",
       "    'term': 'Immunogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'},\n",
       "   {'uri': 'http://edamontology.org/topic_3304', 'term': 'Neurobiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3948', 'term': 'Immunoinformatics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562729'}],\n",
       "  'credit': [{'name': 'João Paulo Matos Santos Lima',\n",
       "    'email': 'jpmslima@imd.ufrn.br',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__immuno-cross',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__immuno-cross',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562729',\n",
       "  'is_preprint': True},\n",
       " {'name': 'CIViCutils',\n",
       "  'description': 'Matching and downstream processing of clinical annotations from CIViC.\\n\\nBackground: With the advent of next-generation sequencing, profiling the genetic landscape of tumors entered clinical diagnostics, bringing the resolution of precision oncology to unprecedented levels. However, the wealth of information generated in a sequencing experiment can be difficult to manage, especially if hundreds of mutations need to be interpreted in a clinical context. Dedicated methods and databases are required that assist in interpreting the importance of a mutation for disease progression, prognosis, and with respect to therapy. Here, the CIViC knowledgebase is a valuable curated resource, however, utilizing CIViC in an efficient way for querying a large number of mutations needs sophisticated downstream methods. Methods:. : To this end, we have developed CIViCutils, a Python package to query, annotate, prioritize, and summarize information from the CIViC database\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.12688/F1000RESEARCH.136986.1',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3226',\n",
       "      'term': 'Variant prioritisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_2421',\n",
       "      'term': 'Database search'},\n",
       "     {'uri': 'http://edamontology.org/operation_3227',\n",
       "      'term': 'Variant calling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0199', 'term': 'Genetic variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.12688/F1000RESEARCH.136986.1'}],\n",
       "  'credit': [{'name': 'Franziska Singer',\n",
       "    'email': 'singer@nexus.ethz.ch',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__civicutils',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__civicutils',\n",
       "  'publication_link': 'https://doi.org/10.12688/F1000RESEARCH.136986.1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'climetrics',\n",
       "  'description': 'An R package to quantify multiple dimensions of climate change.\\n\\nClimate change affects biodiversity in diverse ways, necessitating the exploration of multiple climate dimensions using standardized metrics. However, existing methods for quantifying these metrics are scattered and tools for comparing alternative climate change metrics on the same footing are lacking. To address this gap, we developed “climetrics” which is an extensible and reproducible R package to spatially quantify and explore multiple dimensions of climate change through a unified procedure. Six widely used climate change metrics are currently implemented, including 1) Standardized Local Anomalies; 2) Changes in Probabilities of Local Climate Extremes; 3) Changes in Areas of Analogous Climates; 4) Novel Climates; 5) Changes in Distances to Analogous Climates; and 6) Climate Change Velocity\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.22541/AU.169769525.52686463/V2',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3050',\n",
       "    'term': 'Biodiversity'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.22541/AU.169769525.52686463/V2'}],\n",
       "  'credit': [{'name': 'Shirin Taheri', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__climetrics',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__climetrics',\n",
       "  'publication_link': 'https://doi.org/10.22541/AU.169769525.52686463/V2',\n",
       "  'is_preprint': True},\n",
       " {'name': 'GraphR',\n",
       "  'description': 'Probabilistic Graphical Modeling under Heterogeneity.\\n\\nProbabilistic graphical models are powerful and widely used tools to quantify, visualize and interpret dependencies in complex biological systems such as highthroughput genomics and proteomics. However, most existing graphical modeling methods assume homogeneity within and across samples which restricts their broad applicability to cases where sample-specific heterogeneity exists e.g. tumor heterogeneity. We propose a flexible Bayesian approach called Graph ical R egression (GraphR) which (a) allows direct incorporation of intrinsic factors of sample heterogeneity at different scales through a regression-based formulation, (b) enables sparse network estimation at a sample-specific level, (c) allows identification and uncertainty quantification of potential effects of heterogeneity on network structures, and (d) is computationally efficient through the use of variational Bayes algorithms',\n",
       "  'homepage': 'http://bayesrx.shinyapps.io/GraphR/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_0121', 'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_3922', 'term': 'Proteogenomics'}],\n",
       "  'language': ['R', 'C++'],\n",
       "  'link': [{'url': 'http://github.com/bayesrx/GraphR',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562136'}],\n",
       "  'credit': [{'name': 'Veerabhadran Baladandayuthapani',\n",
       "    'email': 'veerab@umich.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__graphr',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__graphr',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562136',\n",
       "  'is_preprint': True},\n",
       " {'name': 'RESIRE-V',\n",
       "  'description': 'Real space iterative reconstruction for vector tomography (RESIRE-V).\\n\\nTomography has had an important impact on the physical, biological, and medical sciences. To date, most tomographic applications have been focused on 3D scalar reconstructions. However, in some crucial applications, vector tomography is required to reconstruct 3D vector fields such as the electric and magnetic fields. Over the years, several vector tomography methods have been developed. Here, we present the mathematical foundation and algorithmic implementation of REal Space Iterative REconstruction for Vector tomography, termed RESIRE-V. RESIRE-V uses multiple tilt series of projections and iterates between the projections and a 3D reconstruction. Each iteration consists of a forward step using the Radon transform and a backward step using its transpose, then updates the object via gradient descent',\n",
       "  'homepage': 'https://github.com/minhpham0309/RESIRE-V',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3452',\n",
       "    'term': 'Tomography'},\n",
       "   {'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3315', 'term': 'Mathematics'}],\n",
       "  'language': ['MATLAB'],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3435607/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__resire-v',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__resire-v',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3435607/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'COINSTAC',\n",
       "  'description': 'Cortical Similarities in Psychiatric and Mood Disorders Identified in Federated VBM Analysis via COINSTAC.\\n\\nPsychiatric disorders such as schizophrenia, major depressive disorder, and bipolar disorder have areas of significant overlap across multiple domains including genetics, neurochemistry, symptom profiles, and regional gray matter alterations. Various structural neuroimaging studies have identified a combination of shared and disorder-specific patterns of gray matter (GM) deficits across these different disorders, though few direct comparisons have been conducted. Given the overlap in symptom presentations and GM alterations, these disorders may have a common etiology or neuroanatomical basis that may relate to a certain vulnerability for mental illness. Pooling large data or heterogeneous data can ensure representation of several participant factors, providing more accurate results\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/coinstac',\n",
       "  'homepage': 'https://github.com/trendscenter/coinstac-instructions',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3283',\n",
       "      'term': 'Anonymisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3436', 'term': 'Aggregation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3432',\n",
       "      'term': 'Clustering'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3419',\n",
       "    'term': 'Psychiatry'},\n",
       "   {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3334', 'term': 'Neurology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0605', 'term': 'Informatics'}],\n",
       "  'publication': [{'doi': '10.1101/2023.09.28.23296219'}],\n",
       "  'credit': [{'name': 'Kelly Rootes-Murdy',\n",
       "    'email': 'rootesmurdy@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__coinstac',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__coinstac',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.28.23296219',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TopoRoot+',\n",
       "  'description': 'Computing Whorl and Soil Line Traits of Maize Roots from CT Imaging.\\n\\nBackground: The use of 3D imaging techniques, such as X-ray CT, in root phenotyping has become more widespread in recent years. However, due to the complexity of root structure, analyzing the resulting 3D volumes to obtain detailed architectural traits of the root system remains a challenging computational problem. Two types of root features that are notably missing from existing computational image-based phenotyping methods are the whorls of a nodal root system and soil line in an excavated root crown. Knowledge of these features would give biologists deeper insights into the structure of nodal roots and the below- and above-ground root properties. Results:. We developed TopoRoot+, a computational pipeline that computes architectural traits from 3D X-ray CT volumes of excavated maize root crowns\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.21203/RS.3.RS-3463338/V1',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3196',\n",
       "      'term': 'Genotyping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3436',\n",
       "      'term': 'Aggregation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3452',\n",
       "    'term': 'Tomography'},\n",
       "   {'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'}],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3463338/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__toporoot',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__toporoot',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3463338/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'bsAbs',\n",
       "  'description': 'Computational discovery of co-expressed antigens as dual targeting candidates for cancer therapy through bulk, single-cell, and spatial transcriptomics.\\n\\nMotivation Bispecific antibodies (bsAbs) that bind to two distinct surface antigens on cancer cells are emerging as an appealing therapeutic strategy in cancer immunotherapy. However, considering the vast number of surface proteins, experimental identification of potential antigen pairs that are selectively expressed in cancer cells and not in normal cells is both costly and time-consuming. Recent studies have utilized large bulk RNA-seq databases to propose bispecific targets for various cancers. But, co-expressed pairs derived from bulk RNA-seq do not necessarily indicate true co-expression of both markers in the same cell. Single-cell RNA-seq (scRNA-seq) can circumvent this issue but the issues in dropouts and low-coverage of transcripts impede the large-scale characterization of co-expressed pairs. Results',\n",
       "  'homepage': 'https://github.com/Lionir/bsAbsFinder',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0480',\n",
       "      'term': 'Side chain modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3463',\n",
       "      'term': 'Expression correlation analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2830',\n",
       "    'term': 'Immunoproteins and antigens'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.17.562711'}],\n",
       "  'credit': [{'name': 'Evgenii Chekalin',\n",
       "    'email': 'eygen.chekalin@gmail.com',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Bin Chen', 'email': 'chenbi12@msu.edu', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__bsabs',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__bsabs',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.17.562711',\n",
       "  'is_preprint': True},\n",
       " {'name': 'KmerSV',\n",
       "  'description': 'a visualization and annotation tool for structural variants using Human Pangenome derived k-mers.\\n\\nSummary KmerSV is a visualization and annotation tool for structural variants ( SVs ). It can be applied to assembly contigs or long-read sequences. Using k-mers it rapidly generates images and provides genome features of SVs. As an important feature, it utilizes the new Human Pangenome reference which provide haploid specific assemblies, addresses limitations in prior references and improves the discovery of SVs. Availability and implementation. KmerSV is implemented in Python and available at github.com sgtc-stanford kmerSV',\n",
       "  'homepage': 'http://github.com/sgtc-stanford/kmerSV',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_2429', 'term': 'Mapping'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0196',\n",
       "    'term': 'Sequence assembly'},\n",
       "   {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.11.561941'}],\n",
       "  'credit': [{'name': 'Hanlee P. Ji',\n",
       "    'email': 'genomics_ji@stanford.edu',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'HoJoon Lee',\n",
       "    'email': 'hojoon@stanford.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__kmersv',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kmersv',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.11.561941',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AdaptiX',\n",
       "  'description': 'A Transitional XR Framework for Development and Evaluation of Shared Control Applications in Assistive Robotics.\\n\\nWith the ongoing efforts to empower people with mobility impairments and the increase in technological acceptance by the general public, assistive technologies, such as collaborative robotic arms, are gaining popularity. Yet, their widespread success is limited by usability issues, specifically the disparity between user input and software control along the autonomy continuum. To address this, shared control concepts provide opportunities to combine the targeted increase of user autonomy with a certain level of computer assistance. This paper presents the free and open-source AdaptiX XR framework for developing and evaluating shared control applications in a high-resolution simulation environment',\n",
       "  'homepage': 'https://adaptix.robot-research.de',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3454',\n",
       "      'term': 'Phasing'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3318', 'term': 'Physics'}],\n",
       "  'language': ['Java', 'C++'],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3418756/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__adaptix',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__adaptix',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3418756/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'FedGMMAT',\n",
       "  'description': 'Federated Generalized Linear Mixed Model Association Tests.\\n\\nIncreasing genetic and phenotypic data size is critical for understanding the genetic determinants of diseases. Evidently, establishing practical means for collaboration and data sharing among institutions is a fundamental methodological barrier for performing high-powered studies. As the sample sizes become more heterogeneous, complex statistical approaches, such as generalized linear mixed effects models, must be used to correct for confounders that may bias results. On another front, due to the privacy concerns around Protected Health Information (PHI), genetic information is restrictively protected by sharing according to regulations such as Health Insurance Portability and Accountability Act (HIPAA). This limits data sharing among institutions and hampers efforts around executing high-powered collaborative studies',\n",
       "  'homepage': 'https://github.com/Li-Wentao/FedGMMAT',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3920', 'term': 'DNA testing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0625',\n",
       "    'term': 'Genotype and phenotype'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.560753'}],\n",
       "  'credit': [{'name': 'Arif Harmanci',\n",
       "    'email': 'arif.o.harmanci@uth.tmc.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__fedgmmat',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fedgmmat',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.560753',\n",
       "  'is_preprint': True},\n",
       " {'name': 'scDiff',\n",
       "  'description': 'A General Single-Cell Analysis Framework via Conditional Diffusion Generative Models.\\n\\nA bstract. The fast-growing single-cell analysis community extends the horizon of quantitative analysis to numerous computational tasks. While the tasks hold vastly different targets from each other, existing works typically design specific model frameworks according to the downstream objectives. In this work, we propose a general single-cell analysis framework by unifying common computational tasks as posterior estimation problems. In light of conditional diffusion generative models, we introduce scDiff through the proposed framework and study different conditioning strategies. With data-specific conditions, scDiff achieves competitive performance against state-of-the-art in various benchmarking tasks. In addition, we illustrate the flexibility of scDiff by incorporating prior information through large language models and graph neural networks',\n",
       "  'homepage': 'https://github.com/OmicsML/scDiff',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3678',\n",
       "    'term': 'Experimental design and studies'},\n",
       "   {'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.562243'}],\n",
       "  'credit': [{'name': 'Wenzhuo Tang',\n",
       "    'email': 'tangwen2@msu.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__scdiff',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__scdiff',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.562243',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PhysiMeSS',\n",
       "  'description': 'A New PhysiCell Addon for Extracellular Matrix Modelling.\\n\\nThe extracellular matrix is a complex assembly of macro-molecules, such as collagen fibres, which provides structural support for surrounding cells. In the context of cancer metastasis, it represents a barrier for the cells, that the migrating cells needs to degrade in order to leave the primary tumor and invade further tissues. Agent-based frameworks, such as PhysiCell, are often use to represent the spatial dynamics of tumor evolution. However, typically they only implement cells as agents, which are represented by either a circle (2D) or a sphere (3D). In order to accurately represent the extracellular matrix as a network of fibres, we require a new type of agent represented by a segment (2D) or a cylinder (3D). In this article, we present PhysiMeSS, an addon of PhysiCell, which introduces a new type of agent to describe fibres, and their physical interactions with cells and other fibres',\n",
       "  'homepage': 'https://github.com/PhysiMeSS/PhysiMeSS',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_3431',\n",
       "      'term': 'Deposition'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3299',\n",
       "    'term': 'Evolutionary biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.27.564365'}],\n",
       "  'credit': [{'name': 'Vincent Noël',\n",
       "    'email': 'vincent.noel@curie.fr',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Cicely K. Macnamara',\n",
       "    'email': 'Cicely.Macnamara@glasgow.ac.uk',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__physimess',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__physimess',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.27.564365',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ORACLE',\n",
       "  'description': 'Accurate identification of cancer cells in complex pre-clinical models using deep-learning.\\n\\n3D co-cultures are key tools for in vitro biomedical research as they recapitulate more closely the in vivo environment, while allowing control of the density and type of cells included in the analysis, as well as the experimental conditions in which they are maintained. More widespread application of these models is hampered however by the limited technologies available for their analysis. The separation of the contribution of the different cell types, in particular, is a fundamental challenge. In this work, we present ORACLE, a deep neural network trained to distinguish between ovarian cancer and healthy cells based on the shape of their nucleus. The extensive validation that we have conducted includes multiple cell lines and patient derived cultures to characterise the effect of all the major potential confounding factors',\n",
       "  'homepage': 'https://github.com/MarilisaCortesi/ORACLE/tree/main',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3446',\n",
       "      'term': 'Cell migration analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3638', 'term': 'SILAC'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3340',\n",
       "    'term': 'Cell culture collection'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.15.562411'}],\n",
       "  'credit': [{'name': 'Marilisa Cortesi',\n",
       "    'email': 'm.cortesi@unsw.edu.au',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Caroline E. Ford',\n",
       "    'email': 'caroline.ford@unsw.edu.au',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__oracle',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__oracle',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.15.562411',\n",
       "  'is_preprint': True},\n",
       " {'name': 'ARPALData',\n",
       "  'description': 'an R package for retrieving and analyzing air quality and weather data from ARPA Lombardia (Italy).\\n\\nWe present ARPALData, an R package able to help international users in retrieving, handling, and analyzing data for air quality and weather in Lombardy (Northern Italy). The software provides a user-friendly tool that does not require access to institutional platforms and ensures real-time updating of information using a standardized syntax. Also, it provides data according to the standard statistical formats. Eventually, all measurements, metadata and the subsequent analytical tools are provided to the users in the English language, facilitating accessibility to international and domestic users. Data are collected from the open database of the Regional Agency for Environmental Protection for Lombardy, namely ARPA Lombardia\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.21203/RS.3.RS-3380288/V1',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'},\n",
       "     {'uri': 'http://edamontology.org/operation_3435',\n",
       "      'term': 'Standardisation and normalisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3436',\n",
       "      'term': 'Aggregation'}]}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3380288/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__arpaldata',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__arpaldata',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3380288/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'MIC-SQTL',\n",
       "  'description': 'Multimodal joint deconvolution and integrative signature selection in proteomics.\\n\\nDeconvolution is an efficient approach for detecting cell-type-specific (cs) transcriptomic signals without cellular segmentation. However, this type of methods have not been extended to the proteomics research. Here we present a novel algorithm and tool to dissect bulk proteome by leveraging the information shared between transcriptome-proteome. Our tool first identifies potential cell marker proteins by integrating RNA and protein bulk expression profiles and then jointly quantifies the cell abundance in mixture proteomes without using a reference signature matrix, enabling the downstream analyses such as cs-protein Quantitative Trait Loci (cspQTL) mapping. This new method and the cspQTL analysis are implemented in the R package MIC-SQTL that also provides integrative visualization of bulk multimodal samples, available at https: bioconductor.org packages MICSQTL',\n",
       "  'homepage': 'https://bioconductor.org/packages/MICSQTL',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3629',\n",
       "      'term': 'Deisotoping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_0314',\n",
       "      'term': 'Gene expression profiling'},\n",
       "     {'uri': 'http://edamontology.org/operation_0282',\n",
       "      'term': 'Genetic mapping'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0121',\n",
       "    'term': 'Proteomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0080', 'term': 'Sequence analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3520',\n",
       "    'term': 'Proteomics experiment'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.04.560979'}],\n",
       "  'credit': [{'name': 'Qian Li',\n",
       "    'email': 'qian.li@stjude.org',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__mic-sqtl',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__mic-sqtl',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.04.560979',\n",
       "  'is_preprint': True},\n",
       " {'name': 'gtfsort',\n",
       "  'description': 'a tool to efficiently sort GTF files.\\n\\nThe General Transfer Format (GTF) is a widely used format for gene annotation data, integral to various downstream analyses. Efficient management and sorting of GTF data are relevant, as unsorted data can intere with computational efficiency and interpretability. Sorted GTF data, on the other hand, enable more intuitive navigation of genomic structures and can enhance the performance of some bioinformatics tools. This paper presents gtfsort, a tool written in Rust, which efficiently sorts GTF files, outperforming existing software. By reducing computation times and providing structured outputs, gtfsort enhances the interpretation, relationships, and readability of gene annotation data',\n",
       "  'homepage': 'http://github.com/alejandrogzi/gtfsort',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3802',\n",
       "      'term': 'Sorting'},\n",
       "     {'uri': 'http://edamontology.org/operation_2422',\n",
       "      'term': 'Data retrieval'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0091',\n",
       "    'term': 'Bioinformatics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0622', 'term': 'Genomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'},\n",
       "   {'uri': 'http://edamontology.org/topic_0203', 'term': 'Gene expression'}],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.21.563454'}],\n",
       "  'credit': [{'name': 'Alejandro Gonzales-Irribarren',\n",
       "    'email': 'jose.gonzalesdezavala1@unmsm.edu.pe',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__gtfsort',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__gtfsort',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.21.563454',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AID-HS',\n",
       "  'description': 'Automated and Interpretable Detection of Hippocampal Sclerosis in temporal lobe epilepsy.\\n\\nHippocampal Sclerosis (HS) can elude visual detection on MRI scans of patients with temporal lobe epilepsy (TLE), causing delays in surgical treatment and reducing the likelihood of postsurgical seizure-freedom. We developed an open-source software that (1) detects HS from structural MRI scans, (2) generalises across a heterogeneous multicentre cohort of children and adults, and (3) generates individualised predictions for clinical evaluation. We included a cohort of 363 participants (152 patients with HS, 90 disease controls with focal cortical dysplasia, and 121 healthy controls) from four epilepsy centres in the UK, North America, and China. We used the open-source software HippUnfold to extract morphological surface-based features and volumes of the hippocampus from T1w MRI scans',\n",
       "  'homepage': 'https://github.com/mathrip/AID-HS',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3659',\n",
       "      'term': 'Regression analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3444', 'term': 'MRI'},\n",
       "   {'uri': 'http://edamontology.org/topic_3418', 'term': 'Paediatrics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3421', 'term': 'Surgery'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.13.23296991'}],\n",
       "  'credit': [{'name': 'Mathilde Ripart',\n",
       "    'email': 'mathilde.ripart@gmail.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__aid-hs',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__aid-hs',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.13.23296991',\n",
       "  'is_preprint': True},\n",
       " {'name': 'DiabetIA',\n",
       "  'description': 'Building Machine Learning Models for Type 2 Diabetes Complications.\\n\\nBackground Artificial intelligence (AI) models applied to diabetes mellitus research have grown in recent years, particularly in the field of medical imaging. However little work has been done exploring real-world data (RWD) sources such as electronic health records (EHR) mostly due to the lack of reliable public diabetes databases. However, with more than 500 million patients affected worldwide, complications of this condition have catastrophic consequences. In this manuscript we aim to first extract, clean and transform a novel diabetes research database, DiabetIA, and secondly train machine learning (ML) models to predict diabetic complications. Methods. In this study, we used observational retrospective data from the Mexican Institute for Social Security (IMSS) extracting and de-identifying EHR data for almost 2 million patients seen at primary care facilities',\n",
       "  'homepage': 'https://github.com/AmphoraHealth/diabetia',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3431',\n",
       "      'term': 'Deposition'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_3384', 'term': 'Medical imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_3263', 'term': 'Data security'},\n",
       "   {'uri': 'http://edamontology.org/topic_3063',\n",
       "    'term': 'Medical informatics'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'Apache-2.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.22.23297277'}],\n",
       "  'credit': [{'name': 'Anel Gomez-Garcia',\n",
       "    'email': 'anel.gomez@imss.gob.mx',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Cleto Alvarez-Aguilar',\n",
       "    'email': 'cleto.alvarez@umich.mx',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Arturo Lopez-Pineda',\n",
       "    'email': 'arturo@amphora.health',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__diabetia',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__diabetia',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.22.23297277',\n",
       "  'is_preprint': True},\n",
       " {'name': 'BFF',\n",
       "  'description': 'Bayesian Fluorescence Framework for integrative modeling of biomolecules.\\n\\nBayesian fluorescence framework for integrative modeling — IMP.bff documentation.\\n\\nGetting Started Release Highlights for 0.12 GitHub.\\n\\nFluorescence-based structural modeling.\\n\\nJuly 2022. IMP.bff 0.12.0 is available for download (Changelog).',\n",
       "  'homepage': 'https://docs.peulen.xyz/bff',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2476',\n",
       "      'term': 'Molecular dynamics'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_3664',\n",
       "      'term': 'Statistical modelling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0593', 'term': 'NMR'},\n",
       "   {'uri': 'http://edamontology.org/topic_0176', 'term': 'Molecular dynamics'},\n",
       "   {'uri': 'http://edamontology.org/topic_2275',\n",
       "    'term': 'Molecular modelling'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.26.564048'}],\n",
       "  'credit': [{'name': 'Thomas-Otavio Peulen',\n",
       "    'email': 'thomas-otavio.peulen@uni-wuerzburg.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__bff',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__bff',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.26.564048',\n",
       "  'is_preprint': True},\n",
       " {'name': 'TOLGAN',\n",
       "  'description': 'An End-to-end framework for producing Traditional Orient Landscape.\\n\\nWe present TOLGAN that generates traditional oriental landscape (TOL) image from a map that specifies the locations and shapes of the elements composing TOL. Users can create a TOL map by using a user interface or a segmentation scheme from a photograph. We design the generator of TOLGAN as a series of decoding layers where the map is applied between the layers. The generated TOL image is further enhanced through an AdaIN architecture. The discriminator of TOLGAN processes a generated image and its groundtruth TOL artwork image. TOLGAN is trained through a dataset composed of paired TOL artwork images and their TOL maps. We present a tool through which users can produce a TOL map by specifying and organizing the elements of TOL artworks. TOLGAN successfully generates a series of TOL images from the TOL map\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.20944/PREPRINTS202310.1290.V1',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3443',\n",
       "      'term': 'Image analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_2429', 'term': 'Mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3382', 'term': 'Imaging'},\n",
       "   {'uri': 'http://edamontology.org/topic_0621', 'term': 'Model organisms'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'}],\n",
       "  'publication': [{'doi': '10.20944/PREPRINTS202310.1290.V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__tolgan',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__tolgan',\n",
       "  'publication_link': 'https://doi.org/10.20944/PREPRINTS202310.1290.V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'KIET Annual Report',\n",
       "  'description': '산업연구원은 국내외의 산업·기술과 관련된 실물경제의 동향과 정보를 신속히 수집 조사하고 연구.\\n\\n· 한국경제, 반도체 산업 부진과 지정학적 리스크에 더해 고금리 지속에 따른 소비 부진, 보호주의 강화 등 동시다발적 위기에 직면.\\n\\n· 최근 경제 리스크 외에 지정학·환경·사회·기술 리스크 등의 영향 관심 고조.\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE KIET',\n",
       "  'homepage': 'http://www.kiet.re.kr',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3179',\n",
       "    'term': 'ChIP-on-chip'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'}],\n",
       "  'publication': [{'doi': '10.2139/SSRN.4613215'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__kiet-annual-report',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kiet-annual-report',\n",
       "  'publication_link': 'https://doi.org/10.2139/SSRN.4613215',\n",
       "  'is_preprint': True},\n",
       " {'name': 'SaProt',\n",
       "  'description': 'Protein Language Modeling with Structure-aware Vocabulary.\\n\\nABSTRACT Large-scale protein language models (PLMs), such as the ESM family, have achieved remarkable performance in various downstream tasks related to protein structure and function by undergoing unsupervised training on residue sequences. They have become essential tools for researchers and practitioners in biology. However, a limitation of vanilla PLMs is their lack of explicit consideration for protein structure information, which suggests the potential for further improvement. Motivated by this, we introduce the concept of a “structure-aware vocabulary” that integrates residue tokens with structure tokens. The structure tokens are derived by encoding the 3D structure of proteins using Foldseek. We then propose SaProt, a large-scale general-purpose PLM trained on an extensive dataset comprising approximately 40 million protein sequences and structures',\n",
       "  'homepage': 'https://github.com/westlake-repl/SaProt',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0477',\n",
       "      'term': 'Protein modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_2492',\n",
       "      'term': 'Protein interaction prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_2489',\n",
       "      'term': 'Subcellular localisation prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0272',\n",
       "      'term': 'Residue contact prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2814',\n",
       "    'term': 'Protein structure analysis'},\n",
       "   {'uri': 'http://edamontology.org/topic_0623',\n",
       "    'term': 'Gene and protein families'},\n",
       "   {'uri': 'http://edamontology.org/topic_0108', 'term': 'Protein expression'},\n",
       "   {'uri': 'http://edamontology.org/topic_0082',\n",
       "    'term': 'Structure prediction'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.01.560349'}],\n",
       "  'credit': [{'name': 'Fajie Yuan',\n",
       "    'email': 'yuanfajie@westlake.edu.cn',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__saprot',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__saprot',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.01.560349',\n",
       "  'is_preprint': True},\n",
       " {'name': 'VRP assembler',\n",
       "  'description': 'haplotype-resolved<i>de novo</i>assembly of diploid and polyploid genomes using quantum computing.\\n\\nPrecision medicine’s emphasis on individual genetic variants highlights the importance of haplotype-resolved assembly, a computational challenge in bioinformatics given its combinatorial nature. While classical algorithms have made strides in addressing this issue, the potential of quantum computing remains largely untapped. Here, we present the VRP assembler: a novel approach that transforms this task into a vehicle routing problem, a combinatorial optimization problem solvable on a quantum computer. We demonstrate its potential and feasibility through a proof of concept on short synthetic diploid and triploid genomes using a D-Wave quantum annealer. To tackle larger-scale assembly problems, we integrate the VRP assembler with Google’s OR-Tools, achieving a haplotype-resolved assembly across the human major histocompatibility complex (MHC) region',\n",
       "  'homepage': 'https://github.com/yiboch/vrp_assembler/tree/main/data',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_0487',\n",
       "      'term': 'Haplotype mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_3454', 'term': 'Phasing'},\n",
       "     {'uri': 'http://edamontology.org/operation_3184',\n",
       "      'term': 'Sequence assembly visualisation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0196',\n",
       "    'term': 'Sequence assembly'},\n",
       "   {'uri': 'http://edamontology.org/topic_3577',\n",
       "    'term': 'Personalised medicine'},\n",
       "   {'uri': 'http://edamontology.org/topic_3168', 'term': 'Sequencing'}],\n",
       "  'download': [{'url': 'https://github.com/yiboch/vrp_assembler/tree/main/scripts',\n",
       "    'type': 'Source code'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.19.563028'}],\n",
       "  'credit': [{'name': 'Yong Zhang',\n",
       "    'email': 'zhangyong2@genomics.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Yuxiang Li',\n",
       "    'email': 'liyuxiang@genomics.cn',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Xun Xu', 'email': 'xuxun@genomics.cn', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__vrp-assembler',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__vrp-assembler',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.19.563028',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Kairos',\n",
       "  'description': 'Kairos infers<i>in situ</i>horizontal gene transfer in longitudinally sampled microbiomes through microdiversity-aware sequence analysis.\\n\\nHorizontal gene transfer (HGT) occurring within microbiomes is linked to complex environmental and ecological dynamics that are challenging to replicate in controlled settings. Consequently, most extant studies of microbiome HGT are either simplistic experimental settings with tenuous relevance to real microbiomes or correlative studies that assume that HGT potential is a function of the relative abundance of mobile genetic elements (MGEs), the vehicles of HGT. Here we introduce Kairos as a bioinformatic tool deployed in nextflow for detecting HGT events “ in situ, ” i.e., within a microbiome, through analysis of time-series metagenomic sequencing data',\n",
       "  'homepage': 'https://github.com/clb21565/kairos',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'},\n",
       "     {'uri': 'http://edamontology.org/operation_3482',\n",
       "      'term': 'Antimicrobial resistance prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_3198',\n",
       "      'term': 'Read mapping'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3697',\n",
       "    'term': 'Microbial ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3174', 'term': 'Metagenomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3837',\n",
       "    'term': 'Metagenomic sequencing'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'},\n",
       "   {'uri': 'http://edamontology.org/topic_3673',\n",
       "    'term': 'Whole genome sequencing'}],\n",
       "  'language': ['Python', 'Shell'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.24.563791'}],\n",
       "  'credit': [{'name': 'Liqing Zhang',\n",
       "    'email': 'lqzhang@cs.vt.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__kairos',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__kairos',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.24.563791',\n",
       "  'is_preprint': True},\n",
       " {'name': 'VAE',\n",
       "  'description': 'Representational Learning from Healthy Multi-Tissue Human RNA-seq Data such that Latent Space Arithmetics Extracts Disease Modules.\\n\\n1 Developing computational analyses of transcriptomic data has dramatically improved our understanding of complex multifactorial diseases. However, such approaches are limited to small sample sets of disease-affected material, thus being sensitive to statistical biases and noise. Here, we ask if a variational autoencoder (VAE) trained on large groups of healthy, human RNA-seq data of multiple tissues can capture the fundamental healthy gene regulation system such that the learned representation generalizes to account for unseen disease changes. To this end, we trained a multi-scale representation to encode cellular processes ranging from cell types to genegene interactions. Importantly, we found that the learned healthy representations could predict unseen gene expression changes from 25 independent disease datasets',\n",
       "  'homepage': 'https://github.com/ddeweerd/VAE_Transcriptomics/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3223',\n",
       "      'term': 'Differential gene expression profiling'},\n",
       "     {'uri': 'http://edamontology.org/operation_2436',\n",
       "      'term': 'Gene-set enrichment analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_3463',\n",
       "      'term': 'Expression correlation analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0154', 'term': 'Small molecules'}],\n",
       "  'language': ['Python', 'R'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.560661'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__vae',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__vae',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.560661',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Int and in',\n",
       "  'description': 'a machine learning-based web server for split site identification in inteins.\\n\\nInteins are proteins that excise themselves out of host proteins and ligate the flanking polypeptides in an auto-catalytic process called protein splicing. They are gaining momentum in synthetic biology for their ability to post-translationally modify proteins of interest. In nature, inteins are either contiguous or split, in which case the two intein fragments must first form a complex for the splicing to occur. So far, heuristic methods have been employed whenever a new split site in an intein had to be identified. To make the process of split site identification in inteins faster, easier and less costly, we developed Int&in, a web server that uses a gaussian Naïve Bayes machine learning model to predict active and inactive split sites with high accuracy\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.1101/2023.09.27.559783',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3359',\n",
       "      'term': 'Splitting'},\n",
       "     {'uri': 'http://edamontology.org/operation_3896',\n",
       "      'term': 'Active site prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0433',\n",
       "      'term': 'Splice site prediction'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3895',\n",
       "    'term': 'Synthetic biology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0078', 'term': 'Proteins'}],\n",
       "  'publication': [{'doi': '10.1101/2023.09.27.559783'}],\n",
       "  'credit': [{'name': 'Barbara Di Ventura',\n",
       "    'email': 'Barbara.diventura@bio.uni-freiburg.de',\n",
       "    'typeEntity': 'Person'},\n",
       "   {'name': 'Mehmet Ali Öztürk',\n",
       "    'email': 'mehmet.oeztuerk@bioss.uni-freiburg.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__int-and-in',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__int-and-in',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.27.559783',\n",
       "  'is_preprint': True},\n",
       " {'name': 'pyTME',\n",
       "  'description': 'A fast, flexible, and multi-purpose template matching library for cryogenic electron microscopy data.\\n\\nCryogenic electron microscopy (cryo-EM) is a key method in structural and cell biology. Analysis of cryo-EM images requires interpretation of noisy, low-resolution densities which relies on identifying the most probable orientation of macromolecules in a target using template matching. Many method-specific template matching software exist for single-particle cryo-EM, cryo-electron tomography (cryo-ET), or fitting atomic structures into averaged 3D maps of macromolecules. Here, we report the Python Template Matching Engine (pyTME), a software engine that consolidates method-specific template matching problems. The underlying library provides highly efficient template-matching implementation and abstract data structures for storing and manipulating input and output data\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE Python Template Matching Engine',\n",
       "  'homepage': 'https://github.com/KosinskiLab/pyTME',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3452',\n",
       "    'term': 'Tomography'},\n",
       "   {'uri': 'http://edamontology.org/topic_0611',\n",
       "    'term': 'Electron microscopy'},\n",
       "   {'uri': 'http://edamontology.org/topic_2229', 'term': 'Cell biology'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'https://github.com/KosinskiLab/pytme_publication',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.23.563472'}],\n",
       "  'credit': [{'name': 'Jan Kosinski',\n",
       "    'email': 'jan.kosinski@embl.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__pytme',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__pytme',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.23.563472',\n",
       "  'is_preprint': True},\n",
       " {'name': 'EBP',\n",
       "  'description': 'Assessment of Evidence-Based Practice (EBP) among physiotherapists in Cameroon.\\n\\nBackground: Evidence-Based Practice (EBP) is reported to ease effective and adequate decision making for all works of life including health professionals. Investigating the level of implementation of EBP among physiotherapists helps to identify barriers and propose solutions for its extensive implementation. Despite available data on EBP elsewhere, it remains limited or non-existent in Cameroon. We aimed by this study to establish the current state of evidence-based practice among physiotherapists in Cameroon, by assessing knowledge, practice, and as well as barriers to its implementation. Methods A cross-sectional survey was conducted online among French- and English-speaking physiotherapists in Cameroon over a period of six months, between April and July 2023, using the EBP 2 questionnaire\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.21203/RS.3.RS-3425464/V1',\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0089',\n",
       "    'term': 'Ontology and terminology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_3071',\n",
       "    'term': 'Biological databases'}],\n",
       "  'publication': [{'doi': '10.21203/RS.3.RS-3425464/V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__ebp',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__ebp',\n",
       "  'publication_link': 'https://doi.org/10.21203/RS.3.RS-3425464/V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'AngoraPy',\n",
       "  'description': 'A Python Toolkit for Modelling Anthropomorphic Goal-Driven Sensorimotor Systems.\\n\\nA bstract. Goal-driven deep learning is increasingly used to supplement classical modeling approaches in computational neuroscience. The strength of deep neural networks lies in their ability to autonomously learn the connectivity required to solve complex and ecologically valid tasks, obviating the need for hand-engineered or hypothesis-driven connectivity patterns. Consequently, goal-driven models can generate hypotheses about the neurocomputations underlying cortical processing. Whereas goal-driven modeling is becoming increasingly common in perception neuroscience, its application to sensorimotor control is currently hampered by the complexity of the methods required to train models comprising the closed sensation-action loop',\n",
       "  'homepage': 'https://github.com/ccnmaastricht/angorapy',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0481',\n",
       "      'term': 'Loop modelling'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3304',\n",
       "    'term': 'Neurobiology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0610', 'term': 'Ecology'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'GPL-3.0',\n",
       "  'publication': [{'doi': '10.1101/2023.10.05.560998'}],\n",
       "  'credit': [{'name': 'Tonio Weidler',\n",
       "    'email': 'research@tonioweidler.de',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__angorapy',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__angorapy',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.05.560998',\n",
       "  'is_preprint': True},\n",
       " {'name': 'CytoPipeline',\n",
       "  'description': 'An R package suite for building and visualizing automated pre-processing pipelines for flow cytometry data.\\n\\nWith the increase of the dimensionality in conventional flow cytometry data over the past years, there is a growing need to replace or complement traditional manual analysis (i.e. iterative 2D gating) with automated data analysis pipelines. A crucial part of these pipelines consists of pre-processing and applying quality control filtering to the raw data, in order to use high quality events in the downstream analyses. This part can in turn be split into a number of elementary steps : margin events removal, signal compensation, scale transformations, debris and dead cells removal, batch effect correction, etc. However, when designing automated flow cytometry data analysis pipelines, assembling and assessing the pre-processing part can be challenging for a number of reasons. First, each of the involved elementary steps can be implemented using various methods and R packages',\n",
       "  'homepage': 'https://github.com/UCLouvain-CBIO/2023-CytoPipeline-code',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3359',\n",
       "      'term': 'Splitting'},\n",
       "     {'uri': 'http://edamontology.org/operation_0337',\n",
       "      'term': 'Visualisation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0310',\n",
       "      'term': 'Sequence assembly'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3934', 'term': 'Cytometry'},\n",
       "   {'uri': 'http://edamontology.org/topic_0769', 'term': 'Workflows'},\n",
       "   {'uri': 'http://edamontology.org/topic_0196', 'term': 'Sequence assembly'}],\n",
       "  'language': ['R'],\n",
       "  'publication': [{'doi': '10.1101/2023.10.10.561699'}],\n",
       "  'credit': [{'name': 'Laurent Gatto',\n",
       "    'email': 'laurent.gatto@uclouvain.be',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__cytopipeline',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__cytopipeline',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.10.561699',\n",
       "  'is_preprint': True},\n",
       " {'name': 'Nanotope',\n",
       "  'description': 'A Graph Neural Network-Based Model for Nanobody paratope prediction.\\n\\nNanobodies are artificial antibodies derived from the immune system of camelid species, obtained through artificial processing and isolation of antigen-binding proteins. Their small molecular size and high specificity endow nanobodies with extensive potential applications in various fields. However, determining nanobody paratopes through experimental methods is both costly and time-consuming, while traditional computational approaches often lack sufficient accuracy. To enhance the prediction accuracy, we propose Nanotope, a structurebased model capable of efficiently and accurately predicting paratopes, utilizing graph neural networks and an antibody pretrained language model. Our approach primarily leverages sequence features obtained from the AntiBERTy antibody pretrained language model and three-dimensional spatial structure features of nanobodies as inputs',\n",
       "  'homepage': 'https://github.com/WangLabforComputationalBiology/Nanotope',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0416',\n",
       "      'term': 'Epitope mapping'},\n",
       "     {'uri': 'http://edamontology.org/operation_0252',\n",
       "      'term': 'Peptide immunogenicity prediction'},\n",
       "     {'uri': 'http://edamontology.org/operation_0480',\n",
       "      'term': 'Side chain modelling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2830',\n",
       "    'term': 'Immunoproteins and antigens'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'}],\n",
       "  'language': ['Python'],\n",
       "  'publication': [{'doi': '10.1101/2023.09.27.559671'}],\n",
       "  'credit': [{'email': 'wangxin@sztu.edu.cn', 'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__nanotope',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__nanotope',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.09.27.559671',\n",
       "  'is_preprint': True},\n",
       " {'name': 'FedRDR',\n",
       "  'description': 'Federated Reinforcement Distillation-Based Routing Algorithm in UAV-Assisted Resilient Networks for Communication Infrastructure Failures.\\n\\nCommunication infrastructures may be partially or completely destroyed due to the natural disasters, especially the disruption of a few critical network devices, leading to the collapse of communication services. To address this issue, we constructed a layered, multi-domain data transmission architecture based on a UAV-assisted resilient network. Initially, the UAV controllers perceived the network status and learning the temporal- spatial characteristics of air-to-ground networks links. Subsequently, we developed a multi-domain routing algorithm based on federated reinforcement distillation, called FedRDR, enhancing the generalization capabilities of routing decision model through the augmentation of training data samples\\n\\n||| HOMEPAGE MISSING!',\n",
       "  'homepage': 'https://doi.org/10.20944/PREPRINTS202310.1118.V1',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3474',\n",
       "    'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_3524',\n",
       "    'term': 'Simulation experiment'}],\n",
       "  'publication': [{'doi': '10.20944/PREPRINTS202310.1118.V1'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__fedrdr',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__fedrdr',\n",
       "  'publication_link': 'https://doi.org/10.20944/PREPRINTS202310.1118.V1',\n",
       "  'is_preprint': True},\n",
       " {'name': 'HSQC2STRUC',\n",
       "  'description': 'A Machine Learning Model for Protein Secondary Structure Prediction using Unassigned NMR Spectra.\\n\\nDynamic changes in the secondary structure content of proteins can provide valuable insights into protein function or dysfunction. Predicting these dynamic changes is still a significant challenge but is of paramount importance for basic research as well as drug development. Here, we present a machine learning-based model that predicts the secondary structure content of proteins based on their un assigned 1 H, 15 N-HSQC NMR spectra with an RMSE of 0.11 for α -helix, 0.08 for β -sheet and 0.12 for random coil content. Our model has been implemented into an easy-to-use and publicly available web service that estimates secondary structure content based on a provided peak list. Furthermore, a Python version is provided, ready to be integrated into Bruker’s TopSpin software or own scripts',\n",
       "  'homepage': 'https://github.com/bellstedt-lab/hsqc2struc',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0470',\n",
       "      'term': 'Protein secondary structure prediction (coils)'},\n",
       "     {'uri': 'http://edamontology.org/operation_2488',\n",
       "      'term': 'Protein secondary structure comparison'},\n",
       "     {'uri': 'http://edamontology.org/operation_0468',\n",
       "      'term': 'Protein secondary structure prediction (helices)'},\n",
       "     {'uri': 'http://edamontology.org/operation_0319',\n",
       "      'term': 'Protein secondary structure assignment'},\n",
       "     {'uri': 'http://edamontology.org/operation_0303',\n",
       "      'term': 'Fold recognition'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0593', 'term': 'NMR'},\n",
       "   {'uri': 'http://edamontology.org/topic_3542',\n",
       "    'term': 'Protein secondary structure'},\n",
       "   {'uri': 'http://edamontology.org/topic_0082',\n",
       "    'term': 'Structure prediction'},\n",
       "   {'uri': 'http://edamontology.org/topic_3474', 'term': 'Machine learning'},\n",
       "   {'uri': 'http://edamontology.org/topic_0736',\n",
       "    'term': 'Protein folds and structural domains'}],\n",
       "  'language': ['Python', 'Shell'],\n",
       "  'license': 'MIT',\n",
       "  'publication': [{'doi': '10.1101/2023.10.09.561482'}],\n",
       "  'credit': [{'name': 'Peter Bellstedt',\n",
       "    'email': 'peter.bellstedt@usz.ch',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__hsqc2struc',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__hsqc2struc',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.09.561482',\n",
       "  'is_preprint': True},\n",
       " {'name': 'StrastiveVI',\n",
       "  'description': 'Isolating structured salient variations in single-cell transcriptomic data with StrastiveVI.\\n\\nSingle-cell RNA sequencing (scRNA-seq) has provided deeper insights into biological processes by highlighting differences at the cellular level. Within these single-cell omics measurements, researchers are often interested in identifying variations associated with a specific covariate. For instance, in aging research, it becomes vital to differentiate variations related to aging. To address this, we introduce StrastiveVI ( Str uctured Contr astive V ariational Inference; https: github.com suinleelab StrastiveVI ), which effectively separates the variations of interest from other dominant biological signals in scRNA-seq datasets. When deployed on aging and Alzheimer’s disease (AD) datasets, StrastiveVI efficiently isolates aging and AD-associated patterns, distinguishing them from dominant variations linked to sex, tissue, and cell type that are unrelated to aging or AD',\n",
       "  'homepage': 'https://github.com/suinleelab/StrastiveVI',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_3175',\n",
       "    'term': 'Structural variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3308', 'term': 'Transcriptomics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}],\n",
       "  'language': ['Python'],\n",
       "  'license': 'BSD-3-Clause',\n",
       "  'publication': [{'doi': '10.1101/2023.10.06.561320'}],\n",
       "  'credit': [{'name': 'Su-In Lee',\n",
       "    'email': 'suinlee@cs.washington.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__strastivevi',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__strastivevi',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.06.561320',\n",
       "  'is_preprint': True},\n",
       " {'name': 'PhyloAcc-C',\n",
       "  'description': 'A phylogenetic method linking nucleotide substitution rates to rates of continuous trait evolution.\\n\\nPhyloAcc a software to detect the changes of conservation of a genomic region.\\n\\nBayesian estimation of lineage specific substitution rates in conserved non-coding regions while accounting for phylogenetic discordance.\\n\\nPhyloAcc-test-data PhyloAcc-test-data Public.\\n\\nFor those with the most up to date version of Anaconda installed, you can install PhyloAcc with bioconda in a fresh environment with a single command:.\\n\\nHu-etal-2019-data Hu-etal-2019-data Public.\\n\\nFor more help or troubleshooting installation, please see the Install page.',\n",
       "  'homepage': 'https://github.com/phyloacc/PhyloAcc-C',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_2238',\n",
       "      'term': 'Statistical calculation'},\n",
       "     {'uri': 'http://edamontology.org/operation_0324',\n",
       "      'term': 'Phylogenetic analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_0550',\n",
       "      'term': 'DNA substitution modelling'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_0084', 'term': 'Phylogeny'},\n",
       "   {'uri': 'http://edamontology.org/topic_3945',\n",
       "    'term': 'Molecular evolution'},\n",
       "   {'uri': 'http://edamontology.org/topic_0204', 'term': 'Gene regulation'},\n",
       "   {'uri': 'http://edamontology.org/topic_3517', 'term': 'GWAS study'},\n",
       "   {'uri': 'http://edamontology.org/topic_0602',\n",
       "    'term': 'Molecular interactions, pathways and networks'}],\n",
       "  'language': ['R', 'C++'],\n",
       "  'link': [{'url': 'https://phyloacc.github.io/', 'type': ['Other']},\n",
       "   {'url': 'https://github.com/phyloacc', 'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.04.560937'}],\n",
       "  'credit': [{'name': 'Jun S. Liu',\n",
       "    'email': 'jliu@stat.harvard.edu',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__phyloacc-c',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__phyloacc-c',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.04.560937',\n",
       "  'is_preprint': True},\n",
       " {'name': 'DepMap',\n",
       "  'description': 'Connecting Cancers to Chromosomal Locations Through Copy Number Variations.\\n\\nThe latest quarterly release and archive of available datasets.\\n\\n||| NAME EQUAL TO (PUB. DIFFERENT) bio.tools/depmap-library, bio.tools/DepMap\\n\\n||| CORRECT NAME OF TOOL COULD ALSO BE DepMap portal',\n",
       "  'homepage': 'https://depmap.org/portal/download/',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_0283',\n",
       "      'term': 'Linkage analysis'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2640', 'term': 'Oncology'},\n",
       "   {'uri': 'http://edamontology.org/topic_3959', 'term': 'Cytogenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_0102', 'term': 'Mapping'},\n",
       "   {'uri': 'http://edamontology.org/topic_3958',\n",
       "    'term': 'Copy number variation'},\n",
       "   {'uri': 'http://edamontology.org/topic_0634', 'term': 'Pathology'}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.03.560705'}],\n",
       "  'credit': [{'name': 'Saed Sayad',\n",
       "    'email': 'saed@bioada.com',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__depmap',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__depmap',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.03.560705',\n",
       "  'is_preprint': True},\n",
       " {'name': 'scBoolSeq',\n",
       "  'description': 'Linking scRNA-Seq Statistics and Boolean Dynamics.\\n\\nCase_Study_Early-born_Retinal_Neurons.\\n\\nsynthetic scRNA-Seq from Random Network.ipynb.',\n",
       "  'homepage': 'http://nbviewer.org/github/bnediction/scBoolSeq-supplementary',\n",
       "  'function': [{'operation': [{'uri': 'http://edamontology.org/operation_3799',\n",
       "      'term': 'Quantification'},\n",
       "     {'uri': 'http://edamontology.org/operation_3927',\n",
       "      'term': 'Network analysis'},\n",
       "     {'uri': 'http://edamontology.org/operation_2428',\n",
       "      'term': 'Validation'}]}],\n",
       "  'topic': [{'uri': 'http://edamontology.org/topic_2269',\n",
       "    'term': 'Statistics and probability'},\n",
       "   {'uri': 'http://edamontology.org/topic_0749',\n",
       "    'term': 'Transcription factors and regulatory sites'},\n",
       "   {'uri': 'http://edamontology.org/topic_3512', 'term': 'Gene transcripts'},\n",
       "   {'uri': 'http://edamontology.org/topic_3295', 'term': 'Epigenetics'},\n",
       "   {'uri': 'http://edamontology.org/topic_3170', 'term': 'RNA-Seq'}],\n",
       "  'language': ['Python'],\n",
       "  'link': [{'url': 'http://github.com/bnediction/scBoolSeq',\n",
       "    'type': ['Repository']}],\n",
       "  'publication': [{'doi': '10.1101/2023.10.23.563518'}],\n",
       "  'credit': [{'name': 'Loïc Paulevé',\n",
       "    'email': 'loic.pauleve@labri.fr',\n",
       "    'typeEntity': 'Person'}],\n",
       "  'confidence_flag': 'high',\n",
       "  'editPermission': {'type': 'public'},\n",
       "  'biotoolsID': 'pub2tools2023__scboolseq',\n",
       "  'tool_link': 'https://bio-tools-dev.sdu.dk/pub2tools2023__scboolseq',\n",
       "  'publication_link': 'https://doi.org/10.1101/2023.10.23.563518',\n",
       "  'is_preprint': True}]"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "valid_tools"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Identify preprints**\n",
    "\n",
    "7. Identify preprints and get back the same list of tools including _is_preprint_ flag and _publication_link_ for each tool."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "There are 8 newly published tools. 112 preprints remaining.\n"
     ]
    }
   ],
   "source": [
    "from preprints import identify_preprints\n",
    "\n",
    "updated_tools = identify_preprints(rerun = False, tools = valid_tools)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Create .csv file**\n",
    "\n",
    "8. Generate csv file from _to_curate_ first publications and all preprints if _include_preprints_ = True.\n",
    "\n",
    "    Returns:\n",
    "    \n",
    "    * _tools_to_add_: tools to add to database (including preprints if _include_preprints_ = True).\n",
    "    * _tools_left_: tools not in _tools_to_add."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'pub2tools_log' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[4], line 4\u001b[0m\n\u001b[0;32m      1\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mutils\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mutils\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m check_date\n\u001b[0;32m      2\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01mutils\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mcsv_utils\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m generate_csv\n\u001b[1;32m----> 4\u001b[0m file_date \u001b[38;5;241m=\u001b[39m check_date(pub2tools_log)\n\u001b[0;32m      5\u001b[0m tools_to_add, tools_left \u001b[38;5;241m=\u001b[39m generate_csv(updated_tools, to_curate, file_date, include_preprints)\n",
      "\u001b[1;31mNameError\u001b[0m: name 'pub2tools_log' is not defined"
     ]
    }
   ],
   "source": [
    "from utils.utils import check_date\n",
    "from utils.csv_utils import generate_csv\n",
    "\n",
    "file_date = check_date(pub2tools_log)\n",
    "tools_to_add, tools_left = generate_csv(updated_tools, to_curate, file_date, include_preprints)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Create json files**\n",
    "\n",
    "9. Generate json files with tools that will not be curated:\n",
    "\n",
    "    If _separate_preprints_ = True, 2 json files will be generated with:\n",
    "    * Preprints (can be used as input for identify_preprints later as rerun = True)\n",
    "    * Publications\n",
    "\n",
    "    If _separate_preprints_ = False, only 1 file will be generate with the publications and preprints in the same file."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from utils.json_utils import generate_json\n",
    "\n",
    "generate_json(tools_left, file_date, separate_preprints=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "10. **Add tools to curate to dev**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from biotools_dev import add_tools\n",
    "\n",
    "add_tools(tools_to_add, token, WRITE_TO_DB=True)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Rerun\n",
    "\n",
    "Workflow to rerun preprints to check for new publications.\n",
    "\n",
    "Newly published tools in _json_preprints_ will be moved from this file to _json_publications_."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. Define file paths for preprints and publications files:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "json_preprints = \"\"\n",
    "json_publications = \"\""
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "2. Run identify_preprints"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from preprints import identify_preprints\n",
    "\n",
    "updated_tools = identify_preprints(rerun = True, json_prp = json_preprints, json_pub = json_publications)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Run Pub2Tools output and rerun existing preprints"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
